diff --git a/wget-list b/wget-list
index 46c84b9..954776a 100644
--- a/wget-list
+++ b/wget-list
@@ -192,8 +192,8 @@ https://download.osgeo.org/libtiff/tiff-4.6.0.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.3.2.tar.gz
 https://github.com/mypaint/mypaint-brushes/releases/download/v1.3.1/mypaint-brushes-1.3.1.tar.xz
 https://releases.pagure.org/newt/newt-0.52.24.tar.gz
-https://github.com/opencv/opencv/archive/4.8.1/opencv-4.8.1.tar.gz
-https://github.com/opencv/opencv_contrib/archive/4.8.1/opencv_contrib-4.8.1.tar.gz
+https://github.com/opencv/opencv/archive/4.9.0/opencv-4.9.0.tar.gz
+https://github.com/opencv/opencv_contrib/archive/4.9.0/opencv_contrib-4.9.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.0/openjpeg-2.5.0.tar.gz
 https://www.cairographics.org/releases/pixman-0.42.2.tar.gz
 https://poppler.freedesktop.org/poppler-23.12.0.tar.xz
@@ -626,7 +626,7 @@ https://xcb.freedesktop.org/dist/xcb-util-keysyms-0.4.1.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-renderutil-0.3.10.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-wm-0.4.2.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-cursor-0.1.4.tar.xz
-https://mesa.freedesktop.org/archive/mesa-23.3.1.tar.xz
+https://mesa.freedesktop.org/archive/mesa-23.3.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-2.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
@@ -691,8 +691,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-ffmpeg5_fi
 https://download.qt.io/archive/qt/6.6/6.6.1/single/qt-everywhere-src-6.6.1.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.273/Vulkan-Headers-1.3.273.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.273/Vulkan-Loader-1.3.273.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.274/Vulkan-Headers-1.3.274.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.274/Vulkan-Loader-1.3.274.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.42.4.tar.xz
 https://download.gnome.org/sources/gdm/45/gdm-45.0.1.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
@@ -766,7 +766,7 @@ https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz
 https://download.gnome.org/sources/nautilus/45/nautilus-45.2.1.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/42/gnome-bluetooth-42.7.tar.xz
 https://download.gnome.org/sources/gnome-keyring/42/gnome-keyring-42.1.tar.xz
-https://download.gnome.org/sources/gnome-settings-daemon/45/gnome-settings-daemon-45.0.tar.xz
+https://download.gnome.org/sources/gnome-settings-daemon/45/gnome-settings-daemon-45.1.tar.xz
 https://download.gnome.org/sources/tecla/45/tecla-45.0.tar.xz
 https://download.gnome.org/sources/gnome-control-center/45/gnome-control-center-45.2.tar.xz
 https://download.gnome.org/sources/mutter/45/mutter-45.2.tar.xz
@@ -797,7 +797,7 @@ https://download.gnome.org/sources/gnome-weather/45/gnome-weather-45.0.tar.xz
 https://gitlab.gnome.org/GNOME/gucharmap/-/archive/15.1.2/gucharmap-15.1.2.tar.bz2
 https://www.unicode.org/Public/zipped/15.1.0/Unihan.zip
 https://download.gnome.org/sources/seahorse/43/seahorse-43.0.tar.xz
-https://download.gnome.org/sources/snapshot/45/snapshot-45.1.tar.xz
+https://download.gnome.org/sources/snapshot/45/snapshot-45.2.tar.xz
 https://download.gnome.org/sources/vinagre/3.22/vinagre-3.22.0.tar.xz
 https://archive.xfce.org/src/xfce/libxfce4util/4.18/libxfce4util-4.18.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfconf/4.18/xfconf-4.18.3.tar.bz2
