diff --git a/wget-list b/wget-list
index 2dbe2cc67..9f4858b7b 100644
--- a/wget-list
+++ b/wget-list
@@ -6,7 +6,7 @@ https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyru
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.3.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gnupg-2.4.3-emacs-1.patch
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.2.tar.xz
-https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.23.1.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.23.2.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.10.tar.xz
 https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.69.tar.xz
 https://github.com/linux-pam/linux-pam/releases/download/v1.5.3/Linux-PAM-1.5.3.tar.xz
@@ -270,7 +270,7 @@ https://downloads.sourceforge.net/infozip/zip30.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
-https://cmake.org/files/v3.27/cmake-3.27.8.tar.gz
+https://cmake.org/files/v3.27/cmake-3.27.9.tar.gz
 https://doxygen.nl/files/doxygen-1.9.8.src.tar.gz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.4/gc-8.2.4.tar.gz
@@ -451,7 +451,7 @@ https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.2.0.tar.gz
 https://github.com/psf/cachecontrol/archive/refs/tags/v0.13.1/cachecontrol-0.13.1.tar.gz
 https://github.com/cython/cython/releases/download/0.29.36/Cython-0.29.36.tar.gz
-https://files.pythonhosted.org/packages/source/p/python-dbusmock/python-dbusmock-0.29.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/python-dbusmock/python-dbusmock-0.30.0.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.20.1.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.gz
@@ -556,7 +556,7 @@ https://nmap.org/dist/nmap-7.94.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.3.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.2.0.tar.xz
-https://c-ares.haxx.se/download/c-ares-1.22.1.tar.gz
+https://c-ares.haxx.se/download/c-ares-1.23.0.tar.gz
 https://curl.se/download/curl-8.4.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.78/glib-networking-2.78.0.tar.xz
@@ -601,7 +601,7 @@ https://ftp.exim.org/pub/exim/exim4/exim-4.97.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.8.3.tar.gz
 https://ftp.sendmail.org/sendmail.8.17.2.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
-https://downloads.mariadb.org/interstitial/mariadb-10.11.5/source/mariadb-10.11.5.tar.gz
+https://downloads.mariadb.org/interstitial/mariadb-10.11.6/source/mariadb-10.11.6.tar.gz
 https://ftp.postgresql.org/pub/source/v16.1/postgresql-16.1.tar.bz2
 https://sqlite.org/2023/sqlite-autoconf-3440200.tar.gz
 https://sqlite.org/2023/sqlite-doc-3440000.zip
@@ -679,9 +679,9 @@ https://download.gnome.org/sources/pango/1.50/pango-1.50.14.tar.xz
 https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.3.tar.xz
 https://download.qt.io/archive/qt/5.15/5.15.11/single/qt-everywhere-opensource-src-5.15.11.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5.15.11-kf5-1.patch
-https://anduin.linuxfromscratch.org/BLFS/qtwebengine/qtwebengine-5.15.16.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.16-build_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.16-ffmpeg5_fixes-1.patch
+https://anduin.linuxfromscratch.org/BLFS/qtwebengine/qtwebengine-5.15.17.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-build_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-ffmpeg5_fixes-1.patch
 https://download.qt.io/archive/qt/6.6/6.6.1/single/qt-everywhere-src-6.6.1.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
@@ -778,7 +778,7 @@ https://download.gnome.org/sources/evolution/3.50/evolution-3.50.1.tar.xz
 https://download.gnome.org/sources/file-roller/43/file-roller-43.0.tar.xz
 https://download.gnome.org/sources/gnome-calculator/45/gnome-calculator-45.0.2.tar.xz
 https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-3.36.0.tar.xz
-https://download.gnome.org/sources/gnome-disk-utility/45/gnome-disk-utility-45.0.tar.xz
+https://download.gnome.org/sources/gnome-disk-utility/45/gnome-disk-utility-45.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/43/gnome-logs-43.0.tar.xz
 https://download.gnome.org/sources/gnome-maps/45/gnome-maps-45.1.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
@@ -944,7 +944,7 @@ https://gitlab.freedesktop.org/vdpau/libvdpau/-/archive/1.5/libvdpau-1.5.tar.bz2
 https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.0/libvdpau-va-gl-0.4.0.tar.gz
 https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.13.1/libvpx-1.13.1.tar.gz
-https://github.com/mltframework/mlt/releases/download/v7.20.0/mlt-7.20.0.tar.gz
+https://github.com/mltframework/mlt/releases/download/v7.22.0/mlt-7.22.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.4.tar.gz
 https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.0.0/pipewire-1.0.0.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-16.1.tar.xz
@@ -956,7 +956,7 @@ https://www.surina.net/soundtouch/soundtouch-2.3.2.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
 https://taglib.org/releases/taglib-1.13.1.tar.gz
-https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.26.0.tar.bz2
+https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.26.0.tar.xz
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.4.16/wireplumber-0.4.16.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20230215.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/x265/x265-20230215.tar.xz
@@ -1022,7 +1022,7 @@ https://releases.pagure.org/xmlto/xmlto-0.0.28.tar.bz2
 https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview-gtk3/epdfview-gtk3-20200814.tar.xz
 https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.9-src.tar.gz
-https://www.mupdf.com/downloads/archive/mupdf-1.23.6-source.tar.gz
+https://www.mupdf.com/downloads/archive/mupdf-1.23.7-source.tar.gz
 https://downloads.sourceforge.net/paps/paps-0.7.1.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2023/texlive-20230313-source.tar.xz
