diff --git a/wget-list b/wget-list
index 0ddd7363f..ec9a898bc 100644
--- a/wget-list
+++ b/wget-list
@@ -16,8 +16,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
 https://kerberos.org/dist/krb5/1.21/krb5-1.21.2.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.9.1.tar.gz
-https://archive.mozilla.org/pub/security/nss/releases/NSS_3_94_RTM/src/nss-3.94.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.94-standalone-1.patch
+https://archive.mozilla.org/pub/security/nss/releases/NSS_3_95_RTM/src/nss-3.95.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.95-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.5p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.25.3/p11-kit-0.25.3.tar.xz
 https://gitlab.freedesktop.org/polkit/polkit/-/archive/123/polkit-123.tar.gz
@@ -171,7 +171,7 @@ https://sourceforge.net/projects/giflib/files/giflib-5.2.1.tar.gz
 https://github.com/g-truc/glm/archive/0.9.9.8/glm-0.9.9.8.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/8.3.0/harfbuzz-8.3.0.tar.xz
-https://github.com/jasper-software/jasper/archive/version-4.1.0/jasper-4.1.0.tar.gz
+https://github.com/jasper-software/jasper/archive/version-4.1.1/jasper-4.1.1.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.14/lcms2-2.14.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.0.2/libavif-1.0.2.tar.gz
 https://github.com/libexif/libexif/releases/download/v0.6.24/libexif-0.6.24.tar.bz2
@@ -480,7 +480,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.31.0-use_system_cer
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.16.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-7.2.6.tar.gz
-https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-1.3.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-0.7.13.tar.gz
 https://files.pythonhosted.org/packages/source/a/attrs/attrs-22.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/B/Babel/Babel-2.11.0.tar.gz
