diff --git a/wget-list b/wget-list
index f6c26dc..e9064ff 100644
--- a/wget-list
+++ b/wget-list
@@ -44,8 +44,7 @@ https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.5.0.tar.xz
 https://github.com/rhboot/efivar/releases/download/38/efivar-38.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/efivar-38-i686-1.patch
 https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
-https://ftp.gnu.org/gnu/grub/grub-2.06.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/grub-2.06-upstream_fixes-1.patch
+https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
 https://unifoundry.com/pub/unifont/unifont-15.1.04/font-builds/unifont-15.1.04.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.14.tar.bz2
 https://ftp.gnu.org/gnu/ed/ed-1.19.tar.lz
@@ -261,7 +260,7 @@ https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmon-qt/releases/download/v1.0/sysmon-qt-1.0.tar.xz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.4.tar.xz
-https://github.com/systemd/systemd/archive/v254/systemd-254.tar.gz
+https://github.com/systemd/systemd/archive/v255/systemd-255.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-6.2.12.tar.gz
 https://downloads.sourceforge.net/infozip/unzip60.tar.gz
@@ -534,7 +533,6 @@ https://github.com/openjdk/jdk21u/archive/jdk-21.0.1-ga.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.1/jtreg-7.2+1.tar.gz
 https://archive.apache.org/dist/ant/source/apache-ant-1.10.14-src.tar.xz
 https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.0.6/dhcpcd-10.0.6.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/dhcpcd-10.0.6-upstream_fixes-1.patch
 https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.xz
 https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.0.tar.bz2
 https://www.ncftp.com/downloads/ncftp/ncftp-3.2.6-src.tar.xz
@@ -596,7 +594,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/procmail-3.22-consolidated_fix
 https://archive.apache.org/dist/httpd/httpd-2.4.58.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/httpd-2.4.58-blfs_layout-1.patch
 https://downloads.isc.org/isc/kea/2.4.1/kea-2.4.1.tar.gz
-https://github.com/proftpd/proftpd/archive/v1.3.8a/proftpd-1.3.8a.tar.gz
+https://github.com/proftpd/proftpd/archive/v1.3.8b/proftpd-1.3.8b.tar.gz
 https://security.appspot.com/downloads/vsftpd-3.0.5.tar.gz
 https://www.dovecot.org/releases/2.3/dovecot-2.3.21.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-openssl3_fixes-1.patch
