diff --git a/wget-list b/wget-list
index ec9a898bc..2dbe2cc67 100644
--- a/wget-list
+++ b/wget-list
@@ -956,7 +956,7 @@ https://www.surina.net/soundtouch/soundtouch-2.3.2.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
 https://taglib.org/releases/taglib-1.13.1.tar.gz
-https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.24.1.tar.bz2
+https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.26.0.tar.bz2
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.4.16/wireplumber-0.4.16.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20230215.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/x265/x265-20230215.tar.xz
