diff --git a/wget-list b/wget-list
index 02ba591431..8cdf826328 100644
--- a/wget-list
+++ b/wget-list
@@ -17,7 +17,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
 https://kerberos.org/dist/krb5/1.22/krb5-1.22.1.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.10.2.tar.gz
-https://archive.mozilla.org/pub/security/nss/releases/NSS_3_117_RTM/src/nss-3.117.tar.gz
+https://archive.mozilla.org/pub/security/nss/releases/NSS_3_118_1_RTM/src/nss-3.118.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/nss-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-10.2p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.25.10/p11-kit-0.25.10.tar.xz
@@ -82,7 +82,7 @@ https://libopenraw.freedesktop.org/download/exempi-2.6.6.tar.xz
 https://github.com/fastfloat/fast_float/archive/v8.1.0/fast_float-8.1.0.tar.gz
 https://www.fftw.org/fftw-3.3.10.tar.gz
 https://github.com/fmtlib/fmt/archive/12.1.0/fmt-12.1.0.tar.gz
-https://download.gnome.org/sources/glib/2.86/glib-2.86.1.tar.xz
+https://download.gnome.org/sources/glib/2.86/glib-2.86.2.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.86/gobject-introspection-1.86.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
@@ -126,7 +126,7 @@ https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.11/libproxy-0.5.11.tar.gz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.8.1/libqalculate-5.8.1.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.8.2/libqalculate-5.8.2.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.36.0/libqmi-1.36.0.tar.gz
 https://github.com/seccomp/libseccomp/releases/download/v2.6.0/libseccomp-2.6.0.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -299,10 +299,11 @@ https://doxygen.nl/files/doxygen-1.15.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
 https://github.com/bdwgc/bdwgc/releases/download/v8.2.10/gc-8.2.10.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/gc-8.2.10-fork_workaround-1.patch
 https://ftp.gnu.org/gnu/gdb/gdb-16.3.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-2.51.2.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-manpages-2.51.2.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.51.2.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-2.52.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-manpages-2.52.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.52.0.tar.xz
 https://ftp.gnu.org/gnu/guile/guile-3.0.10.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/guile-3.0.10-32bit_fixes-1.patch
 https://github.com/llvm/llvm-project/releases/download/llvmorg-21.1.2/llvm-21.1.2.src.tar.xz
@@ -469,7 +470,7 @@ https://www.python.org/ftp/python/doc/3.13.7/python-3.13.7-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/b/build/build-1.3.0.tar.gz
-https://github.com/psf/cachecontrol/archive/v0.14.3/cachecontrol-0.14.3.tar.gz
+https://github.com/psf/cachecontrol/archive/v0.14.4/cachecontrol-0.14.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.3.0.tar.gz
 https://github.com/cython/cython/releases/download/3.2.1/cython-3.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.37.1.tar.gz
@@ -482,7 +483,7 @@ https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/html5lib-1.1-python_3.14_buildfix-1.patch
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.3.10.tar.gz
-https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.3.4.tar.gz
+https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.3.5.tar.gz
 https://files.pythonhosted.org/packages/source/p/ply/ply-3.11.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.1.3.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
@@ -493,8 +494,9 @@ https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.19.2.tar.gz
 https://download.gnome.org/sources/pygobject/3.54/pygobject-3.54.5.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.2.5.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
-https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.4.2.tar.gz
+https://files.pythonhosted.org/packages/source/p/pytest/pytest-9.0.1.tar.gz
 https://files.pythonhosted.org/packages/b0/25/7998cd2dec731acbd438fbf91bc619603fc5188de0a9a17699a781840452/pyxdg-0.28.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/pyxdg-0.28-python3_14-fixes-1.patch
 https://files.pythonhosted.org/packages/source/P/PyYAML/pyyaml-6.0.3.tar.gz
 https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1.tar.gz
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.32.5.tar.gz
@@ -507,27 +509,30 @@ https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_the
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.4.0.tar.gz
 https://files.pythonhosted.org/packages/source/B/Babel/babel-2.17.0.tar.gz
-https://files.pythonhosted.org/packages/source/c/certifi/certifi-2025.8.3.tar.gz
+https://files.pythonhosted.org/packages/source/c/certifi/certifi-2025.11.12.tar.gz
 https://files.pythonhosted.org/packages/source/c/chardet/chardet-5.2.0.tar.gz
-https://files.pythonhosted.org/packages/source/c/charset-normalizer/charset_normalizer-3.4.3.tar.gz
+https://files.pythonhosted.org/packages/source/c/charset-normalizer/charset_normalizer-3.4.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/commonmark/commonmark-0.9.1.tar.gz
 https://files.pythonhosted.org/packages/source/e/editables/editables-0.5.tar.gz
 https://files.pythonhosted.org/packages/source/h/hatchling/hatchling-1.27.0.tar.gz
 https://files.pythonhosted.org/packages/source/h/hatch-fancy-pypi-readme/hatch_fancy_pypi_readme-25.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/h/hatch-vcs/hatch_vcs-0.5.0.tar.gz
-https://files.pythonhosted.org/packages/source/i/idna/idna-3.10.tar.gz
+https://files.pythonhosted.org/packages/source/i/idna/idna-3.11.tar.gz
 https://files.pythonhosted.org/packages/source/i/imagesize/imagesize-1.4.1.tar.gz
-https://files.pythonhosted.org/packages/source/i/iniconfig/iniconfig-2.1.0.tar.gz
-https://files.pythonhosted.org/packages/source/M/Markdown/markdown-3.8.2.tar.gz
+https://files.pythonhosted.org/packages/source/i/iniconfig/iniconfig-2.3.0.tar.gz
+https://github.com/PyO3/maturin/archive/v1.10.1/maturin-1.10.1.tar.gz
+https://files.pythonhosted.org/packages/source/M/Markdown/markdown-3.10.tar.gz
 https://files.pythonhosted.org/packages/source/m/meson_python/meson_python-0.18.0.tar.gz
-https://files.pythonhosted.org/packages/source/m/msgpack/msgpack-1.1.1.tar.gz
+https://files.pythonhosted.org/packages/source/m/msgpack/msgpack-1.1.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/pathspec/pathspec-0.12.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pluggy/pluggy-1.6.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyproject-hooks/pyproject_hooks-1.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyproject-metadata/pyproject_metadata-0.9.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pytz/pytz-2025.2.tar.gz
 https://files.pythonhosted.org/packages/source/r/roman-numerals-py/roman_numerals_py-3.1.0.tar.gz
-https://files.pythonhosted.org/packages/source/s/setuptools_scm/setuptools_scm-8.3.1.tar.gz
+https://files.pythonhosted.org/packages/source/s/semantic_version/semantic_version-2.10.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/setuptools_rust/setuptools_rust-1.12.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/setuptools_scm/setuptools_scm-9.2.2.tar.gz
 https://files.pythonhosted.org/packages/source/s/smartypants/smartypants-2.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/s/snowballstemmer/snowballstemmer-3.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-applehelp/sphinxcontrib_applehelp-2.0.0.tar.gz
@@ -537,9 +542,10 @@ https://files.pythonhosted.org/packages/source/s/sphinxcontrib-jquery/sphinxcont
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-jsmath/sphinxcontrib-jsmath-1.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-qthelp/sphinxcontrib_qthelp-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib_serializinghtml/sphinxcontrib_serializinghtml-2.0.0.tar.gz
-https://files.pythonhosted.org/packages/source/t/trove_classifiers/trove_classifiers-2025.8.6.13.tar.gz
+https://files.pythonhosted.org/packages/source/t/trove_classifiers/trove_classifiers-2025.11.14.15.tar.gz
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.5.0.tar.gz
+https://github.com/PyO3/uv_build/archive/v0.9.10/uv_build-0.9.10.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.4/ruby-3.4.7.tar.xz
 https://static.rust-lang.org/dist/rustc-1.91.0-src.tar.xz
@@ -577,7 +583,7 @@ https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
 https://ftp.isc.org/isc/bind9/9.20.15/bind-9.20.15.tar.xz
-https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.54.1/downloads/NetworkManager-1.54.1.tar.xz
+https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.54.2/downloads/NetworkManager-1.54.2.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.98.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
@@ -620,7 +626,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/httpd-blfs_layout-1.patch
 https://downloads.isc.org/isc/kea/3.0.1/kea-3.0.1.tar.xz
 https://github.com/proftpd/proftpd/archive/v1.3.9/proftpd-1.3.9.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.2.tar.gz
-https://ftp.exim.org/pub/exim/exim4/exim-4.98.2.tar.xz
+https://ftp.exim.org/pub/exim/exim4/exim-4.99.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.10.5.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/sendmail-8.18.1-gcc15_fixes-1.patch
@@ -666,7 +672,7 @@ https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
 https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-source.tar.gz
 https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-docs-html.tar.gz
-https://downloads.sourceforge.net/freeglut/freeglut-3.6.0.tar.gz
+https://downloads.sourceforge.net/freeglut/freeglut-3.8.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.4.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.2.0.tgz
 https://github.com/KhronosGroup/glslang/archive/16.0.0/glslang-16.0.0.tar.gz
@@ -895,7 +901,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.22-cxx17-1.patc
 https://gitlab.gnome.org/GNOME/balsa/-/archive/2.6.5/balsa-2.6.5.tar.gz
 https://feh.finalrewind.org/feh-3.11.2.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20251009/fontforge-20251009.tar.xz
-https://github.com/freerdp/freerdp/archive/3.17.2/FreeRDP-3.17.2.tar.gz
+https://github.com/freerdp/freerdp/archive/3.18.0/FreeRDP-3.18.0.tar.gz
 https://download.gimp.org/gimp/v3.0/gimp-3.0.6.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
@@ -1003,6 +1009,7 @@ https://github.com/mpv-player/mpv/archive/v0.40.0/mpv-0.40.0.tar.gz
 https://download.videolan.org/vlc/3.0.21/vlc-3.0.21.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.21-taglib-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.21-fedora_ffmpeg7-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.21-ffmpeg8-1.patch
 https://downloads.sourceforge.net/xine/xine-ui-0.99.14.tar.xz
 https://downloads.sourceforge.net/cdrdao/cdrdao-1.2.5.tar.bz2
 https://downloads.sourceforge.net/cdrtools/cdrtools-3.02a09.tar.bz2
@@ -1059,5 +1066,5 @@ https://github.com/mgieseki/dvisvgm/releases/download/3.5/dvisvgm-3.5.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20250225.tar.xz
-https://github.com/elogind/elogind/archive/v255.17/elogind-255.17.tar.gz
+https://github.com/elogind/elogind/archive/V255.22/elogind-255.22.tar.gz
 https://github.com/lfs-book/blocaled/releases/download/v0.7/blocaled-0.7.tar.xz
