diff --git a/wget-list b/wget-list
index 1ef456caf2..f4ffbadae3 100644
--- a/wget-list
+++ b/wget-list
@@ -120,7 +120,7 @@ https://ftp.gnu.org/gnu/libidn/libidn2-2.3.8.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v249/liblinear-249.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.32.0/libmbim-1.32.0.tar.gz
-https://github.com/linux-nvme/libnvme/archive/v1.16/libnvme-1.16.tar.gz
+https://github.com/linux-nvme/libnvme/archive/v1.16.1/libnvme-1.16.1.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.6/libpaper-2.2.6.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
