diff --git a/wget-list b/wget-list
index 61b03d39d8..62b3a15912 100644
--- a/wget-list
+++ b/wget-list
@@ -271,6 +271,8 @@ https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmon-qt/releases/download/v2.0.1/sysmon-qt-2.0.1.tar.xz
+https://github.com/lfs-book/sysmond/releases/download/sysmond-1.0.0/sysmond-1.0.0.tar.xz
+https://github.com/lfs-book/sysmon3/releases/download/sysmon3-3.0.0/sysmon3-3.0.0.tar.xz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.6.tar.xz
 https://github.com/systemd/systemd/archive/v256.5/systemd-256.5.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
