diff --git a/wget-list b/wget-list
index d35571a982..f8ab58b462 100644
--- a/wget-list
+++ b/wget-list
@@ -165,7 +165,7 @@ https://github.com/nemtrif/utfcpp/archive/v4.0.6/utfcpp-4.0.6.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.23.0/downloads/wayland-1.23.0.tar.xz
 https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.38/downloads/wayland-protocols-1.38.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
-https://oligarchy.co.uk/xapian/1.4.26/xapian-core-1.4.26.tar.xz
+https://oligarchy.co.uk/xapian/1.4.27/xapian-core-1.4.27.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
 https://download.gimp.org/pub/babl/0.1/babl-0.1.110.tar.xz
 https://github.com/Exiv2/exiv2/archive/v0.28.3/exiv2-0.28.3.tar.gz
