diff --git a/wget-list b/wget-list
index c056ddfaf4..1b2173211a 100644
--- a/wget-list
+++ b/wget-list
@@ -564,7 +564,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.3/bind-9.20.3.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.4/bind-9.20.4.tar.xz
 https://download.gnome.org/sources/NetworkManager/1.50/NetworkManager-1.50.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.95.tar.bz2
