diff --git a/wget-list b/wget-list
index 590bb85f80..154c58baf5 100644
--- a/wget-list
+++ b/wget-list
@@ -5,7 +5,7 @@ https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.7.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.8.tar.xz
-https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.0.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.1.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.11.tar.xz
 https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.72.tar.xz
 https://github.com/linux-pam/linux-pam/releases/download/v1.7.0/Linux-PAM-1.7.0.tar.xz
@@ -149,7 +149,7 @@ https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.36/src/nspr-4.36.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.44/pcre2-10.44.tar.bz2
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
-https://github.com/protocolbuffers/protobuf/releases/download/v29.0/protobuf-29.0.tar.gz
+https://github.com/protocolbuffers/protobuf/releases/download/v29.1/protobuf-29.1.tar.gz
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.0/protobuf-c-1.5.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.0-consolidated_fixes-1.patch
 https://download.kde.org/stable/qca/2.3.9/qca-2.3.9.tar.xz
@@ -274,7 +274,7 @@ https://github.com/lfs-book/sysmon-qt/releases/download/v2.0.1/sysmon-qt-2.0.1.t
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.6.tar.xz
 https://github.com/systemd/systemd/archive/v256.5/systemd-256.5.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
-https://www.rarlab.com/rar/unrarsrc-7.1.1.tar.gz
+https://www.rarlab.com/rar/unrarsrc-7.1.2.tar.gz
 https://downloads.sourceforge.net/infozip/unzip60.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/unzip-6.0-consolidated_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/unzip-6.0-gcc14-1.patch
@@ -491,7 +491,7 @@ https://files.pythonhosted.org/packages/source/r/requests/requests-2.32.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.32.3-use_system_certs-1.patch
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
 https://github.com/getsentry/sentry-python/releases/download/2.18.0/sentry_sdk-2.18.0.tar.gz
-https://files.pythonhosted.org/packages/source/s/six/six-1.16.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/six/six-1.17.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.1.3.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-3.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
@@ -556,7 +556,7 @@ https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.7-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.3.0.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.21.1.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.21.2.tar.gz
 https://ftp.gnu.org/gnu/wget/wget-1.25.0.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.9.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
@@ -612,7 +612,7 @@ https://www.dovecot.org/releases/2.3/dovecot-2.3.21.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21.1-openssl3_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21.1-security_fix-1.patch
 https://ftp.exim.org/pub/exim/exim4/exim-4.98.tar.xz
-https://ghostarchive.org/postfix/postfix-release/official/postfix-3.9.0.tar.gz
+https://ghostarchive.org/postfix/postfix-release/official/postfix-3.9.1.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-11.4.4/source/mariadb-11.4.4.tar.gz
@@ -682,15 +682,15 @@ https://gitlab.freedesktop.org/libinput/libei/-/archive/1.3.0/libei-1.3.0.tar.gz
 https://gitlab.gnome.org/World/gedit/libgedit-amtk/-/archive/5.9.0/libgedit-amtk-5.9.0.tar.bz2
 https://gitlab.gnome.org/World/gedit/libgedit-gtksourceview/-/archive/299.3.0/libgedit-gtksourceview-299.3.0.tar.bz2
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
-https://dri.freedesktop.org/libdrm/libdrm-2.4.123.tar.xz
+https://dri.freedesktop.org/libdrm/libdrm-2.4.124.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
 https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.3.tar.xz
 https://people.freedesktop.org/~svu/libxklavier-5.4.tar.bz2
 https://download.gnome.org/sources/pango/1.55/pango-1.55.0.tar.xz
 https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.4.tar.xz
 https://download.gnome.org/sources/pangomm/2.54/pangomm-2.54.0.tar.xz
-https://download.qt.io/archive/qt/6.8/6.8.0/single/qt-everywhere-src-6.8.0.tar.xz
-https://download.qt.io/official_releases/qt/6.8/6.8.0/submodules/qtwebengine-everywhere-src-6.8.0.tar.xz
+https://download.qt.io/archive/qt/6.8/6.8.1/single/qt-everywhere-src-6.8.1.tar.xz
+https://download.qt.io/official_releases/qt/6.8/6.8.1/submodules/qtwebengine-everywhere-src-6.8.1.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.301/Vulkan-Headers-1.3.301.tar.gz
 https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.301/Vulkan-Loader-1.3.301.tar.gz
@@ -919,13 +919,12 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://downloads.xiph.org/releases/flac/flac-1.4.3.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.9.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.9.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.9.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.9.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/gst-plugins-bad-1.24.9-x265_4.1-1.patch
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.9.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.9.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.10.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.10.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.10.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.10.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.10.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.10.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/intel-media-24.3.4.tar.gz
@@ -943,7 +942,7 @@ https://get.videolan.org/libdvdcss/1.4.3/libdvdcss-1.4.3.tar.bz2
 https://get.videolan.org/libdvdread/6.1.3/libdvdread-6.1.3.tar.bz2
 https://get.videolan.org/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2
 https://downloads.sourceforge.net/libdv/libdv-1.0.0.tar.gz
-https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.24.9/gst-plugins-rs-gstreamer-1.24.9.tar.gz
+https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.24.10/gst-plugins-rs-gstreamer-1.24.10.tar.gz
 https://downloads.sourceforge.net/mad/libmad-0.15.1b.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libmad-0.15.1b-fixes-1.patch
 https://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
