diff --git a/wget-list b/wget-list
index 5e69ded640..8630c1519e 100644
--- a/wget-list
+++ b/wget-list
@@ -60,7 +60,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/25.12.2/src/kate-25.12.2.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.5.tar.xz
 https://www.nano-editor.org/dist/v8/nano-8.7.1.tar.xz
-https://github.com/vim/vim/archive/v9.2.0000/vim-9.2.0000.tar.gz
+https://github.com/vim/vim/archive/v9.2.0078/vim-9.2.0078.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.1.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
@@ -423,6 +423,7 @@ https://cpan.metacpan.org/authors/id/O/OA/OALDERS/LWP-MediaTypes-6.04.tar.gz
 https://cpan.metacpan.org/authors/id/R/RE/REHSACK/MIME-Base32-1.303.tar.gz
 https://cpan.metacpan.org/authors/id/N/NE/NEZUMI/MIME-Charset-1.013.1.tar.gz
 https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Module-Implementation-0.09.tar.gz
+https://cpan.metacpan.org/authors/id/S/SI/SIMONW/Module-Pluggable-6.3.tar.gz
 https://cpan.metacpan.org/authors/id/H/HA/HAARG/Module-Runtime-0.018.tar.gz
 https://cpan.metacpan.org/authors/id/H/HA/HAARG/MRO-Compat-0.15.tar.gz
 https://cpan.metacpan.org/authors/id/E/ET/ETHER/namespace-autoclean-0.31.tar.gz
@@ -950,7 +951,6 @@ https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.28.1.tar.xz
 https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.28.1.tar.xz
 https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.28.1.tar.xz
 https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.28.1.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/gst-plugins-bad-1.28.1-zxing_cpp_3-1.patch
 https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.28.1.tar.xz
 https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.28.1/gst-plugins-rs-gstreamer-1.28.1.tar.gz
 https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.28.1.tar.xz
