diff --git a/wget-list b/wget-list
index 00e19afe81..6c444d1560 100644
--- a/wget-list
+++ b/wget-list
@@ -151,7 +151,7 @@ https://github.com/hughsie/libxmlb/releases/download/0.3.25/libxmlb-0.3.25.tar.x
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.45.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.18.0/libwacom-2.18.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
-https://github.com/pantoniou/libfyaml/releases/download/v0.9.5/libfyaml-0.9.5.tar.gz
+https://github.com/pantoniou/libfyaml/releases/download/v0.9.6/libfyaml-0.9.6.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_2/log4cplus-2.1.2.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
@@ -613,7 +613,7 @@ https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.1/libslirp-v4.9.1.t
 https://download.gnome.org/sources/libsoup/3.6/libsoup-3.6.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libsoup-3.6.6-upstream_fixes-1.patch
 https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.7.tar.bz2
-https://notroj.github.io/neon/neon-0.37.0.tar.gz
+https://notroj.github.io/neon/neon-0.37.1.tar.gz
 https://github.com/nghttp2/nghttp2/releases/download/v1.68.0/nghttp2-1.68.0.tar.xz
 https://github.com/thkukuk/rpcsvc-proto/releases/download/v1.4.4/rpcsvc-proto-1.4.4.tar.xz
 https://archive.apache.org/dist/serf/serf-1.3.10.tar.bz2
