diff --git a/wget-list b/wget-list
index d5476c46c0..f361f6c79c 100644
--- a/wget-list
+++ b/wget-list
@@ -4,7 +4,7 @@ https://github.com/cracklib/cracklib/releases/download/v2.10.3/cracklib-2.10.3.t
 https://www.kernel.org/pub/linux/utils/cryptsetup/v2.8/cryptsetup-2.8.4.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/cyrus-sasl-2.1.28-gcc15_fixes-1.patch
-https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.5.17.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.5.18.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.12.tar.xz
 https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-2.0.1.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gpgmepp/gpgmepp-2.0.0.tar.xz
@@ -18,7 +18,7 @@ https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.
 https://kerberos.org/dist/krb5/1.22/krb5-1.22.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/mitkrb-1.22.2-upstream_fix-1.patch
 https://ftpmirror.gnu.org/nettle/nettle-3.10.2.tar.gz
-https://archive.mozilla.org/pub/security/nss/releases/NSS_3_120_1_RTM/src/nss-3.120.1.tar.gz
+https://archive.mozilla.org/pub/security/nss/releases/NSS_3_121_RTM/src/nss-3.121.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/nss-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-10.2p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.26.2/p11-kit-0.26.2.tar.xz
@@ -91,7 +91,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/glib-2.86.4-upstream_fixes-1.p
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
 https://download.gnome.org/sources/glibmm/2.86/glibmm-2.86.0.tar.xz
 https://github.com/jstedfast/gmime/releases/download/3.2.15/gmime-3.2.15.tar.xz
-https://github.com/lfs-book/gmmlib/archive/v22.8.2/gmmlib-22.8.2.tar.gz
+https://github.com/lfs-book/gmmlib/archive/v22.9.0/gmmlib-22.9.0.tar.gz
 https://ftpmirror.gnu.org/gsl/gsl-2.8.tar.gz
 https://download.gnome.org/sources/gspell/1.14/gspell-1.14.2.tar.xz
 https://github.com/google/highway/archive/1.3.0/highway-1.3.0.tar.gz
@@ -112,7 +112,7 @@ https://github.com/llvm/llvm-project/releases/download/llvmorg-21.1.8/libclc-21.
 https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
 https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.3.0/downloads/libdisplay-info-0.3.0.tar.xz
-https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.12.0.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.12.1.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.59.tar.bz2
 https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.55.tar.xz
 https://download.gnome.org/sources/libgudev/238/libgudev-238.tar.xz
@@ -120,7 +120,7 @@ https://github.com/hughsie/libgusb/releases/download/0.4.9/libgusb-0.4.9.tar.xz
 https://github.com/libical/libical/releases/download/v3.0.20/libical-3.0.20.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn-1.43.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn2-2.3.8.tar.gz
-https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.8.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v250/liblinear-250.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.34.0/libmbim-1.34.0.tar.gz
 https://github.com/linux-nvme/libnvme/archive/v1.16.1/libnvme-1.16.1.tar.gz
@@ -138,19 +138,19 @@ https://ftpmirror.gnu.org/libsigsegv/libsigsegv-2.15.tar.gz
 https://www.libssh2.org/download/libssh2-1.11.1.tar.gz
 https://www.mirrorservice.org/sites/ftp.i-scream.org/pub/i-scream/libstatgrab/libstatgrab-0.92.1.tar.gz
 https://ftpmirror.gnu.org/libtasn1/libtasn1-4.21.0.tar.gz
-https://ftpmirror.gnu.org/libunistring/libunistring-1.4.1.tar.xz
+https://ftpmirror.gnu.org/libunistring/libunistring-1.4.2.tar.xz
 https://github.com/libunwind/libunwind/releases/download/v1.8.3/libunwind-1.8.3.tar.gz
 https://lttng.org/files/urcu/userspace-rcu-0.15.6.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.29/libusb-1.0.29.tar.bz2
 https://dist.libuv.org/dist/v1.52.0/libuv-v1.52.0.tar.gz
 https://github.com/lfs-book/libxkbcommon/archive/v1.13.1/libxkbcommon-1.13.1.tar.gz
-https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.1.tar.xz
+https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.2.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.25/libxmlb-0.3.25.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.45.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.18.0/libwacom-2.18.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
-https://github.com/pantoniou/libfyaml/releases/download/v0.9.4/libfyaml-0.9.4.tar.gz
+https://github.com/pantoniou/libfyaml/releases/download/v0.9.5/libfyaml-0.9.5.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_2/log4cplus-2.1.2.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
@@ -159,8 +159,9 @@ https://www.linuxfromscratch.org/patches/blfs/svn/node-v22.22.0-python_build_fix
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.38.2/src/nspr-4.38.2.tar.gz
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
-https://github.com/protocolbuffers/protobuf/releases/download/v33.5/protobuf-33.5.tar.gz
+https://github.com/protocolbuffers/protobuf/releases/download/v34.0/protobuf-34.0.tar.gz
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.2/protobuf-c-1.5.2.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.2-protobuf34_fix-1.patch
 https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.13.0/qcoro-0.13.0.tar.gz
 https://github.com/simdutf/simdutf/archive/v8.0.0/simdutf-8.0.0.tar.gz
@@ -171,17 +172,17 @@ https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.341.0/SPIRV-T
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v21.1.4/SPIRV-LLVM-Translator-21.1.4.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.4.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
-https://github.com/martinpitt/umockdev/releases/download/0.19.4/umockdev-0.19.4.tar.xz
+https://github.com/martinpitt/umockdev/releases/download/0.19.5/umockdev-0.19.5.tar.xz
 https://github.com/nemtrif/utfcpp/archive/v4.0.9/utfcpp-4.0.9.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.24.0/downloads/wayland-1.24.0.tar.xz
 https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.47/downloads/wayland-protocols-1.47.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
-https://oligarchy.co.uk/xapian/1.4.30/xapian-core-1.4.30.tar.xz
+https://oligarchy.co.uk/xapian/1.4.31/xapian-core-1.4.31.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
 https://download.gimp.org/pub/babl/0.1/babl-0.1.122.tar.xz
-https://github.com/Exiv2/exiv2/archive/v0.28.7/exiv2-0.28.7.tar.gz
-https://downloads.sourceforge.net/freetype/freetype-2.14.1.tar.xz
-https://downloads.sourceforge.net/freetype/freetype-doc-2.14.1.tar.xz
+https://github.com/Exiv2/exiv2/archive/v0.28.8/exiv2-0.28.8.tar.gz
+https://downloads.sourceforge.net/freetype/freetype-2.14.2.tar.xz
+https://downloads.sourceforge.net/freetype/freetype-doc-2.14.2.tar.xz
 https://gitlab.freedesktop.org/api/v4/projects/890/packages/generic/fontconfig/2.17.1/fontconfig-2.17.1.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.16/fribidi-1.0.16.tar.xz
 https://download.gimp.org/pub/gegl/0.4/gegl-0.4.66.tar.xz
@@ -194,10 +195,10 @@ https://download.gnome.org/sources/glycin/2.0/glycin-2.0.8.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.0.8-xorg_prefix-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.0.8-xbm_xpm-1.patch
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
-https://github.com/harfbuzz/harfbuzz/releases/download/12.3.2/harfbuzz-12.3.2.tar.xz
+https://github.com/harfbuzz/harfbuzz/releases/download/13.0.0/harfbuzz-13.0.0.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.2.8/jasper-version-4.2.8.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.18/lcms2-2.18.tar.gz
-https://github.com/AOMediaCodec/libavif/archive/v1.3.0/libavif-1.3.0.tar.gz
+https://github.com/AOMediaCodec/libavif/archive/v1.4.0/libavif-1.4.0.tar.gz
 https://github.com/libexif/libexif/releases/download/v0.6.25/libexif-0.6.25.tar.bz2
 https://download.gnome.org/sources/libgxps/0.3/libgxps-0.3.2.tar.xz
 https://github.com/strukturag/libheif/releases/download/v1.21.2/libheif-1.21.2.tar.gz
@@ -220,7 +221,7 @@ https://github.com/opencv/opencv/archive/4.13.0/opencv-4.13.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.13.0/opencv_contrib-4.13.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.4/openjpeg-2.5.4.tar.gz
 https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
-https://poppler.freedesktop.org/poppler-26.02.0.tar.xz
+https://poppler.freedesktop.org/poppler-26.03.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v12.3.2/qpdf-12.3.2.tar.gz
@@ -236,7 +237,7 @@ https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.4.tar.gz
 https://github.com/google/shaderc/archive/v2026.1/shaderc-2026.1.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/14.1.2/graphviz-14.1.2.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.35/gtk-doc-1.35.1.tar.xz
-http://www.andre-simon.de/zip/highlight-4.18.tar.bz2
+http://www.andre-simon.de/zip/highlight-4.19.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.33/ibus-1.5.33.tar.gz
 https://www.unicode.org/Public/17.0.0/ucd/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.2-13.tar.xz
@@ -270,7 +271,7 @@ http://fcron.free.fr/archives/fcron-3.4.0.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-gcc15_fixes-1.patch
-https://github.com/vcrhonek/hwdata/archive/v0.404/hwdata-0.404.tar.gz
+https://github.com/vcrhonek/hwdata/archive/v0.405/hwdata-0.405.tar.gz
 https://github.com/lfs-book/LSB-Tools/releases/download/v0.12/LSB-Tools-0.12.tar.gz
 https://github.com/logrotate/logrotate/releases/download/3.22.0/logrotate-3.22.0.tar.xz
 http://ftp.midnight-commander.org/mc-4.8.33.tar.xz
@@ -591,7 +592,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.19/bind-9.20.19.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.20/bind-9.20.20.tar.xz
 https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.56.0/downloads/NetworkManager-1.56.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.98.tar.bz2
@@ -618,7 +619,7 @@ https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.1/libslirp-v4.9.1.t
 https://download.gnome.org/sources/libsoup/3.6/libsoup-3.6.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libsoup-3.6.6-upstream_fixes-1.patch
 https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.7.tar.bz2
-https://notroj.github.io/neon/neon-0.36.0.tar.gz
+https://notroj.github.io/neon/neon-0.37.0.tar.gz
 https://github.com/nghttp2/nghttp2/releases/download/v1.68.0/nghttp2-1.68.0.tar.xz
 https://github.com/thkukuk/rpcsvc-proto/releases/download/v1.4.4/rpcsvc-proto-1.4.4.tar.xz
 https://archive.apache.org/dist/serf/serf-1.3.10.tar.bz2
@@ -655,7 +656,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-4.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20250912.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
-https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.46.tar.xz
+https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.47.tar.xz
 https://www.x.org/pub/individual/xserver/xwayland-24.1.9.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.21.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.21-tearfree_backport-1.patch
@@ -839,16 +840,16 @@ https://archive.xfce.org/src/xfce/garcon/4.20/garcon-4.20.0.tar.bz2
 https://download.gnome.org/sources/libwnck/43/libwnck-43.3.tar.xz
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.20/xfce4-dev-tools-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.5.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.6.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.7.tar.bz2
 https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.7.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.20/thunar-volman-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.20/tumbler-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.20/xfce4-appfinder-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-power-manager/4.20/xfce4-power-manager-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-settings/4.20/xfce4-settings-4.20.3.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-settings/4.20/xfce4-settings-4.20.4.tar.bz2
 https://archive.xfce.org/src/xfce/xfdesktop/4.20/xfdesktop-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfwm4/4.20/xfwm4-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-session/4.20/xfce4-session-4.20.3.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-session/4.20/xfce4-session-4.20.4.tar.bz2
 https://archive.xfce.org/src/apps/parole/4.20/parole-4.20.0.tar.xz
 https://archive.xfce.org/src/apps/xfce4-terminal/1.1/xfce4-terminal-1.1.5.tar.xz
 https://archive.xfce.org/src/apps/xfburn/0.8/xfburn-0.8.0.tar.bz2
@@ -915,7 +916,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.23-cxx17-1.patc
 https://gitlab.gnome.org/GNOME/balsa/-/archive/2.6.5/balsa-2.6.5.tar.gz
 https://feh.finalrewind.org/feh-3.11.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20251009/fontforge-20251009.tar.xz
-https://github.com/freerdp/freerdp/archive/3.22.0/FreeRDP-3.22.0.tar.gz
+https://github.com/freerdp/freerdp/archive/3.23.0/FreeRDP-3.23.0.tar.gz
 https://download.gimp.org/gimp/v3.0/gimp-3.0.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gimp-3.0.6-security_fixes-1.patch
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.2.tar.bz2
@@ -992,7 +993,7 @@ https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.6.0/pipewire-1.6.0.
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.2.tar.xz
 https://www.libsdl.org/release/sdl2-compat-2.32.64.tar.gz
-https://www.libsdl.org/release/SDL3-3.4.0.tar.gz
+https://www.libsdl.org/release/SDL3-3.4.2.tar.gz
 https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.4.0.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
