diff --git a/wget-list b/wget-list
index 6c444d1560..32fac4f01c 100644
--- a/wget-list
+++ b/wget-list
@@ -214,7 +214,7 @@ https://download.gnome.org/sources/librsvg/2.62/librsvg-2.62.0.tar.xz
 https://github.com/fontforge/libspiro/releases/download/20220722/libspiro-dist-20220722.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.7.1.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.6.0.tar.gz
-https://github.com/mypaint/mypaint-brushes/releases/download/v1.3.1/mypaint-brushes-1.3.1.tar.xz
+https://github.com/mypaint/mypaint-brushes/releases/download/v2.0.2/mypaint-brushes-2.0.2.tar.xz
 https://releases.pagure.org/newt/newt-0.52.25.tar.gz
 https://github.com/opencv/opencv/archive/4.13.0/opencv-4.13.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.13.0/opencv_contrib-4.13.0.tar.gz
@@ -911,8 +911,7 @@ https://gitlab.gnome.org/GNOME/balsa/-/archive/2.6.5/balsa-2.6.5.tar.gz
 https://feh.finalrewind.org/feh-3.11.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20251009/fontforge-20251009.tar.xz
 https://github.com/freerdp/freerdp/archive/3.24.0/FreeRDP-3.24.0.tar.gz
-https://download.gimp.org/gimp/v3.0/gimp-3.0.6.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/gimp-3.0.6-security_fixes-1.patch
+https://download.gimp.org/gimp/v3.2/gimp-3.2.0.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.2.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.8.0.tar.gz
 https://inkscape.org/gallery/item/58914/inkscape-1.4.3.tar.xz
