diff --git a/wget-list b/wget-list
index 8da87065c..cb1abf47b 100644
--- a/wget-list
+++ b/wget-list
@@ -46,7 +46,7 @@ https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
 https://unifoundry.com/pub/unifont/unifont-15.1.05/font-builds/unifont-15.1.05.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.15.tar.bz2
 https://ftp.gnu.org/gnu/ed/ed-1.20.2.tar.lz
-https://ftp.gnu.org/gnu/emacs/emacs-29.3.tar.xz
+https://ftp.gnu.org/gnu/emacs/emacs-29.4.tar.xz
 https://download.gnome.org/sources/gedit/47/gedit-47.0.tar.xz
 https://github.com/gedit-technology/libgedit-tepl/releases/download/6.10.0/libgedit-tepl-6.10.0.tar.xz
 https://github.com/gedit-technology/libgedit-gfls/releases/download/0.1.0/libgedit-gfls-0.1.0.tar.xz
