diff --git a/wget-list b/wget-list
index fc1a1a464..5e29a07a1 100644
--- a/wget-list
+++ b/wget-list
@@ -264,7 +264,7 @@ https://www.freedesktop.org/software/ModemManager/ModemManager-1.18.12.tar.xz
 https://download.gnome.org/sources/notification-daemon/3.20/notification-daemon-3.20.0.tar.xz
 https://github.com/p7zip-project/p7zip/archive/v17.04/p7zip-17.04.tar.gz
 http://www.mirbsd.org/MirOS/dist/mir/cpio/paxmirabilis-20201030.tgz
-https://mj.ucw.cz/download/linux/pci/pciutils-3.12.0.tar.gz
+https://mj.ucw.cz/download/linux/pci/pciutils-3.13.0.tar.gz
 https://pm-utils.freedesktop.org/releases/pm-utils-1.4.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/pm-utils-1.4.1-bugfixes-1.patch
 https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/archive/0.21/power-profiles-daemon-0.21.tar.gz
@@ -915,7 +915,7 @@ https://github.com/transmission/transmission/releases/download/4.0.6/transmissio
 https://www.linuxfromscratch.org/patches/blfs/svn/transmission-4.0.6-build_fix-1.patch
 https://github.com/ib/xarchiver/archive/0.5.4.23/xarchiver-0.5.4.23.tar.gz
 https://gitlab.freedesktop.org/xdg/xdg-utils/-/archive/v1.2.1/xdg-utils-v1.2.1.tar.gz
-https://www.jwz.org/xscreensaver/xscreensaver-6.08.tar.gz
+https://www.jwz.org/xscreensaver/xscreensaver-6.09.tar.gz
 https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.11.tar.bz2
 https://www.alsa-project.org/files/pub/plugins/alsa-plugins-1.2.7.1.tar.bz2
 https://www.alsa-project.org/files/pub/utils/alsa-utils-1.2.11.tar.bz2
