diff --git a/wget-list b/wget-list
index 9b8906492..85e2cc5f2 100644
--- a/wget-list
+++ b/wget-list
@@ -103,7 +103,7 @@ https://github.com/storaged-project/libbytesize/releases/download/2.10/libbytesi
 https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.7/libclc-18.1.7.src.tar.xz
 https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
-https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.1.1/downloads//libdisplay-info-0.1.1.tar.xz
+https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.2.0/downloads//libdisplay-info-0.2.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.11.0.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.50.tar.bz2
 https://download.gnome.org/sources/libgrss/0.7/libgrss-0.7.0.tar.xz
