diff --git a/wget-list b/wget-list
index 27a01a07..fe223342 100644
--- a/wget-list
+++ b/wget-list
@@ -28,7 +28,6 @@ ftp://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.4p1.tar.gz
 ftp://ftp.stunnel.org/stunnel/archive/5.x/stunnel-5.70.tar.gz
 ftp://ftp.sudo.ws/pub/sudo/sudo-1.9.14p3.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
-https://github.com/felixonmars/volume_key/archive/volume_key-0.3.12.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.3.3.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
 https://github.com/libfuse/libfuse/releases/download/fuse-3.16.1/fuse-3.16.1.tar.gz
@@ -97,7 +96,7 @@ https://github.com/libarchive/libarchive/releases/download/v3.7.1/libarchive-3.7
 ftp://ftp.gnupg.org/gcrypt/libassuan/libassuan-2.5.6.tar.bz2
 https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/ivmai/libatomic_ops/releases/download/v7.8.0/libatomic_ops-7.8.0.tar.gz
-https://github.com/storaged-project/libblockdev/releases/download/3.0.2-1/libblockdev-3.0.2.tar.gz
+https://github.com/storaged-project/libblockdev/releases/download/3.0.3-1/libblockdev-3.0.3.tar.gz
 https://github.com/storaged-project/libbytesize/releases/download/2.9/libbytesize-2.9.tar.gz
 ftp://ftp.acc.umu.se/pub/gnome/sources/libcloudproviders/0.3/libcloudproviders-0.3.2.tar.xz
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
@@ -164,8 +163,8 @@ https://oligarchy.co.uk/xapian/1.4.23/xapian-core-1.4.23.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
 https://download.gimp.org/pub/babl/0.1/babl-0.1.106.tar.xz
 https://github.com/Exiv2/exiv2/releases/download/v0.28.0/exiv2-0.28.0-Source.tar.gz
-https://downloads.sourceforge.net/freetype/freetype-2.13.1.tar.xz
-https://downloads.sourceforge.net/freetype/freetype-doc-2.13.1.tar.xz
+https://downloads.sourceforge.net/freetype/freetype-2.13.2.tar.xz
+https://downloads.sourceforge.net/freetype/freetype-doc-2.13.2.tar.xz
 https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.14.2.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.13/fribidi-1.0.13.tar.xz
 https://download.gimp.org/pub/gegl/0.4/gegl-0.4.46.tar.xz
@@ -213,7 +212,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/desktop-file-utils-0.26-update
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.1.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/8.1.0/graphviz-8.1.0.tar.bz2
 ftp://ftp.acc.umu.se/pub/gnome/sources/gtk-doc/1.33/gtk-doc-1.33.2.tar.xz
-http://www.andre-simon.de/zip/highlight-4.7.tar.bz2
+http://www.andre-simon.de/zip/highlight-4.8.tar.bz2
 https://github.com/ibus/ibus/releases/download/1.5.28/ibus-1.5.28.tar.gz
 https://www.unicode.org/Public/zipped/15.0.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-15.tar.xz
@@ -236,7 +235,7 @@ https://www.freedesktop.org/software/accountsservice/accountsservice-23.13.9.tar
 https://downloads.sourceforge.net/acpid2/acpid-2.0.34.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/at/at_3.2.5.orig.tar.gz
 https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.8.tar.xz
-https://www.kernel.org/pub/linux/bluetooth/bluez-5.68.tar.xz
+https://www.kernel.org/pub/linux/bluetooth/bluez-5.69.tar.xz
 https://github.com/containers/bubblewrap/releases/download/v0.8.0/bubblewrap-0.8.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-1.4.6.tar.xz
 ftp://ftp.gnu.org/gnu/cpio/cpio-2.14.tar.bz2
@@ -273,7 +272,7 @@ ftp://ftp.gnu.org/gnu/which/which-2.21.tar.gz
 ftp://ftp.info-zip.org/pub/infozip/src/zip30.tgz
 ftp://ftp.gnu.org/gnu/autoconf/autoconf-2.13.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/autoconf-2.13-consolidated_fixes-1.patch
-https://github.com/eqrion/cbindgen/archive/v0.24.3/cbindgen-0.24.3.tar.gz
+https://github.com/mozilla/cbindgen/archive/v0.25.0/cbindgen-0.25.0.tar.gz
 ftp://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
 https://cmake.org/files/v3.27/cmake-3.27.4.tar.gz
@@ -285,7 +284,6 @@ https://www.kernel.org/pub/software/scm/git/git-2.42.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.42.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.42.0.tar.xz
 ftp://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.xz
-https://download.tuxfamily.org/librep/librep_0.92.7.tar.xz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.5/llvm-16.0.5.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party.src.tar.xz
@@ -522,7 +520,7 @@ https://files.pythonhosted.org/packages/source/t/typing_extensions/typing_extens
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.0.7.tar.gz
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.0.2.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.2/ruby-3.2.2.tar.xz
-https://static.rust-lang.org/dist/rustc-1.71.1-src.tar.xz
+https://static.rust-lang.org/dist/rustc-1.72.0-src.tar.xz
 https://downloads.sourceforge.net/scons/SCons-4.5.2.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.2.tar.bz2
@@ -572,8 +570,8 @@ https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.1.1/kdsoap-2.1.1.tar.g
 https://www.nlnetlabs.nl/downloads/ldns/ldns-1.8.3.tar.gz
 https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
 https://netfilter.org/projects/libmnl/files/libmnl-1.0.5.tar.bz2
-https://github.com/thom311/libnl/releases/download/libnl3_7_0/libnl-3.7.0.tar.gz
-https://github.com/thom311/libnl/releases/download/libnl3_7_0/libnl-doc-3.7.0.tar.gz
+https://github.com/thom311/libnl/releases/download/libnl3_8_0/libnl-3.8.0.tar.gz
+https://github.com/thom311/libnl/releases/download/libnl3_8_0/libnl-doc-3.8.0.tar.gz
 ftp://ftp.acc.umu.se/pub/gnome/sources/libnma/1.10/libnma-1.10.6.tar.xz
 https://github.com/thkukuk/libnsl/releases/download/v2.0.0/libnsl-2.0.0.tar.xz
 https://www.tcpdump.org/release/libpcap-1.10.4.tar.gz
@@ -640,8 +638,8 @@ ftp://ftp.x.org/pub/individual/xserver/xorg-server-21.1.8.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.8-tearfree_backport-1.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.1.tar.xz
 ftp://ftp.x.org/pub/individual/driver/xf86-input-evdev-2.10.6.tar.bz2
-https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.23.0/libinput-1.23.0.tar.gz
-ftp://ftp.x.org/pub/individual/driver/xf86-input-libinput-1.3.0.tar.xz
+https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.24.0/libinput-1.24.0.tar.gz
+ftp://ftp.x.org/pub/individual/driver/xf86-input-libinput-1.4.0.tar.xz
 ftp://ftp.x.org/pub/individual/driver/xf86-input-synaptics-1.9.2.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.0/xf86-input-wacom-1.2.0.tar.bz2
 ftp://ftp.x.org/pub/individual/app/twm-1.0.12.tar.xz
@@ -649,7 +647,7 @@ https://invisible-mirror.net/archives/xterm/xterm-384.tgz
 ftp://ftp.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 ftp://ftp.x.org/pub/individual/app/xinit-1.4.2.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/atkmm/2.28/atkmm-2.28.3.tar.xz
-ftp://ftp.acc.umu.se/pub/gnome/sources/at-spi2-core/2.48/at-spi2-core-2.48.3.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/at-spi2-core/2.48/at-spi2-core-2.48.4.tar.xz
 https://download.gnome.org/sources/cairo/1.17/cairo-1.17.6.tar.xz
 https://www.cairographics.org/releases/cairomm-1.14.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/cogl/1.22/cogl-1.22.8.tar.xz
@@ -913,7 +911,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.13.1-configuration_
 https://github.com/transmission/transmission/releases/download/4.0.3/transmission-4.0.3.tar.xz
 https://github.com/ib/xarchiver/archive/0.5.4.21/xarchiver-0.5.4.21.tar.gz
 https://portland.freedesktop.org/download/xdg-utils-1.1.3.tar.gz
-https://www.jwz.org/xscreensaver/xscreensaver-6.06.tar.gz
+https://www.jwz.org/xscreensaver/xscreensaver-6.07.tar.gz
 ftp://ftp.alsa-project.org/pub/lib/alsa-lib-1.2.9.tar.bz2
 ftp://ftp.alsa-project.org/pub/plugins/alsa-plugins-1.2.7.1.tar.bz2
 ftp://ftp.alsa-project.org/pub/utils/alsa-utils-1.2.9.tar.bz2
@@ -1058,6 +1056,6 @@ https://github.com/mgieseki/dvisvgm/releases/download/3.1/dvisvgm-3.1.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20230824.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/bluez-5.68-obexd_without_systemd-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/bluez-5.69-obexd_without_systemd-1.patch
 https://github.com/elogind/elogind/archive/v252.9/elogind-252.9.tar.gz
 https://github.com/lfs-book/blocaled/releases/download/v0.5/blocaled-0.5.tar.xz
