diff --git a/wget-list b/wget-list
index c3cbf492..69c93184 100644
--- a/wget-list
+++ b/wget-list
@@ -284,12 +284,12 @@ https://www.kernel.org/pub/software/scm/git/git-2.42.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.42.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.42.0.tar.xz
 ftp://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.5/llvm-16.0.5.src.tar.xz
-https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake.src.tar.xz
-https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party.src.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.5/clang-16.0.5.src.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/clang-16.0.5-enable_default_ssp-1.patch
-https://github.com/llvm/llvm-project/releases/download/llvmorg-16.0.5/compiler-rt-16.0.5.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.1/llvm-17.0.1.src.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-17.src.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party-17.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.1/clang-17.0.1.src.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/clang-17-enable_default_ssp-1.patch
+https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.1/compiler-rt-17.0.1.src.tar.xz
 https://www.lua.org/ftp/lua-5.4.6.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.6-shared_library-1.patch
 https://www.lua.org/tests/lua-5.4.6-tests.tar.gz
@@ -692,7 +692,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.15-build_fixe
 https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.15-ffmpeg5_fixes-1.patch
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 ftp://ftp.acc.umu.se/pub/gnome/sources/tepl/6.8/tepl-6.8.0.tar.xz
-https://webkitgtk.org/releases/webkitgtk-2.41.92.tar.xz
+https://webkitgtk.org/releases/webkitgtk-2.42.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gdm/44/gdm-44.1.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.8/lightdm-gtk-greeter-2.0.8.tar.gz
@@ -820,7 +820,7 @@ https://archive.xfce.org/src/apps/xfce4-terminal/1.1/xfce4-terminal-1.1.0.tar.bz
 https://archive.xfce.org/src/apps/xfburn/0.7/xfburn-0.7.0.tar.bz2
 https://archive.xfce.org/src/apps/ristretto/0.13/ristretto-0.13.1.tar.bz2
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.18/xfce4-dev-tools-4.18.0.tar.bz2
-https://archive.xfce.org/src/apps/xfce4-notifyd/0.8/xfce4-notifyd-0.8.2.tar.bz2
+https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.0.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.4/xfce4-pulseaudio-plugin-0.4.7.tar.bz2
 https://download.kde.org/stable/frameworks/5.109/kwindowsystem-5.109.0.tar.xz
 https://download.kde.org/stable/frameworks/5.109/kwayland-5.109.0.tar.xz
