diff --git a/wget-list b/wget-list
index ac2d2c7c..63475b67 100644
--- a/wget-list
+++ b/wget-list
@@ -606,7 +606,7 @@ ftp://ftp.porcupine.org/mirrors/postfix-release/official/postfix-3.8.2.tar.gz
 ftp://ftp.sendmail.org/pub/sendmail/sendmail.8.17.2.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/bdb/db-5.3.28.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
-https://downloads.mariadb.org/interstitial/mariadb-10.11.4/source/mariadb-10.11.4.tar.gz
+https://downloads.mariadb.org/interstitial/mariadb-10.11.5/source/mariadb-10.11.5.tar.gz
 https://ftp.postgresql.org/pub/source/v15.4/postgresql-15.4.tar.bz2
 https://sqlite.org/2023/sqlite-autoconf-3430000.tar.gz
 https://sqlite.org/2023/sqlite-doc-3430000.zip
