diff --git a/wget-list b/wget-list
index e507bab9f6..2fd45b43c3 100644
--- a/wget-list
+++ b/wget-list
@@ -47,8 +47,6 @@ https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.19.0.tar.xz
 https://github.com/rhboot/efivar/archive/39/efivar-39.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/efivar-39-upstream_fixes-1.patch
 https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
-https://ftpmirror.gnu.org/grub/grub-2.14.tar.xz
-https://unifoundry.com/pub/unifont/unifont-17.0.03/font-builds/unifont-17.0.03.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.4.0.tar.bz2
 https://ftpmirror.gnu.org/ed/ed-1.22.5.tar.lz
 https://ftpmirror.gnu.org/emacs/emacs-30.2.tar.xz
