diff --git a/wget-list b/wget-list
index f852611f8b..2bd7e3d69d 100644
--- a/wget-list
+++ b/wget-list
@@ -58,7 +58,7 @@ https://gitlab.gnome.org/World/gedit/libgedit-tepl/-/archive/6.13.0/libgedit-tep
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/25.12.2/src/kate-25.12.2.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.7/mousepad-0.7.0.tar.xz
-https://www.nano-editor.org/dist/v8/nano-8.7.1.tar.xz
+https://www.nano-editor.org/dist/v8/nano-9.0.tar.xz
 https://github.com/vim/vim/archive/v9.2.0272/vim-9.2.0272.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.2.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
@@ -249,7 +249,7 @@ https://ftpmirror.gnu.org/time/time-1.9.tar.gz
 https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.3.2/unix-tree-2.3.2.tar.bz2
 https://github.com/lurcher/unixODBC/archive/v2.3.14/unixODBC-2.3.14.tar.gz
 https://github.com/flatpak/xdg-dbus-proxy/releases/download/0.1.6/xdg-dbus-proxy-0.1.6.tar.xz
-https://gitlab.freedesktop.org/xdg/xdg-user-dirs/-/archive/v0.19/xdg-user-dirs-v0.19.tar.gz
+https://gitlab.freedesktop.org/xdg/xdg-user-dirs/-/archive/v0.20/xdg-user-dirs-v0.20.tar.gz
 https://github.com/ip7z/7zip/archive/26.00/7zip-26.00.tar.gz
 https://www.freedesktop.org/software/accountsservice/accountsservice-23.13.9.tar.xz
 https://downloads.sourceforge.net/acpid2/acpid-2.0.34.tar.xz
