diff --git a/wget-list b/wget-list
index 2857b4d367..991864324e 100644
--- a/wget-list
+++ b/wget-list
@@ -230,7 +230,7 @@ https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-ut
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.5.tar.gz
 https://github.com/google/shaderc/archive/v2026.1/shaderc-2026.1.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/14.1.4/graphviz-14.1.4.tar.bz2
-https://download.gnome.org/sources/gtk-doc/1.36/gtk-doc-1.36.0.tar.xz
+https://download.gnome.org/sources/gtk-doc/1.36/gtk-doc-1.36.1.tar.xz
 http://www.andre-simon.de/zip/highlight-4.19.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.33/ibus-1.5.33.tar.gz
 https://www.unicode.org/Public/17.0.0/ucd/UCD.zip
@@ -468,7 +468,7 @@ https://www.python.org/ftp/python/doc/3.14.3/python-3.14.3-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://download.gnome.org/sources/blueprint-compiler/0.18/blueprint-compiler-0.18.0.tar.xz
-https://files.pythonhosted.org/packages/source/b/build/build-1.4.2.tar.gz
+https://files.pythonhosted.org/packages/source/b/build/build-1.4.3.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.4/cachecontrol-0.14.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.4.0.tar.gz
 https://github.com/cython/cython/releases/download/3.2.4/cython-3.2.4.tar.gz
@@ -564,7 +564,7 @@ https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/OpenJDK-21.0.10
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/OpenJDK-21.0.10+6-x86_64-bin.tar.xz
 https://github.com/openjdk/jdk21u/archive/jdk-21.0.10-ga.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/jtreg-8.2.1+1.tar.gz
-https://archive.apache.org/dist/ant/source/apache-ant-1.10.16-src.tar.xz
+https://archive.apache.org/dist/ant/source/apache-ant-1.10.17-src.tar.xz
 https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.5.tar.bz2
 https://www.ncftp.com/public_ftp/ncftp/ncftp-3.3.0-src.tar.gz
 https://downloads.sourceforge.net/project/net-tools/net-tools-2.10.tar.xz
@@ -725,7 +725,7 @@ https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.1.0.tar.xz
 https://download.kde.org/stable/frameworks/6.23/extra-cmake-modules-6.23.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
-https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
+https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.201.1.tar.xz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.20.0.tar.xz
 https://download.kde.org/stable/release-service/25.12.2/src/ark-25.12.2.tar.xz
 https://download.kde.org/stable/release-service/25.12.2/src/dolphin-25.12.2.tar.xz
