diff --git a/wget-list b/wget-list
index b7e375ba04..5103c46561 100644
--- a/wget-list
+++ b/wget-list
@@ -169,7 +169,7 @@ https://archive.mozilla.org/pub/firefox/releases/140.9.0esr/source/firefox-140.9
 https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.9.0-python_3.14_fixes-1.patch
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.341.0/SPIRV-Headers-vulkan-sdk-1.4.341.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.341.0/SPIRV-Tools-vulkan-sdk-1.4.341.0.tar.gz
-https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.0/SPIRV-LLVM-Translator-22.1.0.tar.gz
+https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.1/SPIRV-LLVM-Translator-22.1.1.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.4.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.7/umockdev-0.19.7.tar.xz
@@ -671,14 +671,14 @@ https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
 https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-source.tar.gz
 https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-docs-html.tar.gz
 https://downloads.sourceforge.net/freeglut/freeglut-3.8.0.tar.gz
-https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.5.tar.xz
+https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.6.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.3.1.tgz
 https://github.com/KhronosGroup/glslang/archive/16.2.0/glslang-16.2.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.60.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk/3.24/gtk-3.24.52.tar.xz
-https://download.gnome.org/sources/gtk/4.22/gtk-4.22.1.tar.xz
+https://download.gnome.org/sources/gtk/4.22/gtk-4.22.2.tar.xz
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.10.tar.xz
 https://download.gnome.org/sources/gtkmm/4.22/gtkmm-4.22.0.tar.xz
 https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
@@ -891,11 +891,10 @@ https://github.com/lxqt/qtermwidget/releases/download/2.3.0/qtermwidget-2.3.0.ta
 https://github.com/lxqt/qterminal/releases/download/2.3.0/qterminal-2.3.0.tar.xz
 https://github.com/lxqt/screengrab/releases/download/3.1.0/screengrab-3.1.0.tar.xz
 https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.60.tar.xz
-https://download.documentfoundation.org/libreoffice/src/26.2.1/libreoffice-26.2.1.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/26.2.1/libreoffice-dictionaries-26.2.1.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/26.2.1/libreoffice-help-26.2.1.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/26.2.1/libreoffice-translations-26.2.1.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-26.2.1.2-poppler_26.02-1.patch
+https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-26.2.2.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-dictionaries-26.2.2.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-help-26.2.2.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-translations-26.2.2.2.tar.xz
 https://download.gnome.org/sources/epiphany/49/epiphany-49.3.tar.xz
 https://download.kde.org/stable/release-service/25.12.2/src/falkon-25.12.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.9.0esr-ffmpeg-8.0.patch
@@ -909,7 +908,7 @@ https://gitlab.gnome.org/GNOME/balsa/-/archive/2.6.5/balsa-2.6.5.tar.gz
 https://feh.finalrewind.org/feh-3.11.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20251009/fontforge-20251009.tar.xz
 https://github.com/freerdp/freerdp/archive/3.24.2/FreeRDP-3.24.2.tar.gz
-https://download.gimp.org/gimp/v3.2/gimp-3.2.0.tar.xz
+https://download.gimp.org/gimp/v3.2/gimp-3.2.2.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.2.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.8.1.tar.gz
 https://inkscape.org/gallery/item/58914/inkscape-1.4.3.tar.xz
