diff --git a/wget-list b/wget-list
index 3507ff0ea3..f49371c317 100644
--- a/wget-list
+++ b/wget-list
@@ -680,7 +680,7 @@ https://download.gnome.org/sources/gtkmm/4.22/gtkmm-4.22.0.tar.xz
 https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
 https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
 https://download.gnome.org/sources/gtksourceview/4.8/gtksourceview-4.8.4.tar.xz
-https://download.gnome.org/sources/gtksourceview/5.18/gtksourceview-5.18.0.tar.xz
+https://download.gnome.org/sources/gtksourceview/5.20/gtksourceview-5.20.0.tar.xz
 https://downloads.sourceforge.net/enlightenment/imlib2-1.12.6.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.2/kImageAnnotator-0.7.2.tar.gz
@@ -711,7 +711,7 @@ https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/4.0.0/icewm-4.0.0.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openbox-3.6.1-py3-1.patch
-https://download.gnome.org/sources/adwaita-icon-theme/49/adwaita-icon-theme-49.0.tar.xz
+https://download.gnome.org/sources/adwaita-icon-theme/50/adwaita-icon-theme-50.0.tar.xz
 https://download.kde.org/stable/frameworks/6.23/breeze-icons-6.23.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
@@ -1026,7 +1026,7 @@ https://github.com/OpenPrinting/libcupsfilters/releases/download/2.1.1/libcupsfi
 https://www.linuxfromscratch.org/patches/blfs/svn/libcupsfilters-2.1.1-security_fixes-1.patch
 https://github.com/OpenPrinting/libppd/releases/download/2.1.1/libppd-2.1.1.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/sane/sane-backends-1.4.0.tar.gz
-https://download.gnome.org/sources/simple-scan/49/simple-scan-49.1.tar.xz
+https://download.gnome.org/sources/simple-scan/50/simple-scan-50.0.tar.xz
 https://sourceware.org/ftp/docbook-tools/new-trials/SOURCES/sgml-common-0.6.3.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/sgml-common-0.6.3-manpage-1.patch
 https://archive.docbook.org/sgml/3.1/docbk31.zip
