diff --git a/wget-list b/wget-list
index 192de0ba2..ea6393324 100644
--- a/wget-list
+++ b/wget-list
@@ -93,7 +93,7 @@ https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
 https://s3.amazonaws.com/json-c_releases/releases/json-c-0.17.tar.gz
 https://download.gnome.org/sources/json-glib/1.8/json-glib-1.8.0.tar.xz
 https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git/snapshot/keyutils-1.6.3.tar.gz
-https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.3.0.tar.xz
+https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.4.0.tar.xz
 https://pagure.io/libaio/archive/libaio-0.3.113/libaio-0.3.113.tar.gz
 https://github.com/libarchive/libarchive/releases/download/v3.7.4/libarchive-3.7.4.tar.xz
 https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-3.0.1.tar.bz2
@@ -152,7 +152,7 @@ https://www.gnupg.org/ftp/gcrypt/npth/npth-1.7.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.44/pcre2-10.44.tar.bz2
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
-https://github.com/protocolbuffers/protobuf/releases/download/v27.2/protobuf-27.2.tar.gz
+https://github.com/protocolbuffers/protobuf/releases/download/v27.3/protobuf-27.3.tar.gz
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.0/protobuf-c-1.5.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.0-consolidated_fixes-1.patch
 https://download.kde.org/stable/qca/2.3.9/qca-2.3.9.tar.xz
@@ -711,7 +711,7 @@ https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/3.6.0/icewm-3.6.0.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://download.gnome.org/sources/adwaita-icon-theme/46/adwaita-icon-theme-46.2.tar.xz
-https://download.kde.org/stable/frameworks/6.2/breeze-icons-6.2.0.tar.xz
+https://download.kde.org/stable/frameworks/6.5/breeze-icons-6.5.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-symbolic/3.12/gnome-icon-theme-symbolic-3.12.0.tar.xz
@@ -720,7 +720,7 @@ https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.18.tar.xz
 http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
 https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.0.0.tar.xz
-https://download.kde.org/stable/frameworks/6.2/extra-cmake-modules-6.2.0.tar.xz
+https://download.kde.org/stable/frameworks/6.5/extra-cmake-modules-6.5.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
@@ -734,10 +734,11 @@ https://download.kde.org/stable/release-service/24.05.0/src/konsole-24.05.0.tar.
 https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.05.0-scrollbar-1.patch
 https://download.kde.org/stable/release-service/24.05.0/src/konversation-24.05.0.tar.xz
 https://download.kde.org/stable/release-service/24.05.0/src/libkexiv2-24.05.0.tar.xz
-https://download.kde.org/stable/plasma/6.0.5/plasma-activities-6.0.5.tar.xz
+https://download.kde.org/stable/plasma/6.1.4/plasma-activities-6.1.4.tar.xz
 https://download.kde.org/stable/release-service/24.05.0/src/okular-24.05.0.tar.xz
 https://download.kde.org/stable/release-service/24.05.0/src/libkdcraw-24.05.0.tar.xz
 https://download.kde.org/stable/release-service/24.05.0/src/gwenview-24.05.0.tar.xz
+https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.5.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/46/gsettings-desktop-schemas-46.1.tar.xz
@@ -830,13 +831,13 @@ https://archive.xfce.org/src/apps/ristretto/0.13/ristretto-0.13.2.tar.bz2
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.18/xfce4-dev-tools-4.18.1.tar.bz2
 https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.4.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.4/xfce4-pulseaudio-plugin-0.4.8.tar.bz2
-https://download.kde.org/stable/frameworks/6.2/kwindowsystem-6.2.0.tar.xz
-https://download.kde.org/stable/frameworks/6.2/kconfig-6.2.0.tar.xz
-https://download.kde.org/stable/frameworks/6.2/solid-6.2.0.tar.xz
-https://download.kde.org/stable/frameworks/6.2/kidletime-6.2.0.tar.xz
-http://download.kde.org/stable/plasma/6.0.5/kwayland-6.0.5.tar.xz
-http://download.kde.org/stable/plasma/6.0.5/libkscreen-6.0.5.tar.xz
-http://download.kde.org/stable/plasma/6.0.5/layer-shell-qt-6.0.5.tar.xz
+https://download.kde.org/stable/frameworks/6.5/kwindowsystem-6.5.0.tar.xz
+https://download.kde.org/stable/frameworks/6.5/kconfig-6.5.0.tar.xz
+https://download.kde.org/stable/frameworks/6.5/solid-6.5.0.tar.xz
+https://download.kde.org/stable/frameworks/6.5/kidletime-6.5.0.tar.xz
+http://download.kde.org/stable/plasma/6.1.4/kwayland-6.1.4.tar.xz
+http://download.kde.org/stable/plasma/6.1.4/libkscreen-6.1.4.tar.xz
+http://download.kde.org/stable/plasma/6.1.4/layer-shell-qt-6.1.4.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.4/muparser-2.3.4.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/2.0.0/lxqt-build-tools-2.0.0.tar.xz
 https://github.com/lxqt/libqtxdg/releases/download/4.0.0/libqtxdg-4.0.0.tar.xz
@@ -959,7 +960,7 @@ https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.14.1/libvpx-1.14.1.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.24.0/mlt-7.24.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz
-https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2.1/pipewire-1.2.1.tar.bz2
+https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2.2/pipewire-1.2.2.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.0.tar.xz
 https://github.com/libsdl-org/sdl12-compat/archive/release-1.2.68/sdl12-compat-release-1.2.68.tar.gz
