diff --git a/wget-list b/wget-list
index 97e416242..4e7b159cf 100644
--- a/wget-list
+++ b/wget-list
@@ -51,7 +51,7 @@ https://download.gnome.org/sources/gedit/47/gedit-47.0.tar.xz
 https://github.com/gedit-technology/libgedit-tepl/releases/download/6.10.0/libgedit-tepl-6.10.0.tar.xz
 https://github.com/gedit-technology/libgedit-gfls/releases/download/0.1.0/libgedit-gfls-0.1.0.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/24.05.0/src/kate-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/kate-24.08.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v8/nano-8.1.tar.xz
 https://github.com/vim/vim/archive/v9.1.0660/vim-9.1.0660.tar.gz
@@ -687,6 +687,7 @@ https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3
 https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.3.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.3.0/libei-1.3.0.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
+https://github.com/gedit-technology/libgedit-gtksourceview/releases/download/299.2.1/libgedit-gtksourceview-299.2.1.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
 https://dri.freedesktop.org/libdrm/libdrm-2.4.122.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
@@ -728,19 +729,21 @@ https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.13.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/ark-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/dolphin-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/dolphin-plugins-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/kdenlive-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/khelpcenter-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/konsole-24.05.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.05.0-scrollbar-1.patch
-https://download.kde.org/stable/release-service/24.05.0/src/konversation-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/libkexiv2-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/ark-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/dolphin-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/dolphin-plugins-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/kdenlive-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/khelpcenter-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/konsole-24.08.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.08.0-scrollbar-1.patch
+https://download.kde.org/stable/release-service/24.08.0/src/konversation-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/libkexiv2-24.08.0.tar.xz
 https://download.kde.org/stable/plasma/6.1.4/plasma-activities-6.1.4.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/okular-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/libkdcraw-24.05.0.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/gwenview-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/okular-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/libkdcraw-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/gwenview-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/libkcddb-24.08.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/k3b-24.08.0.tar.xz
 https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.4.0.tar.xz
 https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.5.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
@@ -754,7 +757,7 @@ https://download.gnome.org/sources/yelp-xsl/42/yelp-xsl-42.1.tar.xz
 https://download.gnome.org/sources/geocode-glib/3.26/geocode-glib-3.26.4.tar.xz
 https://download.gnome.org/sources/gjs/1.80/gjs-1.80.2.tar.xz
 https://download.gnome.org/sources/gnome-autoar/0.4/gnome-autoar-0.4.4.tar.xz
-https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.0.tar.xz
+https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.1.tar.xz
 https://download.gnome.org/sources/gnome-menus/3.36/gnome-menus-3.36.0.tar.xz
 https://download.gnome.org/sources/gnome-online-accounts/3.50/gnome-online-accounts-3.50.4.tar.xz
 https://download.gnome.org/sources/grilo/0.3/grilo-0.3.16.tar.xz
@@ -888,7 +891,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-help-
 https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-translations-24.2.5.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.2.5.2-boost_fixes-2.patch
 https://download.gnome.org/sources/epiphany/46/epiphany-46.3.tar.xz
-https://download.kde.org/stable/release-service/24.05.0/src/falkon-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.08.0/src/falkon-24.08.0.tar.xz
 https://archive.mozilla.org/pub/firefox/releases/128.1.0esr/source/firefox-128.1.0esr.source.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.18.2/source/seamonkey-2.53.18.2.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.18.2-cxx17-1.patch
@@ -965,7 +968,7 @@ https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.14.1/libvpx-1.14.1.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.26.0/mlt-7.26.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz
-https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2.2/pipewire-1.2.2.tar.bz2
+https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2.3/pipewire-1.2.3.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.0.tar.xz
 https://github.com/libsdl-org/sdl12-compat/archive/release-1.2.68/sdl12-compat-release-1.2.68.tar.gz
