diff --git a/wget-list b/wget-list
index 0a6c49494..adfe93d24 100644
--- a/wget-list
+++ b/wget-list
@@ -31,7 +31,7 @@ https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.t
 https://github.com/libfuse/libfuse/releases/download/fuse-3.16.2/fuse-3.16.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/jfsutils-1.1.15-gcc10_fix-1.patch
-https://sourceware.org/ftp/lvm2/LVM2.2.03.25.tgz
+https://sourceware.org/ftp/lvm2/LVM2.2.03.26.tgz
 https://www.kernel.org/pub/linux/utils/raid/mdadm/mdadm-4.3.tar.xz
 https://tuxera.com/opensource/ntfs-3g_ntfsprogs-2022.10.3.tgz
 https://downloads.sourceforge.net/gptfdisk/gptfdisk-1.0.10.tar.gz
@@ -101,7 +101,7 @@ https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-3.0.1.tar.bz2
 https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/ivmai/libatomic_ops/releases/download/v7.8.2/libatomic_ops-7.8.2.tar.gz
 https://github.com/storaged-project/libblockdev/releases/download/3.1.1-1/libblockdev-3.1.1.tar.gz
-https://github.com/storaged-project/libbytesize/releases/download/2.10/libbytesize-2.10.tar.gz
+https://github.com/storaged-project/libbytesize/releases/download/2.11/libbytesize-2.11.tar.gz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.7/libclc-18.1.7.src.tar.xz
 https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
