diff --git a/wget-list b/wget-list
index 73336765e..875333d5b 100644
--- a/wget-list
+++ b/wget-list
@@ -287,7 +287,7 @@ https://downloads.sourceforge.net/infozip/zip30.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
-https://cmake.org/files/v3.30/cmake-3.30.1.tar.gz
+https://cmake.org/files/v3.30/cmake-3.30.2.tar.gz
 https://doxygen.nl/files/doxygen-1.11.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.0.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-14.2.0/gcc-14.2.0.tar.xz
@@ -771,17 +771,17 @@ https://download.gnome.org/sources/gnome-backgrounds/46/gnome-backgrounds-46.0.t
 https://download.gnome.org/sources/gvfs/1.54/gvfs-1.54.2.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.3.tar.xz
 https://download.gnome.org/sources/nautilus/46/nautilus-46.2.tar.xz
-https://download.gnome.org/sources/gnome-bluetooth/46/gnome-bluetooth-46.0.tar.xz
+https://download.gnome.org/sources/gnome-bluetooth/46/gnome-bluetooth-46.1.tar.xz
 https://download.gnome.org/sources/gnome-keyring/46/gnome-keyring-46.2.tar.xz
 https://download.gnome.org/sources/gnome-settings-daemon/46/gnome-settings-daemon-46.0.tar.xz
 https://download.gnome.org/sources/tecla/46/tecla-46.0.tar.xz
 https://download.gnome.org/sources/gnome-control-center/46/gnome-control-center-46.3.tar.xz
-https://download.gnome.org/sources/mutter/46/mutter-46.3.1.tar.xz
-https://download.gnome.org/sources/gnome-shell/46/gnome-shell-46.3.1.tar.xz
+https://download.gnome.org/sources/mutter/46/mutter-46.4.tar.xz
+https://download.gnome.org/sources/gnome-shell/46/gnome-shell-46.4.tar.xz
 https://download.gnome.org/sources/gnome-shell-extensions/46/gnome-shell-extensions-46.2.tar.xz
 https://download.gnome.org/sources/gnome-session/46/gnome-session-46.0.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.1.tar.xz
-https://download.gnome.org/sources/gnome-user-docs/46/gnome-user-docs-46.1.tar.xz
+https://download.gnome.org/sources/gnome-user-docs/46/gnome-user-docs-46.4.tar.xz
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
 https://download.gnome.org/sources/baobab/46/baobab-46.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
@@ -881,7 +881,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-24.2.
 https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-dictionaries-24.2.5.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-help-24.2.5.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-translations-24.2.5.2.tar.xz
-https://download.gnome.org/sources/epiphany/46/epiphany-46.2.tar.xz
+https://download.gnome.org/sources/epiphany/46/epiphany-46.3.tar.xz
 https://download.kde.org/stable/release-service/24.05.0/src/falkon-24.05.0.tar.xz
 https://archive.mozilla.org/pub/firefox/releases/128.0esr/source/firefox-128.0esr.source.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.18.2/source/seamonkey-2.53.18.2.source.tar.xz
@@ -985,8 +985,8 @@ https://downloads.sourceforge.net/mpg123/mpg123-1.32.6.tar.bz2
 https://freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-6.0.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.2.tar.gz
-https://ffmpeg.org/releases/ffmpeg-7.0.1.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-7.0.1-chromium_method-1.patch
+https://ffmpeg.org/releases/ffmpeg-7.0.2.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-7.0.2-chromium_method-1.patch
 https://github.com/mpv-player/mpv/archive/v0.38.0/mpv-0.38.0.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/transcode/transcode-1.1.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/transcode-1.1.7-gcc10_fix-1.patch
@@ -1048,4 +1048,4 @@ https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20240416.tar.xz
 https://github.com/elogind/elogind/archive/v255.5/elogind-255.5.tar.gz
-https://github.com/lfs-book/blocaled/releases/download/v0.5/blocaled-0.5.tar.xz
+https://github.com/lfs-book/blocaled/releases/download/v0.6/blocaled-0.6.tar.xz
