diff --git a/wget-list b/wget-list
index 8b493ad6a..9e25a72a3 100644
--- a/wget-list
+++ b/wget-list
@@ -140,6 +140,7 @@ https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
 https://dist.libuv.org/dist/v1.48.0/libuv-v1.48.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.7.0.tar.xz
 https://download.gnome.org/sources/libxml2/2.13/libxml2-2.13.3.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/libxml2-2.13.3-upstream_fix-1.patch
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.19/libxmlb-0.3.19.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.42.tar.xz
