diff --git a/wget-list b/wget-list
index 1d0045c0a..3fa4695e2 100644
--- a/wget-list
+++ b/wget-list
@@ -117,7 +117,7 @@ https://ftp.gnu.org/gnu/libidn/libidn2-2.3.7.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v247/liblinear-247.tar.gz
 https://www.freedesktop.org/software/libmbim/libmbim-1.26.4.tar.xz
-https://github.com/linux-nvme/libnvme/archive/v1.9/libnvme-1.9.tar.gz
+https://github.com/linux-nvme/libnvme/archive/v1.10/libnvme-1.10.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.5/libpaper-2.2.5.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.7.1/libportal-0.7.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.7.1-testsuite_fix-1.patch
@@ -681,7 +681,7 @@ https://downloads.sourceforge.net/enlightenment/imlib2-1.12.3.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.1/kImageAnnotator-0.7.1.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
-https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.2.tar.xz
+https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.3.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.1/libei-1.2.1.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
@@ -698,8 +698,8 @@ https://download.qt.io/archive/qt/6.7/6.7.2/single/qt-everywhere-src-6.7.2.tar.x
 https://download.qt.io/official_releases/qt/6.7/6.7.2/submodules/qtwebengine-everywhere-src-6.7.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-6.7.2-ffmpeg7_fixes-1.patch
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.290/Vulkan-Headers-1.3.290.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.290/Vulkan-Loader-1.3.290.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.292/Vulkan-Headers-1.3.292.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.292/Vulkan-Loader-1.3.292.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.44.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-desktop-portal-1.18.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
@@ -963,7 +963,7 @@ https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2.1/pipewire-1.2.1.
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.0.tar.xz
 https://github.com/libsdl-org/sdl12-compat/archive/release-1.2.68/sdl12-compat-release-1.2.68.tar.gz
-https://www.libsdl.org/release/SDL2-2.30.5.tar.gz
+https://www.libsdl.org/release/SDL2-2.30.6.tar.gz
 https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.3.3.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
