diff --git a/wget-list b/wget-list
index fda6a9394..d12f0de2a 100644
--- a/wget-list
+++ b/wget-list
@@ -288,7 +288,7 @@ https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
 https://cmake.org/files/v3.30/cmake-3.30.2.tar.gz
-https://doxygen.nl/files/doxygen-1.11.0.src.tar.gz
+https://doxygen.nl/files/doxygen-1.12.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.0.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-14.2.0/gcc-14.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.6/gc-8.2.6.tar.gz
@@ -640,7 +640,7 @@ https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240102.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
 https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.42.tar.xz
-https://www.x.org/pub/individual/xserver/xwayland-24.1.1.tar.xz
+https://www.x.org/pub/individual/xserver/xwayland-24.1.2.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.13.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.13-tearfree_backport-2.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.2.tar.xz
@@ -682,7 +682,7 @@ https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.1/kImageAnnotator-0.7.1.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
 https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.3.tar.xz
-https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.1/libei-1.2.1.tar.gz
+https://gitlab.freedesktop.org/libinput/libei/-/archive/1.3.0/libei-1.3.0.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
 https://dri.freedesktop.org/libdrm/libdrm-2.4.122.tar.xz
@@ -981,7 +981,7 @@ https://distfiles.audacious-media-player.org/audacious-plugins-4.4.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
-https://downloads.sourceforge.net/mpg123/mpg123-1.32.6.tar.bz2
+https://downloads.sourceforge.net/mpg123/mpg123-1.32.7.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-6.1.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.2.tar.gz
