diff --git a/wget-list b/wget-list
index f9e541edd..55cfd255b 100644
--- a/wget-list
+++ b/wget-list
@@ -196,7 +196,7 @@ https://github.com/mypaint/libmypaint/releases/download/v1.6.1/libmypaint-1.6.1.
 https://downloads.sourceforge.net/libpng/libpng-1.6.43.tar.xz
 https://downloads.sourceforge.net/sourceforge/libpng-apng/libpng-1.6.43-apng.patch.gz
 https://www.libraw.org/data/LibRaw-0.21.2.tar.gz
-https://download.gnome.org/sources/librsvg/2.58/librsvg-2.58.2.tar.xz
+https://download.gnome.org/sources/librsvg/2.58/librsvg-2.58.3.tar.xz
 https://github.com/fontforge/libspiro/releases/download/20220722/libspiro-dist-20220722.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.6.0.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.4.0.tar.gz
@@ -532,7 +532,7 @@ https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.2.2.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.4.tar.xz
 https://static.rust-lang.org/dist/rustc-1.80.1-src.tar.xz
-https://github.com/rust-lang/rust-bindgen/archive/v0.69.4/rust-bindgen-0.69.4.tar.gz
+https://github.com/rust-lang/rust-bindgen/archive/v0.70.0/rust-bindgen-0.70.0.tar.gz
 https://downloads.sourceforge.net/scons/SCons-4.8.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.3.tar.bz2
