diff --git a/wget-list b/wget-list
index 666f18e2e..0102e169c 100644
--- a/wget-list
+++ b/wget-list
@@ -4,7 +4,7 @@ https://github.com/cracklib/cracklib/releases/download/v2.9.11/cracklib-2.9.11.t
 https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.1.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.5.tar.bz2
-https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.4.tar.xz
+https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.5.tar.xz
 https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.23.2.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.10.tar.xz
 https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.69.tar.xz
@@ -69,7 +69,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/clucene-2.3.3.4-contribs_lib-1
 https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.112.tar.gz
 https://github.com/google/double-conversion/archive/v3.3.0/double-conversion-3.3.0.tar.gz
 https://duktape.org/duktape-2.7.0.tar.xz
-https://github.com/AbiWord/enchant/releases/download/v2.6.8/enchant-2.6.8.tar.gz
+https://github.com/AbiWord/enchant/releases/download/v2.6.9/enchant-2.6.9.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.5.tar.xz
 https://www.fftw.org/fftw-3.3.10.tar.gz
 https://github.com/fmtlib/fmt/archive/refs/tags/10.2.1/fmt-10.2.1.tar.gz
@@ -135,7 +135,7 @@ https://dist.libuv.org/dist/v1.48.0/libuv-v1.48.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.7.0.tar.xz
 https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.6.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
-https://github.com/hughsie/libxmlb/releases/download/0.3.15/libxmlb-0.3.15.tar.xz
+https://github.com/hughsie/libxmlb/releases/download/0.3.16/libxmlb-0.3.16.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.39.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.10.0/libwacom-2.10.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
@@ -245,7 +245,7 @@ http://fcron.free.fr/archives/fcron-3.2.1.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
 https://downloads.sourceforge.net/hdparm/hdparm-9.65.tar.gz
-https://github.com/vcrhonek/hwdata/archive/v0.380/hwdata-0.380.tar.gz
+https://github.com/vcrhonek/hwdata/archive/v0.381/hwdata-0.381.tar.gz
 https://github.com/lfs-book/LSB-Tools/releases/download/v0.12/LSB-Tools-0.12.tar.gz
 https://github.com/lm-sensors/lm-sensors/archive/V3-6-0/lm-sensors-3-6-0.tar.gz
 https://github.com/logrotate/logrotate/releases/download/3.21.0/logrotate-3.21.0.tar.xz
@@ -254,7 +254,7 @@ https://www.freedesktop.org/software/ModemManager/ModemManager-1.18.12.tar.xz
 https://download.gnome.org/sources/notification-daemon/3.20/notification-daemon-3.20.0.tar.xz
 https://github.com/p7zip-project/p7zip/archive/v17.04/p7zip-17.04.tar.gz
 http://www.mirbsd.org/MirOS/dist/mir/cpio/paxmirabilis-20201030.tgz
-https://mj.ucw.cz/download/linux/pci/pciutils-3.11.1.tar.gz
+https://mj.ucw.cz/download/linux/pci/pciutils-3.12.0.tar.gz
 https://pm-utils.freedesktop.org/releases/pm-utils-1.4.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/pm-utils-1.4.1-bugfixes-1.patch
 https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/archive/0.20/power-profiles-daemon-0.20.tar.gz
@@ -288,11 +288,11 @@ https://www.kernel.org/pub/software/scm/git/git-2.44.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.44.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.44.0.tar.xz
 https://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.0/llvm-18.1.0.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.2/llvm-18.1.2.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-18.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party-18.src.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.0/clang-18.1.0.src.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.0/compiler-rt-18.1.0.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.2/clang-18.1.2.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.2/compiler-rt-18.1.2.src.tar.xz
 https://www.lua.org/ftp/lua-5.4.6.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.6-shared_library-1.patch
 https://www.lua.org/tests/lua-5.4.6-tests.tar.gz
@@ -487,7 +487,7 @@ https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.31.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.31.0-use_system_certs-1.patch
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
-https://files.pythonhosted.org/packages/source/s/sentry-sdk/sentry-sdk-1.44.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/sentry-sdk/sentry-sdk-1.44.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.16.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-7.2.6.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-2.0.0.tar.gz
@@ -638,6 +638,7 @@ https://xcb.freedesktop.org/dist/xcb-util-cursor-0.1.4.tar.xz
 https://mesa.freedesktop.org/archive/mesa-24.0.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-2.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
+https://invisible-mirror.net/archives/luit/luit-20240102.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
 https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.41.tar.xz
 https://www.x.org/pub/individual/xserver/xwayland-23.2.4.tar.xz
@@ -654,7 +655,7 @@ https://invisible-mirror.net/archives/xterm/xterm-390.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.2.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
-https://download.gnome.org/sources/at-spi2-core/2.50/at-spi2-core-2.50.2.tar.xz
+https://download.gnome.org/sources/at-spi2-core/2.52/at-spi2-core-2.52.0.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.0.tar.xz
 https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
@@ -758,7 +759,6 @@ https://download.gnome.org/sources/libgtop/2.40/libgtop-2.40.0.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.2.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.1/libshumate-1.1.3.tar.xz
-https://download.gnome.org/sources/libwnck/43/libwnck-43.0.tar.xz
 https://download.gnome.org/sources/evolution-data-server/3.50/evolution-data-server-3.50.4.tar.xz
 https://download.gnome.org/sources/tracker/3.6/tracker-3.6.0.tar.xz
 https://download.gnome.org/sources/tracker-miners/3.6/tracker-miners-3.6.2.tar.xz
@@ -804,13 +804,14 @@ https://download.gnome.org/sources/gnome-weather/46/gnome-weather-46.0.tar.xz
 https://gitlab.gnome.org/GNOME/gucharmap/-/archive/15.1.2/gucharmap-15.1.2.tar.bz2
 https://www.unicode.org/Public/zipped/15.1.0/Unihan.zip
 https://download.gnome.org/sources/seahorse/43/seahorse-43.0.tar.xz
-https://download.gnome.org/sources/snapshot/45/snapshot-46.1.tar.xz
+https://download.gnome.org/sources/snapshot/46/snapshot-46.1.tar.xz
 https://download.gnome.org/sources/vinagre/3.22/vinagre-3.22.0.tar.xz
 https://archive.xfce.org/src/xfce/libxfce4util/4.18/libxfce4util-4.18.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfconf/4.18/xfconf-4.18.3.tar.bz2
 https://archive.xfce.org/src/xfce/libxfce4ui/4.18/libxfce4ui-4.18.6.tar.bz2
 https://archive.xfce.org/src/xfce/exo/4.18/exo-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/garcon/4.18/garcon-4.18.2.tar.bz2
+https://download.gnome.org/sources/libwnck/43/libwnck-43.0.tar.xz
 https://archive.xfce.org/src/xfce/xfce4-panel/4.18/xfce4-panel-4.18.6.tar.bz2
 https://archive.xfce.org/src/xfce/thunar/4.18/thunar-4.18.10.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.18/thunar-volman-4.18.0.tar.bz2
