diff --git a/wget-list b/wget-list
index b48a129f8..a19296f73 100644
--- a/wget-list
+++ b/wget-list
@@ -49,7 +49,7 @@ https://ftp.gnu.org/gnu/ed/ed-1.20.1.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-29.3.tar.xz
 https://download.gnome.org/sources/gedit/46/gedit-46.2.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/24.02.0/src/kate-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/kate-24.02.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v7/nano-7.2.tar.xz
 https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
@@ -718,7 +718,7 @@ https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/3.4.7/icewm-3.4.7.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://download.gnome.org/sources/adwaita-icon-theme/46/adwaita-icon-theme-46.0.tar.xz
-https://download.kde.org/stable/frameworks/6.0/breeze-icons-6.0.0.tar.xz
+https://download.kde.org/stable/frameworks/6.1/breeze-icons-6.1.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-symbolic/3.12/gnome-icon-theme-symbolic-3.12.0.tar.xz
@@ -726,22 +726,22 @@ https://download.gnome.org/sources/gnome-themes-extra/3.28/gnome-themes-extra-3.
 https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.17.tar.xz
 http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
-https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.0.0.tar.xz
-https://download.kde.org/stable/frameworks/6.0/extra-cmake-modules-6.0.0.tar.xz
+https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.1.0.tar.xz
+https://download.kde.org/stable/frameworks/6.1/extra-cmake-modules-6.1.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.12.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/ark-24.02.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/kdenlive-24.02.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/khelpcenter-24.02.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/konsole-24.02.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.02.0-scrollbar-1.patch
-https://download.kde.org/stable/release-service/24.02.0/src/libkexiv2-24.02.0.tar.xz
-https://download.kde.org/stable/plasma/6.0.0/plasma-activities-6.0.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/okular-24.02.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/libkdcraw-24.02.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/gwenview-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/ark-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/kdenlive-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/khelpcenter-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/konsole-24.02.2.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.02.2-scrollbar-1.patch
+https://download.kde.org/stable/release-service/24.02.2/src/libkexiv2-24.02.2.tar.xz
+https://download.kde.org/stable/plasma/6.0.3/plasma-activities-6.0.3.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/okular-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/libkdcraw-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/gwenview-24.02.2.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/46/gsettings-desktop-schemas-46.0.tar.xz
@@ -881,7 +881,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-dicti
 https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-help-24.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-translations-24.2.2.2.tar.xz
 https://download.gnome.org/sources/epiphany/46/epiphany-46.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.0/src/falkon-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/falkon-24.02.2.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.18.2/source/seamonkey-2.53.18.2.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
@@ -980,7 +980,6 @@ https://distfiles.audacious-media-player.org/audacious-4.3.1.tar.bz2
 https://distfiles.audacious-media-player.org/audacious-plugins-4.3.1.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
-https://download.kde.org/stable/release-service/24.02.0/src/kwave-24.02.0.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
 https://downloads.sourceforge.net/mpg123/mpg123-1.32.6.tar.bz2
 https://freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-5.0.tar.xz
