diff --git a/wget-list b/wget-list
index 0102e169c..0a7d7f727 100644
--- a/wget-list
+++ b/wget-list
@@ -54,7 +54,7 @@ https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v7/nano-7.2.tar.xz
 https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
-https://astron.com/pub/tcsh/tcsh-6.24.11.tar.gz
+https://astron.com/pub/tcsh/tcsh-6.24.12.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
 https://download.qemu.org/qemu-8.2.2.tar.xz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.2.tar.xz
@@ -135,14 +135,14 @@ https://dist.libuv.org/dist/v1.48.0/libuv-v1.48.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.7.0.tar.xz
 https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.6.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
-https://github.com/hughsie/libxmlb/releases/download/0.3.16/libxmlb-0.3.16.tar.xz
+https://github.com/hughsie/libxmlb/releases/download/0.3.17/libxmlb-0.3.17.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.39.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.10.0/libwacom-2.10.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_1/log4cplus-2.1.1.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
-https://bitmath.org/code/mtdev/mtdev-1.1.6.tar.bz2
-https://nodejs.org/dist/v20.11.1/node-v20.11.1.tar.xz
+https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
+https://nodejs.org/dist/v20.12.1/node-v20.12.1.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.7.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.43/pcre2-10.43.tar.bz2
@@ -553,7 +553,7 @@ https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p17.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.6.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.6-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.2.7.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.19.5.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.20.0.tar.gz
 https://ftp.gnu.org/gnu/wget/wget-1.24.5.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.7.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
@@ -635,7 +635,7 @@ https://xcb.freedesktop.org/dist/xcb-util-keysyms-0.4.1.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-renderutil-0.3.10.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-wm-0.4.2.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-cursor-0.1.4.tar.xz
-https://mesa.freedesktop.org/archive/mesa-24.0.3.tar.xz
+https://mesa.freedesktop.org/archive/mesa-24.0.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-2.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240102.tgz
@@ -684,7 +684,7 @@ https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3
 https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.0.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.1/libei-1.2.1.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
-https://gedit-technology.net/tarballs/libgedit-gtksourceview/libgedit-gtksourceview-299.0.4.tar.xz
+https://github.com/gedit-technology/libgedit-gtksourceview/releases/download/299.1.0/libgedit-gtksourceview-299.1.0.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
 https://dri.freedesktop.org/libdrm/libdrm-2.4.120.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
@@ -699,7 +699,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5
 https://download.qt.io/archive/qt/6.6/6.6.3/single/qt-everywhere-src-6.6.3.tar.xz
 https://download.qt.io/official_releases/qt/6.6/6.6.3/submodules/qtwebengine-everywhere-src-6.6.3.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
-https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
+https://download.gnome.org/sources/tepl/6.8/tepl-6.9.0.tar.xz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.281/Vulkan-Headers-1.3.281.tar.gz
 https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.281/Vulkan-Loader-1.3.281.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
