diff --git a/wget-list b/wget-list
index c2e52d321..babe43ba2 100644
--- a/wget-list
+++ b/wget-list
@@ -532,7 +532,7 @@ https://files.pythonhosted.org/packages/source/t/typing_extensions/typing_extens
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.0.7.tar.gz
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
-https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.0.tar.xz
+https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.1.tar.xz
 https://static.rust-lang.org/dist/rustc-1.77.0-src.tar.xz
 https://downloads.sourceforge.net/scons/SCons-4.7.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
