diff --git a/wget-list b/wget-list
index 29fb2fc8e..b48a129f8 100644
--- a/wget-list
+++ b/wget-list
@@ -1,7 +1,7 @@
 https://www.linuxfromscratch.org/blfs/downloads/systemd/blfs-systemd-units-20240205.tar.xz
 https://github.com/lfs-book/make-ca/releases/download/v1.13/make-ca-1.13.tar.xz
 https://github.com/cracklib/cracklib/releases/download/v2.9.11/cracklib-2.9.11.tar.xz
-https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.1.tar.xz
+https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.2.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.5.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.5.tar.xz
@@ -9,7 +9,7 @@ https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.23.2.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.10.tar.xz
 https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.69.tar.xz
 https://github.com/linux-pam/linux-pam/releases/download/v1.6.1/Linux-PAM-1.6.1.tar.xz
-https://github.com/linux-pam/linux-pam/releases/download/v1.6.1/Linux-PAM-1.6.0-docs.tar.xz
+https://github.com/linux-pam/linux-pam/releases/download/v1.6.1/Linux-PAM-1.6.1-docs.tar.xz
 https://downloads.sourceforge.net/liboauth/liboauth-1.0.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3.patch
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
@@ -137,7 +137,7 @@ https://dist.libuv.org/dist/v1.48.0/libuv-v1.48.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.7.0.tar.xz
 https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.6.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
-https://github.com/hughsie/libxmlb/releases/download/0.3.17/libxmlb-0.3.17.tar.xz
+https://github.com/hughsie/libxmlb/releases/download/0.3.18/libxmlb-0.3.18.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.39.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.10.0/libwacom-2.10.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
@@ -284,7 +284,7 @@ https://downloads.sourceforge.net/infozip/zip30.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
-https://cmake.org/files/v3.29/cmake-3.29.1.tar.gz
+https://cmake.org/files/v3.29/cmake-3.29.2.tar.gz
 https://doxygen.nl/files/doxygen-1.10.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.0.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
@@ -330,7 +330,6 @@ https://www.cpan.org/authors/id/O/OA/OALDERS/HTTP-Daemon-6.16.tar.gz
 https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.085.tar.gz
 https://www.cpan.org/authors/id/G/GA/GAAS/IO-String-1.08.tar.gz
 https://www.cpan.org/authors/id/R/RJ/RJBS/IPC-Run3-0.049.tar.gz
-https://www.cpan.org/authors/id/I/IS/ISHIGAKI/JSON-4.10.tar.gz
 https://www.cpan.org/authors/id/A/AL/ALINKE/Lingua-Translit-0.29.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.77.tar.gz
 https://www.cpan.org/authors/id/D/DR/DROLSKY/List-AllUtils-0.19.tar.gz
@@ -465,13 +464,13 @@ https://github.com/psf/cachecontrol/archive/refs/tags/v0.14.0/cachecontrol-0.14.
 https://github.com/cython/cython/releases/download/0.29.36/Cython-0.29.36.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python-dbusmock-0.31.1.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
-https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.20.1.tar.gz
+https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.1.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi-docgen-2023.3.tar.gz
 https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-4.9.4.tar.gz
-https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.2.tar.gz
+https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.3.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.4.tar.gz
 https://files.pythonhosted.org/packages/source/p/packaging/packaging-24.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-5.9.8.tar.gz
@@ -493,7 +492,7 @@ https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.31.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.31.0-use_system_certs-1.patch
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
-https://files.pythonhosted.org/packages/source/s/sentry-sdk/sentry-sdk-1.44.1.tar.gz
+https://files.pythonhosted.org/packages/source/s/sentry-sdk/sentry-sdk-1.45.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.16.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-7.2.6.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-2.0.0.tar.gz
@@ -647,9 +646,9 @@ https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240102.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
 https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.41.tar.xz
-https://www.x.org/pub/individual/xserver/xwayland-23.2.5.tar.xz
-https://www.x.org/pub/individual/xserver/xorg-server-21.1.12.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.12-tearfree_backport-1.patch
+https://www.x.org/pub/individual/xserver/xwayland-23.2.6.tar.xz
+https://www.x.org/pub/individual/xserver/xorg-server-21.1.13.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.13-tearfree_backport-1.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.1.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.10.6.tar.bz2
 https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.25.0/libinput-1.25.0.tar.gz
@@ -708,10 +707,10 @@ https://www.freedesktop.org/software/startup-notification/releases/startup-notif
 https://download.gnome.org/sources/tepl/6.9/tepl-6.9.0.tar.xz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.281/Vulkan-Headers-1.3.281.tar.gz
 https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.281/Vulkan-Loader-1.3.281.tar.gz
-https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
+https://webkitgtk.org/releases/webkitgtk-2.44.1.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-desktop-portal-1.18.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
-https://download.gnome.org/sources/gdm/45/gdm-45.0.1.tar.xz
+https://download.gnome.org/sources/gdm/46/gdm-46.0.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
@@ -744,7 +743,7 @@ https://download.kde.org/stable/release-service/24.02.0/src/okular-24.02.0.tar.x
 https://download.kde.org/stable/release-service/24.02.0/src/libkdcraw-24.02.0.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/gwenview-24.02.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
-https://download.gnome.org/sources/gcr/4.2/gcr-4.2.1.tar.xz
+https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/46/gsettings-desktop-schemas-46.0.tar.xz
 https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz
 https://download.gnome.org/sources/rest/0.9/rest-0.9.1.tar.xz
@@ -756,11 +755,11 @@ https://download.gnome.org/sources/gjs/1.80/gjs-1.80.2.tar.xz
 https://download.gnome.org/sources/gnome-autoar/0.4/gnome-autoar-0.4.4.tar.xz
 https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.0.tar.xz
 https://download.gnome.org/sources/gnome-menus/3.36/gnome-menus-3.36.0.tar.xz
-https://download.gnome.org/sources/gnome-online-accounts/3.50/gnome-online-accounts-3.50.0.tar.xz
+https://download.gnome.org/sources/gnome-online-accounts/3.50/gnome-online-accounts-3.50.1.tar.xz
 https://download.gnome.org/sources/grilo/0.3/grilo-0.3.16.tar.xz
 https://download.gnome.org/sources/libgdata/0.18/libgdata-0.18.1.tar.xz
 https://download.gnome.org/sources/libgee/0.20/libgee-0.20.6.tar.xz
-https://download.gnome.org/sources/libgtop/2.40/libgtop-2.40.0.tar.xz
+https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.2.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.2/libshumate-1.2.0.tar.xz
@@ -774,19 +773,18 @@ https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/46/gnome-backgrounds-46.0.tar.xz
 https://download.gnome.org/sources/gvfs/1.54/gvfs-1.54.0.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz
-https://download.gnome.org/sources/nautilus/45/nautilus-45.2.1.tar.xz
-https://download.gnome.org/sources/gnome-bluetooth/42/gnome-bluetooth-42.8.tar.xz
+https://download.gnome.org/sources/nautilus/46/nautilus-46.0.tar.xz
+https://download.gnome.org/sources/gnome-bluetooth/46/gnome-bluetooth-46.0.tar.xz
 https://download.gnome.org/sources/gnome-keyring/46/gnome-keyring-46.1.tar.xz
-https://download.gnome.org/sources/gnome-settings-daemon/45/gnome-settings-daemon-45.1.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/gnome-settings-daemon-45.1-testsuite_fix-1.patch
-https://download.gnome.org/sources/tecla/45/tecla-45.0.tar.xz
-https://download.gnome.org/sources/gnome-control-center/45/gnome-control-center-45.3.tar.xz
-https://download.gnome.org/sources/mutter/45/mutter-45.4.tar.xz
-https://download.gnome.org/sources/gnome-shell/45/gnome-shell-45.4.tar.xz
-https://download.gnome.org/sources/gnome-shell-extensions/45/gnome-shell-extensions-45.2.tar.xz
-https://download.gnome.org/sources/gnome-session/45/gnome-session-45.0.tar.xz
-https://download.gnome.org/sources/gnome-tweaks/45/gnome-tweaks-45.1.tar.xz
-https://download.gnome.org/sources/gnome-user-docs/45/gnome-user-docs-45.1.tar.xz
+https://download.gnome.org/sources/gnome-settings-daemon/46/gnome-settings-daemon-46.0.tar.xz
+https://download.gnome.org/sources/tecla/46/tecla-46.0.tar.xz
+https://download.gnome.org/sources/gnome-control-center/46/gnome-control-center-46.0.1.tar.xz
+https://download.gnome.org/sources/mutter/46/mutter-46.0.tar.xz
+https://download.gnome.org/sources/gnome-shell/46/gnome-shell-46.0.tar.xz
+https://download.gnome.org/sources/gnome-shell-extensions/46/gnome-shell-extensions-46.0.tar.xz
+https://download.gnome.org/sources/gnome-session/46/gnome-session-46.0.tar.xz
+https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.0.tar.xz
+https://download.gnome.org/sources/gnome-user-docs/46/gnome-user-docs-46.0.tar.xz
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
 https://download.gnome.org/sources/baobab/45/baobab-45.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
@@ -809,7 +807,7 @@ https://download.gnome.org/sources/gnome-weather/46/gnome-weather-46.0.tar.xz
 https://gitlab.gnome.org/GNOME/gucharmap/-/archive/15.1.2/gucharmap-15.1.2.tar.bz2
 https://www.unicode.org/Public/zipped/15.1.0/Unihan.zip
 https://download.gnome.org/sources/seahorse/43/seahorse-43.0.tar.xz
-https://download.gnome.org/sources/snapshot/46/snapshot-46.1.tar.xz
+https://download.gnome.org/sources/snapshot/46/snapshot-46.2.tar.xz
 https://download.gnome.org/sources/vinagre/3.22/vinagre-3.22.0.tar.xz
 https://archive.xfce.org/src/xfce/libxfce4util/4.18/libxfce4util-4.18.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfconf/4.18/xfconf-4.18.3.tar.bz2
@@ -882,7 +880,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-24.2.
 https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-dictionaries-24.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-help-24.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.2/libreoffice-translations-24.2.2.2.tar.xz
-https://download.gnome.org/sources/epiphany/45/epiphany-45.3.tar.xz
+https://download.gnome.org/sources/epiphany/46/epiphany-46.0.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/falkon-24.02.0.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.18.2/source/seamonkey-2.53.18.2.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
@@ -971,7 +969,7 @@ https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.3.3.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
-https://taglib.org/releases/taglib-2.0.tar.gz
+https://taglib.org/releases/taglib-2.0.1.tar.gz
 https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.26.1.tar.xz
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.1/wireplumber-0.5.1.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20240216.tar.xz
