diff --git a/wget-list b/wget-list
index a5a3a7019..dd1cbb2d9 100644
--- a/wget-list
+++ b/wget-list
@@ -39,7 +39,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/gptfdisk-1.0.10-convenience-1.
 https://ftp.gnu.org/gnu/parted/parted-3.6.tar.xz
 https://downloads.sourceforge.net/smartmontools/smartmontools-7.4.tar.gz
 https://github.com/libfuse/sshfs/releases/download/sshfs-3.7.3/sshfs-3.7.3.tar.xz
-https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.6.0.tar.xz
+https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.7.0.tar.xz
 https://github.com/rhboot/efivar/archive/39/efivar-39.tar.gz
 https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
 https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
@@ -77,13 +77,13 @@ https://github.com/fmtlib/fmt/archive/10.2.1/fmt-10.2.1.tar.gz
 https://download.gnome.org/sources/glib/2.80/glib-2.80.0.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.80/gobject-introspection-1.80.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
-https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.5.tar.xz
+https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.7.tar.xz
 https://download.gnome.org/sources/glibmm/2.80/glibmm-2.80.0.tar.xz
 https://download.gnome.org/sources/gmime/3.2/gmime-3.2.7.tar.xz
 https://ftp.gnu.org/gnu/gsl/gsl-2.7.1.tar.gz
 https://download.gnome.org/sources/gspell/1.12/gspell-1.12.2.tar.xz
 https://github.com/google/highway/archive/1.1.0/highway-1.1.0.tar.gz
-https://github.com/unicode-org/icu/releases/download/release-74-2/icu4c-74_2-src.tgz
+https://github.com/unicode-org/icu/releases/download/release-75-1/icu4c-75_1-src.tgz
 https://github.com/benhoyt/inih/archive/r58/inih-r58.tar.gz
 https://github.com/intel/gmmlib/archive/intel-gmmlib-22.3.16.tar.gz
 https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
@@ -165,7 +165,7 @@ https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.18.1/umockdev-0.18.1.tar.xz
 https://github.com/nemtrif/utfcpp/archive/v4.0.5/utfcpp-4.0.5.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.22.0/downloads/wayland-1.22.0.tar.xz
-https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.34/downloads/wayland-protocols-1.34.tar.xz
+https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.35/downloads/wayland-protocols-1.35.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/1.4.25/xapian-core-1.4.25.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
@@ -307,8 +307,8 @@ https://www.lua.org/tests/lua-5.4.6-tests.tar.gz
 https://www.lua.org/ftp/lua-5.2.4.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.2.4-shared_library-1.patch
 https://www.mercurial-scm.org/release/mercurial-6.7.2.tar.gz
-https://www.nasm.us/pub/nasm/releasebuilds/2.16.02/nasm-2.16.02.tar.xz
-https://www.nasm.us/pub/nasm/releasebuilds/2.16.02/nasm-2.16.02-xdoc.tar.xz
+https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03.tar.xz
+https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
 https://www.cpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz
 https://www.cpan.org/authors/id/V/VP/VPIT/autovivification-0.18.tar.gz
@@ -569,7 +569,7 @@ https://w1.fi/releases/wpa_supplicant-2.10.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/wpa_supplicant-2.10-security_fix-1.patch
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.18.25/bind-9.18.25.tar.xz
+https://ftp.isc.org/isc/bind9/9.18.26/bind-9.18.26.tar.xz
 https://download.gnome.org/sources/NetworkManager/1.46/NetworkManager-1.46.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.94.tar.bz2
@@ -994,7 +994,7 @@ https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.2.tar.gz
 https://ffmpeg.org/releases/ffmpeg-6.1.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.1.1-chromium_method-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.1.1-vulkan_fixes-1.patch
-https://github.com/mpv-player/mpv/archive/v0.37.0/mpv-0.37.0.tar.gz
+https://github.com/mpv-player/mpv/archive/v0.38.0/mpv-0.38.0.tar.gz
 https://mplayerhq.hu/MPlayer/releases/MPlayer-1.5.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/MPlayer-1.5-upstream_ffmpg6_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0-binutils_2.41-1.patch
