diff --git a/wget-list b/wget-list
index a19296f73..b8bde7a7f 100644
--- a/wget-list
+++ b/wget-list
@@ -49,7 +49,7 @@ https://ftp.gnu.org/gnu/ed/ed-1.20.1.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-29.3.tar.xz
 https://download.gnome.org/sources/gedit/46/gedit-46.2.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/24.02.2/src/kate-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.2/src/kate-24.02.2.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v7/nano-7.2.tar.xz
 https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
@@ -195,7 +195,7 @@ https://www.libraw.org/data/LibRaw-0.21.2.tar.gz
 https://download.gnome.org/sources/librsvg/2.58/librsvg-2.58.0.tar.xz
 https://github.com/fontforge/libspiro/releases/download/20220722/libspiro-dist-20220722.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.6.0.tar.gz
-https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.3.2.tar.gz
+https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.4.0.tar.gz
 https://github.com/mypaint/mypaint-brushes/releases/download/v1.3.1/mypaint-brushes-1.3.1.tar.xz
 https://releases.pagure.org/newt/newt-0.52.24.tar.gz
 https://github.com/opencv/opencv/archive/4.9.0/opencv-4.9.0.tar.gz
@@ -726,7 +726,7 @@ https://download.gnome.org/sources/gnome-themes-extra/3.28/gnome-themes-extra-3.
 https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.17.tar.xz
 http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
-https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.1.0.tar.xz
+https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.0.0.tar.xz
 https://download.kde.org/stable/frameworks/6.1/extra-cmake-modules-6.1.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
@@ -850,7 +850,7 @@ https://downloads.sourceforge.net/lxde/menu-cache-1.1.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/menu-cache-1.1.0-consolidated_fixes-1.patch
 https://github.com/lxqt/libfm-qt/releases/download/1.4.0/libfm-qt-1.4.0.tar.xz
 https://github.com/lxqt/lxqt-themes/releases/download/1.3.0/lxqt-themes-1.3.0.tar.xz
-https://github.com/lxqt/lxqt-qtplugin/releases/download/1.4.0/lxqt-qtplugin-1.4.0.tar.xz
+https://github.com/lxqt/lxqt-qtplugin/releases/download/1.4.1/lxqt-qtplugin-1.4.1.tar.xz
 https://github.com/lxqt/lxqt-about/releases/download/1.4.0/lxqt-about-1.4.0.tar.xz
 https://github.com/lxqt/lxqt-admin/releases/download/1.4.0/lxqt-admin-1.4.0.tar.xz
 https://github.com/lxqt/lxqt-openssh-askpass/releases/download/1.4.0/lxqt-openssh-askpass-1.4.0.tar.xz
@@ -916,13 +916,13 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://downloads.xiph.org/releases/flac/flac-1.4.3.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.1.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.1.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.1.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.1.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.1.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.1.tar.xz
-https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.24.1.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.24.2.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/refs/tags/intel-media-23.4.3.tar.gz
@@ -958,7 +958,7 @@ https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.2/libvdpau-va-gl-0.4.2.ta
 https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.14.0/libvpx-1.14.0.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.22.0/mlt-7.22.0.tar.gz
-https://downloads.xiph.org/releases/opus/opus-1.5.1.tar.gz
+https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz
 https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.0.4/pipewire-1.0.4.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/pipewire-1.0.4-upstream_fix-1.patch
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
