diff --git a/wget-list b/wget-list
index 15b67b2c9..f2c1b645f 100644
--- a/wget-list
+++ b/wget-list
@@ -465,7 +465,7 @@ https://github.com/psf/cachecontrol/archive/v0.14.0/cachecontrol-0.14.0.tar.gz
 https://github.com/cython/cython/releases/download/0.29.36/Cython-0.29.36.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python-dbusmock-0.31.1.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
-https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.1.tar.gz
+https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.2.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi-docgen-2023.3.tar.gz
