diff --git a/wget-list b/wget-list
index e8071d51f..8f29ce977 100644
--- a/wget-list
+++ b/wget-list
@@ -57,6 +57,7 @@ http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.12.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
 https://download.qemu.org/qemu-8.2.2.tar.xz
+https://github.com/abseil-cpp/releases/download/20240116.2/abseil-cpp-20240116.2.tar.gz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.2.tar.xz
 https://archive.apache.org/dist/apr/apr-1.7.4.tar.bz2
 https://archive.apache.org/dist/apr/apr-util-1.6.3.tar.bz2
@@ -147,6 +148,9 @@ https://www.gnupg.org/ftp/gcrypt/npth/npth-1.7.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.43/pcre2-10.43.tar.bz2
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
+https://github.com/protocolbuffers/protobuf/releases/download/v26.1/protobuf-26.1.tar.gz
+https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.0/protobuf-c-1.5.0.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.0-consolidated_fixes-1.patch
 https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.10.0/qcoro-0.10.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/115.9.0esr/source/firefox-115.9.0esr.source.tar.xz
@@ -696,8 +700,8 @@ https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.4.tar.xz
 https://download.qt.io/archive/qt/5.15/5.15.12/single/qt-everywhere-opensource-src-5.15.12.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5.15.12-kf5-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5.15.12-security_fix-1.patch
-https://download.qt.io/archive/qt/6.6/6.6.3/single/qt-everywhere-src-6.6.3.tar.xz
-https://download.qt.io/official_releases/qt/6.6/6.6.3/submodules/qtwebengine-everywhere-src-6.6.3.tar.xz
+https://download.qt.io/archive/qt/6.7/6.7.0/single/qt-everywhere-src-6.7.0.tar.xz
+https://download.qt.io/official_releases/qt/6.7/6.7.0/submodules/qtwebengine-everywhere-src-6.7.0.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.9/tepl-6.9.0.tar.xz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.281/Vulkan-Headers-1.3.281.tar.gz
@@ -757,15 +761,15 @@ https://download.gnome.org/sources/libgee/0.20/libgee-0.20.6.tar.xz
 https://download.gnome.org/sources/libgtop/2.40/libgtop-2.40.0.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.2.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
-https://download.gnome.org/sources/libshumate/1.1/libshumate-1.1.3.tar.xz
-https://download.gnome.org/sources/evolution-data-server/3.50/evolution-data-server-3.50.4.tar.xz
-https://download.gnome.org/sources/tracker/3.6/tracker-3.6.0.tar.xz
-https://download.gnome.org/sources/tracker-miners/3.6/tracker-miners-3.6.2.tar.xz
+https://download.gnome.org/sources/libshumate/1.2/libshumate-1.2.0.tar.xz
+https://download.gnome.org/sources/evolution-data-server/3.52/evolution-data-server-3.52.0.tar.xz
+https://download.gnome.org/sources/tracker/3.7/tracker-3.7.1.tar.xz
+https://download.gnome.org/sources/tracker-miners/3.7/tracker-miners-3.7.1.tar.xz
 https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
-https://download.gnome.org/sources/xdg-desktop-portal-gnome/45/xdg-desktop-portal-gnome-45.1.tar.xz
+https://download.gnome.org/sources/xdg-desktop-portal-gnome/46/xdg-desktop-portal-gnome-46.0.tar.xz
 https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
-https://download.gnome.org/sources/gnome-backgrounds/45/gnome-backgrounds-45.0.tar.xz
+https://download.gnome.org/sources/gnome-backgrounds/46/gnome-backgrounds-46.0.tar.xz
 https://download.gnome.org/sources/gvfs/1.54/gvfs-1.54.0.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz
 https://download.gnome.org/sources/nautilus/45/nautilus-45.2.1.tar.xz
@@ -786,13 +790,13 @@ https://download.gnome.org/sources/baobab/45/baobab-45.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://download.gnome.org/sources/eog/45/eog-45.2.tar.xz
 https://download.gnome.org/sources/evince/45/evince-45.0.tar.xz
-https://download.gnome.org/sources/evolution/3.50/evolution-3.50.4.tar.xz
+https://download.gnome.org/sources/evolution/3.52/evolution-3.52.0.tar.xz
 https://download.gnome.org/sources/file-roller/43/file-roller-43.1.tar.xz
 https://download.gnome.org/sources/gnome-calculator/45/gnome-calculator-45.0.2.tar.xz
 https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-3.36.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/45/gnome-disk-utility-45.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/43/gnome-logs-43.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/45/gnome-maps-45.5.tar.xz
+https://download.gnome.org/sources/gnome-maps/46/gnome-maps-46.0.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/43/gnome-power-manager-43.0.tar.xz
