diff --git a/wget-list b/wget-list
index bbaf19d18..0d88c2d49 100644
--- a/wget-list
+++ b/wget-list
@@ -15,8 +15,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
 https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.10.tar.gz
-https://archive.mozilla.org/pub/security/nss/releases/NSS_3_102_RTM/src/nss-3.102.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.102-standalone-1.patch
+https://archive.mozilla.org/pub/security/nss/releases/NSS_3_102_1_RTM/src/nss-3.102.1.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.102.1-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.8p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.25.5/p11-kit-0.25.5.tar.xz
 https://gitlab.freedesktop.org/polkit/polkit/-/archive/124/polkit-124.tar.gz
@@ -138,7 +138,7 @@ https://lttng.org/files/urcu/userspace-rcu-0.14.0.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
 https://dist.libuv.org/dist/v1.48.0/libuv-v1.48.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.7.0.tar.xz
-https://download.gnome.org/sources/libxml2/2.13/libxml2-2.13.2.tar.xz
+https://download.gnome.org/sources/libxml2/2.13/libxml2-2.13.3.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.19/libxmlb-0.3.19.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.42.tar.xz
@@ -147,7 +147,7 @@ https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_1/log4cplus-2.1.1.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
-https://nodejs.org/dist/v20.15.1/node-v20.15.1.tar.xz
+https://nodejs.org/dist/v20.16.0/node-v20.16.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.7.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.44/pcre2-10.44.tar.bz2
@@ -541,10 +541,10 @@ https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.17.tar.xz
 https://sourceware.org/pub/valgrind/valgrind-3.23.0.tar.bz2
 https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz
-https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.1/OpenJDK-22.0.1+8-i686-bin.tar.xz
-https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.1/OpenJDK-22.0.1+8-x86_64-bin.tar.xz
-https://github.com/openjdk/jdk22u/archive/jdk-22.0.1-ga.tar.gz
-https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.1/jtreg-7.3.1+1.tar.gz
+https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.2/OpenJDK-22.0.2+9-i686-bin.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.2/OpenJDK-22.0.2+9-x86_64-bin.tar.xz
+https://github.com/openjdk/jdk22u/archive/jdk-22.0.2-ga.tar.gz
+https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.2/jtreg-7.3.1+1.tar.gz
 https://archive.apache.org/dist/ant/source/apache-ant-1.10.14-src.tar.xz
 https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.0.8/dhcpcd-10.0.8.tar.xz
 https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.xz
@@ -571,7 +571,7 @@ https://nmap.org/dist/nmap-7.95.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.5.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.2.6.tar.xz
-https://github.com/c-ares/c-ares/releases/download/v1.32.2/c-ares-1.32.2.tar.gz
+https://github.com/c-ares/c-ares/releases/download/v1.32.3/c-ares-1.32.3.tar.gz
 https://curl.se/download/curl-8.9.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.80/glib-networking-2.80.0.tar.xz
@@ -589,7 +589,7 @@ http://libndp.org/files/libndp-1.9.tar.gz
 https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.8.0/libslirp-v4.8.0.tar.bz2
 https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz
 https://download.gnome.org/sources/libsoup/3.4/libsoup-3.4.4.tar.xz
-https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.4.tar.bz2
+https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.5.tar.bz2
 https://notroj.github.io/neon/neon-0.33.0.tar.gz
 https://github.com/nghttp2/nghttp2/releases/download/v1.62.1/nghttp2-1.62.1.tar.xz
 https://github.com/thkukuk/rpcsvc-proto/releases/download/v1.4.4/rpcsvc-proto-1.4.4.tar.xz
@@ -899,8 +899,8 @@ https://inkscape.org/gallery/item/44615/inkscape-1.3.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.3.2-upstream_fixes-2.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
 https://archive.mozilla.org/pub/thunderbird/releases/128.0esr/source/thunderbird-128.0esr.source.tar.xz
-https://github.com/TigerVNC/tigervnc/archive/v1.13.1/tigervnc-1.13.1.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.13.1-configuration_fixes-1.patch
+https://github.com/TigerVNC/tigervnc/archive/v1.14.0/tigervnc-1.14.0.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.14.0-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/transmission-4.0.6-build_fix-1.patch
 https://github.com/ib/xarchiver/archive/0.5.4.23/xarchiver-0.5.4.23.tar.gz
@@ -1034,7 +1034,7 @@ https://pagure.io/xmlto/archive/0.0.29/xmlto-0.0.29.tar.gz
 https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview-gtk3/epdfview-gtk3-20200814.tar.xz
 https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.9-src.tar.gz
-https://www.mupdf.com/downloads/archive/mupdf-1.24.7-source.tar.gz
+https://www.mupdf.com/downloads/archive/mupdf-1.24.8-source.tar.gz
 https://github.com/dov/paps/releases/download/v0.8.0/paps-0.8.0.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312-source.tar.xz
@@ -1043,7 +1043,7 @@ https://ftp.tu-chemnitz.de/pub/tug/historic/systems/texlive/2024/texlive-2024031
 https://downloads.sourceforge.net/asymptote/asymptote-2.90.src.tgz
 https://github.com/plk/biber/archive/v2.20/biber-2.20.tar.gz
 https://sourceforge.net/projects/biblatex/files/biblatex-3.20/biblatex-3.20.tds.tgz
-https://github.com/mgieseki/dvisvgm/releases/download/3.3/dvisvgm-3.3.tar.gz
+https://github.com/mgieseki/dvisvgm/releases/download/3.4/dvisvgm-3.4.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20240416.tar.xz
