diff --git a/wget-list b/wget-list
index e6bbbe320..2484ed936 100644
--- a/wget-list
+++ b/wget-list
@@ -595,7 +595,7 @@ https://download.gnome.org/sources/libnma/1.10/libnma-1.10.6.tar.xz
 https://github.com/thkukuk/libnsl/releases/download/v2.0.1/libnsl-2.0.1.tar.xz
 https://www.tcpdump.org/release/libpcap-1.10.4.tar.gz
 https://github.com/rockdaboot/libpsl/releases/download/0.21.5/libpsl-0.21.5.tar.gz
-http://libndp.org/files/libndp-1.8.tar.gz
+http://libndp.org/files/libndp-1.9.tar.gz
 https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.8.0/libslirp-v4.8.0.tar.bz2
 https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz
 https://download.gnome.org/sources/libsoup/3.4/libsoup-3.4.4.tar.xz
@@ -694,7 +694,7 @@ https://downloads.sourceforge.net/enlightenment/imlib2-1.12.2.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.1/kImageAnnotator-0.7.1.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
-https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.1.tar.xz
+https://download.gnome.org/sources/libadwaita/1.5/libadwaita-1.5.2.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.1/libei-1.2.1.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
@@ -712,8 +712,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5
 https://download.qt.io/archive/qt/6.7/6.7.2/single/qt-everywhere-src-6.7.2.tar.xz
 https://download.qt.io/official_releases/qt/6.7/6.7.2/submodules/qtwebengine-everywhere-src-6.7.2.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.288/Vulkan-Headers-1.3.288.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.288/Vulkan-Loader-1.3.288.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.289/Vulkan-Headers-1.3.289.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.289/Vulkan-Loader-1.3.289.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.44.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-desktop-portal-1.18.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
@@ -816,6 +816,7 @@ https://download.gnome.org/sources/gnome-weather/46/gnome-weather-46.0.tar.xz
 https://gitlab.gnome.org/GNOME/gucharmap/-/archive/15.1.5/gucharmap-15.1.5.tar.bz2
 https://www.unicode.org/Public/zipped/15.1.0/Unihan.zip
 https://download.gnome.org/sources/seahorse/43/seahorse-43.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/seahorse-43.0-upstream_fixes-1.patch
 https://download.gnome.org/sources/snapshot/46/snapshot-46.3.tar.xz
 https://download.gnome.org/sources/vinagre/3.22/vinagre-3.22.0.tar.xz
 https://archive.xfce.org/src/xfce/libxfce4util/4.18/libxfce4util-4.18.2.tar.bz2
@@ -985,7 +986,7 @@ https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
 https://taglib.org/releases/taglib-2.0.1.tar.gz
 https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.26.1.tar.xz
-https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.4/wireplumber-0.5.4.tar.bz2
+https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.5/wireplumber-0.5.5.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20240216.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/x265/x265-20240216.tar.xz
 https://downloads.sourceforge.net/xine/xine-lib-1.2.13.tar.xz
