diff --git a/wget-list b/wget-list
index 0d88c2d49..861352323 100644
--- a/wget-list
+++ b/wget-list
@@ -1,6 +1,6 @@
 https://www.linuxfromscratch.org/blfs/downloads/systemd/blfs-systemd-units-20240205.tar.xz
 https://github.com/lfs-book/make-ca/archive/v1.14/make-ca-1.14.tar.gz
-https://github.com/cracklib/cracklib/releases/download/v2.10.0/cracklib-2.10.0.tar.xz
+https://github.com/cracklib/cracklib/releases/download/v2.10.1/cracklib-2.10.1.tar.xz
 https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.3.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.5.tar.bz2
@@ -483,7 +483,7 @@ https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.18.0.tar.gz
 https://download.gnome.org/sources/pygobject/3.48/pygobject-3.48.2.tar.xz
 https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.1.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
-https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.3.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.3.2.tar.gz
 https://files.pythonhosted.org/packages/b0/25/7998cd2dec731acbd438fbf91bc619603fc5188de0a9a17699a781840452/pyxdg-0.28.tar.gz
 https://files.pythonhosted.org/packages/source/P/PyYAML/PyYAML-6.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1.tar.gz
@@ -565,7 +565,7 @@ https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
 https://ftp.isc.org/isc/bind9/9.20.0/bind-9.20.0.tar.xz
-https://download.gnome.org/sources/NetworkManager/1.48/NetworkManager-1.48.4.tar.xz
+https://download.gnome.org/sources/NetworkManager/1.48/NetworkManager-1.48.6.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.95.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.5.tar.gz
@@ -691,7 +691,7 @@ https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.3.tar.xz
 https://people.freedesktop.org/~svu/libxklavier-5.4.tar.bz2
 https://download.gnome.org/sources/pango/1.54/pango-1.54.0.tar.xz
 https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.4.tar.xz
-https://download.gnome.org/sources/pangomm/2.52/pangomm-2.52.0.tar.xz
+https://download.gnome.org/sources/pangomm/2.54/pangomm-2.54.0.tar.xz
 https://download.qt.io/archive/qt/5.15/5.15.14/single/qt-everywhere-opensource-src-5.15.14.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5.15.14-kf5-1.patch
 https://download.qt.io/archive/qt/6.7/6.7.2/single/qt-everywhere-src-6.7.2.tar.xz
