diff --git a/wget-list b/wget-list
index 3da8cf410..5394837e2 100644
--- a/wget-list
+++ b/wget-list
@@ -76,7 +76,7 @@ https://github.com/AbiWord/enchant/releases/download/v2.8.1/enchant-2.8.1.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.5.tar.xz
 https://www.fftw.org/fftw-3.3.10.tar.gz
 https://github.com/fmtlib/fmt/archive/11.0.1/fmt-11.0.1.tar.gz
-https://download.gnome.org/sources/glib/2.80/glib-2.80.3.tar.xz
+https://download.gnome.org/sources/glib/2.80/glib-2.80.4.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.80/gobject-introspection-1.80.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.7.tar.xz
@@ -541,7 +541,7 @@ https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1
 https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.3.tar.xz
 https://static.rust-lang.org/dist/rustc-1.79.0-src.tar.xz
 https://github.com/rust-lang/rust-bindgen/archive/v0.69.4/rust-bindgen-0.69.4.tar.gz
-https://downloads.sourceforge.net/scons/SCons-4.7.0.tar.gz
+https://downloads.sourceforge.net/scons/SCons-4.8.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.3.tar.bz2
 https://downloads.sourceforge.net/swig/swig-4.2.1.tar.gz
@@ -621,7 +621,7 @@ https://security.appspot.com/downloads/vsftpd-3.0.5.tar.gz
 https://www.dovecot.org/releases/2.3/dovecot-2.3.21.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-openssl3_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-security_fix-1.patch
-https://ftp.exim.org/pub/exim/exim4/exim-4.97.1.tar.xz
+https://ftp.exim.org/pub/exim/exim4/exim-4.98.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.9.0.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
