diff --git a/wget-list b/wget-list
index 861352323..20c6042c3 100644
--- a/wget-list
+++ b/wget-list
@@ -530,7 +530,7 @@ https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.0.7.tar.g
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.4.tar.xz
-https://static.rust-lang.org/dist/rustc-1.79.0-src.tar.xz
+https://static.rust-lang.org/dist/rustc-1.80.0-src.tar.xz
 https://github.com/rust-lang/rust-bindgen/archive/v0.69.4/rust-bindgen-0.69.4.tar.gz
 https://downloads.sourceforge.net/scons/SCons-4.8.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
@@ -553,8 +553,8 @@ https://www.ncftp.com/downloads/ncftp/ncftp-3.2.7-src.tar.xz
 https://downloads.sourceforge.net/project/net-tools/net-tools-2.10.tar.xz
 https://www.kernel.org/pub/linux/utils/nfs-utils/2.6.4/nfs-utils-2.6.4.tar.xz
 https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
-https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.6.tar.bz2
-https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.6-vulnerability_fixes-1.patch
+https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.7.tar.bz2
+https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.7-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.3.0.tar.gz
 https://download.samba.org/pub/samba/stable/samba-4.20.2.tar.gz
 https://ftp.gnu.org/gnu/wget/wget-1.24.5.tar.gz
