diff --git a/wget-list b/wget-list
index 1e5ac7668..d28e87316 100644
--- a/wget-list
+++ b/wget-list
@@ -141,7 +141,7 @@ https://github.com/hughsie/libxmlb/releases/download/0.3.21/libxmlb-0.3.21.tar.x
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.42.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.13.0/libwacom-2.13.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
-https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_1/log4cplus-2.1.1.tar.xz
+https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_2/log4cplus-2.1.2.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
 https://nodejs.org/dist/v20.18.0/node-v20.18.0.tar.xz
@@ -649,7 +649,7 @@ https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.9.2.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.2/xf86-input-wacom-1.2.2.tar.bz2
 https://www.x.org/pub/individual/app/twm-1.0.12.tar.xz
-https://invisible-mirror.net/archives/xterm/xterm-394.tgz
+https://invisible-mirror.net/archives/xterm/xterm-395.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.2.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
