diff --git a/wget-list b/wget-list
index c39de4a18..c7f5bea7e 100644
--- a/wget-list
+++ b/wget-list
@@ -120,7 +120,7 @@ https://github.com/linux-nvme/libnvme/archive/v1.10/libnvme-1.10.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.5/libpaper-2.2.5.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.8.1/libportal-0.8.1.tar.xz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.2.0/libqalculate-5.2.0.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.3.0/libqalculate-5.3.0.tar.gz
 https://www.freedesktop.org/software/libqmi/libqmi-1.30.8.tar.xz
 https://github.com/seccomp/libseccomp/releases/download/v2.5.5/libseccomp-2.5.5.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -145,7 +145,7 @@ https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_1/log4cplus-2.1.1.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
-https://nodejs.org/dist/v20.17.0/node-v20.17.0.tar.xz
+https://nodejs.org/dist/v20.18.0/node-v20.18.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.7.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.44/pcre2-10.44.tar.bz2
@@ -694,12 +694,11 @@ https://people.freedesktop.org/~svu/libxklavier-5.4.tar.bz2
 https://download.gnome.org/sources/pango/1.54/pango-1.54.0.tar.xz
 https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.4.tar.xz
 https://download.gnome.org/sources/pangomm/2.54/pangomm-2.54.0.tar.xz
-https://download.qt.io/archive/qt/6.7/6.7.3/single/qt-everywhere-src-6.7.3.tar.xz
-https://download.qt.io/official_releases/qt/6.7/6.7.3/submodules/qtwebengine-everywhere-src-6.7.3.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-6.7.3-ffmpeg7_fixes-1.patch
+https://download.qt.io/archive/qt/6.7/6.8.0/single/qt-everywhere-src-6.8.0.tar.xz
+https://download.qt.io/official_releases/qt/6.8/6.8.0/submodules/qtwebengine-everywhere-src-6.8.0.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.296/Vulkan-Headers-1.3.296.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.296/Vulkan-Loader-1.3.296.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.297/Vulkan-Headers-1.3.297.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.297/Vulkan-Loader-1.3.297.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.46.1.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.4/xdg-desktop-portal-1.18.4.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
