diff --git a/wget-list b/wget-list
index 13dc6ddd6..8d1e24043 100644
--- a/wget-list
+++ b/wget-list
@@ -138,7 +138,7 @@ https://dist.libuv.org/dist/v1.49.1/libuv-v1.49.1.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.7.0.tar.xz
 https://download.gnome.org/sources/libxml2/2.13/libxml2-2.13.4.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
-https://github.com/hughsie/libxmlb/releases/download/0.3.19/libxmlb-0.3.19.tar.xz
+https://github.com/hughsie/libxmlb/releases/download/0.3.20/libxmlb-0.3.20.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.42.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.13.0/libwacom-2.13.0.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
