diff --git a/wget-list b/wget-list
index 4c0d645c4..f160fbd22 100644
--- a/wget-list
+++ b/wget-list
@@ -550,7 +550,7 @@ https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-22.0.2/jtreg-7.3.1+1.ta
 https://archive.apache.org/dist/ant/source/apache-ant-1.10.15-src.tar.xz
 https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.0.10/dhcpcd-10.0.10.tar.xz
 https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.xz
-https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.0.tar.bz2
+https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.1.tar.bz2
 https://www.ncftp.com/downloads/ncftp/ncftp-3.2.7-src.tar.xz
 https://downloads.sourceforge.net/project/net-tools/net-tools-2.10.tar.xz
 https://www.kernel.org/pub/linux/utils/nfs-utils/2.7.1/nfs-utils-2.7.1.tar.xz
