diff --git a/wget-list b/wget-list
index ed261ee37..18aea11eb 100644
--- a/wget-list
+++ b/wget-list
@@ -489,7 +489,7 @@ https://download.gnome.org/sources/pygobject/3.48/pygobject-3.48.2.tar.xz
 https://download.gnome.org/sources/pygtk/2.24/pygtk-2.24.0.tar.bz2
 https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.1.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
-https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.2.0.tar.gz
+https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.2.1.tar.gz
 https://files.pythonhosted.org/packages/b0/25/7998cd2dec731acbd438fbf91bc619603fc5188de0a9a17699a781840452/pyxdg-0.28.tar.gz
 https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1.tar.gz
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.31.0.tar.gz
