diff --git a/wget-list b/wget-list
index 762e6318a..e04ce6417 100644
--- a/wget-list
+++ b/wget-list
@@ -47,7 +47,7 @@ https://unifoundry.com/pub/unifont/unifont-15.1.04/font-builds/unifont-15.1.04.p
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.15.tar.bz2
 https://ftp.gnu.org/gnu/ed/ed-1.20.2.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-29.3.tar.xz
-https://download.gnome.org/sources/gedit/46/gedit-47.0.tar.xz
+https://download.gnome.org/sources/gedit/47/gedit-47.0.tar.xz
 https://github.com/gedit-technology/libgedit-tepl/releases/download/6.10.0/libgedit-tepl-6.10.0.tar.xz
 https://github.com/gedit-technology/libgedit-gfls/releases/download/0.1.0/libgedit-gfls-0.1.0.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
@@ -293,9 +293,9 @@ https://kernel.org/pub/software/utils/dtc/dtc-1.7.0.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.6/gc-8.2.6.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-14.2.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-2.44.0.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-manpages-2.44.0.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.44.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-2.45.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-manpages-2.45.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.45.0.tar.xz
 https://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.xz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.2/llvm-18.1.2.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-18.src.tar.xz
@@ -767,8 +767,8 @@ https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.2.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.2/libshumate-1.2.1.tar.xz
 https://download.gnome.org/sources/evolution-data-server/3.52/evolution-data-server-3.52.1.tar.xz
-https://download.gnome.org/sources/tracker/3.7/tracker-3.7.1.tar.xz
-https://download.gnome.org/sources/tracker-miners/3.7/tracker-miners-3.7.1.tar.xz
+https://download.gnome.org/sources/tracker/3.7/tracker-3.7.2.tar.xz
+https://download.gnome.org/sources/tracker-miners/3.7/tracker-miners-3.7.2.tar.xz
 https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
 https://download.gnome.org/sources/xdg-desktop-portal-gnome/46/xdg-desktop-portal-gnome-46.1.tar.xz
 https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
@@ -786,13 +786,13 @@ https://download.gnome.org/sources/mutter/46/mutter-46.1.tar.xz
 https://download.gnome.org/sources/gnome-shell/46/gnome-shell-46.1.tar.xz
 https://download.gnome.org/sources/gnome-shell-extensions/46/gnome-shell-extensions-46.1.tar.xz
 https://download.gnome.org/sources/gnome-session/46/gnome-session-46.0.tar.xz
-https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.0.tar.xz
+https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.1.tar.xz
 https://download.gnome.org/sources/gnome-user-docs/46/gnome-user-docs-46.1.tar.xz
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
 https://download.gnome.org/sources/baobab/46/baobab-46.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://download.gnome.org/sources/eog/45/eog-45.3.tar.xz
-https://download.gnome.org/sources/evince/46/evince-46.0.tar.xz
+https://download.gnome.org/sources/evince/46/evince-46.1.tar.xz
 https://download.gnome.org/sources/evolution/3.52/evolution-3.52.1.tar.xz
 https://download.gnome.org/sources/file-roller/44/file-roller-44.1.tar.xz
 https://download.gnome.org/sources/gnome-calculator/46/gnome-calculator-46.0.tar.xz
@@ -924,13 +924,13 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://downloads.xiph.org/releases/flac/flac-1.4.3.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.2.tar.xz
-https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.24.2.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.3.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.24.3.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/intel-media-23.4.3.tar.gz
