diff --git a/wget-list b/wget-list
index f063cb47c..ef0066d26 100644
--- a/wget-list
+++ b/wget-list
@@ -122,7 +122,7 @@ https://github.com/rrthomas/libpaper/releases/download/v2.2.5/libpaper-2.2.5.tar
 https://github.com/flatpak/libportal/releases/download/0.7.1/libportal-0.7.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.7.1-testsuite_fix-1.patch
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.0.0/libqalculate-5.0.0.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.1.1/libqalculate-5.1.1.tar.gz
 https://www.freedesktop.org/software/libqmi/libqmi-1.30.8.tar.xz
 https://github.com/seccomp/libseccomp/releases/download/v2.5.5/libseccomp-2.5.5.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -308,7 +308,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.6-shared_library-1.pat
 https://www.lua.org/tests/lua-5.4.6-tests.tar.gz
 https://www.lua.org/ftp/lua-5.2.4.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.2.4-shared_library-1.patch
-https://www.mercurial-scm.org/release/mercurial-6.7.2.tar.gz
+https://www.mercurial-scm.org/release/mercurial-6.7.3.tar.gz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03.tar.xz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
@@ -473,7 +473,7 @@ https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.g
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi-docgen-2023.3.tar.gz
 https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
-https://files.pythonhosted.org/packages/source/l/lxml/lxml-4.9.4.tar.gz
+https://files.pythonhosted.org/packages/source/l/lxml/lxml-5.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.3.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.4.tar.gz
 https://files.pythonhosted.org/packages/source/p/packaging/packaging-24.0.tar.gz
@@ -495,7 +495,7 @@ https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.31.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.31.0-use_system_certs-1.patch
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
-https://github.com/getsentry/sentry-python/releases/download/2.0.1/sentry_sdk-2.0.1.tar.gz
+https://github.com/getsentry/sentry-python/releases/download/2.1.1/sentry_sdk-2.1.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.16.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-7.3.7.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-2.0.0.tar.gz
