diff --git a/wget-list b/wget-list
index c93d3fe2c..39006c7f5 100644
--- a/wget-list
+++ b/wget-list
@@ -677,7 +677,7 @@ https://download.gnome.org/sources/goffice/0.10/goffice-0.10.57.tar.xz
 https://github.com/steveire/grantlee/releases/download/v5.3.1/grantlee-5.3.1.tar.gz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.33.tar.xz
-https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.41.tar.xz
+https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.42.tar.xz
 https://download.gnome.org/sources/gtk/4.14/gtk-4.14.4.tar.xz
 https://download.gnome.org/sources/gtk-engines/2.20/gtk-engines-2.20.2.tar.bz2
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.9.tar.xz
