diff --git a/wget-list b/wget-list
index c36f44641..fb2c8d35f 100644
--- a/wget-list
+++ b/wget-list
@@ -51,7 +51,7 @@ https://download.gnome.org/sources/gedit/47/gedit-47.0.tar.xz
 https://github.com/gedit-technology/libgedit-tepl/releases/download/6.10.0/libgedit-tepl-6.10.0.tar.xz
 https://github.com/gedit-technology/libgedit-gfls/releases/download/0.1.0/libgedit-gfls-0.1.0.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/24.02.2/src/kate-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/kate-24.05.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v8/nano-8.0.tar.xz
 https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
@@ -557,7 +557,7 @@ https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.0.tar.bz2
 https://www.ncftp.com/downloads/ncftp/ncftp-3.2.7-src.tar.xz
 https://downloads.sourceforge.net/project/net-tools/net-tools-2.10.tar.xz
 https://www.kernel.org/pub/linux/utils/nfs-utils/2.6.4/nfs-utils-2.6.4.tar.xz
-https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p17.tar.gz
+https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.6.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.6-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.3.0.tar.gz
@@ -735,23 +735,23 @@ https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.12.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/ark-24.02.2.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/kdenlive-24.02.2.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/khelpcenter-24.02.2.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/konsole-24.02.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.02.2-scrollbar-1.patch
-https://download.kde.org/stable/release-service/24.02.2/src/libkexiv2-24.02.2.tar.xz
-https://download.kde.org/stable/plasma/6.0.3/plasma-activities-6.0.3.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/okular-24.02.2.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/libkdcraw-24.02.2.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/gwenview-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/ark-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/kdenlive-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/khelpcenter-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/konsole-24.05.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.05.0-scrollbar-1.patch
+https://download.kde.org/stable/release-service/24.05.0/src/libkexiv2-24.05.0.tar.xz
+https://download.kde.org/stable/plasma/6.0.5/plasma-activities-6.0.5.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/okular-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/libkdcraw-24.05.0.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/gwenview-24.05.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/46/gsettings-desktop-schemas-46.0.tar.xz
 https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz
 https://download.gnome.org/sources/rest/0.9/rest-0.9.1.tar.xz
 https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.6.tar.xz
-https://gitlab.gnome.org/GNOME/vte/-/archive/0.76.1/vte-0.76.1.tar.gz
+https://gitlab.gnome.org/GNOME/vte/-/archive/0.76.2/vte-0.76.2.tar.gz
 https://download.gnome.org/sources/yelp-xsl/42/yelp-xsl-42.1.tar.xz
 https://download.gnome.org/sources/geocode-glib/3.26/geocode-glib-3.26.4.tar.xz
 https://download.gnome.org/sources/gjs/1.80/gjs-1.80.2.tar.xz
@@ -774,7 +774,7 @@ https://download.gnome.org/sources/xdg-desktop-portal-gnome/46/xdg-desktop-porta
 https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/46/gnome-backgrounds-46.0.tar.xz
-https://download.gnome.org/sources/gvfs/1.54/gvfs-1.54.0.tar.xz
+https://download.gnome.org/sources/gvfs/1.54/gvfs-1.54.1.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz
 https://download.gnome.org/sources/nautilus/46/nautilus-46.1.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/46/gnome-bluetooth-46.0.tar.xz
@@ -839,9 +839,9 @@ https://download.kde.org/stable/frameworks/6.2/kwindowsystem-6.2.0.tar.xz
 https://download.kde.org/stable/frameworks/6.2/kconfig-6.2.0.tar.xz
 https://download.kde.org/stable/frameworks/6.2/solid-6.1.1.tar.xz
 https://download.kde.org/stable/frameworks/6.2/kidletime-6.2.0.tar.xz
-http://download.kde.org/stable/plasma/6.0.3/kwayland-6.0.3.tar.xz
-http://download.kde.org/stable/plasma/6.0.3/libkscreen-6.0.3.tar.xz
-http://download.kde.org/stable/plasma/6.0.3/layer-shell-qt-6.0.3.tar.xz
+http://download.kde.org/stable/plasma/6.0.5/kwayland-6.0.5.tar.xz
+http://download.kde.org/stable/plasma/6.0.5/libkscreen-6.0.5.tar.xz
+http://download.kde.org/stable/plasma/6.0.5/layer-shell-qt-6.0.5.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.4/muparser-2.3.4.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/2.0.0/lxqt-build-tools-2.0.0.tar.xz
 https://github.com/lxqt/libqtxdg/releases/download/4.0.0/libqtxdg-4.0.0.tar.xz
@@ -887,7 +887,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.3/libreoffice-dicti
 https://download.documentfoundation.org/libreoffice/src/24.2.3/libreoffice-help-24.2.3.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.3/libreoffice-translations-24.2.3.2.tar.xz
 https://download.gnome.org/sources/epiphany/46/epiphany-46.0.tar.xz
-https://download.kde.org/stable/release-service/24.02.2/src/falkon-24.02.2.tar.xz
+https://download.kde.org/stable/release-service/24.05.0/src/falkon-24.05.0.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.18.2/source/seamonkey-2.53.18.2.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.18.2-cxx17-1.patch
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
