diff --git a/wget-list b/wget-list
index 30bcc6ddf..72caeaeaa 100644
--- a/wget-list
+++ b/wget-list
@@ -277,7 +277,7 @@ https://sysstat.github.io/sysstat-packages/sysstat-12.7.5.tar.xz
 https://github.com/systemd/systemd/archive/v255/systemd-255.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/systemd-255-upstream_fixes-1.patch
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
-https://www.rarlab.com/rar/unrarsrc-7.0.8.tar.gz
+https://www.rarlab.com/rar/unrarsrc-7.0.9.tar.gz
 https://downloads.sourceforge.net/infozip/unzip60.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/unzip-6.0-consolidated_fixes-1.patch
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.4/upower-v1.90.4.tar.bz2
@@ -647,7 +647,7 @@ https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240102.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
 https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.41.tar.xz
-https://www.x.org/pub/individual/xserver/xwayland-23.2.6.tar.xz
+https://www.x.org/pub/individual/xserver/xwayland-24.1.0.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.13.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.13-tearfree_backport-1.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.1.tar.xz
