diff --git a/wget-list b/wget-list
index d3f9eec0f..ba76da278 100644
--- a/wget-list
+++ b/wget-list
@@ -288,7 +288,7 @@ https://downloads.sourceforge.net/infozip/zip30.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
-https://cmake.org/files/v3.29/cmake-3.29.2.tar.gz
+https://cmake.org/files/v3.29/cmake-3.29.3.tar.gz
 https://doxygen.nl/files/doxygen-1.10.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.0.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
