diff --git a/wget-list b/wget-list
index 1e05b31d7..f854897c5 100644
--- a/wget-list
+++ b/wget-list
@@ -561,7 +561,7 @@ https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p17.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.6.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.6-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.3.0.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.20.0.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.20.1.tar.gz
 https://ftp.gnu.org/gnu/wget/wget-1.24.5.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.7.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
