diff --git a/wget-list b/wget-list
index 17f89c0281..da88d1215b 100644
--- a/wget-list
+++ b/wget-list
@@ -150,7 +150,7 @@ https://github.com/pantoniou/libfyaml/releases/download/v0.9.6/libfyaml-0.9.6.ta
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_2_0_1/log4cplus-2.2.0.1.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
-https://nodejs.org/dist/v26.9.0/node-v26.9.0.tar.xz
+https://nodejs.org/dist/v26.10.0/node-v26.10.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.40/src/nspr-4.40.tar.gz
 https://github.com/AcademySoftwareFoundation/OpenTimelineIO/archive/v0.18.1/OpenTimelineIO-0.18.1.tar.gz
@@ -575,8 +575,8 @@ https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.19.tar.xz
 https://sourceware.org/pub/valgrind/valgrind-3.27.1.tar.bz2
 https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz
-https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/OpenJDK-21.0.10+6-i686-bin.tar.xz
-https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/OpenJDK-21.0.10+6-x86_64-bin.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.12.1/OpenJDK-21.0.12.1+1-i686-bin.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.12.1/OpenJDK-21.0.12.1+1-x86_64-bin.tar.xz
 https://github.com/openjdk/jdk21u/archive/jdk-21.0.12.1-ga.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.12.1/jtreg-8.2.1+1.tar.gz
 https://archive.apache.org/dist/ant/source/apache-ant-1.10.18-src.tar.xz
@@ -657,7 +657,7 @@ https://xorg.freedesktop.org/archive/individual/proto/xcb-proto-1.17.0.tar.xz
 https://xorg.freedesktop.org/archive/individual/lib/libxcb-1.17.0.tar.xz
 https://www.x.org/pub/individual/lib/libxcvt-0.1.3.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-0.4.1.tar.xz
-https://mesa.freedesktop.org/archive/mesa-26.1.7.tar.xz
+https://mesa.freedesktop.org/archive/mesa-26.2.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-5.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.4.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20260907.tgz
