diff --git a/wget-list b/wget-list
index 60a0e6a27d..83fe195ae1 100644
--- a/wget-list
+++ b/wget-list
@@ -574,7 +574,7 @@ https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.52.0/d
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.97.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
-https://github.com/rfc1036/whois/archive/v5.6.1/whois-5.6.1.tar.gz
+https://github.com/rfc1036/whois/archive/v5.6.2/whois-5.6.2.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.4.7.tar.xz
 https://github.com/c-ares/c-ares/releases/download/v1.34.5/c-ares-1.34.5.tar.gz
 https://curl.se/download/curl-8.14.1.tar.xz
@@ -620,8 +620,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/sendmail-8.18.1-gcc15_fixes-1.
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-11.4.7/source/mariadb-11.4.7.tar.gz
 https://ftp.postgresql.org/pub/source/v17.5/postgresql-17.5.tar.bz2
-https://sqlite.org/2025/sqlite-autoconf-3500000.tar.gz
-https://sqlite.org/2025/sqlite-doc-3500000.zip
+https://sqlite.org/2025/sqlite-autoconf-3500100.tar.gz
+https://sqlite.org/2025/sqlite-doc-3500100.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.10.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.10-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.23.0.tar.gz
@@ -633,7 +633,7 @@ https://xorg.freedesktop.org/archive/individual/proto/xcb-proto-1.17.0.tar.xz
 https://xorg.freedesktop.org/archive/individual/lib/libxcb-1.17.0.tar.xz
 https://www.x.org/pub/individual/lib/libxcvt-0.1.3.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-0.4.1.tar.xz
-https://mesa.freedesktop.org/archive/mesa-25.1.2.tar.xz
+https://mesa.freedesktop.org/archive/mesa-25.1.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-4.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240910.tgz
@@ -960,7 +960,7 @@ https://github.com/intel/libva/archive/2.22.0/libva-2.22.0.tar.gz
 https://gitlab.freedesktop.org/vdpau/libvdpau/-/archive/1.5/libvdpau-1.5.tar.bz2
 https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.2/libvdpau-va-gl-0.4.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
-https://github.com/webmproject/libvpx/archive/v1.15.1/libvpx-1.15.1.tar.gz
+https://github.com/webmproject/libvpx/archive/v1.15.2/libvpx-1.15.2.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.32.0/mlt-7.32.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz
 https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.4.5/pipewire-1.4.5.tar.bz2
