diff --git a/wget-list b/wget-list
index 21218c1435..49c35c60ef 100644
--- a/wget-list
+++ b/wget-list
@@ -93,8 +93,8 @@ https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
 https://download.gnome.org/sources/gspell/1.14/gspell-1.14.0.tar.xz
 https://github.com/google/highway/archive/1.2.0/highway-1.2.0.tar.gz
 https://github.com/unicode-org/icu/releases/download/release-77-1/icu4c-77_1-src.tgz
-https://github.com/benhoyt/inih/archive/r59/inih-r59.tar.gz
-https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
+https://github.com/benhoyt/inih/archive/r60/inih-r60.tar.gz
+https://github.com/akheron/jansson/releases/download/v2.14.1/jansson-2.14.1.tar.bz2
 https://s3.amazonaws.com/json-c_releases/releases/json-c-0.18.tar.gz
 https://download.gnome.org/sources/json-glib/1.10/json-glib-1.10.6.tar.xz
 https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git/snapshot/keyutils-1.6.3.tar.gz
@@ -224,7 +224,7 @@ https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.28.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
 https://github.com/google/shaderc/archive/v2025.2/shaderc-2025.2.tar.gz
-https://gitlab.com/graphviz/graphviz/-/archive/12.2.1/graphviz-12.2.1.tar.bz2
+https://gitlab.com/graphviz/graphviz/-/archive/13.0.0/graphviz-13.0.0.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
 http://www.andre-simon.de/zip/highlight-4.16.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.32/ibus-1.5.32.tar.gz
@@ -460,8 +460,8 @@ https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
 https://www.php.net/distributions/php-8.4.8.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
-https://www.python.org/ftp/python/3.13.4/Python-3.13.4.tar.xz
-https://www.python.org/ftp/python/doc/3.13.4/python-3.13.4-docs-html.tar.bz2
+https://www.python.org/ftp/python/3.13.5/Python-3.13.5.tar.xz
+https://www.python.org/ftp/python/doc/3.13.5/python-3.13.5-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/b/build/build-1.2.2.post1.tar.gz
