diff --git a/wget-list b/wget-list
index 60234c7815..728fa1b6f6 100644
--- a/wget-list
+++ b/wget-list
@@ -704,7 +704,7 @@ https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
-https://github.com/ice-wm/icewm/archive/3.7.5/icewm-3.7.5.tar.gz
+https://github.com/ice-wm/icewm/archive/3.8.0/icewm-3.8.0.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openbox-3.6.1-py3-1.patch
 https://download.gnome.org/sources/adwaita-icon-theme/48/adwaita-icon-theme-48.1.tar.xz
@@ -843,7 +843,7 @@ https://download.kde.org/stable/plasma/6.3.4/kwayland-6.3.4.tar.xz
 https://download.kde.org/stable/plasma/6.3.4/libkscreen-6.3.4.tar.xz
 https://download.kde.org/stable/plasma/6.3.4/layer-shell-qt-6.3.4.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.5/muparser-2.3.5.tar.gz
-https://github.com/lxqt/lxqt-build-tools/releases/download/2.2.0/lxqt-build-tools-2.2.0.tar.xz
+https://github.com/lxqt/lxqt-build-tools/releases/download/2.2.1/lxqt-build-tools-2.2.1.tar.xz
 https://github.com/lxqt/libdbusmenu-lxqt/releases/download/0.3.0/libdbusmenu-lxqt-0.3.0.tar.xz
 https://github.com/lxqt/libqtxdg/releases/download/4.2.0/libqtxdg-4.2.0.tar.xz
 https://github.com/lxqt/lxqt-menu-data/releases/download/2.2.0/lxqt-menu-data-2.2.0.tar.xz
@@ -855,7 +855,7 @@ https://github.com/lxde/menu-cache/archive/1.1.1/menu-cache-1.1.1.tar.gz
 https://github.com/lxqt/libfm-qt/releases/download/2.2.0/libfm-qt-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-globalkeys/releases/download/2.2.0/lxqt-globalkeys-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-qtplugin/releases/download/2.2.0/lxqt-qtplugin-2.2.0.tar.xz
-https://github.com/lxqt/lxqt-panel/releases/download/2.2.1/lxqt-panel-2.2.1.tar.xz
+https://github.com/lxqt/lxqt-panel/releases/download/2.2.2/lxqt-panel-2.2.2.tar.xz
 https://github.com/lxqt/pcmanfm-qt/releases/download/2.2.0/pcmanfm-qt-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-powermanagement/releases/download/2.2.0/lxqt-powermanagement-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-runner/releases/download/2.2.0/lxqt-runner-2.2.0.tar.xz
