diff --git a/wget-list b/wget-list
index fa6fe6373..29390bc7b 100644
--- a/wget-list
+++ b/wget-list
@@ -149,7 +149,7 @@ https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.42/pcre2-10.42.
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://ftp.gnu.org/gnu/pth/pth-2.0.7.tar.gz
 https://download.kde.org/stable/qca/2.3.7/qca-2.3.7.tar.xz
-https://archive.mozilla.org/pub/firefox/releases/115.3.0esr/source/firefox-115.3.0esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/115.4.0esr/source/firefox-115.4.0esr.source.tar.xz
 https://www.samba.org/ftp/talloc/talloc-2.4.1.tar.gz
 https://telepathy.freedesktop.org/releases/telepathy-glib/telepathy-glib-0.24.2.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
