diff --git a/wget-list b/wget-list
index ab2621d3..58ac7cb1 100644
--- a/wget-list
+++ b/wget-list
@@ -191,8 +191,8 @@ https://download.osgeo.org/libtiff/tiff-4.6.0.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.3.2.tar.gz
 https://github.com/mypaint/mypaint-brushes/releases/download/v1.3.1/mypaint-brushes-1.3.1.tar.xz
 https://releases.pagure.org/newt/newt-0.52.23.tar.gz
-https://github.com/opencv/opencv/archive/4.8.0/opencv-4.8.0.tar.gz
-https://github.com/opencv/opencv_contrib/archive/4.8.0/opencv_contrib-4.8.0.tar.gz
+https://github.com/opencv/opencv/archive/4.8.1/opencv-4.8.1.tar.gz
+https://github.com/opencv/opencv_contrib/archive/4.8.1/opencv_contrib-4.8.1.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.0/openjpeg-2.5.0.tar.gz
 https://www.cairographics.org/releases/pixman-0.42.2.tar.gz
 https://poppler.freedesktop.org/poppler-23.09.0.tar.xz
@@ -263,7 +263,7 @@ https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.4.tar.xz
 https://github.com/systemd/systemd/archive/v254/systemd-254.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
-https://www.rarlab.com/rar/unrarsrc-6.2.11.tar.gz
+https://www.rarlab.com/rar/unrarsrc-6.2.12.tar.gz
 https://downloads.sourceforge.net/infozip/unzip60.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/unzip-6.0-consolidated_fixes-1.patch
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.2/upower-v1.90.2.tar.bz2
@@ -554,8 +554,8 @@ https://w1.fi/releases/wpa_supplicant-2.10.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
 ftp://ftp.isc.org/isc/bind9/9.18.19/bind-9.18.19.tar.xz
-ftp://ftp.acc.umu.se/pub/gnome/sources/NetworkManager/1.44/NetworkManager-1.44.0.tar.xz
-ftp://ftp.acc.umu.se/pub/gnome/sources/network-manager-applet/1.32/network-manager-applet-1.32.0.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/NetworkManager/1.44/NetworkManager-1.44.2.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.94.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.3.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
@@ -754,7 +754,7 @@ ftp://ftp.acc.umu.se/pub/gnome/sources/libshumate/1.1/libshumate-1.1.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/libwnck/43/libwnck-43.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/evolution-data-server/3.50/evolution-data-server-3.50.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/tracker/3.6/tracker-3.6.0.tar.xz
-ftp://ftp.acc.umu.se/pub/gnome/sources/tracker-miners/3.6/tracker-miners-3.6.0.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/tracker-miners/3.6/tracker-miners-3.6.1.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gsound/1.0/gsound-1.0.3.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/dconf/0.40/dconf-0.40.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
@@ -929,7 +929,7 @@ https://get.videolan.org/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2
 https://downloads.sourceforge.net/libdv/libdv-1.0.0.tar.gz
 ftp://ftp.mars.org/pub/mpeg/libmad-0.15.1b.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libmad-0.15.1b-fixes-1.patch
-ftp://ftp.mirrorservice.org/sites/distfiles.gentoo.org/distfiles/libmpeg2-0.5.1.tar.gz
+https://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
 ftp://ftp.musicbrainz.org/pub/musicbrainz/historical/libmusicbrainz-2.1.5.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libmusicbrainz-2.1.5-missing-includes-1.patch
 https://github.com/metabrainz/libmusicbrainz/releases/download/release-5.1.0/libmusicbrainz-5.1.0.tar.gz
@@ -946,7 +946,7 @@ https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.13.1/libvpx-1.13.1.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.18.0/mlt-7.18.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.4.tar.gz
-https://github.com/PipeWire/pipewire/archive/0.3.80/pipewire-0.3.80.tar.gz
+https://github.com/PipeWire/pipewire/archive/0.3.81/pipewire-0.3.81.tar.gz
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-16.1.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.0.tar.xz
 https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.68/sdl12-compat-release-1.2.68.tar.gz
@@ -960,7 +960,7 @@ https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.24.1.tar.bz2
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.4.14/wireplumber-0.4.14.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20230215.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/x265/x265-20230215.tar.xz
-ftp://ftp.mirrorservice.org/sites/distfiles.gentoo.org/distfiles/xine-lib-1.2.13.tar.xz
+https://downloads.sourceforge.net/xine/xine-lib-1.2.13.tar.xz
 https://downloads.xvid.com/downloads/xvidcore-1.3.7.tar.gz
 https://distfiles.audacious-media-player.org/audacious-4.3.1.tar.bz2
 https://distfiles.audacious-media-player.org/audacious-plugins-4.3.1.tar.bz2
@@ -978,7 +978,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0-binutils_2.41-1.pat
 ftp://ftp.mplayerhq.hu/MPlayer/releases/MPlayer-1.5.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/MPlayer-1.5-upstream_ffmpg6_fixes-1.patch
 ftp://ftp.mplayerhq.hu/MPlayer/skins/Clearlooks-2.0.tar.bz2
-ftp://ftp.mirrorservice.org/sites/distfiles.gentoo.org/distfiles/transcode-1.1.7.tar.bz2
+https://anduin.linuxfromscratch.org/BLFS/transcode/transcode-1.1.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/transcode-1.1.7-gcc10_fix-1.patch
 https://download.videolan.org/vlc/3.0.18/vlc-3.0.18.tar.xz
 https://downloads.sourceforge.net/xine/xine-ui-0.99.14.tar.xz
@@ -1004,7 +1004,7 @@ https://www.docbook.org/sgml/4.5/docbook-4.5.zip
 https://downloads.sourceforge.net/openjade/OpenSP-1.5.2.tar.gz
 https://downloads.sourceforge.net/openjade/openjade-1.3.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openjade-1.3.2-upstream-1.patch
-ftp://ftp.mirrorservice.org/sites/distfiles.gentoo.org/distfiles/docbook-dsssl-1.79.tar.bz2
+https://downloads.sourceforge.net/docbook/docbook-dsssl-1.79.tar.bz2
 https://downloads.sourceforge.net/docbook/docbook-dsssl-doc-1.79.tar.bz2
 ftp://sourceware.org/pub/docbook-tools/new-trials/SOURCES/docbook-utils-0.6.14.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/docbook-utils-0.6.14-grep_fix-1.patch
