diff --git a/wget-list b/wget-list
index 29390bc7b..4ba43f503 100644
--- a/wget-list
+++ b/wget-list
@@ -801,7 +801,7 @@ https://archive.xfce.org/src/xfce/libxfce4ui/4.18/libxfce4ui-4.18.4.tar.bz2
 https://archive.xfce.org/src/xfce/exo/4.18/exo-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/garcon/4.18/garcon-4.18.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-panel/4.18/xfce4-panel-4.18.5.tar.bz2
-https://archive.xfce.org/src/xfce/thunar/4.18/thunar-4.18.7.tar.bz2
+https://archive.xfce.org/src/xfce/thunar/4.18/thunar-4.18.8.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.18/thunar-volman-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.18/tumbler-4.18.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.18/xfce4-appfinder-4.18.1.tar.bz2
