diff --git a/wget-list b/wget-list
index c6ac8b34..3524073b 100644
--- a/wget-list
+++ b/wget-list
@@ -862,10 +862,10 @@ https://github.com/lxqt/screengrab/releases/download/2.6.0/screengrab-2.6.0.tar.
 https://www.abisource.com/downloads/abiword/3.0.5/source/abiword-3.0.5.tar.gz
 https://www.abisource.com/downloads/abiword/3.0.2/source/abiword-docs-3.0.2.tar.gz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnumeric/1.12/gnumeric-1.12.55.tar.xz
-https://download.documentfoundation.org/libreoffice/src/7.6.1/libreoffice-7.6.1.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/7.6.1/libreoffice-dictionaries-7.6.1.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/7.6.1/libreoffice-help-7.6.1.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/7.6.1/libreoffice-translations-7.6.1.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-7.6.2.1.tar.xz
+https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-dictionaries-7.6.2.1.tar.xz
+https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-help-7.6.2.1.tar.xz
+https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-translations-7.6.2.1.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/epiphany/45/epiphany-45.0.tar.xz
 https://download.kde.org/stable/release-service/23.08.0/src/falkon-23.08.0.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.17.1/source/seamonkey-2.53.17.1.source.tar.xz
