diff --git a/wget-list b/wget-list
index 0f1cd93a36..1c8479b6c4 100644
--- a/wget-list
+++ b/wget-list
@@ -76,7 +76,7 @@ https://github.com/google/double-conversion/archive/v3.4.0/double-conversion-3.4
 https://duktape.org/duktape-2.7.0.tar.xz
 https://github.com/rrthomas/enchant/releases/download/v2.8.16/enchant-2.8.16.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.6.tar.xz
-https://github.com/fastfloat/fast_float/archive/v8.2.5/fast_float-8.2.5.tar.gz
+https://github.com/fastfloat/fast_float/archive/v8.2.6/fast_float-8.2.6.tar.gz
 https://www.fftw.org/fftw-3.3.11.tar.gz
 https://github.com/fmtlib/fmt/archive/12.1.0/fmt-12.1.0.tar.gz
 https://download.gnome.org/sources/glib/2.88/glib-2.88.1.tar.xz
@@ -205,7 +205,7 @@ https://downloads.sourceforge.net/libpng/libpng-1.6.58.tar.xz
 https://downloads.sourceforge.net/sourceforge/libpng-apng/libpng-1.6.58-apng.patch.gz
 https://github.com/fukuchi/libqrencode/archive/v4.1.1/libqrencode-4.1.1.tar.gz
 https://www.libraw.org/data/LibRaw-0.22.1.tar.gz
-https://download.gnome.org/sources/librsvg/2.62/librsvg-2.62.2.tar.xz
+https://download.gnome.org/sources/librsvg/2.62/librsvg-2.62.3.tar.xz
 https://github.com/fontforge/libspiro/releases/download/20220722/libspiro-dist-20220722.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.7.1.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.6.0.tar.gz
@@ -295,7 +295,7 @@ https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmond/releases/download/sysmond-1.0.1/sysmond-1.0.1.tar.xz
 https://github.com/lfs-book/sysmon3/releases/download/3.0.3/sysmon3-3.0.3.tar.xz
 https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-019.tar.xz
-https://github.com/lu-zero/cargo-c/archive/v0.10.22/cargo-c-0.10.22.tar.gz
+https://github.com/lu-zero/cargo-c/archive/v0.10.23/cargo-c-0.10.23.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.29.3/cbindgen-0.29.3.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/clisp/clisp-2.49.95.tar.xz
 https://cmake.org/files/v4.3/cmake-4.3.3.tar.gz
@@ -613,7 +613,7 @@ https://download.gnome.org/sources/libnma/1.10/libnma-1.10.6.tar.xz
 https://github.com/thkukuk/libnsl/releases/download/v2.0.1/libnsl-2.0.1.tar.xz
 https://www.tcpdump.org/release/libpcap-1.10.6.tar.gz
 https://github.com/rockdaboot/libpsl/releases/download/0.21.5/libpsl-0.21.5.tar.gz
-https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.2/libslirp-v4.9.2.tar.bz2
+https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.3/libslirp-v4.9.3.tar.bz2
 https://download.gnome.org/sources/libsoup/3.6/libsoup-3.6.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libsoup-3.6.6-upstream_fixes-1.patch
 https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.7.tar.bz2
@@ -657,9 +657,9 @@ https://www.x.org/pub/individual/data/xbitmaps-1.1.4.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20250912.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
 https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.47.tar.xz
-https://www.x.org/pub/individual/xserver/xwayland-24.1.11.tar.xz
-https://www.x.org/pub/individual/xserver/xorg-server-21.1.22.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.22-tearfree_backport-1.patch
+https://www.x.org/pub/individual/xserver/xwayland-24.1.12.tar.xz
+https://www.x.org/pub/individual/xserver/xorg-server-21.1.23.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.23-tearfree_backport-1.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.6.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
 https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.31.2/libinput-1.31.2.tar.gz
@@ -931,8 +931,8 @@ https://github.com/transmission/transmission/releases/download/4.1.1/transmissio
 https://github.com/ib/xarchiver/archive/0.5.4.26/xarchiver-0.5.4.26.tar.gz
 https://gitlab.freedesktop.org/xdg/xdg-utils/-/archive/v1.2.1/xdg-utils-v1.2.1.tar.gz
 https://www.jwz.org/xscreensaver/xscreensaver-6.15.tar.gz
-https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.15.3.tar.bz2
-https://www.alsa-project.org/files/pub/lib/alsa-ucm-conf-1.2.15.3.tar.bz2
+https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.16.tar.bz2
+https://www.alsa-project.org/files/pub/lib/alsa-ucm-conf-1.2.16.tar.bz2
 https://www.alsa-project.org/files/pub/plugins/alsa-plugins-1.2.12.tar.bz2
 https://www.alsa-project.org/files/pub/utils/alsa-utils-1.2.15.2.tar.bz2
 https://www.alsa-project.org/files/pub/tools/alsa-tools-1.2.15.tar.bz2
@@ -990,7 +990,7 @@ https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.6.6/pipewire-1.6.6.
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.2.tar.xz
 https://www.libsdl.org/release/sdl2-compat-2.32.68.tar.gz
-https://www.libsdl.org/release/SDL3-3.4.8.tar.gz
+https://www.libsdl.org/release/SDL3-3.4.10.tar.gz
 https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.4.1.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
@@ -1006,8 +1006,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/xine-lib-1.2.13-upstream_fixes
 https://www.linuxfromscratch.org/patches/blfs/svn/xine-lib-1.2.13-gcc15_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/xine-lib-1.2.13-ffmpeg8.patch
 https://downloads.xvid.com/downloads/xvidcore-1.3.7.tar.gz
-https://distfiles.audacious-media-player.org/audacious-4.5.1.tar.bz2
-https://distfiles.audacious-media-player.org/audacious-plugins-4.5.1.tar.bz2
+https://distfiles.audacious-media-player.org/audacious-4.6.tar.bz2
+https://distfiles.audacious-media-player.org/audacious-plugins-4.6.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
 https://download.kde.org/stable/release-service/26.04.1/src/kwave-26.04.1.tar.xz
