diff --git a/wget-list b/wget-list
index aabd71571d..d953171615 100644
--- a/wget-list
+++ b/wget-list
@@ -162,8 +162,8 @@ https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/qca-2.3.10-openssl4_fixes-1.patch
 https://github.com/danvratil/qcoro/archive/v0.13.0/qcoro-0.13.0.tar.gz
 https://github.com/simdutf/simdutf/archive/v9.0.0/simdutf-9.0.0.tar.gz
-https://archive.mozilla.org/pub/firefox/releases/140.11.0esr/source/firefox-140.11.0esr.source.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.11.0-python_3.14_fixes-1.patch
+https://archive.mozilla.org/pub/firefox/releases/140.12.0esr/source/firefox-140.12.0esr.source.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.12.0-python_3.14_fixes-1.patch
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.350.0/SPIRV-Headers-vulkan-sdk-1.4.350.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.350.0/SPIRV-Tools-vulkan-sdk-1.4.350.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.2/SPIRV-LLVM-Translator-22.1.2.tar.gz
@@ -907,9 +907,9 @@ https://download.documentfoundation.org/libreoffice/src/26.2.4/libreoffice-trans
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-26.2.4.2-poppler_26.06-1.patch
 https://download.gnome.org/sources/epiphany/50/epiphany-50.4.tar.xz
 https://download.kde.org/stable/release-service/26.04.1/src/falkon-26.04.1.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.11.0esr-glibc-2.43.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.11.0esr-python_3.14_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.11.0esr-llvm_22-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.12.0esr-glibc-2.43.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.12.0esr-python_3.14_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.12.0esr-llvm_22-1.patch
 https://archive.seamonkey-project.org/releases/2.53.23/source/seamonkey-2.53.23.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.23-libvpx_security_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.23-cxx17-1.patch
