diff --git a/wget-list b/wget-list
index f40e38539c..6bba86ec8d 100644
--- a/wget-list
+++ b/wget-list
@@ -87,7 +87,7 @@ https://download.gnome.org/sources/glibmm/2.88/glibmm-2.88.0.tar.xz
 https://github.com/jstedfast/gmime/releases/download/3.2.15/gmime-3.2.15.tar.xz
 https://github.com/lfs-book/gmmlib/archive/v22.10.0/gmmlib-22.10.0.tar.gz
 https://ftpmirror.gnu.org/gsl/gsl-2.8.tar.gz
-https://download.gnome.org/sources/gspell/1.14/gspell-1.14.3.tar.xz
+https://download.gnome.org/sources/gspell/1.14/gspell-1.14.4.tar.xz
 https://github.com/google/highway/archive/1.4.0/highway-1.4.0.tar.gz
 https://github.com/unicode-org/icu/releases/download/release-78.3/icu4c-78.3-sources.tgz
 https://github.com/benhoyt/inih/archive/r62/inih-r62.tar.gz
@@ -200,7 +200,7 @@ https://github.com/mm2/Little-CMS/releases/download/lcms2.19.1/lcms2-2.19.1.tar.
 https://github.com/AOMediaCodec/libavif/archive/v1.4.2/libavif-1.4.2.tar.gz
 https://github.com/libexif/libexif/releases/download/v0.6.26/libexif-0.6.26.tar.bz2
 https://download.gnome.org/sources/libgxps/0.3/libgxps-0.3.2.tar.xz
-https://github.com/strukturag/libheif/releases/download/v1.23.0/libheif-1.23.0.tar.gz
+https://github.com/strukturag/libheif/releases/download/v1.23.1/libheif-1.23.1.tar.gz
 https://github.com/libjpeg-turbo/libjpeg-turbo/releases/download/3.1.4.1/libjpeg-turbo-3.1.4.1.tar.gz
 https://github.com/libjxl/libjxl/archive/v0.11.2/libjxl-0.11.2.tar.gz
 https://gitlab.freedesktop.org/emersion/libliftoff/-/archive/v0.5.0/libliftoff-v0.5.0.tar.gz
@@ -257,8 +257,8 @@ https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.3.2/unix-tree-2.3.2.ta
 https://github.com/lurcher/unixODBC/archive/v2.3.14/unixODBC-2.3.14.tar.gz
 https://github.com/flatpak/xdg-dbus-proxy/releases/download/0.1.7/xdg-dbus-proxy-0.1.7.tar.xz
 https://gitlab.freedesktop.org/xdg/xdg-user-dirs/-/archive/v0.20/xdg-user-dirs-v0.20.tar.gz
-https://github.com/ip7z/7zip/archive/26.01/7zip-26.01.tar.gz
-https://www.freedesktop.org/software/accountsservice/accountsservice-23.13.9.tar.xz
+https://github.com/ip7z/7zip/archive/26.02/7zip-26.02.tar.gz
+https://gitlab.freedesktop.org/accountsservice/accountsservice/-/archive/26.26.9/accountsservice-26.26.9.tar.bz2
 https://downloads.sourceforge.net/acpid2/acpid-2.0.34.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/at/at_3.2.5.orig.tar.gz
 https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.9.tar.xz
@@ -321,7 +321,7 @@ https://anduin.linuxfromscratch.org/BLFS/luajit/luajit-20260213.tar.xz
 https://www.mercurial-scm.org/release/mercurial-7.2.2.tar.gz
 https://www.nasm.us/pub/nasm/releasebuilds/3.01/nasm-3.01.tar.xz
 https://www.nasm.us/pub/nasm/releasebuilds/3.01/nasm-3.01-xdoc.tar.xz
-https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
+https://github.com/NixOS/patchelf/releases/download/0.19.0/patchelf-0.19.0.tar.gz
 https://www.cpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz
 https://www.cpan.org/authors/id/V/VP/VPIT/autovivification-0.18.tar.gz
 https://www.cpan.org/authors/id/B/BR/BRIANDFOY/Business-ISBN-3.014.tar.gz
@@ -341,7 +341,7 @@ https://www.cpan.org/authors/id/L/LE/LEONT/File-Slurper-0.014.tar.gz
 https://www.cpan.org/authors/id/P/PL/PLICEASE/File-Which-1.27.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/HTML-Parser-3.85.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/HTTP-Daemon-6.17.tar.gz
-https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.098.tar.gz
+https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.099.tar.gz
 https://www.cpan.org/authors/id/G/GA/GAAS/IO-String-1.08.tar.gz
 https://www.cpan.org/authors/id/R/RJ/RJBS/IPC-Run3-0.049.tar.gz
 https://www.cpan.org/authors/id/A/AL/ALINKE/Lingua-Translit-0.29.tar.gz
@@ -713,7 +713,7 @@ https://dri.freedesktop.org/libdrm/libdrm-2.4.134.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
 https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.8.tar.xz
 https://people.freedesktop.org/~svu/libxklavier-5.4.tar.bz2
-https://download.gnome.org/sources/pango/1.57/pango-1.57.1.tar.xz
+https://download.gnome.org/sources/pango/1.58/pango-1.58.0.tar.xz
 https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.5.tar.xz
 https://download.gnome.org/sources/pangomm/2.56/pangomm-2.56.2.tar.xz
 https://download.qt.io/archive/qt/6.11/6.11.1/single/qt-everywhere-src-6.11.1.tar.xz
@@ -794,7 +794,7 @@ https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/gweather-locations/2026/gweather-locations-2026.2.tar.xz
 https://download.gnome.org/sources/libgweather/4.6/libgweather-4.6.0.tar.xz
 https://download.gnome.org/sources/libpeas/1.38/libpeas-1.38.1.tar.xz
-https://download.gnome.org/sources/libshumate/1.6/libshumate-1.6.1.tar.xz
+https://download.gnome.org/sources/libshumate/1.6/libshumate-1.6.2.tar.xz
 https://download.gnome.org/sources/evolution-data-server/3.60/evolution-data-server-3.60.2.tar.xz
 https://download.gnome.org/sources/tinysparql/3.11/tinysparql-3.11.1.tar.xz
 https://download.gnome.org/sources/localsearch/3.11/localsearch-3.11.1.tar.xz
@@ -829,7 +829,7 @@ https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-
 https://download.gnome.org/sources/gnome-connections/50/gnome-connections-50.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/46/gnome-disk-utility-46.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/45/gnome-logs-45.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/50/gnome-maps-50.1.tar.xz
+https://download.gnome.org/sources/gnome-maps/50/gnome-maps-50.2.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/50/gnome-power-manager-50.0.tar.xz
@@ -1090,7 +1090,7 @@ https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308
 https://downloads.sourceforge.net/asymptote/asymptote-3.10.src.tgz
 https://github.com/plk/biber/archive/v2.21/biber-2.21.tar.gz
 https://github.com/mgieseki/dvisvgm/releases/download/3.6/dvisvgm-3.6.tar.gz
-https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
+https://ctan.org/indexing/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20251220.tar.xz
 https://github.com/elogind/elogind/archive/v257.14/elogind-257.14.tar.gz
