diff --git a/wget-list b/wget-list
index 3eab8dd211..daa8e86cfb 100644
--- a/wget-list
+++ b/wget-list
@@ -60,7 +60,7 @@ http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.4.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.1.tar.xz
 https://download.qemu.org/qemu-11.0.1.tar.xz
-https://github.com/abseil/abseil-cpp/releases/download/20260107.1/abseil-cpp-20260107.1.tar.gz
+https://github.com/abseil/abseil-cpp/releases/download/20260526.0/abseil-cpp-20260526.0.tar.gz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.1.2.tar.xz
 http://people.freedesktop.org/~hughsient/appstream-glib/releases/appstream-glib-0.8.3.tar.xz
 https://archive.apache.org/dist/apr/apr-1.7.6.tar.bz2
@@ -262,7 +262,7 @@ https://www.freedesktop.org/software/colord/releases/colord-1.4.8.tar.xz
 https://ftpmirror.gnu.org/cpio/cpio-2.15.tar.bz2
 https://www.freedesktop.org/software/cups-pk-helper/releases/cups-pk-helper-0.2.7.tar.xz
 https://dbus.freedesktop.org/releases/dbus/dbus-1.16.2.tar.xz
-http://fcron.free.fr/archives/fcron-3.4.0.src.tar.gz
+http://fcron.free.fr/archives/fcron-3.4.1.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-gcc15_fixes-1.patch
@@ -632,7 +632,7 @@ https://archive.apache.org/dist/httpd/httpd-2.4.67.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/httpd-blfs_layout-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/httpd-2.4.67-openssl4_fixes-2.patch
 https://downloads.isc.org/isc/kea/3.0.2/kea-3.0.2.tar.xz
-https://github.com/proftpd/proftpd/archive/v1.3.9a/proftpd-1.3.9a.tar.gz
+https://github.com/proftpd/proftpd/archive/v1.3.9b/proftpd-1.3.9b.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.2.tar.gz
 https://ftp.exim.org/pub/exim/exim4/exim-4.99.1.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.11.3.tar.gz
@@ -661,7 +661,7 @@ https://www.x.org/pub/individual/xserver/xorg-server-21.1.23.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.23-tearfree_backport-1.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.6.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
-https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.31.2/libinput-1.31.2.tar.gz
+https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.31.3/libinput-1.31.3.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.10.0.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.4/xf86-input-wacom-1.2.4.tar.bz2
@@ -672,8 +672,8 @@ https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.5.tar.xz
 https://download.gnome.org/sources/at-spi2-core/2.60/at-spi2-core-2.60.4.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.4.tar.xz
-https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
-https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
+https://www.cairographics.org/releases/cairomm-1.14.6.tar.xz
+https://www.cairographics.org/releases/cairomm-1.18.1.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
 https://github.com/fltk/fltk/releases/download/release-1.4.5/fltk-1.4.5-source.tar.gz
 https://github.com/fltk/fltk/releases/download/release-1.4.5/fltk-1.4.5-docs-html.tar.gz
@@ -926,14 +926,14 @@ https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.11.0esr-glibc-
 https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.11.0esr-llvm_22-1.patch
 https://github.com/TigerVNC/tigervnc/archive/v1.16.0/tigervnc-1.16.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.16.0-configuration_fixes-1.patch
-https://github.com/transmission/transmission/releases/download/4.1.1/transmission-4.1.1.tar.xz
+https://github.com/transmission/transmission/releases/download/4.1.2/transmission-4.1.2.tar.xz
 https://github.com/ib/xarchiver/archive/0.5.4.26/xarchiver-0.5.4.26.tar.gz
 https://gitlab.freedesktop.org/xdg/xdg-utils/-/archive/v1.2.1/xdg-utils-v1.2.1.tar.gz
 https://www.jwz.org/xscreensaver/xscreensaver-6.15.tar.gz
 https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.16.tar.bz2
 https://www.alsa-project.org/files/pub/lib/alsa-ucm-conf-1.2.16.tar.bz2
 https://www.alsa-project.org/files/pub/plugins/alsa-plugins-1.2.12.tar.bz2
-https://www.alsa-project.org/files/pub/utils/alsa-utils-1.2.15.2.tar.bz2
+https://www.alsa-project.org/files/pub/utils/alsa-utils-1.2.16.tar.bz2
 https://www.alsa-project.org/files/pub/tools/alsa-tools-1.2.15.tar.bz2
 https://www.alsa-project.org/files/pub/firmware/alsa-firmware-1.2.4.tar.bz2
 https://download.gnome.org/sources/audiofile/0.3/audiofile-0.3.6.tar.xz
@@ -965,7 +965,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/libcanberra-0.30-wayland-1.pat
 https://downloads.sourceforge.net/libcddb/libcddb-1.3.2.tar.bz2
 https://ftpmirror.gnu.org/libcdio/libcdio-2.1.0.tar.bz2
 https://ftpmirror.gnu.org/libcdio/libcdio-paranoia-10.2+2.0.2.tar.bz2
-https://github.com/strukturag/libde265/releases/download/v1.1.0/libde265-1.1.0.tar.gz
+https://github.com/strukturag/libde265/releases/download/v1.1.1/libde265-1.1.1.tar.gz
 https://get.videolan.org/libdvdcss/1.5.0/libdvdcss-1.5.0.tar.xz
 https://get.videolan.org/libdvdread/7.0.1/libdvdread-7.0.1.tar.xz
 https://get.videolan.org/libdvdnav/7.0.0/libdvdnav-7.0.0.tar.xz
@@ -1011,7 +1011,7 @@ https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
 https://download.kde.org/stable/release-service/26.04.1/src/kwave-26.04.1.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
-https://downloads.sourceforge.net/mpg123/mpg123-1.33.5.tar.bz2
+https://downloads.sourceforge.net/mpg123/mpg123-1.33.6.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-6.2.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.3.tar.gz
