diff --git a/wget-list b/wget-list
index b3d01a850d..81e797d0ae 100644
--- a/wget-list
+++ b/wget-list
@@ -27,7 +27,7 @@ https://github.com/p11-glue/p11-kit/releases/download/0.26.5/p11-kit-0.26.5.tar.
 https://github.com/polkit-org/polkit/archive/127/polkit-127.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
-https://github.com/shadow-maint/shadow/releases/download/4.20.2/shadow-4.20.2.tar.xz
+https://github.com/shadow-maint/shadow/releases/download/4.20.3/shadow-4.20.3.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.82.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.17p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
@@ -287,7 +287,7 @@ https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://github.com/kennylevinsen/seatd/archive/0.9.3/seatd-0.9.3.tar.gz
 https://sysstat.github.io/sysstat-packages/sysstat-12.8.0.tar.xz
-https://github.com/systemd/systemd/archive/v261.3/systemd-261.3.tar.gz
+https://github.com/systemd/systemd/archive/v262/systemd-262.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.11.2/udisks-2.11.2.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.2.4.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.91.4/upower-v1.91.4.tar.bz2
@@ -322,7 +322,7 @@ https://anduin.linuxfromscratch.org/BLFS/luajit/luajit-20260213.tar.xz
 https://www.mercurial-scm.org/release/mercurial-7.2.4.tar.gz
 https://www.nasm.us/pub/nasm/releasebuilds/3.02/nasm-3.02.tar.xz
 https://www.nasm.us/pub/nasm/releasebuilds/3.02/nasm-3.02-xdoc.tar.xz
-https://github.com/NixOS/patchelf/releases/download/0.19.1/patchelf-0.19.1.tar.gz
+https://github.com/NixOS/patchelf/releases/download/0.19.2/patchelf-0.19.2.tar.gz
 https://www.cpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz
 https://www.cpan.org/authors/id/V/VP/VPIT/autovivification-0.18.tar.gz
 https://www.cpan.org/authors/id/B/BR/BRIANDFOY/Business-ISBN-3.014.tar.gz
@@ -475,9 +475,9 @@ https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
 https://www.php.net/distributions/php-8.5.11.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
-https://www.python.org/ftp/python/3.14.7/Python-3.14.7.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/Python-3.14.7-openssl_4-1.patch
-https://www.python.org/ftp/python/doc/3.14.7/python-3.14.7-docs-html.tar.bz2
+https://www.python.org/ftp/python/3.14.8/Python-3.14.8.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/Python-3.14.8-openssl_4-1.patch
+https://www.python.org/ftp/python/doc/3.14.8/python-3.14.8-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/Python-3.11.1-openssl_4_fixes-1.patch
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
@@ -692,7 +692,7 @@ https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.62.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk/3.24/gtk-3.24.52.tar.xz
-https://download.gnome.org/sources/gtk/4.24/gtk-4.24.0.tar.xz
+https://download.gnome.org/sources/gtk/4.24/gtk-4.24.1.tar.xz
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.11.tar.xz
 https://download.gnome.org/sources/gtkmm/4.24/gtkmm-4.24.0.tar.xz
 https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
@@ -713,7 +713,7 @@ https://people.freedesktop.org/~svu/libxklavier-5.4.tar.bz2
 https://download.gnome.org/sources/pango/1.58/pango-1.58.2.tar.xz
 https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.5.tar.xz
 https://download.gnome.org/sources/pangomm/2.58/pangomm-2.58.0.tar.xz
-https://download.qt.io/archive/qt/6.11/6.11.2/single/qt-everywhere-src-6.11.2.tar.xz
+https://download.qt.io/archive/qt/6.12/6.12.0/single/qt-everywhere-src-6.12.0.tar.xz
 https://download.qt.io/official_releases/qt/6.11/6.11.2/submodules/qtwebengine-everywhere-src-6.11.2.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/vulkan-sdk-1.4.363.0/Vulkan-Headers-vulkan-sdk-1.4.363.0.tar.gz
@@ -818,8 +818,7 @@ https://download.gnome.org/sources/gnome-tweaks/49/gnome-tweaks-49.0.tar.xz
 https://download.gnome.org/sources/gnome-user-docs/51/gnome-user-docs-51.0.tar.xz
 https://download.gnome.org/sources/yelp/49/yelp-49.2.tar.xz
 https://download.gnome.org/sources/baobab/50/baobab-50.0.tar.xz
-https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/brasero-3.12.3-upstream_fixes-1.patch
+https://download.gnome.org/sources/brasero/3.12/brasero-3.12.4.tar.xz
 https://download.gnome.org/sources/evince/48/evince-48.4.tar.xz
 https://download.gnome.org/sources/evolution/3.62/evolution-3.62.0.tar.xz
 https://download.gnome.org/sources/file-roller/44/file-roller-44.7.tar.xz
@@ -949,7 +948,7 @@ https://download.gnome.org/sources/audiofile/0.3/audiofile-0.3.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/audiofile-0.3.6-consolidated_patches-1.patch
 https://code.videolan.org/videolan/dav1d/-/archive/1.5.4/dav1d-1.5.4.tar.gz
 https://github.com/lfs-book/faac/archive/v2.2/faac-2.2.tar.gz
-https://github.com/knik0/faad2/archive/2.11.3/faad2-2.11.3.tar.gz
+https://github.com/knik0/faad2/archive/2.11.4/faad2-2.11.4.tar.gz
 https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://github.com/xiph/flac/releases/download/1.5.0/flac-1.5.0.tar.xz
 https://github.com/dyne/frei0r/archive/v3.6.0/frei0r-3.6.0.tar.gz
