diff --git a/wget-list b/wget-list
index bc2941f869..1f3747dadb 100644
--- a/wget-list
+++ b/wget-list
@@ -149,8 +149,7 @@ https://archive.mozilla.org/pub/nspr/releases/v4.36/src/nspr-4.36.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.44/pcre2-10.44.tar.bz2
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://github.com/protocolbuffers/protobuf/releases/download/v29.3/protobuf-29.3.tar.gz
-https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.0/protobuf-c-1.5.0.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.0-consolidated_fixes-1.patch
+https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.1/protobuf-c-1.5.1.tar.gz
 https://download.kde.org/stable/qca/2.3.9/qca-2.3.9.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.11.0/qcoro-0.11.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/128.6.0esr/source/firefox-128.6.0esr.source.tar.xz
@@ -854,7 +853,7 @@ https://github.com/lxqt/lxqt-qtplugin/releases/download/2.1.0/lxqt-qtplugin-2.1.
 https://github.com/lxqt/lxqt-panel/releases/download/2.1.4/lxqt-panel-2.1.4.tar.xz
 https://github.com/lxqt/pcmanfm-qt/releases/download/2.1.0/pcmanfm-qt-2.1.0.tar.xz
 https://github.com/lxqt/lxqt-powermanagement/releases/download/2.1.0/lxqt-powermanagement-2.1.0.tar.xz
-https://github.com/lxqt/lxqt-runner/releases/download/2.1.1/lxqt-runner-2.1.1.tar.xz
+https://github.com/lxqt/lxqt-runner/releases/download/2.1.2/lxqt-runner-2.1.2.tar.xz
 https://github.com/lxqt/lxqt-themes/releases/download/2.1.0/lxqt-themes-2.1.0.tar.xz
 https://github.com/lxqt/lxqt-admin/releases/download/2.1.0/lxqt-admin-2.1.0.tar.xz
 https://github.com/lxqt/lxqt-about/releases/download/2.1.0/lxqt-about-2.1.0.tar.xz
@@ -865,7 +864,7 @@ https://github.com/lxqt/lxqt-openssh-askpass/releases/download/2.1.0/lxqt-openss
 https://github.com/lxqt/lxqt-session/releases/download/2.1.1/lxqt-session-2.1.1.tar.xz
 https://github.com/lxqt/xdg-desktop-portal-lxqt/releases/download/1.1.0/xdg-desktop-portal-lxqt-1.1.0.tar.xz
 https://github.com/lxqt/obconf-qt/releases/download/0.16.5/obconf-qt-0.16.5.tar.xz
-https://github.com/lxqt/lximage-qt/releases/download/2.1.0/lximage-qt-2.1.0.tar.xz
+https://github.com/lxqt/lximage-qt/releases/download/2.1.1/lximage-qt-2.1.1.tar.xz
 https://github.com/lxqt/lxqt-archiver/releases/download/1.1.0/lxqt-archiver-1.1.0.tar.xz
 https://github.com/lxqt/lxqt-notificationd/releases/download/2.1.1/lxqt-notificationd-2.1.1.tar.xz
 https://github.com/lxqt/pavucontrol-qt/releases/download/2.1.0/pavucontrol-qt-2.1.0.tar.xz
