diff --git a/wget-list b/wget-list
index 01c44e7046..849af7be5f 100644
--- a/wget-list
+++ b/wget-list
@@ -606,6 +606,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/procmail-3.24-consolidated_fix
 https://archive.apache.org/dist/httpd/httpd-2.4.63.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/httpd-2.4.63-blfs_layout-1.patch
 https://downloads.isc.org/isc/kea/2.6.1/kea-2.6.1.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/kea-2.6.1-fix_boost_1_87-1.patch
 https://github.com/proftpd/proftpd/archive/v1.3.8b/proftpd-1.3.8b.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.0.tar.gz
 https://ftp.exim.org/pub/exim/exim4/exim-4.98.tar.xz
