diff --git a/wget-list b/wget-list
index fba60b933f..7ee961e221 100644
--- a/wget-list
+++ b/wget-list
@@ -98,7 +98,7 @@ https://github.com/libarchive/libarchive/releases/download/v3.7.7/libarchive-3.7
 https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-3.0.1.tar.bz2
 https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/ivmai/libatomic_ops/releases/download/v7.8.2/libatomic_ops-7.8.2.tar.gz
-https://github.com/storaged-project/libblockdev/releases/download/3.2.1/libblockdev-3.2.1.tar.gz
+https://github.com/storaged-project/libblockdev/releases/download/3.3.0/libblockdev-3.3.0.tar.gz
 https://github.com/storaged-project/libbytesize/releases/download/2.11/libbytesize-2.11.tar.gz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.7/libclc-19.1.7.src.tar.xz
 https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
@@ -117,7 +117,7 @@ https://github.com/cjlin1/liblinear/archive/v248/liblinear-248.tar.gz
 https://www.freedesktop.org/software/libmbim/libmbim-1.26.4.tar.xz
 https://github.com/linux-nvme/libnvme/archive/v1.11.1/libnvme-1.11.1.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.6/libpaper-2.2.6.tar.gz
-https://github.com/flatpak/libportal/releases/download/0.9.0/libportal-0.9.0.tar.xz
+https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar.xz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
 https://github.com/Qalculate/libqalculate/releases/download/v5.5.0/libqalculate-5.5.0.tar.gz
 https://www.freedesktop.org/software/libqmi/libqmi-1.30.8.tar.xz
@@ -203,7 +203,7 @@ https://www.cairographics.org/releases/pixman-0.44.2.tar.gz
 https://poppler.freedesktop.org/poppler-25.02.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
-https://github.com/qpdf/qpdf/releases/download/v11.10.0/qpdf-11.10.0.tar.gz
+https://github.com/qpdf/qpdf/releases/download/v11.10.1/qpdf-11.10.1.tar.gz
 https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.bz2
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
@@ -815,14 +815,14 @@ https://archive.xfce.org/src/xfce/tumbler/4.20/tumbler-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.20/xfce4-appfinder-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-power-manager/4.20/xfce4-power-manager-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-settings/4.20/xfce4-settings-4.20.1.tar.bz2
-https://archive.xfce.org/src/xfce/xfdesktop/4.20/xfdesktop-4.20.0.tar.bz2
+https://archive.xfce.org/src/xfce/xfdesktop/4.20/xfdesktop-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfwm4/4.20/xfwm4-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-session/4.20/xfce4-session-4.20.0.tar.bz2
 https://archive.xfce.org/src/apps/parole/4.18/parole-4.18.2.tar.bz2
 https://archive.xfce.org/src/apps/xfce4-terminal/1.1/xfce4-terminal-1.1.4.tar.bz2
 https://archive.xfce.org/src/apps/xfburn/0.7/xfburn-0.7.2.tar.bz2
 https://archive.xfce.org/src/apps/ristretto/0.13/ristretto-0.13.3.tar.bz2
-https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.6.tar.bz2
+https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.7.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.4/xfce4-pulseaudio-plugin-0.4.9.tar.bz2
 https://download.kde.org/stable/frameworks/6.9/kwindowsystem-6.9.0.tar.xz
 https://download.kde.org/stable/frameworks/6.9/kconfig-6.9.0.tar.xz
@@ -891,7 +891,7 @@ https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
 https://inkscape.org/gallery/item/53679/inkscape-1.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4-poppler_fixes-2.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/128.7.0esr/source/thunderbird-128.7.0esr.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/128.7.1esr/source/thunderbird-128.7.1esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.14.1/tigervnc-1.14.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.14.1-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
