diff --git a/wget-list b/wget-list
index ea44756ee3..249fb717c0 100644
--- a/wget-list
+++ b/wget-list
@@ -5,7 +5,7 @@ https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.7.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.9.tar.xz
-https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.1.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.2.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.11.tar.xz
 https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.73.tar.xz
 https://github.com/linux-pam/linux-pam/releases/download/v1.7.0/Linux-PAM-1.7.0.tar.xz
@@ -58,7 +58,7 @@ https://github.com/vim/vim/archive/v9.1.1071/vim-9.1.1071.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.15.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
-https://download.qemu.org/qemu-9.2.0.tar.xz
+https://download.qemu.org/qemu-9.2.1.tar.xz
 https://github.com/abseil/abseil-cpp/releases/download/20250127.0/abseil-cpp-20250127.0.tar.gz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.4.tar.xz
 http://people.freedesktop.org/~hughsient/appstream-glib/releases/appstream-glib-0.8.3.tar.xz
@@ -456,7 +456,7 @@ https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.2.0.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.2/cachecontrol-0.14.2.tar.gz
-https://github.com/cython/cython/releases/download/3.0.11-1/cython-3.0.11.tar.gz
+https://github.com/cython/cython/releases/download/3.0.12-1/cython-3.0.12.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.34.2.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.2.tar.gz
@@ -886,7 +886,7 @@ https://feh.finalrewind.org/feh-3.10.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20230101/fontforge-20230101.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/fontforge-20230101-security_fixes-1.patch
 https://github.com/freerdp/freerdp/archive/3.11.1/FreeRDP-3.11.1.tar.gz
-https://download.gimp.org/gimp/v3.0/gimp-3.0.0-RC2.tar.xz
+https://download.gimp.org/gimp/v3.0/gimp-3.0.0-RC3.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/gimp/gimp3-help-20240711.tar.xz
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
 https://inkscape.org/gallery/item/53679/inkscape-1.4.tar.xz
@@ -911,7 +911,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/audiofile-0.3.6-consolidated_p
 https://github.com/knik0/faac/archive/1_30/faac-1_30.tar.gz
 https://github.com/knik0/faad2/archive/2.11.1/faad2-2.11.1.tar.gz
 https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
-https://downloads.xiph.org/releases/flac/flac-1.5.0.tar.xz
+https://github.com/xiph/flac/releases/download/1.5.0/flac-1.5.0.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
 https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.12.tar.xz
@@ -1037,7 +1037,7 @@ https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312-source.tar.xz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312-texmf.tar.xz
 https://ftp.tu-chemnitz.de/pub/tug/historic/systems/texlive/2024/texlive-20240312-extra.tar.xz
-https://downloads.sourceforge.net/asymptote/asymptote-2.98.src.tgz
+https://downloads.sourceforge.net/asymptote/asymptote-2.99.src.tgz
 https://github.com/plk/biber/archive/v2.20/biber-2.20.tar.gz
 https://github.com/mgieseki/dvisvgm/releases/download/3.4.3/dvisvgm-3.4.3.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
