diff --git a/wget-list b/wget-list
index aff3c93a6f..68a454b5c3 100644
--- a/wget-list
+++ b/wget-list
@@ -152,8 +152,7 @@ https://archive.mozilla.org/pub/nspr/releases/v4.36/src/nspr-4.36.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.45/pcre2-10.45.tar.bz2
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://github.com/protocolbuffers/protobuf/releases/download/v30.2/protobuf-30.2.tar.gz
-https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.1/protobuf-c-1.5.1.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.1-upstream-protobuf-30.patch
+https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.2/protobuf-c-1.5.2.tar.gz
 https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.12.0/qcoro-0.12.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/128.8.0esr/source/firefox-128.8.0esr.source.tar.xz
@@ -207,7 +206,7 @@ https://www.cairographics.org/releases/pixman-0.44.2.tar.gz
 https://poppler.freedesktop.org/poppler-25.04.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
-https://github.com/qpdf/qpdf/releases/download/v12.0.0/qpdf-12.0.0.tar.gz
+https://github.com/qpdf/qpdf/releases/download/v12.1.0/qpdf-12.1.0.tar.gz
 https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.bz2
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
@@ -667,14 +666,14 @@ https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.59.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk/3.24/gtk-3.24.49.tar.xz
-https://download.gnome.org/sources/gtk/4.18/gtk-4.18.3.tar.xz
+https://download.gnome.org/sources/gtk/4.18/gtk-4.18.4.tar.xz
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.10.tar.xz
 https://download.gnome.org/sources/gtkmm/4.18/gtkmm-4.18.0.tar.xz
 https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
 https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
 https://download.gnome.org/sources/gtksourceview/4.8/gtksourceview-4.8.4.tar.xz
 https://download.gnome.org/sources/gtksourceview/5.16/gtksourceview-5.16.0.tar.xz
-https://downloads.sourceforge.net/enlightenment/imlib2-1.12.4.tar.xz
+https://downloads.sourceforge.net/enlightenment/imlib2-1.12.5.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.1/kImageAnnotator-0.7.1.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
