diff --git a/wget-list b/wget-list
index dfac96b079..256950b672 100644
--- a/wget-list
+++ b/wget-list
@@ -86,7 +86,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
 https://download.gnome.org/sources/glibmm/2.84/glibmm-2.84.0.tar.xz
 https://download.gnome.org/sources/gmime/3.2/gmime-3.2.7.tar.xz
-https://github.com/lfs-book/gmmlib/archive/v22.5.5/gmmlib-22.5.5.tar.gz
+https://github.com/lfs-book/gmmlib/archive/v22.7.1/gmmlib-22.7.1.tar.gz
 https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
 https://download.gnome.org/sources/gspell/1.14/gspell-1.14.0.tar.xz
 https://github.com/google/highway/archive/1.2.0/highway-1.2.0.tar.gz
@@ -121,6 +121,7 @@ https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.32.0/libmbim
 https://github.com/linux-nvme/libnvme/archive/v1.13/libnvme-1.13.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.6/libpaper-2.2.6.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
 https://github.com/Qalculate/libqalculate/releases/download/v5.5.2/libqalculate-5.5.2.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.36.0/libqmi-1.36.0.tar.gz
@@ -555,7 +556,7 @@ https://www.kernel.org/pub/linux/utils/nfs-utils/2.8.3/nfs-utils-2.8.3.tar.xz
 https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.7.tar.bz2
 https://www.samba.org/ftp/rsync/src/rsync-3.4.1.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.22.0.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.22.1.tar.gz
 https://ftp.gnu.org/gnu/wget/wget-1.25.0.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.9.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
@@ -700,7 +701,7 @@ https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
-https://github.com/ice-wm/icewm/archive/3.7.3/icewm-3.7.3.tar.gz
+https://github.com/ice-wm/icewm/archive/3.7.4/icewm-3.7.4.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openbox-3.6.1-py3-1.patch
 https://download.gnome.org/sources/adwaita-icon-theme/48/adwaita-icon-theme-48.0.tar.xz
@@ -760,8 +761,8 @@ https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libgweather-4.4.4-upstream_fix-1.patch
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.4/libshumate-1.4.0.tar.xz
-https://download.gnome.org/sources/evolution-data-server/3.56/evolution-data-server-3.56.0.tar.xz
-https://download.gnome.org/sources/tinysparql/3.9/tinysparql-3.9.1.tar.xz
+https://download.gnome.org/sources/evolution-data-server/3.56/evolution-data-server-3.56.1.tar.xz
+https://download.gnome.org/sources/tinysparql/3.9/tinysparql-3.9.2.tar.xz
 https://download.gnome.org/sources/localsearch/3.9/localsearch-3.9.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/localsearch-3.9.0-libxml2_2_14-1.patch
 https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
@@ -771,18 +772,17 @@ https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/48/gnome-backgrounds-48.0.tar.xz
 https://download.gnome.org/sources/gvfs/1.56/gvfs-1.56.1.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.3.tar.xz
-https://download.gnome.org/sources/nautilus/48/nautilus-48.0.tar.xz
+https://download.gnome.org/sources/nautilus/48/nautilus-48.1.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/47/gnome-bluetooth-47.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-bluetooth-47.1-build_fix-1.patch
 https://download.gnome.org/sources/gnome-keyring/48/gnome-keyring-48.0.tar.xz
-https://download.gnome.org/sources/gnome-settings-daemon/48/gnome-settings-daemon-48.0.tar.xz
+https://download.gnome.org/sources/gnome-settings-daemon/48/gnome-settings-daemon-48.1.tar.xz
 https://download.gnome.org/sources/tecla/48/tecla-48.0.2.tar.xz
-https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.0.tar.xz
-https://download.gnome.org/sources/mutter/48/mutter-48.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/mutter-48.0-upstream_fixes-1.patch
-https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.0.tar.xz
-https://download.gnome.org/sources/gnome-shell-extensions/48/gnome-shell-extensions-48.0.tar.xz
-https://download.gnome.org/sources/gnome-session/47/gnome-session-47.0.1.tar.xz
+https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.1.tar.xz
+https://download.gnome.org/sources/mutter/48/mutter-48.2.tar.xz
+https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.1.tar.xz
+https://download.gnome.org/sources/gnome-shell-extensions/48/gnome-shell-extensions-48.1.tar.xz
+https://download.gnome.org/sources/gnome-session/48/gnome-session-48.0.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.1.tar.xz
 https://download.gnome.org/sources/gnome-user-docs/48/gnome-user-docs-48.1.tar.xz
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
@@ -792,19 +792,19 @@ https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/brasero-3.12.3-upstream_fixes-1.patch
 https://download.gnome.org/sources/eog/47/eog-47.0.tar.xz
 https://download.gnome.org/sources/evince/48/evince-48.0.tar.xz
-https://download.gnome.org/sources/evolution/3.56/evolution-3.56.0.tar.xz
+https://download.gnome.org/sources/evolution/3.56/evolution-3.56.1.tar.xz
 https://download.gnome.org/sources/file-roller/44/file-roller-44.5.tar.xz
-https://download.gnome.org/sources/gnome-calculator/48/gnome-calculator-48.0.2.tar.xz
+https://download.gnome.org/sources/gnome-calculator/48/gnome-calculator-48.1.tar.xz
 https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-3.36.2.tar.xz
 https://download.gnome.org/sources/gnome-connections/48/gnome-connections-48.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/46/gnome-disk-utility-46.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/45/gnome-logs-45.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/48/gnome-maps-48.1.tar.xz
+https://download.gnome.org/sources/gnome-maps/48/gnome-maps-48.2.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/43/gnome-power-manager-43.0.tar.xz
 https://download.gnome.org/sources/gnome-screenshot/41/gnome-screenshot-41.0.tar.xz
-https://download.gnome.org/sources/gnome-system-monitor/48/gnome-system-monitor-48.0.tar.xz
+https://download.gnome.org/sources/gnome-system-monitor/48/gnome-system-monitor-48.1.tar.xz
 https://gitlab.gnome.org/GNOME/gnome-terminal/-/archive/3.56.0/gnome-terminal-3.56.0.tar.gz
 https://download.gnome.org/sources/gnome-weather/48/gnome-weather-48.0.tar.xz
 https://gitlab.gnome.org/GNOME/gucharmap/-/archive/16.0.2/gucharmap-16.0.2.tar.bz2
@@ -885,7 +885,7 @@ https://download.documentfoundation.org/libreoffice/src/25.2.2/libreoffice-dicti
 https://download.documentfoundation.org/libreoffice/src/25.2.2/libreoffice-help-25.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/25.2.2/libreoffice-translations-25.2.2.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-25.2.2.2-icu76_fixes-1.patch
-https://download.gnome.org/sources/epiphany/48/epiphany-48.0.tar.xz
+https://download.gnome.org/sources/epiphany/48/epiphany-48.1.tar.xz
 https://download.kde.org/stable/release-service/24.12.2/src/falkon-24.12.2.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.20/source/seamonkey-2.53.20.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.20-cxx17-1.patch
@@ -894,7 +894,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.p
 https://feh.finalrewind.org/feh-3.10.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20230101/fontforge-20230101.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/fontforge-20230101-security_fixes-1.patch
-https://github.com/freerdp/freerdp/archive/3.14.1/FreeRDP-3.14.1.tar.gz
+https://github.com/freerdp/freerdp/archive/3.15.0/FreeRDP-3.15.0.tar.gz
 https://download.gimp.org/gimp/v3.0/gimp-3.0.2.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
@@ -931,7 +931,7 @@ https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.26.0.t
 https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.26.0.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
-https://github.com/lfs-book/intel-media-driver/archive/v24.4.4/intel-media-driver-24.4.4.tar.gz
+https://github.com/lfs-book/intel-media-driver/archive/v25.1.4/intel-media-driver-25.1.4.tar.gz
 https://github.com/intel/intel-vaapi-driver/releases/download/2.4.1/intel-vaapi-driver-2.4.1.tar.bz2
 https://distfiles.adelielinux.org/source/a52dec/a52dec-0.8.0.tar.gz
 https://downloads.xiph.org/releases/ao/libao-1.2.0.tar.gz
@@ -1045,7 +1045,7 @@ https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-source.tar.xz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-texmf.tar.xz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-extra.tar.xz
-https://downloads.sourceforge.net/asymptote/asymptote-3.01.src.tgz
+https://downloads.sourceforge.net/asymptote/asymptote-3.02.src.tgz
 https://github.com/plk/biber/archive/v2.20/biber-2.20.tar.gz
 https://github.com/mgieseki/dvisvgm/releases/download/3.4.4/dvisvgm-3.4.4.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
