diff --git a/wget-list b/wget-list
index c252913f59..d85cdce4d2 100644
--- a/wget-list
+++ b/wget-list
@@ -22,7 +22,7 @@ https://github.com/p11-glue/p11-kit/releases/download/0.25.5/p11-kit-0.25.5.tar.
 https://github.com/polkit-org/polkit/archive/126/polkit-126.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
-https://github.com/shadow-maint/shadow/releases/download/4.17.3/shadow-4.17.3.tar.xz
+https://github.com/shadow-maint/shadow/releases/download/4.17.4/shadow-4.17.4.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.74.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.16p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
@@ -56,7 +56,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/24.12.2/src/kate-24.12.2.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.4.tar.xz
 https://www.nano-editor.org/dist/v8/nano-8.3.tar.xz
-https://github.com/vim/vim/archive/v9.1.1166/vim-9.1.1166.tar.gz
+https://github.com/vim/vim/archive/v9.1.1263/vim-9.1.1263.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.15.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
@@ -98,7 +98,7 @@ https://s3.amazonaws.com/json-c_releases/releases/json-c-0.18.tar.gz
 https://download.gnome.org/sources/json-glib/1.10/json-glib-1.10.6.tar.xz
 https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git/snapshot/keyutils-1.6.3.tar.gz
 https://pagure.io/libaio/archive/libaio-0.3.113/libaio-0.3.113.tar.gz
-https://github.com/libarchive/libarchive/releases/download/v3.7.8/libarchive-3.7.8.tar.xz
+https://github.com/libarchive/libarchive/releases/download/v3.7.9/libarchive-3.7.9.tar.xz
 https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-3.0.2.tar.bz2
 https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/ivmai/libatomic_ops/releases/download/v7.8.2/libatomic_ops-7.8.2.tar.gz
@@ -554,7 +554,7 @@ https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.x
 https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.3.tar.bz2
 https://www.ncftp.com/public_ftp/ncftp/ncftp-3.2.9-src.tar.gz
 https://downloads.sourceforge.net/project/net-tools/net-tools-2.10.tar.xz
-https://www.kernel.org/pub/linux/utils/nfs-utils/2.8.2/nfs-utils-2.8.2.tar.xz
+https://www.kernel.org/pub/linux/utils/nfs-utils/2.8.3/nfs-utils-2.8.3.tar.xz
 https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.7.tar.bz2
 https://www.samba.org/ftp/rsync/src/rsync-3.4.1.tar.gz
@@ -612,7 +612,7 @@ https://downloads.isc.org/isc/kea/2.6.2/kea-2.6.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/kea-2.6.2-fix_boost_1_87-1.patch
 https://github.com/proftpd/proftpd/archive/v1.3.9/proftpd-1.3.9.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.0.tar.gz
-https://ftp.exim.org/pub/exim/exim4/exim-4.98.1.tar.xz
+https://ftp.exim.org/pub/exim/exim4/exim-4.98.2.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.10.1.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
@@ -703,7 +703,7 @@ https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
-https://github.com/ice-wm/icewm/archive/3.7.2/icewm-3.7.2.tar.gz
+https://github.com/ice-wm/icewm/archive/3.7.3/icewm-3.7.3.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openbox-3.6.1-py3-1.patch
 https://download.gnome.org/sources/adwaita-icon-theme/48/adwaita-icon-theme-48.0.tar.xz
