diff --git a/wget-list b/wget-list
index 81158a71d5..02778ea138 100644
--- a/wget-list
+++ b/wget-list
@@ -28,7 +28,7 @@ https://www.sudo.ws/dist/sudo-1.9.16p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.14.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
-https://github.com/libfuse/libfuse/releases/download/fuse-3.16.2/fuse-3.16.2.tar.gz
+https://github.com/libfuse/libfuse/releases/download/fuse-3.17.2/fuse-3.17.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/jfsutils-1.1.15-gcc10_fix-1.patch
 https://sourceware.org/ftp/lvm2/LVM2.2.03.31.tgz
@@ -147,7 +147,7 @@ https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_2/log4cplus-2.1.2.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
-https://nodejs.org/dist/v22.14.0/node-v22.14.0.tar.xz
+https://nodejs.org/dist/v22.15.0/node-v22.15.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.36/src/nspr-4.36.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.45/pcre2-10.45.tar.bz2
@@ -219,7 +219,7 @@ https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.28.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
-https://github.com/google/shaderc/archive/v2025.1/shaderc-2025.1.tar.gz
+https://github.com/google/shaderc/archive/v2025.2/shaderc-2025.2.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/12.2.1/graphviz-12.2.1.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
 http://www.andre-simon.de/zip/highlight-4.15.tar.bz2
@@ -885,7 +885,7 @@ https://download.documentfoundation.org/libreoffice/src/25.2.2/libreoffice-dicti
 https://download.documentfoundation.org/libreoffice/src/25.2.2/libreoffice-help-25.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/25.2.2/libreoffice-translations-25.2.2.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-25.2.2.2-icu76_fixes-1.patch
-https://download.gnome.org/sources/epiphany/48/epiphany-48.1.tar.xz
+https://download.gnome.org/sources/epiphany/48/epiphany-48.2.tar.xz
 https://download.kde.org/stable/release-service/24.12.2/src/falkon-24.12.2.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.20/source/seamonkey-2.53.20.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.20-cxx17-1.patch
