diff --git a/wget-list b/wget-list
index 152b35dbf1..dfac96b079 100644
--- a/wget-list
+++ b/wget-list
@@ -291,7 +291,7 @@ https://doxygen.nl/files/doxygen-1.13.2.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-14.2.0/gcc-14.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
-https://ftp.gnu.org/gnu/gdb/gdb-16.2.tar.xz
+https://ftp.gnu.org/gnu/gdb/gdb-16.3.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-2.49.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.49.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.49.0.tar.xz
