diff --git a/wget-list b/wget-list
index f087319c5..7eac2c208 100644
--- a/wget-list
+++ b/wget-list
@@ -274,7 +274,7 @@ https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmon-qt/releases/download/v2.0.1/sysmon-qt-2.0.1.tar.xz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.6.tar.xz
-https://github.com/systemd/systemd/archive/v256.4/systemd-256.4.tar.gz
+https://github.com/systemd/systemd/archive/v256.5/systemd-256.5.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.0.9.tar.gz
 https://downloads.sourceforge.net/infozip/unzip60.tar.gz
@@ -293,7 +293,7 @@ https://cmake.org/files/v3.30/cmake-3.30.3.tar.gz
 https://doxygen.nl/files/doxygen-1.12.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.1.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-14.2.0/gcc-14.2.0.tar.xz
-https://github.com/ivmai/bdwgc/releases/download/v8.2.6/gc-8.2.6.tar.gz
+https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-15.1.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-2.46.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.46.0.tar.xz
@@ -457,8 +457,8 @@ https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
 https://www.php.net/distributions/php-8.3.11.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
-https://www.python.org/ftp/python/3.12.5/Python-3.12.5.tar.xz
-https://www.python.org/ftp/python/doc/3.12.5/python-3.12.5-docs-html.tar.bz2
+https://www.python.org/ftp/python/3.12.6/Python-3.12.6.tar.xz
+https://www.python.org/ftp/python/doc/3.12.6/python-3.12.6-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.2.0.tar.gz
@@ -700,7 +700,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-6.7.2-ffmpeg7_fixe
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.295/Vulkan-Headers-1.3.295.tar.gz
 https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.3.295/Vulkan-Loader-1.3.295.tar.gz
-https://webkitgtk.org/releases/webkitgtk-2.44.3.tar.xz
+https://webkitgtk.org/releases/webkitgtk-2.44.4.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-desktop-portal-1.18.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
 https://download.gnome.org/sources/gdm/46/gdm-46.2.tar.xz
