diff --git a/wget-list b/wget-list
index 6958e67bc..18908526b 100644
--- a/wget-list
+++ b/wget-list
@@ -886,11 +886,11 @@ https://github.com/lxqt/screengrab/releases/download/2.8.0/screengrab-2.8.0.tar.
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-3.0.5.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-docs-3.0.2.tar.gz
 https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.57.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-24.2.5.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-dictionaries-24.2.5.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-help-24.2.5.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.5/libreoffice-translations-24.2.5.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.2.5.2-boost_fixes-2.patch
+https://download.documentfoundation.org/libreoffice/src/24.8.0/libreoffice-24.8.0.3.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.0/libreoffice-dictionaries-24.8.0.3.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.0/libreoffice-help-24.8.0.3.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.0/libreoffice-translations-24.8.0.3.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.0.3-boost_fixes-1.patch
 https://download.gnome.org/sources/epiphany/46/epiphany-46.3.tar.xz
 https://download.kde.org/stable/release-service/24.08.0/src/falkon-24.08.0.tar.xz
 https://archive.mozilla.org/pub/firefox/releases/128.1.0esr/source/firefox-128.1.0esr.source.tar.xz
