diff --git a/wget-list b/wget-list
index f48d250c8..53bd99620 100644
--- a/wget-list
+++ b/wget-list
@@ -744,7 +744,7 @@ https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.4.0.tar.xz
 https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.5.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz
-https://download.gnome.org/sources/gsettings-desktop-schemas/46/gsettings-desktop-schemas-46.1.tar.xz
+https://download.gnome.org/sources/gsettings-desktop-schemas/47/gsettings-desktop-schemas-47.1.tar.xz
 https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz
 https://download.gnome.org/sources/rest/0.9/rest-0.9.1.tar.xz
 https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.6.tar.xz
