diff --git a/wget-list b/wget-list
index 53bd99620..456a8d0a7 100644
--- a/wget-list
+++ b/wget-list
@@ -636,7 +636,7 @@ https://xcb.freedesktop.org/dist/xcb-util-keysyms-0.4.1.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-renderutil-0.3.10.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-wm-0.4.2.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-cursor-0.1.5.tar.xz
-https://mesa.freedesktop.org/archive/mesa-24.2.2.tar.xz
+https://mesa.freedesktop.org/archive/mesa-24.2.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-3.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240910.tgz
