diff --git a/wget-list b/wget-list
index d295e9b5a..e86123086 100644
--- a/wget-list
+++ b/wget-list
@@ -222,7 +222,7 @@ https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
 https://github.com/google/shaderc/archive/v2024.2/shaderc-2024.2.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/12.1.1/graphviz-12.1.1.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
-http://www.andre-simon.de/zip/highlight-4.13.tar.bz2
+http://www.andre-simon.de/zip/highlight-4.14.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.30/ibus-1.5.30.tar.gz
 https://www.unicode.org/Public/zipped/15.1.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-36.tar.xz
@@ -342,7 +342,7 @@ https://www.cpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz
 https://www.cpan.org/authors/id/E/ET/ETJ/Log-Log4perl-1.57.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.14.tar.gz
 https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4234.tar.gz
-https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.46.tar.gz
+https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.47.tar.gz
 https://www.cpan.org/authors/id/J/JT/JTBRAUN/Parse-RecDescent-1.967015.tar.gz
 https://www.cpan.org/authors/id/W/WB/WBRASWELL/Parse-Yapp-1.21.tar.gz
 https://www.cpan.org/authors/id/L/LE/LEONT/PerlIO-utf8_strict-0.010.tar.gz
@@ -539,7 +539,7 @@ https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.3.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/subversion-1.14.3-upstream_fixes-1.patch
 https://downloads.sourceforge.net/swig/swig-4.2.1.tar.gz
-https://downloads.sourceforge.net/tcl/tk8.6.14-src.tar.gz
+https://downloads.sourceforge.net/tcl/tk8.6.15-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.17.tar.xz
 https://sourceware.org/pub/valgrind/valgrind-3.23.0.tar.bz2
@@ -567,7 +567,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.0/bind-9.20.0.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.2/bind-9.20.2.tar.xz
 https://download.gnome.org/sources/NetworkManager/1.48/NetworkManager-1.48.10.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.95.tar.bz2
@@ -741,7 +741,7 @@ https://download.kde.org/stable/release-service/24.08.0/src/gwenview-24.08.0.tar
 https://download.kde.org/stable/release-service/24.08.0/src/libkcddb-24.08.0.tar.xz
 https://download.kde.org/stable/release-service/24.08.0/src/k3b-24.08.0.tar.xz
 https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.4.0.tar.xz
-https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.5.0.tar.xz
+https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.6.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.3/gcr-4.3.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/47/gsettings-desktop-schemas-47.1.tar.xz
