diff --git a/wget-list b/wget-list
index 67d7633aa9..2c8dbb758c 100644
--- a/wget-list
+++ b/wget-list
@@ -631,7 +631,7 @@ https://xorg.freedesktop.org/archive/individual/proto/xcb-proto-1.17.0.tar.xz
 https://xorg.freedesktop.org/archive/individual/lib/libxcb-1.17.0.tar.xz
 https://www.x.org/pub/individual/lib/libxcvt-0.1.3.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-0.4.1.tar.xz
-https://mesa.freedesktop.org/archive/mesa-24.3.3.tar.xz
+https://mesa.freedesktop.org/archive/mesa-24.3.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-4.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240910.tgz
@@ -1038,7 +1038,7 @@ https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview-gtk3/epdfview-gtk3-20200814.tar.xz
 https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.10-src.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/fop-2.10-upstream_fixes-1.patch
-https://www.mupdf.com/downloads/archive/mupdf-1.25.2-source.tar.gz
+https://www.mupdf.com/downloads/archive/mupdf-1.25.3-source.tar.gz
 https://github.com/dov/paps/releases/download/v0.8.0/paps-0.8.0.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312-source.tar.xz
