diff --git a/wget-list b/wget-list
index 3ec9ad2f6d..79c6c48950 100644
--- a/wget-list
+++ b/wget-list
@@ -22,7 +22,7 @@ https://github.com/p11-glue/p11-kit/releases/download/0.25.5/p11-kit-0.25.5.tar.
 https://github.com/polkit-org/polkit/archive/125/polkit-125.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
-https://github.com/shadow-maint/shadow/releases/download/4.16.0/shadow-4.16.0.tar.xz
+https://github.com/shadow-maint/shadow/releases/download/4.17.0/shadow-4.17.0.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.73.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.16p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
@@ -246,7 +246,7 @@ https://www.freedesktop.org/software/colord/releases/colord-1.4.7.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/colord-1.4.7-upstream_fixes-1.patch
 https://ftp.gnu.org/gnu/cpio/cpio-2.15.tar.bz2
 https://www.freedesktop.org/software/cups-pk-helper/releases/cups-pk-helper-0.2.7.tar.xz
-https://dbus.freedesktop.org/releases/dbus/dbus-1.14.10.tar.xz
+https://dbus.freedesktop.org/releases/dbus/dbus-1.16.0.tar.xz
 http://fcron.free.fr/archives/fcron-3.2.1.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
@@ -816,7 +816,7 @@ https://download.gnome.org/sources/libwnck/43/libwnck-43.1.tar.xz
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.20/xfce4-dev-tools-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.0.tar.bz2
+https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.20/thunar-volman-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.20/tumbler-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.20/xfce4-appfinder-4.20.0.tar.bz2
