diff --git a/wget-list b/wget-list
index b0afb656b7..3ec9ad2f6d 100644
--- a/wget-list
+++ b/wget-list
@@ -76,7 +76,7 @@ https://duktape.org/duktape-2.7.0.tar.xz
 https://github.com/rrthomas/enchant/releases/download/v2.8.2/enchant-2.8.2.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.5.tar.xz
 https://www.fftw.org/fftw-3.3.10.tar.gz
-https://github.com/fmtlib/fmt/archive/11.1.0/fmt-11.1.0.tar.gz
+https://github.com/fmtlib/fmt/archive/11.1.1/fmt-11.1.1.tar.gz
 https://download.gnome.org/sources/glib/2.82/glib-2.82.4.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.82/gobject-introspection-1.82.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
@@ -288,7 +288,7 @@ https://github.com/mozilla/cbindgen/archive/v0.27.0/cbindgen-0.27.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
 https://cmake.org/files/v3.31/cmake-3.31.3.tar.gz
-https://doxygen.nl/files/doxygen-1.12.0.src.tar.gz
+https://doxygen.nl/files/doxygen-1.13.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-14.2.0/gcc-14.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
