diff --git a/wget-list b/wget-list
index 922ef80ce3..d5219b7ae6 100644
--- a/wget-list
+++ b/wget-list
@@ -45,7 +45,7 @@ https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
 https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
 https://unifoundry.com/pub/unifont/unifont-16.0.01/font-builds/unifont-16.0.01.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.16.tar.bz2
-https://ftp.gnu.org/gnu/ed/ed-1.20.2.tar.lz
+https://ftp.gnu.org/gnu/ed/ed-1.21.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-29.4.tar.xz
 https://download.gnome.org/sources/gedit/48/gedit-48.1.tar.xz
 https://gitlab.gnome.org/World/gedit/libgedit-tepl/-/archive/6.12.0/libgedit-tepl-6.12.0.tar.bz2
@@ -113,7 +113,7 @@ https://github.com/libical/libical/releases/download/v3.0.19/libical-3.0.19.tar.
 https://ftp.gnu.org/gnu/libidn/libidn-1.42.tar.gz
 https://ftp.gnu.org/gnu/libidn/libidn2-2.3.7.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
-https://github.com/cjlin1/liblinear/archive/v247/liblinear-247.tar.gz
+https://github.com/cjlin1/liblinear/archive/v248/liblinear-248.tar.gz
 https://www.freedesktop.org/software/libmbim/libmbim-1.26.4.tar.xz
 https://github.com/linux-nvme/libnvme/archive/v1.11.1/libnvme-1.11.1.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.5/libpaper-2.2.5.tar.gz
@@ -153,13 +153,13 @@ https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.0/protobuf-c-1.5
 https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.0-consolidated_fixes-1.patch
 https://download.kde.org/stable/qca/2.3.9/qca-2.3.9.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.11.0/qcoro-0.11.0.tar.gz
-https://archive.mozilla.org/pub/firefox/releases/128.5.2esr/source/firefox-128.5.2esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/128.6.0esr/source/firefox-128.6.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.3.296.0/SPIRV-Headers-1.3.296.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.3.296.0/SPIRV-Tools-1.3.296.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v19.1.3/SPIRV-LLVM-Translator-19.1.3.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.2.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
-https://github.com/martinpitt/umockdev/releases/download/0.19.0/umockdev-0.19.0.tar.xz
+https://github.com/martinpitt/umockdev/releases/download/0.19.1/umockdev-0.19.1.tar.xz
 https://github.com/nemtrif/utfcpp/archive/v4.0.6/utfcpp-4.0.6.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.23.0/downloads/wayland-1.23.0.tar.xz
 https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.39/downloads/wayland-protocols-1.39.tar.xz
@@ -201,7 +201,7 @@ https://github.com/opencv/opencv/archive/4.10.0/opencv-4.10.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.10.0/opencv_contrib-4.10.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.3/openjpeg-2.5.3.tar.gz
 https://www.cairographics.org/releases/pixman-0.44.2.tar.gz
-https://poppler.freedesktop.org/poppler-24.12.0.tar.xz
+https://poppler.freedesktop.org/poppler-25.01.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v11.9.1/qpdf-11.9.1.tar.gz
@@ -218,7 +218,7 @@ https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
 https://github.com/google/shaderc/archive/v2024.3/shaderc-2024.3.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/12.2.1/graphviz-12.2.1.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
-http://www.andre-simon.de/zip/highlight-4.14.tar.bz2
+http://www.andre-simon.de/zip/highlight-4.15.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.31/ibus-1.5.31.tar.gz
 https://www.unicode.org/Public/zipped/16.0.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-36.tar.xz
@@ -288,7 +288,7 @@ https://github.com/mozilla/cbindgen/archive/v0.27.0/cbindgen-0.27.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
 https://cmake.org/files/v3.31/cmake-3.31.3.tar.gz
-https://doxygen.nl/files/doxygen-1.13.0.src.tar.gz
+https://doxygen.nl/files/doxygen-1.13.1.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-14.2.0/gcc-14.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
@@ -312,7 +312,7 @@ https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
 https://www.cpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz
 https://www.cpan.org/authors/id/V/VP/VPIT/autovivification-0.18.tar.gz
-https://www.cpan.org/authors/id/B/BR/BRIANDFOY/Business-ISBN-3.009.tar.gz
+https://www.cpan.org/authors/id/B/BR/BRIANDFOY/Business-ISBN-3.011.tar.gz
 https://www.cpan.org/authors/id/B/BR/BRIANDFOY/Business-ISMN-1.204.tar.gz
 https://www.cpan.org/authors/id/B/BD/BDFOY/Business-ISSN-1.005.tar.gz
 https://www.cpan.org/authors/id/K/KA/KASEI/Class-Accessor-0.51.tar.gz
@@ -348,7 +348,7 @@ https://www.cpan.org/authors/id/R/RA/RAAB/SGMLSpm-1.1.tar.gz
 https://www.cpan.org/authors/id/S/SA/SALVA/Sort-Key-1.33.tar.gz
 https://www.cpan.org/authors/id/D/DA/DANBOO/Test-Command-0.11.tar.gz
 https://www.cpan.org/authors/id/D/DC/DCANTRELL/Test-Differences-0.71.tar.gz
-https://www.cpan.org/authors/id/A/AM/AMBS/Text-BibTeX-0.89.tar.gz
+https://www.cpan.org/authors/id/A/AM/AMBS/Text-BibTeX-0.90.tar.gz
 https://www.cpan.org/authors/id/I/IS/ISHIGAKI/Text-CSV-2.04.tar.gz
 https://www.cpan.org/authors/id/S/SY/SYP/Text-Roman-3.5.tar.gz
 https://www.cpan.org/authors/id/S/SA/SADAHIRO/Unicode-Collate-1.31.tar.gz
@@ -383,7 +383,7 @@ https://cpan.metacpan.org/authors/id/G/GA/GAAS/Encode-Locale-1.05.tar.gz
 https://cpan.metacpan.org/authors/id/D/DO/DOY/Eval-Closure-0.14.tar.gz
 https://cpan.metacpan.org/authors/id/D/DR/DROLSKY/Exception-Class-1.45.tar.gz
 https://cpan.metacpan.org/authors/id/T/TO/TOBYINK/Exporter-Tiny-1.006002.tar.gz
-https://cpan.metacpan.org/authors/id/A/AM/AMBS/ExtUtils-LibBuilder-0.08.tar.gz
+https://cpan.metacpan.org/authors/id/A/AM/AMBS/ExtUtils-LibBuilder-0.09.tar.gz
 https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/FFI-CheckLib-0.31.tar.gz
 https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/File-chdir-0.1011.tar.gz
 https://cpan.metacpan.org/authors/id/D/DM/DMUEY/File-Copy-Recursive-0.45.tar.gz
@@ -460,7 +460,7 @@ https://www.python.org/ftp/python/doc/3.13.1/python-3.13.1-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.2.0.tar.gz
-https://github.com/psf/cachecontrol/archive/v0.14.1/cachecontrol-0.14.1.tar.gz
+https://github.com/psf/cachecontrol/archive/v0.14.2/cachecontrol-0.14.2.tar.gz
 https://github.com/cython/cython/releases/download/3.0.11-1/cython-3.0.11.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.34.2.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
@@ -479,7 +479,7 @@ https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
 https://download.gnome.org/sources/pyatspi/2.46/pyatspi-2.46.1.tar.xz
 https://github.com/pygobject/pycairo/releases/download/v1.26.1/pycairo-1.26.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pygdbmi/pygdbmi-0.11.0.0.tar.gz
-https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.18.0.tar.gz
+https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.19.1.tar.gz
 https://download.gnome.org/sources/pygobject/3.50/pygobject-3.50.0.tar.xz
 https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
@@ -556,7 +556,7 @@ https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.7-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.3.0.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.21.2.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.21.3.tar.gz
 https://ftp.gnu.org/gnu/wget/wget-1.25.0.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.9.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
@@ -598,7 +598,7 @@ https://archive.apache.org/dist/serf/serf-1.3.10.tar.bz2
 https://tecnocode.co.uk/downloads/uhttpmock/uhttpmock-0.11.0.tar.xz
 http://links.twibright.com/download/links-2.30.tar.bz2
 https://invisible-mirror.net/archives/lynx/tarballs/lynx2.9.2.tar.bz2
-https://downloads.sourceforge.net/fetchmail/fetchmail-6.5.1.tar.xz
+https://downloads.sourceforge.net/fetchmail/fetchmail-6.5.2.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/mailx/heirloom-mailx_12.5.orig.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/heirloom-mailx-12.5-fixes-1.patch
 https://bitbucket.org/mutt/mutt/downloads/mutt-2.2.13.tar.gz
@@ -641,14 +641,14 @@ https://www.x.org/pub/individual/xserver/xorg-server-21.1.15.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.15-tearfree_backport-2.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.3.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
-https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.27.0/libinput-1.27.0.tar.gz
+https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.27.1/libinput-1.27.1.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
-https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.9.2.tar.xz
+https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.10.0.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.3/xf86-input-wacom-1.2.3.tar.bz2
 https://www.x.org/pub/individual/app/twm-1.0.12.tar.xz
 https://invisible-mirror.net/archives/xterm/xterm-397.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
-https://www.x.org/pub/individual/app/xinit-1.4.2.tar.xz
+https://www.x.org/pub/individual/app/xinit-1.4.3.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.36/atkmm-2.36.3.tar.xz
 https://download.gnome.org/sources/at-spi2-core/2.54/at-spi2-core-2.54.0.tar.xz
@@ -669,7 +669,7 @@ https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.43.tar.xz
 https://download.gnome.org/sources/gtk/4.16/gtk-4.16.7.tar.xz
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.9.tar.xz
 https://download.gnome.org/sources/gtkmm/4.16/gtkmm-4.16.0.tar.xz
-https://download.gnome.org/sources/gtk-vnc/1.3/gtk-vnc-1.3.1.tar.xz
+https://download.gnome.org/sources/gtk-vnc/1.4/gtk-vnc-1.4.0.tar.xz
 https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
 https://download.gnome.org/sources/gtksourceview/4.8/gtksourceview-4.8.4.tar.xz
 https://download.gnome.org/sources/gtksourceview/5.14/gtksourceview-5.14.2.tar.xz
@@ -720,7 +720,7 @@ https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.15.0.tar.xz
 https://download.kde.org/stable/release-service/24.12.0/src/ark-24.12.0.tar.xz
-https://download.kde.org/stable/release-service/24.12.0/src/dolphin-24.12.0.tar.xz
+https://download.kde.org/stable/release-service/24.12.0/src/dolphin-24.12.0.1.tar.xz
 https://download.kde.org/stable/release-service/24.12.0/src/dolphin-plugins-24.12.0.tar.xz
 https://download.kde.org/stable/release-service/24.12.0/src/kdenlive-24.12.0.tar.xz
 https://download.kde.org/stable/release-service/24.12.0/src/kmix-24.12.0.tar.xz
@@ -758,7 +758,7 @@ https://download.gnome.org/sources/libgee/0.20/libgee-0.20.6.tar.xz
 https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.4.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
-https://download.gnome.org/sources/libshumate/1.3/libshumate-1.3.0.tar.xz
+https://download.gnome.org/sources/libshumate/1.3/libshumate-1.3.1.tar.xz
 https://download.gnome.org/sources/evolution-data-server/3.54/evolution-data-server-3.54.3.tar.xz
 https://download.gnome.org/sources/tinysparql/3.8/tinysparql-3.8.2.tar.xz
 https://download.gnome.org/sources/localsearch/3.8/localsearch-3.8.2.tar.xz
@@ -794,7 +794,7 @@ https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-
 https://download.gnome.org/sources/gnome-connections/46/gnome-connections-46.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/46/gnome-disk-utility-46.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/45/gnome-logs-45.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/47/gnome-maps-47.2.tar.xz
+https://download.gnome.org/sources/gnome-maps/47/gnome-maps-47.3.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/43/gnome-power-manager-43.0.tar.xz
@@ -876,32 +876,29 @@ https://github.com/lxqt/screengrab/releases/download/2.9.0/screengrab-2.9.0.tar.
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-3.0.5.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-docs-3.0.2.tar.gz
 https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.57.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-24.8.3.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-dictionaries-24.8.3.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-help-24.8.3.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-translations-24.8.3.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-boost_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-icu76_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-poppler_24.10-2.patch
+https://download.documentfoundation.org/libreoffice/src/24.8.4/libreoffice-24.8.4.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.4/libreoffice-dictionaries-24.8.4.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.4/libreoffice-help-24.8.4.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.4/libreoffice-translations-24.8.4.2.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.4.2-boost_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.4.2-icu76_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.4.2-poppler_24.10-2.patch
 https://download.gnome.org/sources/epiphany/47/epiphany-47.2.tar.xz
 https://download.kde.org/stable/release-service/24.12.0/src/falkon-24.12.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-128.5.2esr-upstream_fixes-1.patch
-https://archive.seamonkey-project.org/releases/2.53.19/source/seamonkey-2.53.19.source.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.19-cxx17-1.patch
+https://archive.seamonkey-project.org/releases/2.53.20/source/seamonkey-2.53.20.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
 https://feh.finalrewind.org/feh-3.10.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20230101/fontforge-20230101.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/fontforge-20230101-security_fixes-1.patch
 https://github.com/freerdp/freerdp/archive/3.10.2/FreeRDP-3.10.2.tar.gz
-https://download.gimp.org/gimp/v3.0/gimp-3.0.0-RC1.tar.xz
+https://download.gimp.org/gimp/v3.0/gimp-3.0.0-RC2.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/gimp/gimp3-help-20240711.tar.xz
 https://downloads.sourceforge.net/gparted/gparted-1.6.0.tar.gz
 https://inkscape.org/gallery/item/53679/inkscape-1.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4-poppler_fixes-1.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/128.5.2esr/source/thunderbird-128.5.2esr.source.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-128.5.2esr-upstream_fixes-1.patch
+https://archive.mozilla.org/pub/thunderbird/releases/128.6.0esr/source/thunderbird-128.6.0esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.14.1/tigervnc-1.14.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.14.1-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
@@ -923,12 +920,12 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://downloads.xiph.org/releases/flac/flac-1.4.3.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.10.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.10.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.10.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.10.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.10.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.10.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.24.11.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.24.11.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.24.11.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.24.11.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.24.11.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.24.11.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/intel-media-24.3.4.tar.gz
@@ -946,7 +943,7 @@ https://get.videolan.org/libdvdcss/1.4.3/libdvdcss-1.4.3.tar.bz2
 https://get.videolan.org/libdvdread/6.1.3/libdvdread-6.1.3.tar.bz2
 https://get.videolan.org/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2
 https://downloads.sourceforge.net/libdv/libdv-1.0.0.tar.gz
-https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.24.10/gst-plugins-rs-gstreamer-1.24.10.tar.gz
+https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.24.11/gst-plugins-rs-gstreamer-1.24.11.tar.gz
 https://downloads.sourceforge.net/mad/libmad-0.15.1b.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libmad-0.15.1b-fixes-1.patch
 https://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
@@ -1048,7 +1045,7 @@ https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312
 https://ftp.tu-chemnitz.de/pub/tug/historic/systems/texlive/2024/texlive-20240312-extra.tar.xz
 https://downloads.sourceforge.net/asymptote/asymptote-2.95.src.tgz
 https://github.com/plk/biber/archive/v2.20/biber-2.20.tar.gz
-https://github.com/mgieseki/dvisvgm/releases/download/3.4.2/dvisvgm-3.4.2.tar.gz
+https://github.com/mgieseki/dvisvgm/releases/download/3.4.3/dvisvgm-3.4.3.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20241209.tar.xz
