diff --git a/wget-list b/wget-list
index 0311ccb752..acc1c2655e 100644
--- a/wget-list
+++ b/wget-list
@@ -646,7 +646,7 @@ https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.9.2.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.3/xf86-input-wacom-1.2.3.tar.bz2
 https://www.x.org/pub/individual/app/twm-1.0.12.tar.xz
-https://invisible-mirror.net/archives/xterm/xterm-396.tgz
+https://invisible-mirror.net/archives/xterm/xterm-397.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.2.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
@@ -759,7 +759,7 @@ https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.4.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.3/libshumate-1.3.0.tar.xz
-https://download.gnome.org/sources/evolution-data-server/3.54/evolution-data-server-3.54.2.tar.xz
+https://download.gnome.org/sources/evolution-data-server/3.54/evolution-data-server-3.54.3.tar.xz
 https://download.gnome.org/sources/tinysparql/3.8/tinysparql-3.8.1.tar.xz
 https://download.gnome.org/sources/localsearch/3.8/localsearch-3.8.1.tar.xz
 https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
@@ -787,7 +787,7 @@ https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/brasero-3.12.3-upstream_fixes-1.patch
 https://download.gnome.org/sources/eog/47/eog-47.0.tar.xz
 https://download.gnome.org/sources/evince/46/evince-46.3.1.tar.xz
-https://download.gnome.org/sources/evolution/3.54/evolution-3.54.2.tar.xz
+https://download.gnome.org/sources/evolution/3.54/evolution-3.54.3.tar.xz
 https://download.gnome.org/sources/file-roller/44/file-roller-44.4.tar.xz
 https://download.gnome.org/sources/gnome-calculator/47/gnome-calculator-47.0.tar.xz
 https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-3.36.0.tar.xz
