diff --git a/wget-list b/wget-list
index 40efbd3..bbac69a 100644
--- a/wget-list
+++ b/wget-list
@@ -63,7 +63,7 @@ http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 ftp://ftp.astron.com/pub/tcsh/tcsh-6.24.10.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
 https://www.zsh.org/pub/zsh-5.9-doc.tar.xz
-https://download.qemu.org/qemu-8.0.3.tar.xz
+https://download.qemu.org/qemu-8.1.0.tar.xz
 ftp://ftp.mirrorservice.org/sites/ftp.apache.org/apr/apr-1.7.4.tar.bz2
 ftp://ftp.mirrorservice.org/sites/ftp.apache.org/apr/apr-util-1.6.3.tar.bz2
 ftp://ftp.gnu.org/gnu/aspell/aspell-0.60.8.tar.gz
@@ -551,7 +551,7 @@ https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p17.tar.gz
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.6.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/rpcbind-1.2.6-vulnerability_fixes-1.patch
 https://www.samba.org/ftp/rsync/src/rsync-3.2.7.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.18.5.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.18.6.tar.gz
 ftp://ftp.gnu.org/gnu/wget/wget-1.21.4.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-5.19.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
@@ -712,7 +712,7 @@ https://github.com/ice-wm/icewm/archive/3.4.1/icewm-3.4.1.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://download.tuxfamily.org/sawfish/sawfish_1.13.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/adwaita-icon-theme/44/adwaita-icon-theme-44.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/breeze-icons-5.106.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/breeze-icons-5.109.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-icon-theme-symbolic/3.12/gnome-icon-theme-symbolic-3.12.0.tar.xz
@@ -720,8 +720,8 @@ ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-themes-extra/3.28/gnome-themes-extr
 https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.17.tar.xz
 http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
-https://download.kde.org/stable/frameworks/5.106/oxygen-icons5-5.106.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/extra-cmake-modules-5.106.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/oxygen-icons5-5.109.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/extra-cmake-modules-5.109.0.tar.xz
 https://download.kde.org/stable/phonon/4.11.1/phonon-4.11.1.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-gstreamer/4.10.0/phonon-backend-gstreamer-4.10.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.11.3/phonon-backend-vlc-0.11.3.tar.xz
@@ -778,7 +778,7 @@ ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-backgrounds/44/gnome-backgrounds-44
 ftp://ftp.acc.umu.se/pub/gnome/sources/gvfs/1.50/gvfs-1.50.6.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/nautilus/44/nautilus-44.2.1.tar.xz
-ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-bluetooth/42/gnome-bluetooth-42.5.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-bluetooth/42/gnome-bluetooth-42.6.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-keyring/42/gnome-keyring-42.1.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-settings-daemon/44/gnome-settings-daemon-44.1.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-control-center/44/gnome-control-center-44.3.tar.xz
@@ -805,7 +805,7 @@ ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-nettool/42/gnome-nettool-42.0.tar.x
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-power-manager/43/gnome-power-manager-43.0.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-screenshot/41/gnome-screenshot-41.0.tar.xz
-ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-system-monitor/44/gnome-system-monitor-44.0.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-system-monitor/45/gnome-system-monitor-45.0.tar.xz
 https://gitlab.gnome.org/GNOME/gnome-terminal/-/archive/3.48.2/gnome-terminal-3.48.2.tar.gz
 ftp://ftp.acc.umu.se/pub/gnome/sources/gnome-weather/44/gnome-weather-44.0.tar.xz
 https://gitlab.gnome.org/GNOME/gucharmap/-/archive/15.0.4/gucharmap-15.0.4.tar.bz2
@@ -821,14 +821,14 @@ https://archive.xfce.org/src/xfce/xfce4-panel/4.18/xfce4-panel-4.18.4.tar.bz2
 https://archive.xfce.org/src/xfce/thunar/4.18/thunar-4.18.6.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.18/thunar-volman-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.18/tumbler-4.18.1.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-appfinder/4.18/xfce4-appfinder-4.18.0.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-appfinder/4.18/xfce4-appfinder-4.18.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-power-manager/4.18/xfce4-power-manager-4.18.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-settings/4.18/xfce4-settings-4.18.3.tar.bz2
 https://archive.xfce.org/src/xfce/xfdesktop/4.18/xfdesktop-4.18.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfwm4/4.18/xfwm4-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-session/4.18/xfce4-session-4.18.3.tar.bz2
 https://archive.xfce.org/src/apps/parole/4.18/parole-4.18.0.tar.bz2
-https://archive.xfce.org/src/apps/xfce4-terminal/1.0/xfce4-terminal-1.0.4.tar.bz2
+https://archive.xfce.org/src/apps/xfce4-terminal/1.1/xfce4-terminal-1.1.0.tar.bz2
 https://archive.xfce.org/src/apps/xfburn/0.7/xfburn-0.7.0.tar.bz2
 https://archive.xfce.org/src/apps/ristretto/0.13/ristretto-0.13.1.tar.bz2
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.18/xfce4-dev-tools-4.18.0.tar.bz2
@@ -851,12 +851,12 @@ https://downloads.sourceforge.net/lxde/lxrandr-0.3.2.tar.xz
 https://downloads.sourceforge.net/lxde/lxtask-0.1.10.tar.xz
 ftp://ftp.acc.umu.se/pub/gnome/sources/vte/0.28/vte-0.28.2.tar.xz
 https://downloads.sourceforge.net/lxde/lxterminal-0.4.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/kwindowsystem-5.106.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/kwayland-5.106.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/kconfig-5.106.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/solid-5.106.0.tar.xz
-https://download.kde.org/stable/frameworks/5.106/kidletime-5.106.0.tar.xz
-http://download.kde.org/stable/plasma/5.27.5/libkscreen-5.27.5.tar.xz
+https://download.kde.org/stable/frameworks/5.109/kwindowsystem-5.109.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/kwayland-5.109.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/kconfig-5.109.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/solid-5.109.0.tar.xz
+https://download.kde.org/stable/frameworks/5.109/kidletime-5.109.0.tar.xz
+http://download.kde.org/stable/plasma/5.27.7/libkscreen-5.27.7.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.4/muparser-2.3.4.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/0.13.0/lxqt-build-tools-0.13.0.tar.xz
 https://github.com/lxqt/libqtxdg/releases/download/3.11.0/libqtxdg-3.11.0.tar.xz
@@ -1049,7 +1049,7 @@ https://releases.pagure.org/xmlto/xmlto-0.0.28.tar.bz2
 ftp://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview/epdfview-0.1.8.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/epdfview-0.1.8-fixes-2.patch
-https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.8-src.tar.gz
+https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.9-src.tar.gz
 https://www.mupdf.com/downloads/archive/mupdf-1.22.2-source.tar.gz
 https://downloads.sourceforge.net/paps/paps-0.7.1.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
