diff --git a/wget-list b/wget-list
index 59c6e19..4d38bca 100644
--- a/wget-list
+++ b/wget-list
@@ -534,7 +534,7 @@ https://downloads.sourceforge.net/swig/swig-4.1.1.tar.gz
 ftp://ftp.acc.umu.se/pub/gnome/sources/sysprof/3.48/sysprof-3.48.0.tar.xz
 https://downloads.sourceforge.net/tcl/tk8.6.13-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
-ftp://ftp.acc.umu.se/pub/gnome/sources/vala/0.56/vala-0.56.10.tar.xz
+ftp://ftp.acc.umu.se/pub/gnome/sources/vala/0.56/vala-0.56.11.tar.xz
 ftp://sourceware.org/pub/valgrind/valgrind-3.21.0.tar.bz2
 https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-20.0.2/OpenJDK-20.0.2+9-i686-bin.tar.xz
