diff --git a/wget-list b/wget-list
index e5f39fce75..7a1e801464 100644
--- a/wget-list
+++ b/wget-list
@@ -856,7 +856,7 @@ https://github.com/lxqt/lxqt-globalkeys/releases/download/2.2.0/lxqt-globalkeys-
 https://github.com/lxqt/lxqt-qtplugin/releases/download/2.2.0/lxqt-qtplugin-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-panel/releases/download/2.2.2/lxqt-panel-2.2.2.tar.xz
 https://github.com/lxqt/pcmanfm-qt/releases/download/2.2.0/pcmanfm-qt-2.2.0.tar.xz
-https://github.com/lxqt/lxqt-powermanagement/releases/download/2.2.0/lxqt-powermanagement-2.2.0.tar.xz
+https://github.com/lxqt/lxqt-powermanagement/releases/download/2.2.1/lxqt-powermanagement-2.2.1.tar.xz
 https://github.com/lxqt/lxqt-runner/releases/download/2.2.0/lxqt-runner-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-themes/releases/download/2.2.0/lxqt-themes-2.2.0.tar.xz
 https://github.com/lxqt/lxqt-admin/releases/download/2.2.0/lxqt-admin-2.2.0.tar.xz
@@ -900,7 +900,7 @@ https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
 https://inkscape.org/gallery/item/56344/inkscape-1.4.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4.2-poppler_fixes-1.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/128.11.1esr/source/thunderbird-128.11.1esr.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/140.0esr/source/thunderbird-140.0esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.15.0/tigervnc-1.15.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.15.0-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
