diff --git a/wget-list b/wget-list
index 2e37732c64..1a323e5066 100644
--- a/wget-list
+++ b/wget-list
@@ -161,8 +161,8 @@ https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.2/protobuf-c-1.5
 https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.12.0/qcoro-0.12.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/128.11.0esr/source/firefox-128.11.0esr.source.tar.xz
-https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.313.0/SPIRV-Headers-vulkan-sdk-1.4.313.0.tar.gz
-https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.313.0/SPIRV-Tools-vulkan-sdk-1.4.313.0.tar.gz
+https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.321.0/SPIRV-Headers-vulkan-sdk-1.4.321.0.tar.gz
+https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.321.0/SPIRV-Tools-vulkan-sdk-1.4.321.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v20.1.4/SPIRV-LLVM-Translator-20.1.4.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.3.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
@@ -597,7 +597,7 @@ https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.1/libslirp-v4.9.1.t
 https://download.gnome.org/sources/libsoup/3.6/libsoup-3.6.5.tar.xz
 https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.6.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/libtirpc-1.3.6-gcc15_fixes-1.patch
-https://notroj.github.io/neon/neon-0.34.2.tar.gz
+https://notroj.github.io/neon/neon-0.35.0.tar.gz
 https://github.com/nghttp2/nghttp2/releases/download/v1.66.0/nghttp2-1.66.0.tar.xz
 https://github.com/thkukuk/rpcsvc-proto/releases/download/v1.4.4/rpcsvc-proto-1.4.4.tar.xz
 https://archive.apache.org/dist/serf/serf-1.3.10.tar.bz2
@@ -695,8 +695,8 @@ https://download.gnome.org/sources/pangomm/2.56/pangomm-2.56.1.tar.xz
 https://download.qt.io/archive/qt/6.9/6.9.1/single/qt-everywhere-src-6.9.1.tar.xz
 https://download.qt.io/official_releases/qt/6.9/6.9.1/submodules/qtwebengine-everywhere-src-6.9.1.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.4.313/Vulkan-Headers-1.4.313.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.4.313/Vulkan-Loader-1.4.313.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.4.321/Vulkan-Headers-1.4.321.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.4.321/Vulkan-Loader-1.4.321.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.48.3.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.20.3/xdg-desktop-portal-1.20.3.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.3/xdg-desktop-portal-gtk-1.15.3.tar.xz
@@ -980,8 +980,8 @@ https://bitbucket.org/multicoreware/x265_git/downloads/x265_4.1.tar.gz
 https://downloads.sourceforge.net/xine/xine-lib-1.2.13.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xine-lib-1.2.13-upstream_fixes-1.patch
 https://downloads.xvid.com/downloads/xvidcore-1.3.7.tar.gz
-https://distfiles.audacious-media-player.org/audacious-4.4.2.tar.bz2
-https://distfiles.audacious-media-player.org/audacious-plugins-4.4.2.tar.bz2
+https://distfiles.audacious-media-player.org/audacious-4.5.tar.bz2
+https://distfiles.audacious-media-player.org/audacious-plugins-4.5.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
 https://download.kde.org/stable/release-service/25.04.0/src/kwave-25.04.0.tar.xz
