diff --git a/wget-list b/wget-list
index 7a1e801464..3f8fa43840 100644
--- a/wget-list
+++ b/wget-list
@@ -1,7 +1,7 @@
 https://www.linuxfromscratch.org/blfs/downloads/systemd/blfs-systemd-units-20241211.tar.xz
 https://github.com/lfs-book/make-ca/archive/v1.16.1/make-ca-1.16.1.tar.gz
 https://github.com/cracklib/cracklib/releases/download/v2.10.3/cracklib-2.10.3.tar.xz
-https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz
+https://www.kernel.org/pub/linux/utils/cryptsetup/v2.8/cryptsetup-2.8.0.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/cyrus-sasl-2.1.28-gcc15_fixes-1.patch
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.8.tar.bz2
@@ -26,14 +26,14 @@ https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
 https://github.com/shadow-maint/shadow/releases/download/4.18.0/shadow-4.18.0.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.75.tar.gz
-https://www.sudo.ws/dist/sudo-1.9.17.tar.gz
+https://www.sudo.ws/dist/sudo-1.9.17p1.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.15.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
 https://github.com/libfuse/libfuse/releases/download/fuse-3.17.2/fuse-3.17.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/jfsutils-1.1.15-gcc10_fix-1.patch
-https://sourceware.org/ftp/lvm2/LVM2.2.03.32.tgz
+https://sourceware.org/ftp/lvm2/LVM2.2.03.33.tgz
 https://www.kernel.org/pub/linux/utils/raid/mdadm/mdadm-4.3.tar.xz
 https://github.com/tuxera/ntfs-3g/archive/2022.10.3/ntfs-3g-2022.10.3.tar.gz
 https://downloads.sourceforge.net/gptfdisk/gptfdisk-1.0.10.tar.gz
@@ -105,7 +105,7 @@ https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/bdwgc/libatomic_ops/releases/download/v7.8.2/libatomic_ops-7.8.2.tar.gz
 https://github.com/storaged-project/libblockdev/releases/download/3.3.1/libblockdev-3.3.1.tar.gz
 https://github.com/storaged-project/libbytesize/releases/download/2.11/libbytesize-2.11.tar.gz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.1/libclc-20.1.1.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.7/libclc-20.1.7.src.tar.xz
 https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
 https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.2.0/downloads/libdisplay-info-0.2.0.tar.xz
@@ -126,7 +126,7 @@ https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.10/libproxy-0.5.10.tar.gz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.5.2/libqalculate-5.5.2.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.6.0/libqalculate-5.6.0.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.36.0/libqmi-1.36.0.tar.gz
 https://github.com/seccomp/libseccomp/releases/download/v2.6.0/libseccomp-2.6.0.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -162,7 +162,7 @@ https://github.com/danvratil/qcoro/archive/v0.12.0/qcoro-0.12.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/128.11.0esr/source/firefox-128.11.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.313.0/SPIRV-Headers-vulkan-sdk-1.4.313.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.313.0/SPIRV-Tools-vulkan-sdk-1.4.313.0.tar.gz
-https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v20.1.3/SPIRV-LLVM-Translator-20.1.3.tar.gz
+https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v20.1.4/SPIRV-LLVM-Translator-20.1.4.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.3.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.1/umockdev-0.19.1.tar.xz
@@ -224,14 +224,14 @@ https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.28.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
 https://github.com/google/shaderc/archive/v2025.3/shaderc-2025.3.tar.gz
-https://gitlab.com/graphviz/graphviz/-/archive/13.0.1/graphviz-13.0.1.tar.bz2
+https://gitlab.com/graphviz/graphviz/-/archive/13.1.0/graphviz-13.1.0.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
 http://www.andre-simon.de/zip/highlight-4.16.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.32/ibus-1.5.32.tar.gz
 https://www.unicode.org/Public/zipped/16.0.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-47.tar.xz
 https://salsa.debian.org/iso-codes-team/iso-codes/-/archive/v4.18.0/iso-codes-v4.18.0.tar.gz
-https://github.com/lsof-org/lsof/releases/download/4.99.4/lsof-4.99.4.tar.gz
+https://github.com/lsof-org/lsof/releases/download/4.99.5/lsof-4.99.5.tar.gz
 https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.3.1.tar.bz2
 https://ftp.gnu.org/gnu/screen/screen-5.0.1.tar.gz
 https://gitlab.freedesktop.org/xdg/shared-mime-info/-/archive/2.4/shared-mime-info-2.4.tar.gz
@@ -287,7 +287,7 @@ https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmond/releases/download/sysmond-1.0.0/sysmond-1.0.0.tar.xz
 https://github.com/lfs-book/sysmon3/releases/download/sysmon3-3.0.1/sysmon3-3.0.1.tar.xz
 https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-018.tar.xz
-https://github.com/lu-zero/cargo-c/archive/v0.10.13/cargo-c-0.10.13.tar.gz
+https://github.com/lu-zero/cargo-c/archive/v0.10.14/cargo-c-0.10.14.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.29.0/cbindgen-0.29.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
@@ -301,11 +301,11 @@ https://www.kernel.org/pub/software/scm/git/git-2.50.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.50.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.50.0.tar.xz
 https://ftp.gnu.org/gnu/guile/guile-3.0.10.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.1/llvm-20.1.1.src.tar.xz
-https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-20.1.1.src.tar.xz
-https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party-20.1.1.src.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.1/clang-20.1.1.src.tar.xz
-https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.1/compiler-rt-20.1.1.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.7/llvm-20.1.7.src.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-20.1.7.src.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party-20.1.7.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.7/clang-20.1.7.src.tar.xz
+https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.7/compiler-rt-20.1.7.src.tar.xz
 https://www.lua.org/ftp/lua-5.4.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.8-shared_library-1.patch
 https://www.lua.org/tests/lua-5.4.8-tests.tar.gz
@@ -535,7 +535,8 @@ https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.g
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.3.0.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.4/ruby-3.4.4.tar.xz
-https://static.rust-lang.org/dist/rustc-1.87.0-src.tar.xz
+https://static.rust-lang.org/dist/rustc-1.88.0-src.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/rustc-1.88.0-testsuite_fix-1.patch
 https://github.com/rust-lang/rust-bindgen/archive/v0.72.0/rust-bindgen-0.72.0.tar.gz
 https://downloads.sourceforge.net/scons/SCons-4.9.1.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
@@ -747,14 +748,14 @@ https://download.gnome.org/sources/gsettings-desktop-schemas/48/gsettings-deskto
 https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.7.tar.xz
 https://download.gnome.org/sources/rest/0.9/rest-0.9.1.tar.xz
 https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.6.tar.xz
-https://gitlab.gnome.org/GNOME/vte/-/archive/0.80.2/vte-0.80.2.tar.gz
+https://gitlab.gnome.org/GNOME/vte/-/archive/0.80.3/vte-0.80.3.tar.gz
 https://download.gnome.org/sources/yelp-xsl/42/yelp-xsl-42.4.tar.xz
 https://download.gnome.org/sources/geocode-glib/3.26/geocode-glib-3.26.4.tar.xz
 https://download.gnome.org/sources/gjs/1.84/gjs-1.84.2.tar.xz
 https://download.gnome.org/sources/gnome-autoar/0.4/gnome-autoar-0.4.5.tar.xz
 https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.3.tar.xz
 https://download.gnome.org/sources/gnome-menus/3.36/gnome-menus-3.36.0.tar.xz
-https://download.gnome.org/sources/gnome-online-accounts/3.54/gnome-online-accounts-3.54.3.tar.xz
+https://download.gnome.org/sources/gnome-online-accounts/3.54/gnome-online-accounts-3.54.4.tar.xz
 https://download.gnome.org/sources/libgee/0.20/libgee-0.20.8.tar.xz
 https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.4.tar.xz
@@ -772,15 +773,15 @@ https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/48/gnome-backgrounds-48.2.1.tar.xz
 https://download.gnome.org/sources/gvfs/1.56/gvfs-1.56.1.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.5.tar.xz
-https://download.gnome.org/sources/nautilus/48/nautilus-48.2.tar.xz
+https://download.gnome.org/sources/nautilus/48/nautilus-48.3.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/47/gnome-bluetooth-47.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-bluetooth-47.1-build_fix-1.patch
 https://download.gnome.org/sources/gnome-keyring/48/gnome-keyring-48.0.tar.xz
 https://download.gnome.org/sources/gnome-settings-daemon/48/gnome-settings-daemon-48.1.tar.xz
 https://download.gnome.org/sources/tecla/48/tecla-48.0.2.tar.xz
-https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.2.tar.xz
-https://download.gnome.org/sources/mutter/48/mutter-48.3.1.tar.xz
-https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.2.tar.xz
+https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.3.tar.xz
+https://download.gnome.org/sources/mutter/48/mutter-48.4.tar.xz
+https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.3.tar.xz
 https://download.gnome.org/sources/gnome-shell-extensions/48/gnome-shell-extensions-48.2.tar.xz
 https://download.gnome.org/sources/gnome-session/48/gnome-session-48.0.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.1.tar.xz
@@ -790,7 +791,7 @@ https://download.gnome.org/sources/baobab/48/baobab-48.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/brasero-3.12.3-upstream_fixes-1.patch
 https://download.gnome.org/sources/eog/47/eog-47.0.tar.xz
-https://download.gnome.org/sources/evince/48/evince-48.0.tar.xz
+https://download.gnome.org/sources/evince/48/evince-48.1.tar.xz
 https://download.gnome.org/sources/evolution/3.56/evolution-3.56.2.tar.xz
 https://download.gnome.org/sources/file-roller/44/file-roller-44.5.tar.xz
 https://download.gnome.org/sources/gnome-calculator/48/gnome-calculator-48.1.tar.xz
@@ -922,12 +923,12 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://github.com/xiph/flac/releases/download/1.5.0/flac-1.5.0.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.26.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.26.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.26.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.26.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.26.2.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.26.2.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.26.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.26.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.26.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.26.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.26.3.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.26.3.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/lfs-book/intel-media-driver/archive/v25.1.4/intel-media-driver-25.1.4.tar.gz
@@ -945,7 +946,7 @@ https://get.videolan.org/libdvdcss/1.4.3/libdvdcss-1.4.3.tar.bz2
 https://get.videolan.org/libdvdread/6.1.3/libdvdread-6.1.3.tar.bz2
 https://get.videolan.org/libdvdnav/6.1.1/libdvdnav-6.1.1.tar.bz2
 https://downloads.sourceforge.net/libdv/libdv-1.0.0.tar.gz
-https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.26.2/gst-plugins-rs-gstreamer-1.26.2.tar.gz
+https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.26.3/gst-plugins-rs-gstreamer-1.26.3.tar.gz
 https://downloads.sourceforge.net/mad/libmad-0.15.1b.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libmad-0.15.1b-fixes-1.patch
 https://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
@@ -971,7 +972,7 @@ https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.4.0.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
-https://taglib.org/releases/taglib-2.1.tar.gz
+https://taglib.org/releases/taglib-2.1.1.tar.gz
 https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.30.1.tar.xz
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.10/wireplumber-0.5.10.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20250212.tar.xz
