diff --git a/wget-list b/wget-list
index a99d27f80b..21fe005ce6 100644
--- a/wget-list
+++ b/wget-list
@@ -78,7 +78,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/clucene-2.3.3.4-contribs_lib-1
 https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.114.tar.gz
 https://github.com/google/double-conversion/archive/v3.3.1/double-conversion-3.3.1.tar.gz
 https://duktape.org/duktape-2.7.0.tar.xz
-https://github.com/rrthomas/enchant/releases/download/v2.8.10/enchant-2.8.10.tar.gz
+https://github.com/rrthomas/enchant/releases/download/v2.8.11/enchant-2.8.11.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.6.tar.xz
 https://github.com/fastfloat/fast_float/archive/v8.0.2/fast_float-8.0.2.tar.gz
 https://www.fftw.org/fftw-3.3.10.tar.gz
