diff --git a/wget-list b/wget-list
index 8e60bccb34..d7e60801d6 100644
--- a/wget-list
+++ b/wget-list
@@ -59,7 +59,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/25.04.0/src/kate-25.04.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.5.tar.xz
 https://www.nano-editor.org/dist/v8/nano-8.5.tar.xz
-https://github.com/vim/vim/archive/v9.1.1497/vim-9.1.1497.tar.gz
+https://github.com/vim/vim/archive/v9.1.1552/vim-9.1.1552.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
