diff --git a/wget-list b/wget-list
index 3985c1489f..3d3e8cfc68 100644
--- a/wget-list
+++ b/wget-list
@@ -71,7 +71,7 @@ https://archive.apache.org/dist/apr/apr-util-1.6.3.tar.bz2
 https://ftp.gnu.org/gnu/aspell/aspell-0.60.8.1.tar.gz
 https://ftp.gnu.org/gnu/aspell/dict/en/aspell6-en-2020.12.07-0.tar.bz2
 https://github.com/boostorg/boost/releases/download/boost-1.89.0/boost-1.89.0-b2-nodocs.tar.xz
-https://github.com/google/brotli/archive/v1.1.0/brotli-1.1.0.tar.gz
+https://github.com/google/brotli/archive/v1.2.0/brotli-1.2.0.tar.gz
 https://downloads.sourceforge.net/clucene/clucene-core-2.3.3.4.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/clucene-2.3.3.4-contribs_lib-1.patch
 https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.114.tar.gz
@@ -126,7 +126,7 @@ https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.11/libproxy-0.5.11.tar.gz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.8.0/libqalculate-5.8.0.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.8.1/libqalculate-5.8.1.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.36.0/libqmi-1.36.0.tar.gz
 https://github.com/seccomp/libseccomp/releases/download/v2.6.0/libseccomp-2.6.0.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -501,7 +501,7 @@ https://files.pythonhosted.org/packages/source/s/six/six-1.17.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.2.3.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-3.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
-https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.3.0.tar.gz
+https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.4.0.tar.gz
 https://files.pythonhosted.org/packages/source/B/Babel/babel-2.17.0.tar.gz
 https://files.pythonhosted.org/packages/source/c/certifi/certifi-2025.8.3.tar.gz
 https://files.pythonhosted.org/packages/source/c/chardet/chardet-5.2.0.tar.gz
@@ -572,7 +572,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.13/bind-9.20.13.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.15/bind-9.20.15.tar.xz
 https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.54.1/downloads/NetworkManager-1.54.1.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.98.tar.bz2
@@ -616,7 +616,7 @@ https://downloads.isc.org/isc/kea/3.0.1/kea-3.0.1.tar.xz
 https://github.com/proftpd/proftpd/archive/v1.3.9/proftpd-1.3.9.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.0.tar.gz
 https://ftp.exim.org/pub/exim/exim4/exim-4.98.2.tar.xz
-https://ghostarchive.org/postfix/postfix-release/official/postfix-3.10.4.tar.gz
+https://ghostarchive.org/postfix/postfix-release/official/postfix-3.10.5.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/sendmail-8.18.1-gcc15_fixes-1.patch
 https://git.openldap.org/openldap/openldap/-/archive/LMDB_0.9.33.tar.bz2
@@ -639,9 +639,9 @@ https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20250912.tgz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
 https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.46.tar.xz
-https://www.x.org/pub/individual/xserver/xwayland-24.1.8.tar.xz
-https://www.x.org/pub/individual/xserver/xorg-server-21.1.18.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.18-tearfree_backport-1.patch
+https://www.x.org/pub/individual/xserver/xwayland-24.1.9.tar.xz
+https://www.x.org/pub/individual/xserver/xorg-server-21.1.20.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.20-tearfree_backport-1.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.5.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
 https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.29.2/libinput-1.29.2.tar.gz
