diff --git a/wget-list b/wget-list
index b3a06dff47..81a8302f5b 100644
--- a/wget-list
+++ b/wget-list
@@ -141,8 +141,7 @@ https://lttng.org/files/urcu/userspace-rcu-0.15.3.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.29/libusb-1.0.29.tar.bz2
 https://dist.libuv.org/dist/v1.51.0/libuv-v1.51.0.tar.gz
 https://github.com/xkbcommon/libxkbcommon/archive/xkbcommon-1.12.0.tar.gz
-https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libxml2-2.15.0-upstream_fix-1.patch
+https://download.gnome.org/sources/libxml2/2.15/libxml2-2.15.1.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.24/libxmlb-0.3.24.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.43.tar.xz
