diff --git a/wget-list b/wget-list
index 195f96b83f..f153bd8cd7 100644
--- a/wget-list
+++ b/wget-list
@@ -25,7 +25,7 @@ https://github.com/polkit-org/polkit/archive/126/polkit-126.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
 https://github.com/shadow-maint/shadow/releases/download/4.18.0/shadow-4.18.0.tar.xz
-https://www.stunnel.org/downloads/archive/5.x/stunnel-5.75.tar.gz
+https://www.stunnel.org/downloads/archive/5.x/stunnel-5.76.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.17p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.17.tar.xz
@@ -199,7 +199,7 @@ https://downloads.sourceforge.net/libpng/libpng-1.6.50.tar.xz
 https://downloads.sourceforge.net/sourceforge/libpng-apng/libpng-1.6.47-apng.patch.gz
 https://github.com/fukuchi/libqrencode/archive/v4.1.1/libqrencode-4.1.1.tar.gz
 https://www.libraw.org/data/LibRaw-0.21.4.tar.gz
-https://download.gnome.org/sources/librsvg/2.61/librsvg-2.61.1.tar.xz
+https://download.gnome.org/sources/librsvg/2.61/librsvg-2.61.2.tar.xz
 https://github.com/fontforge/libspiro/releases/download/20220722/libspiro-dist-20220722.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.7.1.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.6.0.tar.gz
@@ -295,7 +295,7 @@ https://cmake.org/files/v4.1/cmake-4.1.2.tar.gz
 https://doxygen.nl/files/doxygen-1.15.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
-https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
+https://github.com/ivmai/bdwgc/releases/download/v8.2.10/gc-8.2.10.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-16.3.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-2.51.1.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.51.1.tar.xz
@@ -480,7 +480,7 @@ https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.3.10.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.3.4.tar.gz
 https://files.pythonhosted.org/packages/source/p/ply/ply-3.11.tar.gz
-https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.1.0.tar.gz
+https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.1.1.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
 https://download.gnome.org/sources/pyatspi/2.58/pyatspi-2.58.0.tar.xz
 https://github.com/pygobject/pycairo/releases/download/v1.28.0/pycairo-1.28.0.tar.gz
@@ -895,7 +895,7 @@ https://download.gimp.org/gimp/v3.0/gimp-3.0.6.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
 https://inkscape.org/gallery/item/56344/inkscape-1.4.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4.2-poppler_fixes-4.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4.2-poppler_fixes-5.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
 https://archive.mozilla.org/pub/thunderbird/releases/140.4.0esr/source/thunderbird-140.4.0esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.15.0/tigervnc-1.15.0.tar.gz
