diff --git a/wget-list b/wget-list
index 59d7aafc77..971bda8abd 100644
--- a/wget-list
+++ b/wget-list
@@ -45,7 +45,7 @@ https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.16.0.tar.xz
 https://github.com/rhboot/efivar/archive/39/efivar-39.tar.gz
 https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
 https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
-https://unifoundry.com/pub/unifont/unifont-16.0.04/font-builds/unifont-16.0.04.pcf.gz
+https://unifoundry.com/pub/unifont/unifont-17.0.01/font-builds/unifont-17.0.01.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.17.tar.bz2
 https://ftp.gnu.org/gnu/ed/ed-1.22.2.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-30.2.tar.xz
@@ -166,7 +166,7 @@ https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.328.1/SPIRV-T
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v21.1.1/SPIRV-LLVM-Translator-21.1.1.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.3.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
-https://github.com/martinpitt/umockdev/releases/download/0.19.3/umockdev-0.19.3.tar.xz
+https://github.com/martinpitt/umockdev/releases/download/0.19.4/umockdev-0.19.4.tar.xz
 https://github.com/nemtrif/utfcpp/archive/v4.0.8/utfcpp-4.0.8.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.24.0/downloads/wayland-1.24.0.tar.xz
 https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.45/downloads/wayland-protocols-1.45.tar.xz
