diff --git a/wget-list b/wget-list
index f3e3274e95..1e7d4313e4 100644
--- a/wget-list
+++ b/wget-list
@@ -86,7 +86,7 @@ https://download.gnome.org/sources/glib/2.86/glib-2.86.0.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.86/gobject-introspection-1.86.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
-https://download.gnome.org/sources/glibmm/2.84/glibmm-2.84.0.tar.xz
+https://download.gnome.org/sources/glibmm/2.86/glibmm-2.86.0.tar.xz
 https://github.com/jstedfast/gmime/releases/download/3.2.15/gmime-3.2.15.tar.xz
 https://github.com/lfs-book/gmmlib/archive/v22.8.1/gmmlib-22.8.1.tar.gz
 https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
@@ -242,7 +242,7 @@ https://github.com/htacg/tidy-html5/archive/5.8.0/tidy-html5-5.8.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tidy-html5-5.8.0-cmake4_fixes-1.patch
 https://ftp.gnu.org/gnu/time/time-1.9.tar.gz
 https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.2.1/unix-tree-2.2.1.tar.bz2
-https://github.com/lurcher/unixODBC/releases/download/2.3.12/unixODBC-2.3.12.tar.gz
+https://github.com/lurcher/unixODBC/archive/v2.3.13/unixODBC-2.3.13.tar.gz
 https://github.com/flatpak/xdg-dbus-proxy/releases/download/0.1.6/xdg-dbus-proxy-0.1.6.tar.xz
 https://gitlab.freedesktop.org/xdg/xdg-user-dirs/-/archive/v0.19/xdg-user-dirs-v0.19.tar.gz
 https://github.com/ip7z/7zip/archive/25.01/7zip-25.01.tar.gz
@@ -274,7 +274,7 @@ https://download.librdf.org/source/raptor2-2.0.16.tar.gz
 https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.8.tar.xz
-https://github.com/systemd/systemd/archive/v257.8/systemd-257.8.tar.gz
+https://github.com/systemd/systemd/archive/v258/systemd-258.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.2/udisks-2.10.2.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.1.10.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.10/upower-v1.90.10.tar.bz2
@@ -578,7 +578,7 @@ https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-a
 https://nmap.org/dist/nmap-7.98.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.4/whois-5.6.4.tar.gz
-https://www.wireshark.org/download/src/all-versions/wireshark-4.4.8.tar.xz
+https://www.wireshark.org/download/src/all-versions/wireshark-4.4.9.tar.xz
 https://github.com/c-ares/c-ares/releases/download/v1.34.5/c-ares-1.34.5.tar.gz
 https://curl.se/download/curl-8.16.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.8.0/geoclue-2.8.0.tar.bz2
@@ -668,8 +668,8 @@ https://github.com/KhronosGroup/glslang/archive/16.0.0/glslang-16.0.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.59.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
-https://download.gnome.org/sources/gtk/3.24/gtk-3.24.50.tar.xz
-https://download.gnome.org/sources/gtk/4.20/gtk-4.20.1.tar.xz
+https://download.gnome.org/sources/gtk/3.24/gtk-3.24.51.tar.xz
+https://download.gnome.org/sources/gtk/4.20/gtk-4.20.2.tar.xz
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.10.tar.xz
 https://download.gnome.org/sources/gtkmm/4.20/gtkmm-4.20.0.tar.xz
 https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
