diff --git a/wget-list b/wget-list
index c9371d49fa..3bd225c0e8 100644
--- a/wget-list
+++ b/wget-list
@@ -454,7 +454,7 @@ https://cpan.metacpan.org/authors/id/E/ET/ETHER/Try-Tiny-0.32.tar.gz
 https://cpan.metacpan.org/authors/id/V/VP/VPIT/Variable-Magic-0.64.tar.gz
 https://cpan.metacpan.org/authors/id/G/GA/GAAS/WWW-RobotRules-6.02.tar.gz
 https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0210.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/XML-LibXML-2.0210-testsuite_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/XML-LibXML-2.0210-testsuite_fixes-2.patch
 https://cpan.metacpan.org/authors/id/P/PE/PERIGRIN/XML-NamespaceSupport-1.12.tar.gz
 https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-1.02.tar.gz
 https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
