diff --git a/wget-list b/wget-list
index 3bd225c0e8..195f96b83f 100644
--- a/wget-list
+++ b/wget-list
@@ -62,7 +62,7 @@ https://github.com/vim/vim/archive/v9.1.1806/vim-9.1.1806.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
-https://download.qemu.org/qemu-10.1.1.tar.xz
+https://download.qemu.org/qemu-10.1.2.tar.xz
 https://github.com/abseil/abseil-cpp/releases/download/20250814.1/abseil-cpp-20250814.1.tar.gz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.1.1.tar.xz
 http://people.freedesktop.org/~hughsient/appstream-glib/releases/appstream-glib-0.8.3.tar.xz
@@ -82,7 +82,7 @@ https://libopenraw.freedesktop.org/download/exempi-2.6.6.tar.xz
 https://github.com/fastfloat/fast_float/archive/v8.1.0/fast_float-8.1.0.tar.gz
 https://www.fftw.org/fftw-3.3.10.tar.gz
 https://github.com/fmtlib/fmt/archive/12.0.0/fmt-12.0.0.tar.gz
-https://download.gnome.org/sources/glib/2.86/glib-2.86.0.tar.xz
+https://download.gnome.org/sources/glib/2.86/glib-2.86.1.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.86/gobject-introspection-1.86.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
@@ -224,7 +224,7 @@ https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.28.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.3.tar.gz
 https://github.com/google/shaderc/archive/v2025.4/shaderc-2025.4.tar.gz
-https://gitlab.com/graphviz/graphviz/-/archive/14.0.1/graphviz-14.0.1.tar.bz2
+https://gitlab.com/graphviz/graphviz/-/archive/14.0.2/graphviz-14.0.2.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.35/gtk-doc-1.35.1.tar.xz
 http://www.andre-simon.de/zip/highlight-4.17.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.32/ibus-1.5.32.tar.gz
@@ -287,12 +287,12 @@ https://mj.ucw.cz/download/linux/pci/pciutils-3.14.0.tar.gz
 https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmond/releases/download/sysmond-1.0.0/sysmond-1.0.0.tar.xz
 https://github.com/lfs-book/sysmon3/releases/download/sysmon3-3.0.1/sysmon3-3.0.1.tar.xz
-https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-018.tar.xz
+https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-019.tar.xz
 https://github.com/lu-zero/cargo-c/archive/v0.10.16/cargo-c-0.10.16.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.29.0/cbindgen-0.29.0.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/clisp/clisp-2.49.95.tar.xz
 https://cmake.org/files/v4.1/cmake-4.1.2.tar.gz
-https://doxygen.nl/files/doxygen-1.14.0.src.tar.gz
+https://doxygen.nl/files/doxygen-1.15.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
@@ -338,7 +338,7 @@ https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.095.tar.gz
 https://www.cpan.org/authors/id/G/GA/GAAS/IO-String-1.08.tar.gz
 https://www.cpan.org/authors/id/R/RJ/RJBS/IPC-Run3-0.049.tar.gz
 https://www.cpan.org/authors/id/A/AL/ALINKE/Lingua-Translit-0.29.tar.gz
-https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.80.tar.gz
+https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.81.tar.gz
 https://www.cpan.org/authors/id/D/DR/DROLSKY/List-AllUtils-0.19.tar.gz
 https://www.cpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz
 https://www.cpan.org/authors/id/E/ET/ETJ/Log-Log4perl-1.57.tar.gz
@@ -543,7 +543,7 @@ https://github.com/rust-lang/rust-bindgen/archive/refs/tags/v0.72.1/rust-bindgen
 https://downloads.sourceforge.net/scons/SCons-4.10.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.5.tar.bz2
-https://downloads.sourceforge.net/swig/swig-4.3.1.tar.gz
+https://downloads.sourceforge.net/swig/swig-4.4.0.tar.gz
 https://downloads.sourceforge.net/tcl/tk8.6.17-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.18.tar.xz
@@ -642,9 +642,9 @@ https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.46.tar
 https://www.x.org/pub/individual/xserver/xwayland-24.1.8.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.18.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.18-tearfree_backport-1.patch
-https://www.freedesktop.org/software/libevdev/libevdev-1.13.4.tar.xz
+https://www.freedesktop.org/software/libevdev/libevdev-1.13.5.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
-https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.29.1/libinput-1.29.1.tar.gz
+https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.29.2/libinput-1.29.2.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.10.0.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.3/xf86-input-wacom-1.2.3.tar.bz2
