diff --git a/wget-list b/wget-list
index e2c0a4232b..ecf842c094 100644
--- a/wget-list
+++ b/wget-list
@@ -665,7 +665,7 @@ https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.10.0.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.4/xf86-input-wacom-1.2.4.tar.bz2
 https://www.x.org/pub/individual/app/twm-1.0.13.1.tar.xz
-https://invisible-mirror.net/archives/xterm/xterm-405.tgz
+https://invisible-mirror.net/archives/xterm/xterm-408.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
@@ -679,7 +679,7 @@ https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-source.t
 https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-docs-html.tar.gz
 https://downloads.sourceforge.net/freeglut/freeglut-3.8.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.4.tar.xz
-https://downloads.sourceforge.net/glew/glew-2.2.0.tgz
+https://downloads.sourceforge.net/glew/glew-2.3.0.tgz
 https://github.com/KhronosGroup/glslang/archive/16.1.0/glslang-16.1.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.59.tar.xz
@@ -871,7 +871,7 @@ https://github.com/lxde/menu-cache/archive/1.1.1/menu-cache-1.1.1.tar.gz
 https://github.com/lxqt/libfm-qt/releases/download/2.3.1/libfm-qt-2.3.1.tar.xz
 https://github.com/lxqt/lxqt-globalkeys/releases/download/2.3.0/lxqt-globalkeys-2.3.0.tar.xz
 https://github.com/lxqt/lxqt-qtplugin/releases/download/2.3.0/lxqt-qtplugin-2.3.0.tar.xz
-https://github.com/lxqt/lxqt-panel/releases/download/2.3.1/lxqt-panel-2.3.1.tar.xz
+https://github.com/lxqt/lxqt-panel/releases/download/2.3.2/lxqt-panel-2.3.2.tar.xz
 https://github.com/lxqt/pcmanfm-qt/releases/download/2.3.0/pcmanfm-qt-2.3.0.tar.xz
 https://github.com/lxqt/lxqt-powermanagement/releases/download/2.3.0/lxqt-powermanagement-2.3.0.tar.xz
 https://github.com/lxqt/lxqt-runner/releases/download/2.3.0/lxqt-runner-2.3.0.tar.xz
@@ -883,7 +883,7 @@ https://github.com/lxqt/lxqt-policykit/releases/download/2.3.0/lxqt-policykit-2.
 https://github.com/lxqt/lxqt-sudo/releases/download/2.3.0/lxqt-sudo-2.3.0.tar.xz
 https://github.com/lxqt/lxqt-openssh-askpass/releases/download/2.3.0/lxqt-openssh-askpass-2.3.0.tar.xz
 https://github.com/lxqt/lxqt-session/releases/download/2.3.0/lxqt-session-2.3.0.tar.xz
-https://github.com/lxqt/lxqt-wayland-session/releases/download/0.3.0/lxqt-wayland-session-0.3.0.tar.xz
+https://github.com/lxqt/lxqt-wayland-session/releases/download/0.3.1/lxqt-wayland-session-0.3.1.tar.xz
 https://github.com/lxqt/xdg-desktop-portal-lxqt/releases/download/1.3.0/xdg-desktop-portal-lxqt-1.3.0.tar.xz
 https://github.com/lxqt/obconf-qt/releases/download/0.16.5/obconf-qt-0.16.5.tar.xz
 https://github.com/lxqt/lximage-qt/releases/download/2.3.0/lximage-qt-2.3.0.tar.xz
@@ -1011,7 +1011,7 @@ https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.g
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.3.tar.gz
 https://ffmpeg.org/releases/ffmpeg-8.0.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-8.0.1-chromium_method-1.patch
-https://github.com/mpv-player/mpv/archive/v0.40.0/mpv-0.40.0.tar.gz
+https://github.com/mpv-player/mpv/archive/v0.41.0/mpv-0.41.0.tar.gz
 https://download.videolan.org/vlc/3.0.22/vlc-3.0.22.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.22-ffmpeg8-1.patch
 https://downloads.sourceforge.net/xine/xine-ui-0.99.14.tar.xz
