diff --git a/wget-list b/wget-list
index ecf842c094..b2baec86fb 100644
--- a/wget-list
+++ b/wget-list
@@ -665,7 +665,7 @@ https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.10.0.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.4/xf86-input-wacom-1.2.4.tar.bz2
 https://www.x.org/pub/individual/app/twm-1.0.13.1.tar.xz
-https://invisible-mirror.net/archives/xterm/xterm-408.tgz
+https://invisible-mirror.net/archives/xterm/xterm-406.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
