diff --git a/wget-list b/wget-list
index 86a0b7ab71..7d71dfda35 100644
--- a/wget-list
+++ b/wget-list
@@ -169,7 +169,7 @@ https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v21.1.3/SPIRV-LLVM
 https://www.samba.org/ftp/talloc/talloc-2.4.3.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.4/umockdev-0.19.4.tar.xz
-https://github.com/nemtrif/utfcpp/archive/v4.0.8/utfcpp-4.0.8.tar.gz
+https://github.com/nemtrif/utfcpp/archive/v4.09/utfcpp-4.09.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.24.0/downloads/wayland-1.24.0.tar.xz
 https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.47/downloads/wayland-protocols-1.47.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
