diff --git a/wget-list b/wget-list
index 1603966d7f..10ac9b51c5 100644
--- a/wget-list
+++ b/wget-list
@@ -279,7 +279,7 @@ https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/archive/0.30/power
 https://download.librdf.org/source/raptor2-2.0.16.tar.gz
 https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
-https://sysstat.github.io/sysstat-packages/sysstat-12.7.8.tar.xz
+https://sysstat.github.io/sysstat-packages/sysstat-12.7.9.tar.xz
 https://github.com/systemd/systemd/archive/v258.1/systemd-258.1.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.11.0/udisks-2.11.0.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.2.2.tar.gz
@@ -557,7 +557,7 @@ https://downloads.sourceforge.net/scons/SCons-4.10.1.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.5.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/subversion-1.14.5-upstream_fixes-1.patch
-https://downloads.sourceforge.net/swig/swig-4.4.0.tar.gz
+https://downloads.sourceforge.net/swig/swig-4.4.1.tar.gz
 https://downloads.sourceforge.net/tcl/tk8.6.17-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.18.tar.xz
@@ -594,7 +594,7 @@ https://nmap.org/dist/nmap-7.98.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.5/whois-5.6.5.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.6.2.tar.xz
-https://github.com/c-ares/c-ares/releases/download/v1.34.5/c-ares-1.34.5.tar.gz
+https://github.com/c-ares/c-ares/releases/download/v1.34.6/c-ares-1.34.6.tar.gz
 https://curl.se/download/curl-8.17.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.8.0/geoclue-2.8.0.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.80/glib-networking-2.80.1.tar.xz
@@ -699,7 +699,7 @@ https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3
 https://download.gnome.org/sources/libadwaita/1.8/libadwaita-1.8.2.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.5.0/libei-1.5.0.tar.bz2
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
-https://dri.freedesktop.org/libdrm/libdrm-2.4.129.tar.xz
+https://dri.freedesktop.org/libdrm/libdrm-2.4.130.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
 https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.7.tar.xz
 https://people.freedesktop.org/~svu/libxklavier-5.4.tar.bz2
