diff --git a/wget-list b/wget-list
index 03a9cf8801..0d8b904082 100644
--- a/wget-list
+++ b/wget-list
@@ -99,7 +99,7 @@ https://s3.amazonaws.com/json-c_releases/releases/json-c-0.18.tar.gz
 https://download.gnome.org/sources/json-glib/1.10/json-glib-1.10.8.tar.xz
 https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git/snapshot/keyutils-1.6.3.tar.gz
 https://pagure.io/libaio/archive/libaio-0.3.113/libaio-0.3.113.tar.gz
-https://github.com/libarchive/libarchive/releases/download/v3.8.3/libarchive-3.8.3.tar.xz
+https://github.com/libarchive/libarchive/releases/download/v3.8.4/libarchive-3.8.4.tar.xz
 https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-3.0.2.tar.bz2
 https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/bdwgc/libatomic_ops/releases/download/v7.10.0/libatomic_ops-7.10.0.tar.gz
@@ -164,7 +164,7 @@ https://archive.mozilla.org/pub/firefox/releases/140.5.0esr/source/firefox-140.5
 https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.5.0-python_3.14_fixes-1.patch
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.328.1/SPIRV-Headers-vulkan-sdk-1.4.328.1.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.328.1/SPIRV-Tools-vulkan-sdk-1.4.328.1.tar.gz
-https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v21.1.2/SPIRV-LLVM-Translator-21.1.2.tar.gz
+https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v21.1.3/SPIRV-LLVM-Translator-21.1.3.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.3.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.4/umockdev-0.19.4.tar.xz
@@ -215,7 +215,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/opencv-4.12.0-ffmpeg8-1.patch
 https://github.com/opencv/opencv_contrib/archive/4.12.0/opencv_contrib-4.12.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.4/openjpeg-2.5.4.tar.gz
 https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
-https://poppler.freedesktop.org/poppler-25.11.0.tar.xz
+https://poppler.freedesktop.org/poppler-25.12.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v12.2.0/qpdf-12.2.0.tar.gz
@@ -284,7 +284,7 @@ https://github.com/storaged-project/udisks/releases/download/udisks-2.11.0/udisk
 https://www.rarlab.com/rar/unrarsrc-7.2.2.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.91.0/upower-v1.91.0.tar.bz2
 https://ftp.gnu.org/gnu/which/which-2.23.tar.gz
-https://github.com/jordansissel/xdotool/releases/download/v3.20251130.1/xdotool-3.20251130.1.tar.gz
+https://github.com/jordansissel/xdotool/releases/download/v4.20251130.1/xdotool-4.20251130.1.tar.gz
 https://downloads.sourceforge.net/infozip/zip30.tar.gz
 https://downloads.sourceforge.net/hdparm/hdparm-9.65.tar.gz
 https://github.com/hramrach/lm-sensors/archive/V3-6-2/lm-sensors-3-6-2.tar.gz
@@ -306,8 +306,7 @@ https://ftp.gnu.org/gnu/gdb/gdb-16.3.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-2.52.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.52.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.52.0.tar.xz
-https://ftp.gnu.org/gnu/guile/guile-3.0.10.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/guile-3.0.10-32bit_fixes-1.patch
+https://ftp.gnu.org/gnu/guile/guile-3.0.11.tar.xz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-21.1.2/llvm-21.1.2.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-21.1.2.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-third-party-21.1.2.src.tar.xz
@@ -657,7 +656,7 @@ https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.46.tar
 https://www.x.org/pub/individual/xserver/xwayland-24.1.9.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.21.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.21-tearfree_backport-1.patch
-https://www.freedesktop.org/software/libevdev/libevdev-1.13.5.tar.xz
+https://www.freedesktop.org/software/libevdev/libevdev-1.13.6.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
 https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.30.0/libinput-1.30.0.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
