diff --git a/wget-list b/wget-list
index d1168d086b..3a50b99c2e 100644
--- a/wget-list
+++ b/wget-list
@@ -614,7 +614,7 @@ https://ghostarchive.org/postfix/postfix-release/official/postfix-3.9.0.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-11.4.4/source/mariadb-11.4.4.tar.gz
-https://ftp.postgresql.org/pub/source/v17.0/postgresql-17.0.tar.bz2
+https://ftp.postgresql.org/pub/source/v17.1/postgresql-17.1.tar.bz2
 https://sqlite.org/2024/sqlite-autoconf-3470000.tar.gz
 https://sqlite.org/2024/sqlite-doc-3470000.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.8.tgz
@@ -660,7 +660,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/cairo-1.18.2-upstream_fixes-1.
 https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
 https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
-https://fltk.org/pub/fltk/1.3.9/fltk-1.3.9-source.tar.gz
+https://fltk.org/pub/fltk/1.3.10/fltk-1.3.10-source.tar.gz
 https://downloads.sourceforge.net/freeglut/freeglut-3.6.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.12.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.2.0.tgz
