diff --git a/wget-list b/wget-list
index 3a50b99c2e..d060212d80 100644
--- a/wget-list
+++ b/wget-list
@@ -875,12 +875,12 @@ https://github.com/lxqt/screengrab/releases/download/2.9.0/screengrab-2.9.0.tar.
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-3.0.5.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-docs-3.0.2.tar.gz
 https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.57.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.2/libreoffice-24.8.2.1.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.2/libreoffice-dictionaries-24.8.2.1.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.2/libreoffice-help-24.8.2.1.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.8.2/libreoffice-translations-24.8.2.1.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.2.1-boost_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.2.1-icu76_fixes-1.patch
+https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-24.8.3.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-dictionaries-24.8.3.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-help-24.8.3.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-translations-24.8.3.2.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-boost_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-icu76_fixes-1.patch
 https://download.gnome.org/sources/epiphany/47/epiphany-47.2.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/falkon-24.08.3.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.19/source/seamonkey-2.53.19.source.tar.xz
