diff --git a/wget-list b/wget-list
index b35c50b28b..3689bc256e 100644
--- a/wget-list
+++ b/wget-list
@@ -7,7 +7,7 @@ https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.6.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.8.tar.xz
 https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.0.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.11.tar.xz
-https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.71.tar.xz
+https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.72.tar.xz
 https://github.com/linux-pam/linux-pam/releases/download/v1.7.0/Linux-PAM-1.7.0.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/Linux-PAM/Linux-PAM-1.7.0-docs.tar.xz
 https://downloads.sourceforge.net/liboauth/liboauth-1.0.3.tar.gz
