diff --git a/wget-list b/wget-list
index 159d454be7..edb7d24d52 100644
--- a/wget-list
+++ b/wget-list
@@ -3,7 +3,7 @@ https://github.com/lfs-book/make-ca/archive/v1.14/make-ca-1.14.tar.gz
 https://github.com/cracklib/cracklib/releases/download/v2.10.2/cracklib-2.10.2.tar.xz
 https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
-https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.5.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.6.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.7.1.tar.xz
 https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.23.2.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.10.tar.xz
@@ -192,7 +192,7 @@ https://github.com/mypaint/libmypaint/releases/download/v1.6.1/libmypaint-1.6.1.
 https://downloads.sourceforge.net/libpng/libpng-1.6.44.tar.xz
 https://downloads.sourceforge.net/sourceforge/libpng-apng/libpng-1.6.43-apng.patch.gz
 https://www.libraw.org/data/LibRaw-0.21.3.tar.gz
-https://download.gnome.org/sources/librsvg/2.59/librsvg-2.59.1.tar.xz
+https://download.gnome.org/sources/librsvg/2.59/librsvg-2.59.2.tar.xz
 https://github.com/fontforge/libspiro/releases/download/20220722/libspiro-dist-20220722.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.7.0.tar.gz
 https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.4.0.tar.gz
@@ -242,7 +242,7 @@ https://downloads.sourceforge.net/acpid2/acpid-2.0.34.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/at/at_3.2.5.orig.tar.gz
 https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.9.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/bluez-5.78.tar.xz
-https://github.com/containers/bubblewrap/releases/download/v0.10.0/bubblewrap-0.10.0.tar.xz
+https://github.com/containers/bubblewrap/releases/download/v0.11.0/bubblewrap-0.11.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-1.4.7.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/colord-1.4.7-upstream_fixes-1.patch
 https://ftp.gnu.org/gnu/cpio/cpio-2.15.tar.bz2
@@ -579,8 +579,8 @@ https://www.nlnetlabs.nl/downloads/ldns/ldns-1.8.4.tar.gz
 https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
 https://netfilter.org/projects/libmnl/files/libmnl-1.0.5.tar.bz2
 http://libndp.org/files/libndp-1.9.tar.gz
-https://github.com/thom311/libnl/releases/download/libnl3_10_0/libnl-3.10.0.tar.gz
-https://github.com/thom311/libnl/releases/download/libnl3_10_0/libnl-doc-3.10.0.tar.gz
+https://github.com/thom311/libnl/releases/download/libnl3_11_0/libnl-3.11.0.tar.gz
+https://github.com/thom311/libnl/releases/download/libnl3_11_0/libnl-doc-3.11.0.tar.gz
 https://download.gnome.org/sources/libnma/1.10/libnma-1.10.6.tar.xz
 https://github.com/thkukuk/libnsl/releases/download/v2.0.1/libnsl-2.0.1.tar.xz
 https://www.tcpdump.org/release/libpcap-1.10.5.tar.gz
@@ -596,7 +596,7 @@ https://archive.apache.org/dist/serf/serf-1.3.10.tar.bz2
 https://tecnocode.co.uk/downloads/uhttpmock/uhttpmock-0.11.0.tar.xz
 http://links.twibright.com/download/links-2.30.tar.bz2
 https://invisible-mirror.net/archives/lynx/tarballs/lynx2.9.2.tar.bz2
-https://downloads.sourceforge.net/fetchmail/fetchmail-6.4.39.tar.xz
+https://downloads.sourceforge.net/fetchmail/fetchmail-6.5.0.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/mailx/heirloom-mailx_12.5.orig.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/heirloom-mailx-12.5-fixes-1.patch
 https://bitbucket.org/mutt/mutt/downloads/mutt-2.2.13.tar.gz
@@ -633,7 +633,7 @@ https://xcb.freedesktop.org/dist/xcb-util-keysyms-0.4.1.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-renderutil-0.3.10.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-wm-0.4.2.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-cursor-0.1.5.tar.xz
-https://mesa.freedesktop.org/archive/mesa-24.2.5.tar.xz
+https://mesa.freedesktop.org/archive/mesa-24.2.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-3.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://invisible-mirror.net/archives/luit/luit-20240910.tgz
@@ -958,7 +958,7 @@ https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.14.1/libvpx-1.14.1.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.28.0/mlt-7.28.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.5.2.tar.gz
-https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2./pipewire-1.2..tar.bz2
+https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.2.6/pipewire-1.2.6.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.0.tar.xz
 https://github.com/libsdl-org/sdl12-compat/archive/release-1.2.68/sdl12-compat-release-1.2.68.tar.gz
@@ -1026,6 +1026,8 @@ https://docbook.org/xml/5.1/docbook-v5.1-os.zip
 https://github.com/docbook/xslt10-stylesheets/releases/download/release/1.79.2/docbook-xsl-nons-1.79.2.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/docbook-xsl-nons-1.79.2-stack_fix-1.patch
 https://github.com/docbook/xslt10-stylesheets/releases/download/release/1.79.2/docbook-xsl-doc-1.79.2.tar.bz2
+https://github.com/docbook/xslt10-stylesheets/releases/download/release/1.79.2/docbook-xsl-1.79.2.tar.bz2
+https://www.linuxfromscratch.org/patches/blfs/svn/docbook-xsl-1.79.2-stack_fix-1.patch
 https://files.itstool.org/itstool/itstool-2.0.7.tar.bz2
 https://pagure.io/xmlto/archive/0.0.29/xmlto-0.0.29.tar.gz
 https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
