diff --git a/wget-list b/wget-list
index d060212d80..ecc81de59e 100644
--- a/wget-list
+++ b/wget-list
@@ -881,6 +881,7 @@ https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-help-
 https://download.documentfoundation.org/libreoffice/src/24.8.3/libreoffice-translations-24.8.3.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-boost_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-icu76_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.8.3.2-poppler_24.10-1.patch
 https://download.gnome.org/sources/epiphany/47/epiphany-47.2.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/falkon-24.08.3.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.19/source/seamonkey-2.53.19.source.tar.xz
@@ -1033,7 +1034,7 @@ https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview-gtk3/epdfview-gtk3-20200814.tar.xz
 https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.10-src.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/fop-2.10-upstream_fixes-1.patch
-https://www.mupdf.com/downloads/archive/mupdf-1.24.10-source.tar.gz
+https://www.mupdf.com/downloads/archive/mupdf-1.24.11-source.tar.gz
 https://github.com/dov/paps/releases/download/v0.8.0/paps-0.8.0.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312-source.tar.xz
