diff --git a/wget-list b/wget-list
index ecc81de59e..4b25d28d7a 100644
--- a/wget-list
+++ b/wget-list
@@ -644,7 +644,7 @@ https://www.x.org/pub/individual/xserver/xorg-server-21.1.14.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.14-tearfree_backport-2.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.3.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
-https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.26.2/libinput-1.26.2.tar.gz
+https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.27.0/libinput-1.27.0.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.9.2.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.3/xf86-input-wacom-1.2.3.tar.bz2
@@ -726,6 +726,7 @@ https://download.kde.org/stable/release-service/24.08.3/src/ark-24.08.3.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/dolphin-24.08.3.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/dolphin-plugins-24.08.3.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/kdenlive-24.08.3.tar.xz
+https://download.kde.org/stable/release-service/24.08.3/src/kio-extras-24.08.3.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/khelpcenter-24.08.3.tar.xz
 https://download.kde.org/stable/release-service/24.08.3/src/konsole-24.08.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.08.3-scrollbar-1.patch
@@ -897,7 +898,7 @@ https://anduin.linuxfromscratch.org/BLFS/gimp/gimp3-help-20240711.tar.xz
 https://downloads.sourceforge.net/gparted/gparted-1.6.0.tar.gz
 https://inkscape.org/gallery/item/53679/inkscape-1.4.tar.xz
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/128.4.3esr/source/thunderbird-128.4.3esr.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/128.4.4esr/source/thunderbird-128.4.4esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.14.1/tigervnc-1.14.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.14.1-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
