diff --git a/wget-list b/wget-list
index 3689bc256e..d1168d086b 100644
--- a/wget-list
+++ b/wget-list
@@ -54,7 +54,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/24.08.3/src/kate-24.08.3.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.3.tar.bz2
 https://www.nano-editor.org/dist/v8/nano-8.2.tar.xz
-https://github.com/vim/vim/archive/v9.1.0660/vim-9.1.0660.tar.gz
+https://github.com/vim/vim/archive/v9.1.0866/vim-9.1.0866.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.13.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
@@ -116,7 +116,7 @@ https://ftp.gnu.org/gnu/libidn/libidn2-2.3.7.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v247/liblinear-247.tar.gz
 https://www.freedesktop.org/software/libmbim/libmbim-1.26.4.tar.xz
-https://github.com/linux-nvme/libnvme/archive/v1.11/libnvme-1.11.tar.gz
+https://github.com/linux-nvme/libnvme/archive/v1.11.1/libnvme-1.11.1.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.5/libpaper-2.2.5.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.8.1/libportal-0.8.1.tar.xz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
