diff --git a/wget-list b/wget-list
index 838e7f3ffb..cc3b3dddf6 100644
--- a/wget-list
+++ b/wget-list
@@ -976,7 +976,7 @@ https://github.com/mltframework/mlt/releases/download/v7.36.1/mlt-7.36.1.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.6.1.tar.gz
 https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.4.10/pipewire-1.4.10.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
-https://www.kernel.org/pub/linux/bluetooth/sbc-2.1.tar.xz
+https://www.kernel.org/pub/linux/bluetooth/sbc-2.2.tar.xz
 https://www.libsdl.org/release/sdl2-compat-2.32.64.tar.gz
 https://www.libsdl.org/release/SDL3-3.4.0.tar.gz
 https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
