diff --git a/wget-list b/wget-list
index 3136c947ac..b03c1b6466 100644
--- a/wget-list
+++ b/wget-list
@@ -190,6 +190,7 @@ https://github.com/Dav1dde/glad/archive/v2.0.8/glad-2.0.8.tar.gz
 https://github.com/g-truc/glm/archive/1.0.3/glm-1.0.3.tar.gz
 https://download.gnome.org/sources/glycin/2.0/glycin-2.0.7.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.0.7-xorg_prefix-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.0.7-xbm_xpm-1.patch
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/12.3.2/harfbuzz-12.3.2.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.2.8/jasper-version-4.2.8.tar.gz
@@ -230,7 +231,7 @@ https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.28.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.4.tar.gz
 https://github.com/google/shaderc/archive/v2026.1/shaderc-2026.1.tar.gz
-https://gitlab.com/graphviz/graphviz/-/archive/14.1.1/graphviz-14.1.1.tar.bz2
+https://gitlab.com/graphviz/graphviz/-/archive/14.1.2/graphviz-14.1.2.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.35/gtk-doc-1.35.1.tar.xz
 http://www.andre-simon.de/zip/highlight-4.18.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.33/ibus-1.5.33.tar.gz
@@ -247,7 +248,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/sharutils-4.15.2-consolidated-
 https://github.com/htacg/tidy-html5/archive/5.8.0/tidy-html5-5.8.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tidy-html5-5.8.0-cmake4_fixes-1.patch
 https://ftpmirror.gnu.org/time/time-1.9.tar.gz
-https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.3.0/unix-tree-2.3.0.tar.bz2
+https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.3.1/unix-tree-2.3.1.tar.bz2
 https://github.com/lurcher/unixODBC/archive/v2.3.14/unixODBC-2.3.14.tar.gz
 https://github.com/flatpak/xdg-dbus-proxy/releases/download/0.1.6/xdg-dbus-proxy-0.1.6.tar.xz
 https://gitlab.freedesktop.org/xdg/xdg-user-dirs/-/archive/v0.19/xdg-user-dirs-v0.19.tar.gz
@@ -486,7 +487,7 @@ https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/html5lib-1.1-python_3.14_buildfix-1.patch
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.3.10.tar.gz
-https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.4.1.tar.gz
+https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.4.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/ply/ply-3.11.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.2.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
@@ -511,7 +512,7 @@ https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-9.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-3.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.4.0.tar.gz
-https://files.pythonhosted.org/packages/source/B/Babel/babel-2.17.0.tar.gz
+https://files.pythonhosted.org/packages/source/B/Babel/babel-2.18.0.tar.gz
 https://files.pythonhosted.org/packages/source/c/certifi/certifi-2025.11.12.tar.gz
 https://files.pythonhosted.org/packages/source/c/chardet/chardet-5.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/c/charset-normalizer/charset_normalizer-3.4.4.tar.gz
@@ -551,7 +552,7 @@ https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.6.0.tar.gz
 https://files.pythonhosted.org/packages/source/u/uv_build/uv_build-0.9.10.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/4.0/ruby-4.0.1.tar.xz
-https://static.rust-lang.org/dist/rustc-1.92.0-src.tar.xz
+https://static.rust-lang.org/dist/rustc-1.93.0-src.tar.xz
 https://github.com/rust-lang/rust-bindgen/archive/refs/tags/v0.72.1/rust-bindgen-0.72.1.tar.gz
 https://downloads.sourceforge.net/scons/SCons-4.10.1.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
@@ -784,7 +785,7 @@ https://download.gnome.org/sources/xdg-desktop-portal-gnome/49/xdg-desktop-porta
 https://download.gnome.org/sources/dconf/0.49/dconf-0.49.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/49/dconf-editor-49.0.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/49/gnome-backgrounds-49.0.tar.xz
-https://download.gnome.org/sources/gvfs/1.58/gvfs-1.58.0.tar.xz
+https://download.gnome.org/sources/gvfs/1.58/gvfs-1.58.1.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.6.tar.xz
 https://download.gnome.org/sources/nautilus/49/nautilus-49.3.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/47/gnome-bluetooth-47.1.tar.xz
