diff --git a/wget-list b/wget-list
index 5fd1da7f26..efc01ca2ff 100644
--- a/wget-list
+++ b/wget-list
@@ -106,14 +106,14 @@ https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
 https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.3.0/downloads/libdisplay-info-0.3.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.12.2.tar.bz2
-https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.60.tar.bz2
-https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.57.tar.xz
+https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.61.tar.bz2
+https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.58.tar.xz
 https://download.gnome.org/sources/libgudev/238/libgudev-238.tar.xz
 https://github.com/hughsie/libgusb/releases/download/0.4.9/libgusb-0.4.9.tar.xz
-https://github.com/libical/libical/releases/download/v3.0.20/libical-3.0.20.tar.gz
+https://github.com/libical/libical/releases/download/v4.0.0/libical-4.0.0.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn-1.43.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn2-2.3.8.tar.gz
-https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.8.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.7.0.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v250/liblinear-250.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.34.0/libmbim-1.34.0.tar.gz
 https://github.com/linux-nvme/libnvme/archive/v1.16.1/libnvme-1.16.1.tar.gz
@@ -186,12 +186,12 @@ https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.1.1-xorg_prefix-1.pat
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/14.2.0/harfbuzz-14.2.0.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.2.9/jasper-version-4.2.9.tar.gz
-https://github.com/mm2/Little-CMS/releases/download/lcms2.19/lcms2-2.19.tar.gz
+https://github.com/mm2/Little-CMS/releases/download/lcms2.19.1/lcms2-2.19.1.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.4.1/libavif-1.4.1.tar.gz
 https://github.com/libexif/libexif/releases/download/v0.6.26/libexif-0.6.26.tar.bz2
 https://download.gnome.org/sources/libgxps/0.3/libgxps-0.3.2.tar.xz
 https://github.com/strukturag/libheif/releases/download/v1.21.2/libheif-1.21.2.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/libheif-1.21.2-svt_av1_build_fix-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/libheif-1.21.2-svt_av1_build_fix-2.patch
 https://github.com/libjpeg-turbo/libjpeg-turbo/releases/download/3.1.4.1/libjpeg-turbo-3.1.4.1.tar.gz
 https://github.com/libjxl/libjxl/archive/v0.11.2/libjxl-0.11.2.tar.gz
 https://downloads.sourceforge.net/libmng/libmng-2.0.3.tar.xz
@@ -210,7 +210,7 @@ https://github.com/opencv/opencv/archive/4.13.0/opencv-4.13.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.13.0/opencv_contrib-4.13.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.4/openjpeg-2.5.4.tar.gz
 https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
-https://poppler.freedesktop.org/poppler-26.04.0.tar.xz
+https://poppler.freedesktop.org/poppler-26.05.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v12.3.2/qpdf-12.3.2.tar.gz
@@ -262,7 +262,7 @@ http://fcron.free.fr/archives/fcron-3.4.0.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-gcc15_fixes-1.patch
-https://github.com/vcrhonek/hwdata/archive/v0.406/hwdata-0.406.tar.gz
+https://github.com/vcrhonek/hwdata/archive/v0.407/hwdata-0.407.tar.gz
 https://github.com/lfs-book/LSB-Tools/releases/download/v0.12/LSB-Tools-0.12.tar.gz
 https://github.com/logrotate/logrotate/releases/download/3.22.0/logrotate-3.22.0.tar.xz
 http://ftp.midnight-commander.org/mc-4.8.33.tar.xz
@@ -457,7 +457,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/XML-LibXML-2.0210-testsuite_fi
 https://cpan.metacpan.org/authors/id/P/PE/PERIGRIN/XML-NamespaceSupport-1.12.tar.gz
 https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-1.02.tar.gz
 https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
-https://www.php.net/distributions/php-8.5.5.tar.xz
+https://www.php.net/distributions/php-8.5.6.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
 https://www.python.org/ftp/python/3.14.4/Python-3.14.4.tar.xz
@@ -541,7 +541,7 @@ https://files.pythonhosted.org/packages/source/s/sphinxcontrib-qthelp/sphinxcont
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib_serializinghtml/sphinxcontrib_serializinghtml-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/t/trove_classifiers/trove_classifiers-2026.1.14.14.tar.gz
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.gz
-https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.6.3.tar.gz
+https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.7.0.tar.gz
 https://files.pythonhosted.org/packages/source/u/uv_build/uv_build-0.10.2.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/4.0/ruby-4.0.3.tar.xz
@@ -626,6 +626,7 @@ https://ftp.exim.org/pub/exim/exim4/exim-4.99.1.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.11.2.tar.gz
 https://git.openldap.org/openldap/openldap/-/archive/LMDB_0.9.35/LMDB_0.9.35.tar.bz2
 https://downloads.mariadb.org/interstitial/mariadb-11.8.6/source/mariadb-11.8.6.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/mariadb-11.8.6-openssl4_fixes-1.patch
 https://ftp.postgresql.org/pub/source/v18.3/postgresql-18.3.tar.bz2
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.13.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.13-consolidated-1.patch
@@ -669,7 +670,7 @@ https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.6.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.3.1.tgz
 https://github.com/KhronosGroup/glslang/archive/16.3.0/glslang-16.3.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
-https://download.gnome.org/sources/goffice/0.10/goffice-0.10.60.tar.xz
+https://download.gnome.org/sources/goffice/0.10/goffice-0.10.61.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk/3.24/gtk-3.24.52.tar.xz
 https://download.gnome.org/sources/gtk/4.22/gtk-4.22.4.tar.xz
@@ -881,7 +882,7 @@ https://github.com/lxqt/qps/releases/download/2.13.0/qps-2.13.0.tar.xz
 https://github.com/lxqt/qtermwidget/releases/download/2.4.0/qtermwidget-2.4.0.tar.xz
 https://github.com/lxqt/qterminal/releases/download/2.4.0/qterminal-2.4.0.tar.xz
 https://github.com/lxqt/screengrab/releases/download/3.2.0/screengrab-3.2.0.tar.xz
-https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.60.tar.xz
+https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.61.tar.xz
 https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-26.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-dictionaries-26.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-help-26.2.2.2.tar.xz
@@ -901,13 +902,13 @@ https://github.com/freerdp/freerdp/archive/3.26.0/FreeRDP-3.26.0.tar.gz
 https://download.gimp.org/gimp/v3.2/gimp-3.2.4.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.2.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.8.1.tar.gz
-https://inkscape.org/gallery/item/58914/inkscape-1.4.3.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4.3-poppler-26_fixes-2.patch
+https://inkscape.org/gallery/item/59505/inkscape-1.4.4.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4.4-poppler_fixes-1.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/140.10.1esr/source/thunderbird-140.10.1esr.source.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.1esr-python_3.14_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.1esr-glibc-2.43.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.1esr-llvm_22-1.patch
+https://archive.mozilla.org/pub/thunderbird/releases/140.10.2esr/source/thunderbird-140.10.2esr.source.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.2esr-python_3.14_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.2esr-glibc-2.43.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.2esr-llvm_22-1.patch
 https://github.com/TigerVNC/tigervnc/archive/v1.16.0/tigervnc-1.16.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.16.0-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.1.1/transmission-4.1.1.tar.xz
@@ -927,7 +928,7 @@ https://github.com/lfs-book/faac/archive/v1.50/faac-1.50.tar.gz
 https://github.com/knik0/faad2/archive/2.11.2/faad2-2.11.2.tar.gz
 https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://github.com/xiph/flac/releases/download/1.5.0/flac-1.5.0.tar.xz
-https://github.com/dyne/frei0r/archive/v3.1.1/frei0r-3.1.1.tar.gz
+https://github.com/dyne/frei0r/archive/v3.1.3/frei0r-3.1.3.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
 https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.28.2.tar.xz
 https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.28.2.tar.xz
