diff --git a/wget-list b/wget-list
index d8bf682e6a..e67727535f 100644
--- a/wget-list
+++ b/wget-list
@@ -51,7 +51,7 @@ https://www.bennewitz.com/bluefish/stable/source/bluefish-2.4.1.tar.bz2
 https://ftpmirror.gnu.org/ed/ed-1.22.5.tar.lz
 https://ftpmirror.gnu.org/emacs/emacs-30.2.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/25.12.2/src/kate-25.12.2.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/kate-26.04.1.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.7/mousepad-0.7.0.tar.xz
 https://www.nano-editor.org/dist/v9/nano-9.0.tar.xz
 https://github.com/vim/vim/archive/v9.2.0481/vim-9.2.0481.tar.gz
@@ -117,7 +117,7 @@ https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.8.0.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v250/liblinear-250.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.34.0/libmbim-1.34.0.tar.gz
 https://github.com/linux-nvme/libnvme/archive/v1.16.1/libnvme-1.16.1.tar.gz
-https://github.com/rrthomas/libpaper/releases/download/v2.2.7/libpaper-2.2.7.tar.gz
+https://github.com/rrthomas/libpaper/releases/download/v2.2.8/libpaper-2.2.8.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.12/libproxy-0.5.12.tar.gz
@@ -150,6 +150,8 @@ https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
 https://nodejs.org/dist/v24.15.0/node-v24.15.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.39/src/nspr-4.39.tar.gz
+https://github.com/AcademySoftwareFoundation/OpenTimelineIO/archive/v0.18.1/OpenTimelineIO-0.18.1.tar.gz
+https://anduin.linuxfromscratch.org/BLFS/OpenTimelineIO/OpenTimelineIO-rapidjson-20260513.tar.xz
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://github.com/protocolbuffers/protobuf/releases/download/v34.1/protobuf-34.1.tar.gz
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.2/protobuf-c-1.5.2.tar.gz
@@ -166,7 +168,7 @@ https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.2/SPIRV-LLVM
 https://www.samba.org/ftp/talloc/talloc-2.4.4.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.7/umockdev-0.19.7.tar.xz
-https://github.com/nemtrif/utfcpp/archive/v4.0.9/utfcpp-4.0.9.tar.gz
+https://github.com/nemtrif/utfcpp/archive/v4.1.0/utfcpp-4.1.0.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.25.0/downloads/wayland-1.25.0.tar.xz
 https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.48/downloads/wayland-protocols-1.48.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
@@ -186,6 +188,7 @@ https://download.gnome.org/sources/glycin/2.1/glycin-2.1.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.1.1-xorg_prefix-1.patch
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/14.2.0/harfbuzz-14.2.0.tar.xz
+https://github.com/AcademySoftwareFoundation/Imath/releases/download/v3.2.2/Imath-3.2.2.tar.gz
 https://github.com/jasper-software/jasper/archive/version-4.2.9/jasper-version-4.2.9.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.19.1/lcms2-2.19.1.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.4.1/libavif-1.4.1.tar.gz
@@ -243,7 +246,7 @@ https://ftpmirror.gnu.org/sharutils/sharutils-4.15.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/sharutils-4.15.2-consolidated-1.patch
 https://github.com/htacg/tidy-html5/archive/5.8.0/tidy-html5-5.8.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tidy-html5-5.8.0-cmake4_fixes-1.patch
-https://ftpmirror.gnu.org/time/time-1.9.tar.gz
+https://ftpmirror.gnu.org/time/time-1.10.tar.gz
 https://gitlab.com/OldManProgrammer/unix-tree/-/archive/2.3.2/unix-tree-2.3.2.tar.bz2
 https://github.com/lurcher/unixODBC/archive/v2.3.14/unixODBC-2.3.14.tar.gz
 https://github.com/flatpak/xdg-dbus-proxy/releases/download/0.1.7/xdg-dbus-proxy-0.1.7.tar.xz
@@ -481,7 +484,7 @@ https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi_docgen-2026.1.tar.gz
 https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/html5lib-1.1-python_3.14_buildfix-1.patch
-https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.1.0.tar.gz
+https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.1.1.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.3.12.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.4.4.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.2.2.tar.gz
@@ -582,7 +585,7 @@ https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
 https://ftp.isc.org/isc/bind9/9.20.20/bind-9.20.20.tar.xz
-https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.56.0/downloads/NetworkManager-1.56.0.tar.xz
+https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.56.1/downloads/NetworkManager-1.56.1.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.99.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/nmap-7.99-openssl4_fixes-1.patch
@@ -689,7 +692,7 @@ https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.2/kImageAnnotator-0.7.2.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
 https://download.gnome.org/sources/libadwaita/1.9/libadwaita-1.9.0.tar.xz
-https://gitlab.freedesktop.org/libinput/libei/-/archive/1.5.0/libei-1.5.0.tar.bz2
+https://gitlab.freedesktop.org/libinput/libei/-/archive/1.6.0/libei-1.6.0.tar.bz2
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
 https://dri.freedesktop.org/libdrm/libdrm-2.4.133.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
@@ -731,29 +734,30 @@ https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.201.1.tar.xz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.21.0.tar.xz
 https://github.com/dmtx/libdmtx/archive/v0.7.8/libdmtx-0.7.8.tar.gz
-https://download.kde.org/stable/release-service/25.12.2/src/ark-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/dolphin-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/dolphin-plugins-25.12.2.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/ark-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/dolphin-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/dolphin-plugins-26.04.1.tar.xz
 https://github.com/KDAB/KDDockWidgets/releases/download/v2.4.0/KDDockWidgets-2.4.0.tar.gz
-https://download.kde.org/stable/release-service/25.12.2/src/kdenlive-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/kmix-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/khelpcenter-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/konsole-25.12.2.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/kdenlive-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/kmix-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/khelpcenter-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/konsole-26.04.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/konsole-adjust_scrollbar-1.patch
-https://download.kde.org/stable/release-service/25.12.2/src/konversation-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/libkexiv2-25.12.2.tar.xz
-https://download.kde.org/stable/plasma/6.6.1/plasma-activities-6.6.1.tar.xz
-https://download.kde.org/stable/plasma/6.6.1/plasma-activities-stats-6.6.1.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/kio-extras-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/okular-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/libkdcraw-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/gwenview-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/libkcddb-25.12.2.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/k3b-25.12.2.tar.xz
-https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.10.0.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/konversation-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/libkexiv2-26.04.1.tar.xz
+https://download.kde.org/stable/plasma/6.6.5/plasma-activities-6.6.5.tar.xz
+https://download.kde.org/stable/plasma/6.6.5/plasma-activities-stats-6.6.5.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/kio-extras-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/okular-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/libkdcraw-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/gwenview-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/libkcddb-26.04.1.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/k3b-26.04.1.tar.xz
+https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.12.1.tar.xz
 https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.8.1.tar.xz
-https://github.com/frankosterfeld/qtkeychain/archive/0.15.0/qtkeychain-0.15.0.tar.gz
-https://github.com/KDE/kquickimageeditor/archive/v0.6.0/kquickimageeditor-0.6.0.tar.gz
+https://github.com/frankosterfeld/qtkeychain/archive/0.16.0/qtkeychain-0.16.0.tar.gz
+https://github.com/KDE/kquickimageeditor/archive/v0.6.1/kquickimageeditor-0.6.1.tar.gz
+https://gitlab.freedesktop.org/bolt/bolt/-/archive/0.9.11/bolt-0.9.11.tar.gz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.4/gcr-4.4.0.1.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/50/gsettings-desktop-schemas-50.1.tar.xz
@@ -847,11 +851,11 @@ https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.7.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.5/xfce4-pulseaudio-plugin-0.5.1.tar.xz
 https://download.kde.org/stable/frameworks/6.26/kwindowsystem-6.26.0.tar.xz
 https://download.kde.org/stable/frameworks/6.26/kconfig-6.26.0.tar.xz
-https://download.kde.org/stable/frameworks/6.26/solid-6.23.0.tar.xz
+https://download.kde.org/stable/frameworks/6.26/solid-6.26.0.tar.xz
 https://download.kde.org/stable/frameworks/6.26/kidletime-6.26.0.tar.xz
-https://download.kde.org/stable/plasma/6.6.1/kwayland-6.6.1.tar.xz
-https://download.kde.org/stable/plasma/6.6.1/libkscreen-6.6.1.tar.xz
-https://download.kde.org/stable/plasma/6.6.1/layer-shell-qt-6.6.1.tar.xz
+https://download.kde.org/stable/plasma/6.6.5/kwayland-6.6.5.tar.xz
+https://download.kde.org/stable/plasma/6.6.5/libkscreen-6.6.5.tar.xz
+https://download.kde.org/stable/plasma/6.6.5/layer-shell-qt-6.6.5.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.5/muparser-2.3.5.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/2.4.0/lxqt-build-tools-2.4.0.tar.xz
 https://github.com/lxqt/libdbusmenu-lxqt/releases/download/0.4.0/libdbusmenu-lxqt-0.4.0.tar.xz
@@ -894,7 +898,7 @@ https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-dicti
 https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-help-26.2.2.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-translations-26.2.2.2.tar.xz
 https://download.gnome.org/sources/epiphany/49/epiphany-49.3.tar.xz
-https://download.kde.org/stable/release-service/25.12.2/src/falkon-25.12.2.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/falkon-26.04.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.11.0esr-glibc-2.43.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.11.0esr-python_3.14_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.11.0esr-llvm_22-1.patch
@@ -911,10 +915,10 @@ https://downloads.sourceforge.net/gparted/gparted-1.8.1.tar.gz
 https://inkscape.org/gallery/item/59505/inkscape-1.4.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4.4-poppler_fixes-1.patch
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/140.10.2esr/source/thunderbird-140.10.2esr.source.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.2esr-python_3.14_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.2esr-glibc-2.43.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.10.2esr-llvm_22-1.patch
+https://archive.mozilla.org/pub/thunderbird/releases/140.11.0esr/source/thunderbird-140.11.0esr.source.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.11.0esr-python_3.14_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.11.0esr-glibc-2.43.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.11.0esr-llvm_22-1.patch
 https://github.com/TigerVNC/tigervnc/archive/v1.16.0/tigervnc-1.16.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.16.0-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.1.1/transmission-4.1.1.tar.xz
@@ -1000,7 +1004,7 @@ https://distfiles.audacious-media-player.org/audacious-4.5.1.tar.bz2
 https://distfiles.audacious-media-player.org/audacious-plugins-4.5.1.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
-https://download.kde.org/stable/release-service/25.12.2/src/kwave-25.12.2.tar.xz
+https://download.kde.org/stable/release-service/26.04.1/src/kwave-26.04.1.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
 https://downloads.sourceforge.net/mpg123/mpg123-1.33.5.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-6.2.tar.xz
