diff --git a/wget-list b/wget-list
index dcb1adb2d9..f3ba8811c5 100644
--- a/wget-list
+++ b/wget-list
@@ -476,7 +476,7 @@ https://download.gnome.org/sources/blueprint-compiler/0.20/blueprint-compiler-0.
 https://files.pythonhosted.org/packages/source/b/build/build-1.5.0.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.4/cachecontrol-0.14.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.4.0.tar.gz
-https://github.com/cython/cython/releases/download/3.2.4/cython-3.2.4.tar.gz
+https://github.com/cython/cython/releases/download/3.2.5/cython-3.2.5.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.38.1.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.4.0.tar.xz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.22.4.tar.gz
@@ -593,7 +593,7 @@ https://nmap.org/dist/nmap-7.99.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/nmap-7.99-openssl4_fixes-1.patch
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.6/whois-5.6.6.tar.gz
-https://www.wireshark.org/download/src/all-versions/wireshark-4.6.5.tar.xz
+https://www.wireshark.org/download/src/all-versions/wireshark-4.6.6.tar.xz
 https://github.com/c-ares/c-ares/releases/download/v1.34.6/c-ares-1.34.6.tar.gz
 https://curl.se/download/curl-8.20.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.8.1/geoclue-2.8.1.tar.bz2
@@ -610,7 +610,7 @@ https://download.gnome.org/sources/libnma/1.10/libnma-1.10.6.tar.xz
 https://github.com/thkukuk/libnsl/releases/download/v2.0.1/libnsl-2.0.1.tar.xz
 https://www.tcpdump.org/release/libpcap-1.10.6.tar.gz
 https://github.com/rockdaboot/libpsl/releases/download/0.21.5/libpsl-0.21.5.tar.gz
-https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.1/libslirp-v4.9.1.tar.bz2
+https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.9.2/libslirp-v4.9.2.tar.bz2
 https://download.gnome.org/sources/libsoup/3.6/libsoup-3.6.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libsoup-3.6.6-upstream_fixes-1.patch
 https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.7.tar.bz2
@@ -668,7 +668,7 @@ https://invisible-mirror.net/archives/xterm/xterm-410.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
-https://download.gnome.org/sources/at-spi2-core/2.60/at-spi2-core-2.60.3.tar.xz
+https://download.gnome.org/sources/at-spi2-core/2.60/at-spi2-core-2.60.4.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.4.tar.xz
 https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
 https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
