diff --git a/wget-list b/wget-list
index 9e6ec27671..d775a7163e 100644
--- a/wget-list
+++ b/wget-list
@@ -54,7 +54,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/25.12.2/src/kate-25.12.2.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.7/mousepad-0.7.0.tar.xz
 https://www.nano-editor.org/dist/v9/nano-9.0.tar.xz
-https://github.com/vim/vim/archive/v9.2.0340/vim-9.2.0340.tar.gz
+https://github.com/vim/vim/archive/v9.2.0421/vim-9.2.0421.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.3.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
@@ -84,7 +84,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
 https://download.gnome.org/sources/glibmm/2.88/glibmm-2.88.0.tar.xz
 https://github.com/jstedfast/gmime/releases/download/3.2.15/gmime-3.2.15.tar.xz
-https://github.com/lfs-book/gmmlib/archive/v22.9.0/gmmlib-22.9.0.tar.gz
+https://github.com/lfs-book/gmmlib/archive/v22.10.0/gmmlib-22.10.0.tar.gz
 https://ftpmirror.gnu.org/gsl/gsl-2.8.tar.gz
 https://download.gnome.org/sources/gspell/1.14/gspell-1.14.3.tar.xz
 https://github.com/google/highway/archive/1.4.0/highway-1.4.0.tar.gz
@@ -616,7 +616,7 @@ https://invisible-mirror.net/archives/lynx/tarballs/lynx2.9.2.tar.bz2
 https://downloads.sourceforge.net/fetchmail/fetchmail-6.6.3.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/mailx/heirloom-mailx_12.5.orig.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/heirloom-mailx-12.5-fixes-1.patch
-http://ftp.mutt.org/pub/mutt/mutt-2.3.1.tar.gz
+http://ftp.mutt.org/pub/mutt/mutt-2.3.2.tar.gz
 https://github.com/BuGlessRB/procmail/archive/v3.24/procmail-3.24.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/procmail-3.24-consolidated_fixes-1.patch
 https://archive.apache.org/dist/httpd/httpd-2.4.66.tar.bz2
@@ -942,7 +942,7 @@ https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/archive/gstreamer-1.28
 https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.28.2.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
-https://github.com/lfs-book/intel-media-driver/archive/v25.4.6/intel-media-driver-25.4.6.tar.gz
+https://github.com/lfs-book/intel-media-driver/archive/v26.1.5/intel-media-driver-26.1.5.tar.gz
 https://github.com/intel/intel-vaapi-driver/releases/download/2.4.1/intel-vaapi-driver-2.4.1.tar.bz2
 https://distfiles.adelielinux.org/source/a52dec/a52dec-0.8.0.tar.gz
 https://downloads.xiph.org/releases/ao/libao-1.2.0.tar.gz
