diff --git a/wget-list b/wget-list
index ab3d6653c7..a9caba4c3b 100644
--- a/wget-list
+++ b/wget-list
@@ -123,7 +123,7 @@ https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.12/libproxy-0.5.12.tar.gz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.10.0/libqalculate-5.10.0.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.11.0/libqalculate-5.11.0.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.38.0/libqmi-1.38.0.tar.gz
 https://github.com/seccomp/libseccomp/releases/download/v2.6.0/libseccomp-2.6.0.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -230,9 +230,9 @@ https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-ut
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.5.tar.gz
 https://github.com/editorconfig/editorconfig-core-c/archive/v0.12.11/editorconfig-core-c-0.12.11.tar.gz
 https://github.com/google/shaderc/archive/v2026.2/shaderc-2026.2.tar.gz
-https://gitlab.com/graphviz/graphviz/-/archive/14.1.5/graphviz-14.1.5.tar.bz2
+https://gitlab.com/graphviz/graphviz/-/archive/15.0.0/graphviz-15.0.0.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.36/gtk-doc-1.36.1.tar.xz
-http://www.andre-simon.de/zip/highlight-4.19.tar.bz2
+http://www.andre-simon.de/zip/highlight-4.20.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.34/ibus-1.5.34.tar.gz
 https://www.unicode.org/Public/17.0.0/ucd/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.2-13.tar.xz
@@ -637,7 +637,7 @@ https://downloads.isc.org/isc/kea/3.0.2/kea-3.0.2.tar.xz
 https://github.com/proftpd/proftpd/archive/v1.3.9a/proftpd-1.3.9a.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.2.tar.gz
 https://ftp.exim.org/pub/exim/exim4/exim-4.99.1.tar.xz
-https://ghostarchive.org/postfix/postfix-release/official/postfix-3.11.2.tar.gz
+https://ghostarchive.org/postfix/postfix-release/official/postfix-3.11.3.tar.gz
 https://git.openldap.org/openldap/openldap/-/archive/LMDB_0.9.35/LMDB_0.9.35.tar.bz2
 https://downloads.mariadb.org/interstitial/mariadb-11.8.7/source/mariadb-11.8.7.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/mariadb-11.8.7-openssl4_fixes-1.patch
@@ -839,7 +839,7 @@ https://archive.xfce.org/src/xfce/exo/4.20/exo-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/garcon/4.20/garcon-4.20.0.tar.bz2
 https://download.gnome.org/sources/libwnck/43/libwnck-43.3.tar.xz
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.20/xfce4-dev-tools-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.5.tar.bz2
+https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.6.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.7.tar.bz2
 https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.8.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.20/thunar-volman-4.20.0.tar.bz2
