diff --git a/wget-list b/wget-list
index 947e2f7395..5fd1da7f26 100644
--- a/wget-list
+++ b/wget-list
@@ -35,7 +35,7 @@ https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.t
 https://github.com/libfuse/libfuse/releases/download/fuse-3.18.2/fuse-3.18.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/jfsutils-1.1.15-gcc10_fix-1.patch
-https://sourceware.org/ftp/lvm2/LVM2.2.03.39.tgz
+https://sourceware.org/ftp/lvm2/LVM2.2.03.40.tgz
 https://www.kernel.org/pub/linux/utils/raid/mdadm/mdadm-4.4.tar.xz
 https://github.com/tuxera/ntfs-3g/archive/2026.2.25/ntfs-3g-2026.2.25.tar.gz
 https://downloads.sourceforge.net/gptfdisk/gptfdisk-1.0.10.tar.gz
@@ -157,8 +157,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/protobuf-c-1.5.2-protobuf34_fi
 https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.13.0/qcoro-0.13.0.tar.gz
 https://github.com/simdutf/simdutf/archive/v9.0.0/simdutf-9.0.0.tar.gz
-https://archive.mozilla.org/pub/firefox/releases/140.10.1esr/source/firefox-140.10.1esr.source.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.10.1-python_3.14_fixes-1.patch
+https://archive.mozilla.org/pub/firefox/releases/140.10.2esr/source/firefox-140.10.2esr.source.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.10.2-python_3.14_fixes-1.patch
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.341.0/SPIRV-Headers-vulkan-sdk-1.4.341.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.341.0/SPIRV-Tools-vulkan-sdk-1.4.341.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.2/SPIRV-LLVM-Translator-22.1.2.tar.gz
@@ -300,9 +300,9 @@ https://www.linuxfromscratch.org/patches/blfs/svn/dtc-1.7.2-glibc_fixes-1.patch
 https://ftpmirror.gnu.org/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
 https://github.com/bdwgc/bdwgc/releases/download/v8.2.12/gc-8.2.12.tar.gz
 https://ftpmirror.gnu.org/gdb/gdb-17.1.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-2.53.0.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-manpages-2.53.0.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.53.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-2.54.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-manpages-2.54.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.54.0.tar.xz
 https://ftpmirror.gnu.org/guile/guile-3.0.11.tar.xz
 https://www.lua.org/ftp/lua-5.4.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.8-shared_library-1.patch
@@ -556,8 +556,7 @@ https://downloads.sourceforge.net/swig/swig-4.4.1.tar.gz
 https://downloads.sourceforge.net/tcl/tk8.6.17-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.19.tar.xz
-https://sourceware.org/pub/valgrind/valgrind-3.26.0.tar.bz2
-https://www.linuxfromscratch.org/patches/blfs/svn/valgrind-3.26.0-upstream_fixes-1.patch
+https://sourceware.org/pub/valgrind/valgrind-3.27.0.tar.bz2
 https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/OpenJDK-21.0.10+6-i686-bin.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.10/OpenJDK-21.0.10+6-x86_64-bin.tar.xz
@@ -587,7 +586,7 @@ https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.6/whois-5.6.6.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.6.5.tar.xz
 https://github.com/c-ares/c-ares/releases/download/v1.34.6/c-ares-1.34.6.tar.gz
-https://curl.se/download/curl-8.19.0.tar.xz
+https://curl.se/download/curl-8.20.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.8.1/geoclue-2.8.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.80/glib-networking-2.80.1.tar.xz
 https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.3.0/kdsoap-2.3.0.tar.gz
@@ -889,17 +888,16 @@ https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-help-
 https://download.documentfoundation.org/libreoffice/src/26.2.2/libreoffice-translations-26.2.2.2.tar.xz
 https://download.gnome.org/sources/epiphany/49/epiphany-49.3.tar.xz
 https://download.kde.org/stable/release-service/25.12.2/src/falkon-25.12.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.10.1esr-glibc-2.43.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.10.1esr-python_3.14_fixes-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.10.1esr-llvm_22-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.10.2esr-glibc-2.43.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.10.2esr-python_3.14_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/firefox-140.10.2esr-llvm_22-1.patch
 https://archive.seamonkey-project.org/releases/2.53.23/source/seamonkey-2.53.23.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.23-libvpx_security_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.23-cxx17-1.patch
 https://gitlab.gnome.org/GNOME/balsa/-/archive/2.6.5/balsa-2.6.5.tar.gz
 https://feh.finalrewind.org/feh-3.12.1.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20251009/fontforge-20251009.tar.xz
-https://github.com/freerdp/freerdp/archive/3.25.0/FreeRDP-3.25.0.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/FreeRDP-3.25.0-upstream_fixes-1.patch
+https://github.com/freerdp/freerdp/archive/3.26.0/FreeRDP-3.26.0.tar.gz
 https://download.gimp.org/gimp/v3.2/gimp-3.2.4.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.2.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.8.1.tar.gz
@@ -1001,8 +999,8 @@ https://downloads.sourceforge.net/mpg123/mpg123-1.33.5.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-6.2.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.3.tar.gz
-https://ffmpeg.org/releases/ffmpeg-8.1.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-8.1-chromium_method-1.patch
+https://ffmpeg.org/releases/ffmpeg-8.1.1.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-8.1.1-chromium_method-1.patch
 https://github.com/mpv-player/mpv/archive/v0.41.0/mpv-0.41.0.tar.gz
 https://download.videolan.org/vlc/3.0.23/vlc-3.0.23.tar.xz
 https://downloads.sourceforge.net/xine/xine-ui-0.99.14.tar.xz
