diff --git a/wget-list b/wget-list
index bd550cc507..71ae91b040 100644
--- a/wget-list
+++ b/wget-list
@@ -78,7 +78,7 @@ https://download.gnome.org/sources/glib/2.84/glib-2.84.0.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.82/gobject-introspection-1.82.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.7.tar.xz
-https://download.gnome.org/sources/glibmm/2.82/glibmm-2.82.0.tar.xz
+https://download.gnome.org/sources/glibmm/2.84/glibmm-2.84.0.tar.xz
 https://download.gnome.org/sources/gmime/3.2/gmime-3.2.7.tar.xz
 https://github.com/lfs-book/gmmlib/archive/v22.5.5/gmmlib-22.5.5.tar.gz
 https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
@@ -147,7 +147,7 @@ https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.45/pcre2-10.45.
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://github.com/protocolbuffers/protobuf/releases/download/v29.3/protobuf-29.3.tar.gz
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.1/protobuf-c-1.5.1.tar.gz
-https://download.kde.org/stable/qca/2.3.9/qca-2.3.9.tar.xz
+https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.11.0/qcoro-0.11.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/128.8.0esr/source/firefox-128.8.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.304.1/SPIRV-Headers-vulkan-sdk-1.4.304.1.tar.gz
@@ -200,7 +200,7 @@ https://www.cairographics.org/releases/pixman-0.44.2.tar.gz
 https://poppler.freedesktop.org/poppler-25.03.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
-https://github.com/qpdf/qpdf/releases/download/v11.10.1/qpdf-11.10.1.tar.gz
+https://github.com/qpdf/qpdf/releases/download/v12.0.0/qpdf-12.0.0.tar.gz
 https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.bz2
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
@@ -642,8 +642,7 @@ https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.36/atkmm-2.36.3.tar.xz
 https://download.gnome.org/sources/at-spi2-core/2.54/at-spi2-core-2.54.1.tar.xz
-https://www.cairographics.org/releases/cairo-1.18.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/cairo-1.18.2-upstream_fixes-1.patch
+https://www.cairographics.org/releases/cairo-1.18.4.tar.xz
 https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
 https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
