diff --git a/wget-list b/wget-list
index 9bc9164ccf..ca59cfcfe2 100644
--- a/wget-list
+++ b/wget-list
@@ -22,7 +22,7 @@ https://github.com/p11-glue/p11-kit/releases/download/0.25.5/p11-kit-0.25.5.tar.
 https://github.com/polkit-org/polkit/archive/126/polkit-126.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
-https://github.com/shadow-maint/shadow/releases/download/4.17.2/shadow-4.17.2.tar.xz
+https://github.com/shadow-maint/shadow/releases/download/4.17.3/shadow-4.17.3.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.74.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.16p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
@@ -134,7 +134,7 @@ https://lttng.org/files/urcu/userspace-rcu-0.15.0.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
 https://dist.libuv.org/dist/v1.50.0/libuv-v1.50.0.tar.gz
 https://github.com/xkbcommon/libxkbcommon/archive/xkbcommon-1.8.0/libxkbcommon-xkbcommon-1.8.0.tar.gz
-https://download.gnome.org/sources/libxml2/2.13/libxml2-2.13.5.tar.xz
+https://download.gnome.org/sources/libxml2/2.13/libxml2-2.13.6.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.21/libxmlb-0.3.21.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.42.tar.xz
@@ -166,7 +166,7 @@ https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/1.4.27/xapian-core-1.4.27.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
 https://download.gimp.org/pub/babl/0.1/babl-0.1.110.tar.xz
-https://github.com/Exiv2/exiv2/archive/v0.28.4/exiv2-0.28.4.tar.gz
+https://github.com/Exiv2/exiv2/archive/v0.28.5/exiv2-0.28.5.tar.gz
 https://downloads.sourceforge.net/freetype/freetype-2.13.3.tar.xz
 https://downloads.sourceforge.net/freetype/freetype-doc-2.13.3.tar.xz
 https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.16.0.tar.xz
@@ -270,7 +270,7 @@ https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
 https://github.com/lfs-book/sysmond/releases/download/sysmond-1.0.0/sysmond-1.0.0.tar.xz
 https://github.com/lfs-book/sysmon3/releases/download/sysmon3-3.0.0/sysmon3-3.0.0.tar.xz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.7.tar.xz
-https://github.com/systemd/systemd/archive/v257/systemd-257.tar.gz
+https://github.com/systemd/systemd/archive/v257.3/systemd-257.3.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.1.3.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.7/upower-v1.90.7.tar.bz2
@@ -278,7 +278,7 @@ https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-018.tar.xz
 https://ftp.gnu.org/gnu/which/which-2.23.tar.gz
 https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz
 https://downloads.sourceforge.net/infozip/zip30.tar.gz
-https://github.com/lu-zero/cargo-c/archive/v0.10.9/cargo-c-0.10.9.tar.gz
+https://github.com/lu-zero/cargo-c/archive/v0.10.11/cargo-c-0.10.11.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.28.0/cbindgen-0.28.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
@@ -328,13 +328,13 @@ https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.089.tar.gz
 https://www.cpan.org/authors/id/G/GA/GAAS/IO-String-1.08.tar.gz
 https://www.cpan.org/authors/id/R/RJ/RJBS/IPC-Run3-0.049.tar.gz
 https://www.cpan.org/authors/id/A/AL/ALINKE/Lingua-Translit-0.29.tar.gz
-https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.77.tar.gz
+https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.78.tar.gz
 https://www.cpan.org/authors/id/D/DR/DROLSKY/List-AllUtils-0.19.tar.gz
 https://www.cpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz
 https://www.cpan.org/authors/id/E/ET/ETJ/Log-Log4perl-1.57.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.14.tar.gz
 https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4234.tar.gz
-https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.49.tar.gz
+https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.50.tar.gz
 https://www.cpan.org/authors/id/J/JT/JTBRAUN/Parse-RecDescent-1.967015.tar.gz
 https://www.cpan.org/authors/id/W/WB/WBRASWELL/Parse-Yapp-1.21.tar.gz
 https://www.cpan.org/authors/id/L/LE/LEONT/PerlIO-utf8_strict-0.010.tar.gz
@@ -525,7 +525,7 @@ https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.g
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.3.0.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.4/ruby-3.4.2.tar.xz
-https://static.rust-lang.org/dist/rustc-1.84.1-src.tar.xz
+https://static.rust-lang.org/dist/rustc-1.85.0-src.tar.xz
 https://github.com/rust-lang/rust-bindgen/archive/v0.71.1/rust-bindgen-0.71.1.tar.gz
 https://downloads.sourceforge.net/scons/SCons-4.8.1.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
@@ -605,14 +605,14 @@ https://downloads.isc.org/isc/kea/2.6.1/kea-2.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/kea-2.6.1-fix_boost_1_87-1.patch
 https://github.com/proftpd/proftpd/archive/v1.3.8b/proftpd-1.3.8b.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.0.tar.gz
-https://ftp.exim.org/pub/exim/exim4/exim-4.98.tar.xz
+https://ftp.exim.org/pub/exim/exim4/exim-4.98.1.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.9.2.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-11.4.5/source/mariadb-11.4.5.tar.gz
-https://ftp.postgresql.org/pub/source/v17.3/postgresql-17.3.tar.bz2
-https://sqlite.org/2025/sqlite-autoconf-3490000.tar.gz
-https://sqlite.org/2025/sqlite-doc-3490000.zip
+https://ftp.postgresql.org/pub/source/v17.4/postgresql-17.4.tar.bz2
+https://sqlite.org/2025/sqlite-autoconf-3490100.tar.gz
+https://sqlite.org/2025/sqlite-doc-3490100.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.9.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.9-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.22.0.tar.gz
