diff --git a/wget-list b/wget-list
index c2adad31bc..8f25005098 100644
--- a/wget-list
+++ b/wget-list
@@ -272,7 +272,7 @@ https://github.com/lfs-book/sysmon3/releases/download/sysmon3-3.0.1/sysmon3-3.0.
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.7.tar.xz
 https://github.com/systemd/systemd/archive/v257.3/systemd-257.3.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
-https://www.rarlab.com/rar/unrarsrc-7.1.3.tar.gz
+https://www.rarlab.com/rar/unrarsrc-7.1.5.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.7/upower-v1.90.7.tar.bz2
 https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-018.tar.xz
 https://ftp.gnu.org/gnu/which/which-2.23.tar.gz
@@ -462,7 +462,7 @@ https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.3.2.tar.gz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.2.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
-https://files.pythonhosted.org/packages/source/g/gi-docgen/gi_docgen-2024.1.tar.gz
+https://files.pythonhosted.org/packages/source/g/gi-docgen/gi_docgen-2025.3.tar.gz
 https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-5.3.1.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.3.9.tar.gz
@@ -487,7 +487,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.32.3-use_system_cer
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
 https://github.com/getsentry/sentry-python/releases/download/2.22.0/sentry_sdk-2.22.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.17.0.tar.gz
-https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.2.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-3.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.1.0.tar.gz
@@ -559,7 +559,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.5/bind-9.20.5.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.6/bind-9.20.6.tar.xz
 https://download.gnome.org/sources/NetworkManager/1.50/NetworkManager-1.50.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.95.tar.bz2
@@ -651,8 +651,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/cairo-1.18.2-upstream_fixes-1.
 https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
 https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.1.tar.xz
-https://github.com/fltk/fltk/releases/download/release-1.4.1/fltk-1.4.1-source.tar.gz
-https://github.com/fltk/fltk/releases/download/release-1.4.1/fltk-1.4.1-docs-html.tar.gz
+https://github.com/fltk/fltk/releases/download/release-1.4.2/fltk-1.4.2-source.tar.gz
+https://github.com/fltk/fltk/releases/download/release-1.4.2/fltk-1.4.2-docs-html.tar.gz
 https://downloads.sourceforge.net/freeglut/freeglut-3.6.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.12.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.2.0.tgz
@@ -726,8 +726,8 @@ https://download.kde.org/stable/release-service/24.12.2/src/konsole-24.12.2.tar.
 https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.12.2-scrollbar-1.patch
 https://download.kde.org/stable/release-service/24.12.2/src/konversation-24.12.2.tar.xz
 https://download.kde.org/stable/release-service/24.12.2/src/libkexiv2-24.12.2.tar.xz
-https://download.kde.org/stable/plasma/6.3.1/plasma-activities-6.3.1.tar.xz
-https://download.kde.org/stable/plasma/6.3.1/plasma-activities-stats-6.3.1.tar.xz
+https://download.kde.org/stable/plasma/6.3.2/plasma-activities-6.3.2.tar.xz
+https://download.kde.org/stable/plasma/6.3.2/plasma-activities-stats-6.3.2.tar.xz
 https://download.kde.org/stable/release-service/24.12.2/src/kio-extras-24.12.2.tar.xz
 https://download.kde.org/stable/release-service/24.12.2/src/okular-24.12.2.tar.xz
 https://download.kde.org/stable/release-service/24.12.2/src/libkdcraw-24.12.2.tar.xz
@@ -831,9 +831,9 @@ https://download.kde.org/stable/frameworks/6.11/kwindowsystem-6.11.0.tar.xz
 https://download.kde.org/stable/frameworks/6.11/kconfig-6.11.0.tar.xz
 https://download.kde.org/stable/frameworks/6.11/solid-6.11.0.tar.xz
 https://download.kde.org/stable/frameworks/6.11/kidletime-6.11.0.tar.xz
-https://download.kde.org/stable/plasma/6.3.1/kwayland-6.3.1.tar.xz
-https://download.kde.org/stable/plasma/6.3.1/libkscreen-6.3.1.tar.xz
-https://download.kde.org/stable/plasma/6.3.1/layer-shell-qt-6.3.1.tar.xz
+https://download.kde.org/stable/plasma/6.3.2/kwayland-6.3.2.tar.xz
+https://download.kde.org/stable/plasma/6.3.2/libkscreen-6.3.2.tar.xz
+https://download.kde.org/stable/plasma/6.3.2/layer-shell-qt-6.3.2.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.5/muparser-2.3.5.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/2.1.0/lxqt-build-tools-2.1.0.tar.xz
 https://github.com/lxqt/libdbusmenu-lxqt/releases/download/0.2.0/libdbusmenu-lxqt-0.2.0.tar.xz
@@ -909,7 +909,7 @@ https://www.alsa-project.org/files/pub/tools/alsa-tools-1.2.11.tar.bz2
 https://www.alsa-project.org/files/pub/firmware/alsa-firmware-1.2.4.tar.bz2
 https://download.gnome.org/sources/audiofile/0.3/audiofile-0.3.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/audiofile-0.3.6-consolidated_patches-1.patch
-https://github.com/knik0/faac/archive/1_30/faac-1_30.tar.gz
+https://github.com/lfs-book/faac/archive/v1.31/faac-1.31.tar.gz
 https://github.com/knik0/faad2/archive/2.11.1/faad2-2.11.1.tar.gz
 https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://github.com/xiph/flac/releases/download/1.5.0/flac-1.5.0.tar.xz
