diff --git a/wget-list b/wget-list
index ca59cfcfe2..b87a5ebdb8 100644
--- a/wget-list
+++ b/wget-list
@@ -76,8 +76,8 @@ https://duktape.org/duktape-2.7.0.tar.xz
 https://github.com/rrthomas/enchant/releases/download/v2.8.2/enchant-2.8.2.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.5.tar.xz
 https://www.fftw.org/fftw-3.3.10.tar.gz
-https://github.com/fmtlib/fmt/archive/11.1.3/fmt-11.1.3.tar.gz
-https://download.gnome.org/sources/glib/2.82/glib-2.82.4.tar.xz
+https://github.com/fmtlib/fmt/archive/11.1.4/fmt-11.1.4.tar.gz
+https://download.gnome.org/sources/glib/2.82/glib-2.82.5.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.82/gobject-introspection-1.82.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.7.tar.xz
@@ -130,7 +130,7 @@ https://www.mirrorservice.org/sites/ftp.i-scream.org/pub/i-scream/libstatgrab/li
 https://ftp.gnu.org/gnu/libtasn1/libtasn1-4.20.0.tar.gz
 https://ftp.gnu.org/gnu/libunistring/libunistring-1.3.tar.xz
 https://github.com/libunwind/libunwind/releases/download/v1.8.1/libunwind-1.8.1.tar.gz
-https://lttng.org/files/urcu/userspace-rcu-0.15.0.tar.bz2
+https://lttng.org/files/urcu/userspace-rcu-0.15.1.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
 https://dist.libuv.org/dist/v1.50.0/libuv-v1.50.0.tar.gz
 https://github.com/xkbcommon/libxkbcommon/archive/xkbcommon-1.8.0/libxkbcommon-xkbcommon-1.8.0.tar.gz
@@ -177,7 +177,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/giflib-5.2.2-upstream_fixes-1.
 https://github.com/Dav1dde/glad/archive/v2.0.8/glad-2.0.8.tar.gz
 https://github.com/g-truc/glm/archive/1.0.1/glm-1.0.1.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
-https://github.com/harfbuzz/harfbuzz/releases/download/10.2.0/harfbuzz-10.2.0.tar.xz
+https://github.com/harfbuzz/harfbuzz/releases/download/10.3.0/harfbuzz-10.3.0.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.2.4/jasper-version-4.2.4.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.17/lcms2-2.17.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.1.1/libavif-1.1.1.tar.gz
@@ -301,7 +301,7 @@ https://www.lua.org/ftp/lua-5.4.7.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.7-shared_library-1.patch
 https://www.lua.org/tests/lua-5.4.7-tests.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/luajit/luajit-20250212.tar.xz
-https://www.mercurial-scm.org/release/mercurial-6.9.1.tar.gz
+https://www.mercurial-scm.org/release/mercurial-6.9.2.tar.gz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03.tar.xz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
@@ -485,9 +485,9 @@ https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.32.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/requests-2.32.3-use_system_certs-1.patch
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
-https://github.com/getsentry/sentry-python/releases/download/2.21.0/sentry_sdk-2.21.0.tar.gz
+https://github.com/getsentry/sentry-python/releases/download/2.22.0/sentry_sdk-2.22.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.17.0.tar.gz
-https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.1.3.tar.gz
+https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-3.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.1.0.tar.gz
@@ -541,7 +541,7 @@ https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-23.0.2/OpenJDK-23.0.2+7
 https://github.com/openjdk/jdk23u/archive/jdk-23.0.2-ga.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-23.0.2/jtreg-7.3.1+1.tar.gz
 https://archive.apache.org/dist/ant/source/apache-ant-1.10.15-src.tar.xz
-https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.2.0/dhcpcd-10.2.0.tar.xz
+https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.2.2/dhcpcd-10.2.2.tar.xz
 https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.xz
 https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.2.tar.bz2
 https://www.ncftp.com/downloads/ncftp/ncftp-3.2.7-src.tar.xz
