diff --git a/wget-list b/wget-list
index 18d40819eb..17d17f8789 100644
--- a/wget-list
+++ b/wget-list
@@ -45,7 +45,7 @@ https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
 https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
 https://unifoundry.com/pub/unifont/unifont-16.0.01/font-builds/unifont-16.0.01.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.16.tar.bz2
-https://ftp.gnu.org/gnu/ed/ed-1.21.tar.lz
+https://ftp.gnu.org/gnu/ed/ed-1.21.1.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-30.1.tar.xz
 https://download.gnome.org/sources/gedit/48/gedit-48.1.tar.xz
 https://gitlab.gnome.org/World/gedit/libgedit-amtk/-/archive/5.9.1/libgedit-amtk-5.9.1.tar.bz2
@@ -176,7 +176,7 @@ https://downloads.sourceforge.net/freetype/freetype-2.13.3.tar.xz
 https://downloads.sourceforge.net/freetype/freetype-doc-2.13.3.tar.xz
 https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.16.0.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.16/fribidi-1.0.16.tar.xz
-https://download.gimp.org/pub/gegl/0.4/gegl-0.4.56.tar.xz
+https://download.gimp.org/pub/gegl/0.4/gegl-0.4.58.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/giflib-5.2.2-upstream_fixes-1.patch
 https://github.com/Dav1dde/glad/archive/v2.0.8/glad-2.0.8.tar.gz
@@ -277,7 +277,7 @@ https://github.com/lfs-book/sysmon3/releases/download/sysmon3-3.0.1/sysmon3-3.0.
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.7.tar.xz
 https://github.com/systemd/systemd/archive/v257.3/systemd-257.3.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
-https://www.rarlab.com/rar/unrarsrc-7.1.5.tar.gz
+https://www.rarlab.com/rar/unrarsrc-7.1.6.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.7/upower-v1.90.7.tar.bz2
 https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-018.tar.xz
 https://ftp.gnu.org/gnu/which/which-2.23.tar.gz
@@ -481,7 +481,7 @@ https://github.com/pygobject/pycairo/releases/download/v1.26.1/pycairo-1.26.1.ta
 https://files.pythonhosted.org/packages/source/p/pygdbmi/pygdbmi-0.11.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.19.1.tar.gz
 https://download.gnome.org/sources/pygobject/3.52/pygobject-3.52.3.tar.gz
-https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.2.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.2.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
 https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.3.5.tar.gz
 https://files.pythonhosted.org/packages/b0/25/7998cd2dec731acbd438fbf91bc619603fc5188de0a9a17699a781840452/pyxdg-0.28.tar.gz
@@ -638,7 +638,7 @@ https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.44.tar
 https://www.x.org/pub/individual/xserver/xwayland-24.1.6.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.16.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.16-tearfree_backport-1.patch
-https://www.freedesktop.org/software/libevdev/libevdev-1.13.3.tar.xz
+https://www.freedesktop.org/software/libevdev/libevdev-1.13.4.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.11.0.tar.xz
 https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.28.0/libinput-1.28.0.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.5.0.tar.xz
@@ -666,7 +666,7 @@ https://download.gnome.org/sources/goffice/0.10/goffice-0.10.59.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk/3.24/gtk-3.24.49.tar.xz
 https://download.gnome.org/sources/gtk/4.18/gtk-4.18.2.tar.xz
-https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.9.tar.xz
+https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.10.tar.xz
 https://download.gnome.org/sources/gtkmm/4.18/gtkmm-4.18.0.tar.xz
 https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
 https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
@@ -696,7 +696,7 @@ https://webkitgtk.org/releases/webkitgtk-2.48.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/webkitgtk-2.48.0-build_fix-1.patch
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.20.0/xdg-desktop-portal-1.20.0.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.3/xdg-desktop-portal-gtk-1.15.3.tar.xz
-https://download.gnome.org/sources/gdm/47/gdm-47.0.tar.xz
+https://download.gnome.org/sources/gdm/48/gdm-48.0.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
@@ -766,21 +766,23 @@ https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
 https://download.gnome.org/sources/xdg-desktop-portal-gnome/48/xdg-desktop-portal-gnome-48.0.tar.xz
 https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
-https://download.gnome.org/sources/gnome-backgrounds/47/gnome-backgrounds-47.0.tar.xz
+https://download.gnome.org/sources/gnome-backgrounds/48/gnome-backgrounds-48.0.tar.xz
 https://download.gnome.org/sources/gvfs/1.56/gvfs-1.56.1.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.3.tar.xz
-https://download.gnome.org/sources/nautilus/47/nautilus-47.2.tar.xz
+https://download.gnome.org/sources/nautilus/48/nautilus-48.0.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/47/gnome-bluetooth-47.1.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/gnome-bluetooth-47.1-build_fix-1.patch
 https://download.gnome.org/sources/gnome-keyring/48/gnome-keyring-48.0.tar.xz
-https://download.gnome.org/sources/gnome-settings-daemon/47/gnome-settings-daemon-47.2.tar.xz
-https://download.gnome.org/sources/tecla/47/tecla-47.0.tar.xz
-https://download.gnome.org/sources/gnome-control-center/47/gnome-control-center-47.4.tar.xz
-https://download.gnome.org/sources/mutter/47/mutter-47.6.tar.xz
-https://download.gnome.org/sources/gnome-shell/47/gnome-shell-47.5.tar.xz
-https://download.gnome.org/sources/gnome-shell-extensions/47/gnome-shell-extensions-47.4.tar.xz
+https://download.gnome.org/sources/gnome-settings-daemon/48/gnome-settings-daemon-48.0.tar.xz
+https://download.gnome.org/sources/tecla/48/tecla-48.0.2.tar.xz
+https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.0.tar.xz
+https://download.gnome.org/sources/mutter/48/mutter-48.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/mutter-48.0-upstream_fixes-1.patch
+https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.0.tar.xz
+https://download.gnome.org/sources/gnome-shell-extensions/48/gnome-shell-extensions-48.0.tar.xz
 https://download.gnome.org/sources/gnome-session/47/gnome-session-47.0.1.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.1.tar.xz
-https://download.gnome.org/sources/gnome-user-docs/47/gnome-user-docs-47.2.tar.xz
+https://download.gnome.org/sources/gnome-user-docs/47/gnome-user-docs-47.5.tar.xz
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
 https://download.gnome.org/sources/baobab/47/baobab-47.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
@@ -890,7 +892,7 @@ https://feh.finalrewind.org/feh-3.10.3.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20230101/fontforge-20230101.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/fontforge-20230101-security_fixes-1.patch
 https://github.com/freerdp/freerdp/archive/3.13.0/FreeRDP-3.13.0.tar.gz
-https://download.gimp.org/gimp/v3.0/gimp-3.0.0.tar.xz
+https://download.gimp.org/gimp/v3.0/gimp-3.0.2.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
 https://inkscape.org/gallery/item/53679/inkscape-1.4.tar.xz
