diff --git a/wget-list b/wget-list
index a8cebe294..118b0a54b 100644
--- a/wget-list
+++ b/wget-list
@@ -103,7 +103,7 @@ https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.10.2.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.47.tar.bz2
 https://download.gnome.org/sources/libgrss/0.7/libgrss-0.7.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libgrss-0.7.0-bugfixes-2.patch
-https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.50.tar.xz
+https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.51.tar.xz
 https://download.gnome.org/sources/libgudev/238/libgudev-238.tar.xz
 https://github.com/hughsie/libgusb/releases/download/0.4.7/libgusb-0.4.7.tar.xz
 https://github.com/libical/libical/releases/download/v3.0.17/libical-3.0.17.tar.gz
@@ -141,7 +141,7 @@ https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_1_0/log4cplus-2.1.0.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.6.tar.bz2
-https://nodejs.org/dist/v18.18.2/node-v18.18.2.tar.xz
+https://nodejs.org/dist/v20.9.0/node-v20.9.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.6.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.bz2
@@ -194,7 +194,7 @@ https://github.com/opencv/opencv/archive/4.8.1/opencv-4.8.1.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.8.1/opencv_contrib-4.8.1.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.0/openjpeg-2.5.0.tar.gz
 https://www.cairographics.org/releases/pixman-0.42.2.tar.gz
-https://poppler.freedesktop.org/poppler-23.10.0.tar.xz
+https://poppler.freedesktop.org/poppler-23.11.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v11.6.3/qpdf-11.6.3.tar.gz
@@ -258,6 +258,7 @@ https://download.librdf.org/source/raptor2-2.0.16.tar.gz
 https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://sg.danny.cz/sg/p/sg3_utils-1.48.tar.xz
+https://github.com/lfs-book/sysmon-qt/releases/download/v1.0/sysmon-qt-1.0.tar.xz 
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.4.tar.xz
 https://github.com/systemd/systemd/archive/v254/systemd-254.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
@@ -278,9 +279,9 @@ https://doxygen.nl/files/doxygen-1.9.8.src.tar.gz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.4/gc-8.2.4.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-13.2.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-2.42.0.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-manpages-2.42.0.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.42.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-2.42.1.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-manpages-2.42.1.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.42.1.tar.xz
 https://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.xz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.1/llvm-17.0.1.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-17.src.tar.xz
@@ -600,14 +601,14 @@ https://www.dovecot.org/releases/2.3/dovecot-2.3.21.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-openssl3_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-security_fix-1.patch
 https://ftp.exim.org/pub/exim/exim4/exim-4.96.2.tar.xz
-https://ghostarchive.org/postfix/postfix-release/official/postfix-3.8.2.tar.gz
+https://ghostarchive.org/postfix/postfix-release/official/postfix-3.8.3.tar.gz
 https://ftp.sendmail.org/sendmail.8.17.2.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/bdb/db-5.3.28.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-10.11.5/source/mariadb-10.11.5.tar.gz
 https://ftp.postgresql.org/pub/source/v16.0/postgresql-16.0.tar.bz2
-https://sqlite.org/2023/sqlite-autoconf-3430200.tar.gz
-https://sqlite.org/2023/sqlite-doc-3430200.zip
+https://sqlite.org/2023/sqlite-autoconf-3440000.tar.gz
+https://sqlite.org/2023/sqlite-doc-3440000.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.6.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.6-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.18.0.tar.gz
@@ -652,7 +653,7 @@ https://downloads.sourceforge.net/freeglut/freeglut-3.4.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.10.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.2.0.tgz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
-https://download.gnome.org/sources/goffice/0.10/goffice-0.10.55.tar.xz
+https://download.gnome.org/sources/goffice/0.10/goffice-0.10.56.tar.xz
 https://github.com/steveire/grantlee/releases/download/v5.3.1/grantlee-5.3.1.tar.gz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.33.tar.xz
@@ -693,7 +694,7 @@ https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.8/lightdm-gtk-greeter-2.0.8.tar.gz
 https://github.com/sddm/sddm/archive/v0.20.0/sddm-0.20.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
-https://github.com/ice-wm/icewm/archive/3.4.3/icewm-3.4.3.tar.gz
+https://github.com/ice-wm/icewm/archive/3.4.4/icewm-3.4.4.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://download.gnome.org/sources/adwaita-icon-theme/45/adwaita-icon-theme-45.0.tar.xz
 https://download.kde.org/stable/frameworks/5.109/breeze-icons-5.109.0.tar.xz
@@ -858,7 +859,7 @@ https://github.com/lxqt/qterminal/releases/download/1.3.0/qterminal-1.3.0.tar.xz
 https://github.com/lxqt/screengrab/releases/download/2.6.0/screengrab-2.6.0.tar.xz
 https://www.abisource.com/downloads/abiword/3.0.5/source/abiword-3.0.5.tar.gz
 https://www.abisource.com/downloads/abiword/3.0.2/source/abiword-docs-3.0.2.tar.gz
-https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.55.tar.xz
+https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.56.tar.xz
 https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-7.6.2.1.tar.xz
 https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-dictionaries-7.6.2.1.tar.xz
 https://download.documentfoundation.org/libreoffice/src/7.6.2/libreoffice-help-7.6.2.1.tar.xz
@@ -945,11 +946,11 @@ https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.13.1/libvpx-1.13.1.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.20.0/mlt-7.20.0.tar.gz
 https://downloads.xiph.org/releases/opus/opus-1.4.tar.gz
-https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/0.3.83/pipewire-0.3.83.tar.bz2
+https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/0.3.84/pipewire-0.3.84.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-16.1.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.0.tar.xz
 https://github.com/libsdl-org/sdl12-compat/archive/refs/tags/release-1.2.68/sdl12-compat-release-1.2.68.tar.gz
-https://www.libsdl.org/release/SDL2-2.28.4.tar.gz
+https://www.libsdl.org/release/SDL2-2.28.5.tar.gz
 https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.3.2.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
@@ -991,7 +992,7 @@ https://files.libburnia-project.org/releases/libisofs-1.5.6.tar.gz
 https://github.com/OpenPrinting/cups/releases/download/v2.4.7/cups-2.4.7-source.tar.gz
 https://github.com/OpenPrinting/cups-browsed/releases/download/2.0.0/cups-browsed-2.0.0.tar.xz
 https://github.com/OpenPrinting/cups-filters/releases/download/2.0.0/cups-filters-2.0.0.tar.xz
-https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10020/ghostscript-10.02.0.tar.xz
+https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10021/ghostscript-10.02.1.tar.xz
 https://downloads.sourceforge.net/gs-fonts/ghostscript-fonts-std-8.11.tar.gz
 https://downloads.sourceforge.net/gs-fonts/gnu-gs-fonts-other-6.0.tar.gz
 https://downloads.sourceforge.net/gimp-print/gutenprint-5.3.4.tar.xz
