diff --git a/wget-list b/wget-list
index eb31b81c0..5280be685 100644
--- a/wget-list
+++ b/wget-list
@@ -172,8 +172,6 @@ https://github.com/g-truc/glm/archive/0.9.9.8/glm-0.9.9.8.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/8.2.2/harfbuzz-8.2.2.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.1.0/jasper-4.1.0.tar.gz
-https://downloads.sourceforge.net/lcms/lcms-1.19.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/lcms-1.19-cve_2013_4276-1.patch
 https://github.com/mm2/Little-CMS/releases/download/lcms2.14/lcms2-2.14.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.0.1/libavif-1.0.1.tar.gz
 https://github.com/libexif/libexif/releases/download/v0.6.24/libexif-0.6.24.tar.bz2
@@ -269,8 +267,6 @@ https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.2/upower-v1.90.2.ta
 https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-017.tar.xz
 https://ftp.gnu.org/gnu/which/which-2.21.tar.gz
 https://downloads.sourceforge.net/infozip/zip30.tar.gz
-https://ftp.gnu.org/gnu/autoconf/autoconf-2.13.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/autoconf-2.13-consolidated_fixes-1.patch
 https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
@@ -607,7 +603,7 @@ https://ftp.sendmail.org/sendmail.8.17.2.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/bdb/db-5.3.28.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-10.11.5/source/mariadb-10.11.5.tar.gz
-https://ftp.postgresql.org/pub/source/v16.0/postgresql-16.0.tar.bz2
+https://ftp.postgresql.org/pub/source/v16.1/postgresql-16.1.tar.bz2
 https://sqlite.org/2023/sqlite-autoconf-3440000.tar.gz
 https://sqlite.org/2023/sqlite-doc-3440000.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.6.tgz
@@ -974,13 +970,12 @@ https://downloads.sourceforge.net/mpg123/mpg123-1.32.3.tar.bz2
 https://freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-5.0.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.2.tar.gz
-https://ffmpeg.org/releases/ffmpeg-6.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0-chromium_method-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0-binutils_2.41-1.patch
-https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0-texinfo7_fixes-1.patch
+https://ffmpeg.org/releases/ffmpeg-6.0.1.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0.1-chromium_method-1.patch
 https://github.com/mpv-player/mpv/archive/refs/tags/v0.36.0/mpv-0.36.0.tar.gz
 https://mplayerhq.hu/MPlayer/releases/MPlayer-1.5.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/MPlayer-1.5-upstream_ffmpg6_fixes-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.0-binutils_2.41-1.patch
 https://mplayerhq.hu/MPlayer/skins/Clearlooks-2.0.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/transcode/transcode-1.1.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/transcode-1.1.7-gcc10_fix-1.patch
