diff --git a/wget-list b/wget-list
index a08487935..fac6dd55e 100644
--- a/wget-list
+++ b/wget-list
@@ -209,7 +209,7 @@ https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.1.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/9.0.0/graphviz-9.0.0.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.33/gtk-doc-1.33.2.tar.xz
 http://www.andre-simon.de/zip/highlight-4.10.tar.bz2
-https://github.com/ibus/ibus/releases/download/1.5.29/ibus-1.5.29.tar.gz
+https://github.com/ibus/ibus/archive/refs/tags/1.5.29/ibus-1.5.29.tar.gz
 https://www.unicode.org/Public/zipped/15.1.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-15.tar.xz
 https://ftp.debian.org/debian/pool/main/i/iso-codes/iso-codes_4.15.0.orig.tar.xz
@@ -275,9 +275,9 @@ https://doxygen.nl/files/doxygen-1.9.8.src.tar.gz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.4/gc-8.2.4.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-13.2.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-2.42.1.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-manpages-2.42.1.tar.xz
-https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.42.1.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-2.43.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-manpages-2.43.0.tar.xz
+https://www.kernel.org/pub/software/scm/git/git-htmldocs-2.43.0.tar.xz
 https://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.xz
 https://github.com/llvm/llvm-project/releases/download/llvmorg-17.0.1/llvm-17.0.1.src.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/llvm/llvm-cmake-17.src.tar.xz
@@ -600,7 +600,6 @@ https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-security_fix-1.
 https://ftp.exim.org/pub/exim/exim4/exim-4.97.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.8.3.tar.gz
 https://ftp.sendmail.org/sendmail.8.17.2.tar.gz
-https://anduin.linuxfromscratch.org/BLFS/bdb/db-5.3.28.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-10.11.5/source/mariadb-10.11.5.tar.gz
 https://ftp.postgresql.org/pub/source/v16.1/postgresql-16.1.tar.bz2
@@ -671,7 +670,7 @@ https://gitlab.freedesktop.org/libinput/libei/-/archive/1.1.0/libei-1.1.0.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
 https://gedit-technology.net/tarballs/libgedit-gtksourceview/libgedit-gtksourceview-299.0.4.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.2.tar.xz
-https://dri.freedesktop.org/libdrm/libdrm-2.4.117.tar.xz
+https://dri.freedesktop.org/libdrm/libdrm-2.4.118.tar.xz
 https://download.gnome.org/sources/libepoxy/1.5/libepoxy-1.5.10.tar.xz
 https://download.gnome.org/sources/libglade/2.6/libglade-2.6.4.tar.bz2
 https://download.gnome.org/sources/libnotify/0.8/libnotify-0.8.3.tar.xz
@@ -879,7 +878,7 @@ https://inkscape.org/gallery/item/44467/inkscape-1.3.1.tar.xz
 https://downloads.sourceforge.net/pidgin/pidgin-2.14.12.tar.bz2
 https://downloads.sourceforge.net/rox/rox-filer-2.11.tar.bz2
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/115.4.3/source/thunderbird-115.4.3.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/115.5.0/source/thunderbird-115.5.0.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.13.1/tigervnc-1.13.1.tar.gz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.13.1-configuration_fixes-1.patch
@@ -915,7 +914,7 @@ https://github.com/intel/media-driver/archive/refs/tags/intel-media-23.4.0.tar.g
 https://github.com/intel/intel-vaapi-driver/releases/download/2.4.1/intel-vaapi-driver-2.4.1.tar.bz2
 https://liba52.sourceforge.net/files/a52dec-0.7.4.tar.gz
 https://downloads.xiph.org/releases/ao/libao-1.2.0.tar.gz
-https://storage.googleapis.com/aom-releases/libaom-3.7.0.tar.gz
+https://storage.googleapis.com/aom-releases/libaom-3.7.1.tar.gz
 https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz
 https://0pointer.de/lennart/projects/libcanberra/libcanberra-0.30.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libcanberra-0.30-wayland-1.patch
