diff --git a/wget-list b/wget-list
index f796d393c..4dc14c002 100644
--- a/wget-list
+++ b/wget-list
@@ -25,7 +25,7 @@ https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
 https://github.com/shadow-maint/shadow/releases/download/4.14.2/shadow-4.14.2.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.71.tar.gz
-https://www.sudo.ws/dist/sudo-1.9.15.tar.gz
+https://www.sudo.ws/dist/sudo-1.9.15p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.5.3.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
@@ -170,7 +170,7 @@ https://download.gimp.org/pub/gegl/0.4/gegl-0.4.46.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.1.tar.gz
 https://github.com/g-truc/glm/archive/0.9.9.8/glm-0.9.9.8.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
-https://github.com/harfbuzz/harfbuzz/releases/download/8.2.2/harfbuzz-8.2.2.tar.xz
+https://github.com/harfbuzz/harfbuzz/releases/download/8.3.0/harfbuzz-8.3.0.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.1.0/jasper-4.1.0.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.14/lcms2-2.14.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.0.1/libavif-1.0.1.tar.gz
@@ -685,7 +685,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.16-build_fixe
 https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.16-ffmpeg5_fixes-1.patch
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
-https://webkitgtk.org/releases/webkitgtk-2.42.1.tar.xz
+https://webkitgtk.org/releases/webkitgtk-2.42.2.tar.xz
 https://download.gnome.org/sources/gdm/45/gdm-45.0.1.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.8/lightdm-gtk-greeter-2.0.8.tar.gz
