diff --git a/wget-list b/wget-list
index f5c311eca..22fdf3757 100644
--- a/wget-list
+++ b/wget-list
@@ -27,7 +27,7 @@ https://github.com/shadow-maint/shadow/releases/download/4.14.2/shadow-4.14.2.ta
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.71.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.15p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
-https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.6.tar.xz
+https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.6.1.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
 https://github.com/libfuse/libfuse/releases/download/fuse-3.16.2/fuse-3.16.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
@@ -521,7 +521,7 @@ https://archive.apache.org/dist/subversion/subversion-1.14.2.tar.bz2
 https://downloads.sourceforge.net/swig/swig-4.1.1.tar.gz
 https://downloads.sourceforge.net/tcl/tk8.6.13-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
-https://download.gnome.org/sources/vala/0.56/vala-0.56.13.tar.xz
+https://download.gnome.org/sources/vala/0.56/vala-0.56.14.tar.xz
 https://sourceware.org/pub/valgrind/valgrind-3.22.0.tar.bz2
 https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-21.0.1/OpenJDK-21.0.1+12-i686-bin.tar.xz
@@ -556,7 +556,7 @@ https://downloads.sourceforge.net/traceroute/traceroute-2.1.3.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.0.10.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/wireshark-4.0.10-py_3.12_fix-1.patch
-https://c-ares.haxx.se/download/c-ares-1.21.0.tar.gz
+https://c-ares.haxx.se/download/c-ares-1.22.0.tar.gz
 https://curl.se/download/curl-8.4.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.78/glib-networking-2.78.0.tar.xz
@@ -900,13 +900,13 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.2.tar.gz
 https://downloads.xiph.org/releases/flac/flac-1.4.3.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.6.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.22.6.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.22.6.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.22.6.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.22.6.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.22.6.tar.xz
-https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.22.6.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.7.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.22.7.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.22.7.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.22.7.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.22.7.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.22.7.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.22.7.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/refs/tags/intel-media-23.4.0.tar.gz
