diff --git a/wget-list b/wget-list
index c7b1545f..9b156f62 100644
--- a/wget-list
+++ b/wget-list
@@ -890,7 +890,7 @@ https://download.gimp.org/pub/gimp/help/gimp-help-2.10.34.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.5.0.tar.gz
 https://github.com/hexchat/hexchat/releases/download/v2.16.2/hexchat-2.16.2.tar.xz
 https://inkscape.org/gallery/item/44615/inkscape-1.3.2.tar.xz
-https://downloads.sourceforge.net/pidgin/pidgin-2.14.12.tar.bz2
+https://downloads.sourceforge.net/pidgin/pidgin-2.14.13.tar.bz2
 https://downloads.sourceforge.net/rox/rox-filer-2.11.tar.bz2
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
 https://archive.mozilla.org/pub/thunderbird/releases/115.8.0/source/thunderbird-115.8.0.source.tar.xz
