diff --git a/wget-list b/wget-list
index 9b156f62..888859a2 100644
--- a/wget-list
+++ b/wget-list
@@ -701,7 +701,7 @@ https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-deskt
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
 https://download.gnome.org/sources/gdm/45/gdm-45.0.1.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
-https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.8/lightdm-gtk-greeter-2.0.8.tar.gz
+https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.20.0/sddm-0.20.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/3.4.5/icewm-3.4.5.tar.gz
@@ -734,6 +734,7 @@ https://download.kde.org/stable/release-service/23.08.5/src/libkexiv2-23.08.5.ta
 https://download.kde.org/stable/release-service/23.08.5/src/okular-23.08.5.tar.xz
 https://download.kde.org/stable/release-service/23.08.5/src/libkdcraw-23.08.5.tar.xz
 https://download.kde.org/stable/release-service/23.08.5/src/gwenview-23.08.5.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/gwenview-23.08.5-build_fixes-1.patch
 https://download.kde.org/stable/release-service/23.08.5/src/libkcddb-23.08.5.tar.xz
 https://download.kde.org/stable/release-service/23.08.5/src/k3b-23.08.5.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
@@ -924,7 +925,7 @@ https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/refs/tags/intel-media-23.4.3.tar.gz
 https://github.com/intel/intel-vaapi-driver/releases/download/2.4.1/intel-vaapi-driver-2.4.1.tar.bz2
-https://liba52.sourceforge.net/files/a52dec-0.7.4.tar.gz
+http://ftp.osuosl.org/pub/blfs/conglomeration/a52dec/a52dec-0.7.4.tar.gz
 https://downloads.xiph.org/releases/ao/libao-1.2.0.tar.gz
 https://storage.googleapis.com/aom-releases/libaom-3.8.1.tar.gz
 https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz
@@ -951,7 +952,7 @@ https://github.com/libsndfile/libsamplerate/releases/download/0.2.2/libsamplerat
 https://github.com/libsndfile/libsndfile/releases/download/1.2.2/libsndfile-1.2.2.tar.xz
 https://github.com/intel/libva/releases/download/2.20.0/libva-2.20.0.tar.bz2
 https://gitlab.freedesktop.org/vdpau/libvdpau/-/archive/1.5/libvdpau-1.5.tar.bz2
-https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.0/libvdpau-va-gl-0.4.0.tar.gz
+https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.2/libvdpau-va-gl-0.4.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.14.0/libvpx-1.14.0.tar.gz
 https://github.com/mltframework/mlt/releases/download/v7.22.0/mlt-7.22.0.tar.gz
