diff --git a/wget-list b/wget-list
index 949e08e3..1043a7e6 100644
--- a/wget-list
+++ b/wget-list
@@ -244,7 +244,7 @@ http://fcron.free.fr/archives/fcron-3.2.1.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
 https://downloads.sourceforge.net/hdparm/hdparm-9.65.tar.gz
-https://github.com/lfs-book/LSB-Tools/releases/download/v0.10/LSB-Tools-0.10.tar.gz
+https://github.com/lfs-book/LSB-Tools/releases/download/v0.12/LSB-Tools-0.12.tar.gz
 https://github.com/lm-sensors/lm-sensors/archive/V3-6-0/lm-sensors-3-6-0.tar.gz
 https://github.com/logrotate/logrotate/releases/download/3.21.0/logrotate-3.21.0.tar.xz
 http://ftp.midnight-commander.org/mc-4.8.31.tar.xz
