diff --git a/wget-list b/wget-list
index 7bfd3df5..6c746db4 100644
--- a/wget-list
+++ b/wget-list
@@ -182,7 +182,7 @@ https://download.gnome.org/sources/libgxps/0.3/libgxps-0.3.2.tar.xz
 https://downloads.sourceforge.net/libjpeg-turbo/libjpeg-turbo-3.0.1.tar.gz
 https://downloads.sourceforge.net/libmng/libmng-2.0.3.tar.xz
 https://github.com/mypaint/libmypaint/releases/download/v1.6.1/libmypaint-1.6.1.tar.xz
-https://downloads.sourceforge.net/libpng/libpng-1.6.41.tar.xz
+https://downloads.sourceforge.net/libpng/libpng-1.6.42.tar.xz
 https://downloads.sourceforge.net/sourceforge/libpng-apng/libpng-1.6.40-apng.patch.gz
 https://www.libraw.org/data/LibRaw-0.21.2.tar.gz
 https://download.gnome.org/sources/librsvg/2.57/librsvg-2.57.1.tar.xz
@@ -462,7 +462,7 @@ https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.g
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi-docgen-2023.3.tar.gz
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-4.9.4.tar.gz
-https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.0.tar.gz
+https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.2.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.3.tar.gz
 https://files.pythonhosted.org/packages/source/p/packaging/packaging-23.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
@@ -607,8 +607,8 @@ https://ftp.sendmail.org/sendmail.8.17.2.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-10.11.6/source/mariadb-10.11.6.tar.gz
 https://ftp.postgresql.org/pub/source/v16.1/postgresql-16.1.tar.bz2
-https://sqlite.org/2024/sqlite-autoconf-3450000.tar.gz
-https://sqlite.org/2024/sqlite-doc-3450000.zip
+https://sqlite.org/2024/sqlite-autoconf-3450100.tar.gz
+https://sqlite.org/2024/sqlite-doc-3450100.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.6.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.6-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.19.0.tar.gz
@@ -896,11 +896,10 @@ https://github.com/transmission/transmission/releases/download/4.0.5/transmissio
 https://github.com/ib/xarchiver/archive/0.5.4.22/xarchiver-0.5.4.22.tar.gz
 https://portland.freedesktop.org/download/xdg-utils-1.1.3.tar.gz
 https://www.jwz.org/xscreensaver/xscreensaver-6.08.tar.gz
-https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.10.tar.bz2
-https://www.linuxfromscratch.org/patches/blfs/svn/alsa-lib-1.2.10-upstream_fixes-1.patch
+https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.11.tar.bz2
 https://www.alsa-project.org/files/pub/plugins/alsa-plugins-1.2.7.1.tar.bz2
-https://www.alsa-project.org/files/pub/utils/alsa-utils-1.2.10.tar.bz2
-https://www.alsa-project.org/files/pub/tools/alsa-tools-1.2.5.tar.bz2
+https://www.alsa-project.org/files/pub/utils/alsa-utils-1.2.11.tar.bz2
+https://www.alsa-project.org/files/pub/tools/alsa-tools-1.2.11.tar.bz2
 https://www.alsa-project.org/files/pub/firmware/alsa-firmware-1.2.4.tar.bz2
 https://download.gnome.org/sources/audiofile/0.3/audiofile-0.3.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/audiofile-0.3.6-consolidated_patches-1.patch
