diff --git a/wget-list b/wget-list
index 26a6000f..11437ce3 100644
--- a/wget-list
+++ b/wget-list
@@ -39,7 +39,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/gptfdisk-1.0.9-convenience-1.p
 https://ftp.gnu.org/gnu/parted/parted-3.6.tar.xz
 https://downloads.sourceforge.net/smartmontools/smartmontools-7.4.tar.gz
 https://github.com/libfuse/sshfs/releases/download/sshfs-3.7.3/sshfs-3.7.3.tar.xz
-https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.5.0.tar.xz
+https://www.kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-6.6.0.tar.xz
 https://github.com/rhboot/efivar/releases/download/38/efivar-38.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/efivar-38-i686-1.patch
 https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
@@ -325,7 +325,7 @@ https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.76.tar.gz
 https://www.cpan.org/authors/id/D/DR/DROLSKY/List-AllUtils-0.19.tar.gz
 https://www.cpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz
 https://www.cpan.org/authors/id/E/ET/ETJ/Log-Log4perl-1.57.tar.gz
-https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.12.tar.gz
+https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.13.tar.gz
 https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4232.tar.gz
 https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.43.tar.gz
 https://www.cpan.org/authors/id/J/JT/JTBRAUN/Parse-RecDescent-1.967015.tar.gz
@@ -463,7 +463,7 @@ https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi-docgen-2023.3.tar.gz
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-4.9.4.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.2.tar.gz
-https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.3.tar.gz
+https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.4.tar.gz
 https://files.pythonhosted.org/packages/source/p/packaging/packaging-23.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
 https://download.gnome.org/sources/pyatspi/2.46/pyatspi-2.46.1.tar.xz
@@ -498,13 +498,13 @@ https://files.pythonhosted.org/packages/source/i/idna/idna-3.4.tar.gz
 https://files.pythonhosted.org/packages/source/i/imagesize/imagesize-1.4.1.tar.gz
 https://files.pythonhosted.org/packages/source/i/iniconfig/iniconfig-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/M/Markdown/Markdown-3.4.1.tar.gz
-https://files.pythonhosted.org/packages/source/m/meson_python/meson_python-0.12.0.tar.gz
+https://files.pythonhosted.org/packages/source/m/meson_python/meson_python-0.15.0.tar.gz
 https://files.pythonhosted.org/packages/source/m/msgpack/msgpack-1.0.4.tar.gz
 https://files.pythonhosted.org/packages/source/p/pathspec/pathspec-0.10.3.tar.gz
 https://files.pythonhosted.org/packages/source/p/pluggy/pluggy-1.4.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/py/py-1.11.0.tar.gz
-https://files.pythonhosted.org/packages/source/p/pyproject-metadata/pyproject-metadata-0.7.0.tar.gz
-https://files.pythonhosted.org/packages/source/p/pytz/pytz-2022.7.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pyproject-metadata/pyproject-metadata-0.7.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pytz/pytz-2024.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/setuptools_scm/setuptools_scm-7.1.0.tar.gz
 https://github.com/leohemsted/smartypants.py/archive/v2.0.1/smartypants-2.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/snowballstemmer/snowballstemmer-2.2.0.tar.gz
@@ -671,7 +671,7 @@ https://github.com/ksnip/kColorPicker/archive/refs/tags/v0.3.0/kColorPicker-0.3.
 https://github.com/ksnip/kImageAnnotator/archive/refs/tags/v0.7.0/kImageAnnotator-0.7.0.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
 https://download.gnome.org/sources/libadwaita/1.4/libadwaita-1.4.2.tar.xz
-https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.0/libei-1.2.0.tar.gz
+https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.1/libei-1.2.1.tar.gz
 https://gedit-technology.net/tarballs/libgedit-amtk/libgedit-amtk-5.8.0.tar.xz
 https://gedit-technology.net/tarballs/libgedit-gtksourceview/libgedit-gtksourceview-299.0.4.tar.xz
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.2.tar.xz
