diff --git a/wget-list b/wget-list
index 9fc98711..5ff7e6ae 100644
--- a/wget-list
+++ b/wget-list
@@ -26,7 +26,7 @@ https://github.com/shadow-maint/shadow/releases/download/4.14.5/shadow-4.14.5.ta
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.72.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.15p5.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
-https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.7.tar.xz
+https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.7.1.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
 https://github.com/libfuse/libfuse/releases/download/fuse-3.16.2/fuse-3.16.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
@@ -46,11 +46,11 @@ https://github.com/rhboot/efibootmgr/archive/18/efibootmgr-18.tar.gz
 https://ftp.gnu.org/gnu/grub/grub-2.12.tar.xz
 https://unifoundry.com/pub/unifont/unifont-15.1.04/font-builds/unifont-15.1.04.pcf.gz
 https://www.bennewitz.com/bluefish/stable/source/bluefish-2.2.14.tar.bz2
-https://ftp.gnu.org/gnu/ed/ed-1.20.tar.lz
+https://ftp.gnu.org/gnu/ed/ed-1.20.1.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-29.2.tar.xz
 https://download.gnome.org/sources/gedit/46/gedit-46.1.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/23.08.4/src/kate-23.08.4.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/kate-23.08.5.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v7/nano-7.2.tar.xz
 https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
@@ -444,7 +444,7 @@ https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/XML-LibXML-2.0208.tar.gz
 https://cpan.metacpan.org/authors/id/P/PE/PERIGRIN/XML-NamespaceSupport-1.12.tar.gz
 https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-1.02.tar.gz
 https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
-https://www.php.net/distributions/php-8.3.2.tar.xz
+https://www.php.net/distributions/php-8.3.3.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
 https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
@@ -563,7 +563,7 @@ https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-a
 https://nmap.org/dist/nmap-7.94.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.5.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
-https://www.wireshark.org/download/src/all-versions/wireshark-4.2.2.tar.xz
+https://www.wireshark.org/download/src/all-versions/wireshark-4.2.3.tar.xz
 https://c-ares.haxx.se/download/c-ares-1.26.0.tar.gz
 https://curl.se/download/curl-8.6.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
@@ -692,7 +692,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-opensource-src-5
 https://anduin.linuxfromscratch.org/BLFS/qtwebengine/qtwebengine-5.15.17.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-build_fixes-2.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-ffmpeg5_fixes-1.patch
-https://download.qt.io/archive/qt/6.6/6.6.1/single/qt-everywhere-src-6.6.1.tar.xz
+https://download.qt.io/archive/qt/6.6/6.6.2/single/qt-everywhere-src-6.6.2.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.277/Vulkan-Headers-1.3.277.tar.gz
@@ -708,7 +708,7 @@ https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/3.4.5/icewm-3.4.5.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://download.gnome.org/sources/adwaita-icon-theme/45/adwaita-icon-theme-45.0.tar.xz
-https://download.kde.org/stable/frameworks/5.113/breeze-icons-5.113.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/breeze-icons-5.115.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-symbolic/3.12/gnome-icon-theme-symbolic-3.12.0.tar.xz
@@ -716,27 +716,27 @@ https://download.gnome.org/sources/gnome-themes-extra/3.28/gnome-themes-extra-3.
 https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.17.tar.xz
 http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
-https://download.kde.org/stable/frameworks/5.113/oxygen-icons-5.113.0.tar.xz
-https://download.kde.org/stable/frameworks/5.113/extra-cmake-modules-5.113.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/oxygen-icons-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/extra-cmake-modules-5.115.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.114.0.tar.xz
 https://launchpad.net/ubuntu/+archive/primary/+files/libdbusmenu-qt_0.9.3+16.04.20160218.orig.tar.gz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.12.0.tar.xz
 https://download.kde.org/stable/kuserfeedback/kuserfeedback-1.3.0.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/ark-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/kdenlive-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/kmix-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/kio-extras-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/khelpcenter-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/konsole-23.08.4.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/konsole-23.08.4-scrollbar-1.patch
-https://download.kde.org/stable/release-service/23.08.4/src/libkexiv2-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/okular-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/libkdcraw-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/gwenview-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/libkcddb-23.08.4.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/k3b-23.08.4.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/ark-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/kdenlive-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/kmix-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/kio-extras-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/khelpcenter-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/konsole-23.08.5.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/konsole-23.08.5-scrollbar-1.patch
+https://download.kde.org/stable/release-service/23.08.5/src/libkexiv2-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/okular-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/libkdcraw-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/gwenview-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/libkcddb-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/k3b-23.08.5.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.2/gcr-4.2.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/45/gsettings-desktop-schemas-45.0.tar.xz
@@ -829,11 +829,11 @@ https://archive.xfce.org/src/apps/ristretto/0.13/ristretto-0.13.2.tar.bz2
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.18/xfce4-dev-tools-4.18.1.tar.bz2
 https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.4.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.4/xfce4-pulseaudio-plugin-0.4.8.tar.bz2
-https://download.kde.org/stable/frameworks/5.113/kwindowsystem-5.113.0.tar.xz
-https://download.kde.org/stable/frameworks/5.113/kwayland-5.113.0.tar.xz
-https://download.kde.org/stable/frameworks/5.113/kconfig-5.113.0.tar.xz
-https://download.kde.org/stable/frameworks/5.113/solid-5.113.0.tar.xz
-https://download.kde.org/stable/frameworks/5.113/kidletime-5.113.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/kwindowsystem-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/kwayland-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/kconfig-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/solid-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/5.115/kidletime-5.115.0.tar.xz
 http://download.kde.org/stable/plasma/5.27.10/libkscreen-5.27.10.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.4/muparser-2.3.4.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/0.13.0/lxqt-build-tools-0.13.0.tar.xz
@@ -879,7 +879,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-help-
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-translations-24.2.0.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.2.0.3-consolidated_fixes-1.patch
 https://download.gnome.org/sources/epiphany/45/epiphany-45.2.tar.xz
-https://download.kde.org/stable/release-service/23.08.4/src/falkon-23.08.4.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/falkon-23.08.5.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.18/source/seamonkey-2.53.18.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
@@ -976,7 +976,7 @@ https://distfiles.audacious-media-player.org/audacious-4.3.1.tar.bz2
 https://distfiles.audacious-media-player.org/audacious-plugins-4.3.1.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
-https://download.kde.org/stable/release-service/23.08.4/src/kwave-23.08.4.tar.xz
+https://download.kde.org/stable/release-service/23.08.5/src/kwave-23.08.5.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
 https://downloads.sourceforge.net/mpg123/mpg123-1.32.4.tar.bz2
 https://freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-5.0.tar.xz
@@ -992,6 +992,7 @@ https://mplayerhq.hu/MPlayer/skins/Clearlooks-2.0.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/transcode/transcode-1.1.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/transcode-1.1.7-gcc10_fix-1.patch
 https://download.videolan.org/vlc/3.0.20/vlc-3.0.20.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.20-taglib-1.patch
 https://downloads.sourceforge.net/xine/xine-ui-0.99.14.tar.xz
 https://downloads.sourceforge.net/cdrdao/cdrdao-1.2.4.tar.bz2
 https://downloads.sourceforge.net/cdrtools/cdrtools-3.02a09.tar.bz2
