diff --git a/wget-list b/wget-list
index 1043a7e6..a1d093d6 100644
--- a/wget-list
+++ b/wget-list
@@ -1,4 +1,4 @@
-https://www.linuxfromscratch.org/blfs/downloads/systemd/blfs-systemd-units-20240128.tar.xz
+https://www.linuxfromscratch.org/blfs/downloads/systemd/blfs-systemd-units-20240205.tar.xz
 https://github.com/lfs-book/make-ca/releases/download/v1.13/make-ca-1.13.tar.xz
 https://github.com/cracklib/cracklib/releases/download/v2.9.11/cracklib-2.9.11.tar.xz
 https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.0.tar.xz
@@ -70,7 +70,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/clucene-2.3.3.4-contribs_lib-1
 https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.112.tar.gz
 https://github.com/google/double-conversion/archive/v3.3.0/double-conversion-3.3.0.tar.gz
 https://duktape.org/duktape-2.7.0.tar.xz
-https://github.com/AbiWord/enchant/releases/download/v2.6.5/enchant-2.6.5.tar.gz
+https://github.com/AbiWord/enchant/releases/download/v2.6.6/enchant-2.6.6.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.5.tar.xz
 https://www.fftw.org/fftw-3.3.10.tar.gz
 https://github.com/fmtlib/fmt/archive/refs/tags/10.2.1/fmt-10.2.1.tar.gz
@@ -131,7 +131,7 @@ https://lttng.org/files/urcu/userspace-rcu-0.14.0.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
 https://dist.libuv.org/dist/v1.47.0/libuv-v1.47.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.6.0.tar.xz
-https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.4.tar.xz
+https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.5.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.15/libxmlb-0.3.15.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.39.tar.xz
@@ -151,7 +151,7 @@ https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
 https://archive.mozilla.org/pub/firefox/releases/115.7.0esr/source/firefox-115.7.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Headers-1.3.275.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Tools-1.3.275.0.tar.gz
-https://www.samba.org/ftp/talloc/talloc-2.4.1.tar.gz
+https://www.samba.org/ftp/talloc/talloc-2.4.2.tar.gz
 https://telepathy.freedesktop.org/releases/telepathy-glib/telepathy-glib-0.24.2.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.17.18/umockdev-0.17.18.tar.xz
@@ -170,7 +170,7 @@ https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.15.0.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.13/fribidi-1.0.13.tar.xz
 https://download.gimp.org/pub/gegl/0.4/gegl-0.4.46.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.1.tar.gz
-https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.4/glad-2.0.4.tar.gz
+https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.5/glad-2.0.5.tar.gz
 https://github.com/g-truc/glm/archive/1.0.0/glm-1.0.0.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/8.3.0/harfbuzz-8.3.0.tar.xz
@@ -273,10 +273,10 @@ https://downloads.sourceforge.net/infozip/zip30.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.26.0/cbindgen-0.26.0.tar.gz
 https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
-https://cmake.org/files/v3.28/cmake-3.28.1.tar.gz
+https://cmake.org/files/v3.28/cmake-3.28.2.tar.gz
 https://doxygen.nl/files/doxygen-1.10.0.src.tar.gz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
-https://github.com/ivmai/bdwgc/releases/download/v8.2.4/gc-8.2.4.tar.gz
+https://github.com/ivmai/bdwgc/releases/download/v8.2.6/gc-8.2.6.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-14.1.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-2.43.0.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-manpages-2.43.0.tar.xz
@@ -579,7 +579,7 @@ https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.7.0/libslirp-v4.7.0.t
 https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz
 https://download.gnome.org/sources/libsoup/3.4/libsoup-3.4.4.tar.xz
 https://downloads.sourceforge.net/libtirpc/libtirpc-1.3.4.tar.bz2
-https://notroj.github.io/neon/neon-0.32.5.tar.gz
+https://notroj.github.io/neon/neon-0.33.0.tar.gz
 https://github.com/nghttp2/nghttp2/releases/download/v1.59.0/nghttp2-1.59.0.tar.xz
 https://github.com/thkukuk/rpcsvc-proto/releases/download/v1.4.4/rpcsvc-proto-1.4.4.tar.xz
 https://archive.apache.org/dist/serf/serf-1.3.10.tar.bz2
@@ -609,8 +609,8 @@ https://downloads.mariadb.org/interstitial/mariadb-10.11.6/source/mariadb-10.11.
 https://ftp.postgresql.org/pub/source/v16.1/postgresql-16.1.tar.bz2
 https://sqlite.org/2024/sqlite-autoconf-3450100.tar.gz
 https://sqlite.org/2024/sqlite-doc-3450100.zip
-https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.6.tgz
-https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.6-consolidated-1.patch
+https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.7.tgz
+https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.7-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.19.0.tar.gz
 https://www.x.org/pub/individual/util/util-macros-1.20.0.tar.xz
 https://xorg.freedesktop.org/archive/individual/proto/xorgproto-2023.2.tar.xz
@@ -629,7 +629,7 @@ https://mesa.freedesktop.org/archive/mesa-23.3.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-2.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
-https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.40.tar.xz
+https://www.x.org/pub/individual/data/xkeyboard-config/xkeyboard-config-2.41.tar.xz
 https://www.x.org/pub/individual/xserver/xwayland-23.2.4.tar.xz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.11.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.11-tearfree_backport-1.patch
@@ -692,7 +692,7 @@ https://www.freedesktop.org/software/startup-notification/releases/startup-notif
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.277/Vulkan-Headers-1.3.277.tar.gz
 https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.277/Vulkan-Loader-1.3.277.tar.gz
-https://webkitgtk.org/releases/webkitgtk-2.42.4.tar.xz
+https://webkitgtk.org/releases/webkitgtk-2.42.5.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-desktop-portal-1.18.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
 https://download.gnome.org/sources/gdm/45/gdm-45.0.1.tar.xz
