diff --git a/wget-list b/wget-list
index a61b7a9b..cbe307b0 100644
--- a/wget-list
+++ b/wget-list
@@ -129,7 +129,7 @@ https://ftp.gnu.org/gnu/libunistring/libunistring-1.1.tar.xz
 https://download.savannah.nongnu.org/releases/libunwind/libunwind-1.6.2.tar.gz
 https://lttng.org/files/urcu/userspace-rcu-0.14.0.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
-https://dist.libuv.org/dist/v1.47.0/libuv-v1.47.0.tar.gz
+https://dist.libuv.org/dist/v1.48.0/libuv-v1.48.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.6.0.tar.xz
 https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.5.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
@@ -162,13 +162,13 @@ https://wpewebkit.org/releases/wpebackend-fdo-1.14.2.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/1.4.24/xapian-core-1.4.24.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
-https://download.gimp.org/pub/babl/0.1/babl-0.1.106.tar.xz
+https://download.gimp.org/pub/babl/0.1/babl-0.1.108.tar.xz
 https://github.com/Exiv2/exiv2/archive/v0.28.1/exiv2-0.28.1.tar.gz
 https://downloads.sourceforge.net/freetype/freetype-2.13.2.tar.xz
 https://downloads.sourceforge.net/freetype/freetype-doc-2.13.2.tar.xz
 https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.15.0.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.13/fribidi-1.0.13.tar.xz
-https://download.gimp.org/pub/gegl/0.4/gegl-0.4.46.tar.xz
+https://download.gimp.org/pub/gegl/0.4/gegl-0.4.48.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.1.tar.gz
 https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.5/glad-2.0.5.tar.gz
 https://github.com/g-truc/glm/archive/1.0.0/glm-1.0.0.tar.gz
@@ -209,7 +209,7 @@ https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.27.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
-https://gitlab.com/graphviz/graphviz/-/archive/9.0.0/graphviz-9.0.0.tar.bz2
+https://gitlab.com/graphviz/graphviz/-/archive/10.0.1/graphviz-10.0.1.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.33/gtk-doc-1.33.2.tar.xz
 http://www.andre-simon.de/zip/highlight-4.10.tar.bz2
 https://github.com/ibus/ibus/archive/refs/tags/1.5.29/ibus-1.5.29.tar.gz
@@ -521,8 +521,8 @@ https://files.pythonhosted.org/packages/source/t/typing_extensions/typing_extens
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.0.7.tar.gz
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.0.2.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.0.tar.xz
-https://static.rust-lang.org/dist/rustc-1.75.0-src.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/rustc-1.75.0-testsuite_fix-1.patch
+https://static.rust-lang.org/dist/rustc-1.76.0-src.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/rustc-1.76.0-testsuite_fix-1.patch
 https://downloads.sourceforge.net/scons/SCons-4.6.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.3.tar.bz2
@@ -555,7 +555,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.10.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.18.21/bind-9.18.21.tar.xz
+https://ftp.isc.org/isc/bind9/9.18.24/bind-9.18.24.tar.xz
 https://download.gnome.org/sources/NetworkManager/1.44/NetworkManager-1.44.2.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.94.tar.bz2
@@ -613,7 +613,7 @@ https://sqlite.org/2024/sqlite-autoconf-3450100.tar.gz
 https://sqlite.org/2024/sqlite-doc-3450100.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.7.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.7-consolidated-1.patch
-https://nlnetlabs.nl/downloads/unbound/unbound-1.19.0.tar.gz
+https://nlnetlabs.nl/downloads/unbound/unbound-1.19.1.tar.gz
 https://www.x.org/pub/individual/util/util-macros-1.20.0.tar.xz
 https://xorg.freedesktop.org/archive/individual/proto/xorgproto-2023.2.tar.xz
 https://www.x.org/pub/individual/lib/libXau-1.0.11.tar.xz
@@ -656,7 +656,7 @@ https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.10.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.2.0.tgz
 https://github.com/KhronosGroup/glslang/archive/refs/tags/14.0.0/glslang-14.0.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
-https://download.gnome.org/sources/goffice/0.10/goffice-0.10.56.tar.xz
+https://download.gnome.org/sources/goffice/0.10/goffice-0.10.57.tar.xz
 https://github.com/steveire/grantlee/releases/download/v5.3.1/grantlee-5.3.1.tar.gz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.33.tar.xz
@@ -869,7 +869,7 @@ https://github.com/lxqt/qterminal/releases/download/1.4.0/qterminal-1.4.0.tar.xz
 https://github.com/lxqt/screengrab/releases/download/2.7.0/screengrab-2.7.0.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-3.0.5.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-docs-3.0.2.tar.gz
-https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.56.tar.xz
+https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.57.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-24.2.0.3.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-dictionaries-24.2.0.3.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-help-24.2.0.3.tar.xz
@@ -885,7 +885,7 @@ https://github.com/fontforge/fontforge/releases/download/20230101/fontforge-2023
 https://download.gimp.org/pub/gimp/v2.10/gimp-2.10.36.tar.bz2
 https://download.gimp.org/pub/gimp/help/gimp-help-2.10.34.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.5.0.tar.gz
-https:///github.com/hexchat/hexchat/releases/download/v2.16.2/hexchat-2.16.2.tar.xz
+https://github.com/hexchat/hexchat/releases/download/v2.16.2/hexchat-2.16.2.tar.xz
 https://inkscape.org/gallery/item/44615/inkscape-1.3.2.tar.xz
 https://downloads.sourceforge.net/pidgin/pidgin-2.14.12.tar.bz2
 https://downloads.sourceforge.net/rox/rox-filer-2.11.tar.bz2
