diff --git a/wget-list b/wget-list
index 0967794612..df9bce3fe8 100644
--- a/wget-list
+++ b/wget-list
@@ -624,7 +624,7 @@ https://downloads.mariadb.org/interstitial/mariadb-11.8.3/source/mariadb-11.8.3.
 https://ftp.postgresql.org/pub/source/v17.6/postgresql-17.6.tar.bz2
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.10.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.10-consolidated-1.patch
-https://nlnetlabs.nl/downloads/unbound/unbound-1.23.1.tar.gz
+https://nlnetlabs.nl/downloads/unbound/unbound-1.24.0.tar.gz
 https://www.x.org/pub/individual/util/util-macros-1.20.2.tar.xz
 https://xorg.freedesktop.org/archive/individual/proto/xorgproto-2024.1.tar.xz
 https://www.x.org/pub/individual/lib/libXau-1.0.12.tar.xz
