diff --git a/wget-list b/wget-list
index 6f05f36f7b..26f8a78afd 100644
--- a/wget-list
+++ b/wget-list
@@ -28,7 +28,7 @@ https://github.com/polkit-org/polkit/archive/127/polkit-127.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
 https://github.com/shadow-maint/shadow/releases/download/4.19.4/shadow-4.19.4.tar.xz
-https://www.stunnel.org/downloads/archive/5.x/stunnel-5.78.tar.gz
+https://www.stunnel.org/downloads/archive/5.x/stunnel-5.79.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.17p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v7.0.tar.xz
@@ -55,7 +55,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/26.04.1/src/kate-26.04.1.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.7/mousepad-0.7.0.tar.xz
 https://www.nano-editor.org/dist/v9/nano-9.1.tar.xz
-https://github.com/vim/vim/archive/v9.2.0640/vim-9.2.0640.tar.gz
+https://github.com/vim/vim/archive/v9.2.0752/vim-9.2.0752.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.4.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.1.tar.xz
@@ -82,8 +82,8 @@ https://github.com/fmtlib/fmt/archive/12.2.0/fmt-12.2.0.tar.gz
 https://download.gnome.org/sources/glib/2.88/glib-2.88.2.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.86/gobject-introspection-1.86.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
-https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
-https://download.gnome.org/sources/glibmm/2.88/glibmm-2.88.0.tar.xz
+https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.9.tar.xz
+https://download.gnome.org/sources/glibmm/2.88/glibmm-2.88.1.tar.xz
 https://github.com/jstedfast/gmime/releases/download/3.2.15/gmime-3.2.15.tar.xz
 https://github.com/lfs-book/gmmlib/archive/v22.10.0/gmmlib-22.10.0.tar.gz
 https://ftpmirror.gnu.org/gsl/gsl-2.8.tar.gz
@@ -287,7 +287,7 @@ https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://github.com/kennylevinsen/seatd/archive/0.9.3/seatd-0.9.3.tar.gz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.9.tar.xz
-https://github.com/systemd/systemd/archive/v260.2/systemd-260.2.tar.gz
+https://github.com/systemd/systemd/archive/v261.1/systemd-261.1.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.11.1/udisks-2.11.1.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.2.4.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.91.2/upower-v1.91.2.tar.bz2
@@ -319,8 +319,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.8-shared_library-1.pat
 https://www.lua.org/tests/lua-5.4.8-tests.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/luajit/luajit-20260213.tar.xz
 https://www.mercurial-scm.org/release/mercurial-7.2.2.tar.gz
-https://www.nasm.us/pub/nasm/releasebuilds/3.01/nasm-3.01.tar.xz
-https://www.nasm.us/pub/nasm/releasebuilds/3.01/nasm-3.01-xdoc.tar.xz
+https://www.nasm.us/pub/nasm/releasebuilds/3.02/nasm-3.02.tar.xz
+https://www.nasm.us/pub/nasm/releasebuilds/3.02/nasm-3.02-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.19.0/patchelf-0.19.0.tar.gz
 https://www.cpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz
 https://www.cpan.org/authors/id/V/VP/VPIT/autovivification-0.18.tar.gz
@@ -706,7 +706,7 @@ https://downloads.sourceforge.net/enlightenment/imlib2-1.12.6.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.2/kImageAnnotator-0.7.2.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
-https://download.gnome.org/sources/libadwaita/1.9/libadwaita-1.9.1.tar.xz
+https://download.gnome.org/sources/libadwaita/1.9/libadwaita-1.9.2.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.6.0/libei-1.6.0.tar.bz2
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
 https://dri.freedesktop.org/libdrm/libdrm-2.4.134.tar.xz
@@ -784,7 +784,7 @@ https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.7.t
 https://gitlab.gnome.org/GNOME/vte/-/archive/0.84.0/vte-0.84.0.tar.gz
 https://download.gnome.org/sources/yelp-xsl/49/yelp-xsl-49.0.tar.xz
 https://download.gnome.org/sources/geocode-glib/3.26/geocode-glib-3.26.4.tar.xz
-https://download.gnome.org/sources/gjs/1.88/gjs-1.88.0.tar.xz
+https://download.gnome.org/sources/gjs/1.88/gjs-1.88.1.tar.xz
 https://download.gnome.org/sources/gnome-autoar/0.4/gnome-autoar-0.4.5.tar.xz
 https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.5.tar.xz
 https://download.gnome.org/sources/gnome-menus/3.38/gnome-menus-3.38.1.tar.xz
@@ -942,7 +942,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/thunderbird-140.12.0esr-llvm_2
 https://github.com/TigerVNC/tigervnc/archive/v1.16.2/tigervnc-1.16.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.16.2-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.1.2/transmission-4.1.2.tar.xz
-https://github.com/ib/xarchiver/archive/0.5.4.26/xarchiver-0.5.4.26.tar.gz
+https://github.com/ib/xarchiver/archive/0.5.4.27/xarchiver-0.5.4.27.tar.gz
 https://gitlab.freedesktop.org/xdg/xdg-utils/-/archive/v1.2.1/xdg-utils-v1.2.1.tar.gz
 https://www.jwz.org/xscreensaver/xscreensaver-6.15.tar.gz
 https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.16.1.tar.bz2
