diff --git a/wget-list b/wget-list
index ae8205824b..a307b653a1 100644
--- a/wget-list
+++ b/wget-list
@@ -4,7 +4,7 @@ https://github.com/cracklib/cracklib/releases/download/v2.10.3/cracklib-2.10.3.t
 https://www.kernel.org/pub/linux/utils/cryptsetup/v2.8/cryptsetup-2.8.6.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/cyrus-sasl-2.1.28-gcc15_fixes-1.patch
-https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.5.20.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.5.21.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.13.tar.xz
 https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-2.1.2.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gpgmepp/gpgmepp-2.1.0.tar.xz
@@ -220,7 +220,7 @@ https://github.com/opencv/opencv/archive/4.13.0/opencv-4.13.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.13.0/opencv_contrib-4.13.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.4/openjpeg-2.5.4.tar.gz
 https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
-https://poppler.freedesktop.org/poppler-26.06.0.tar.xz
+https://poppler.freedesktop.org/poppler-26.07.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v12.3.2/qpdf-12.3.2.tar.gz
@@ -289,7 +289,7 @@ https://sysstat.github.io/sysstat-packages/sysstat-12.7.9.tar.xz
 https://github.com/systemd/systemd/archive/v261.1/systemd-261.1.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.11.1/udisks-2.11.1.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.2.4.tar.gz
-https://gitlab.freedesktop.org/upower/upower/-/archive/v1.91.2/upower-v1.91.2.tar.bz2
+https://gitlab.freedesktop.org/upower/upower/-/archive/v1.91.3/upower-v1.91.3.tar.bz2
 https://ftpmirror.gnu.org/which/which-2.25.tar.gz
 https://github.com/jordansissel/xdotool/archive/v4.20260303.1/xdotool-4.20260303.1.tar.gz
 https://downloads.sourceforge.net/infozip/zip30.tar.gz
@@ -605,7 +605,7 @@ https://github.com/rfc1036/whois/archive/v5.6.6/whois-5.6.6.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.6.6.tar.xz
 https://github.com/c-ares/c-ares/releases/download/v1.34.6/c-ares-1.34.6.tar.gz
 https://curl.se/download/curl-8.21.0.tar.xz
-https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.8.1/geoclue-2.8.1.tar.bz2
+https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.8.2/geoclue-2.8.2.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.80/glib-networking-2.80.1.tar.xz
 https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.3.0/kdsoap-2.3.0.tar.gz
 https://download.kde.org/stable/kdsoap-ws-discovery-client/kdsoap-ws-discovery-client-0.4.0.tar.xz
@@ -993,7 +993,7 @@ https://downloads.xiph.org/releases/ogg/libogg-1.3.6.tar.xz
 https://github.com/haasn/libplacebo/archive/v7.360.1/libplacebo-7.360.1.tar.gz
 https://github.com/libsndfile/libsamplerate/releases/download/0.2.2/libsamplerate-0.2.2.tar.xz
 https://github.com/libsndfile/libsndfile/releases/download/1.2.2/libsndfile-1.2.2.tar.xz
-https://github.com/intel/libva/archive/2.23.0/libva-2.23.0.tar.gz
+https://github.com/intel/libva/archive/2.24.0/libva-2.24.0.tar.gz
 https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
 https://github.com/webmproject/libvpx/archive/v1.16.0/libvpx-1.16.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libvpx-1.16.0-security_fix-1.patch
