diff --git a/wget-list b/wget-list
index fbb68695d3..2689fd2a21 100644
--- a/wget-list
+++ b/wget-list
@@ -59,9 +59,9 @@ https://github.com/vim/vim/archive/v9.2.0752/vim-9.2.0752.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.5.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.2.tar.xz
-https://download.qemu.org/qemu-11.0.2.tar.xz
+https://download.qemu.org/qemu-11.0.3.tar.xz
 https://github.com/abseil/abseil-cpp/releases/download/20260526.0/abseil-cpp-20260526.0.tar.gz
-https://www.freedesktop.org/software/appstream/releases/AppStream-1.1.4.tar.xz
+https://www.freedesktop.org/software/appstream/releases/AppStream-1.1.5.tar.xz
 http://people.freedesktop.org/~hughsient/appstream-glib/releases/appstream-glib-0.8.3.tar.xz
 https://archive.apache.org/dist/apr/apr-1.7.6.tar.bz2
 https://archive.apache.org/dist/apr/apr-util-1.6.3.tar.bz2
@@ -112,7 +112,7 @@ https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.61.tar.bz2
 https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.58.tar.xz
 https://download.gnome.org/sources/libgudev/238/libgudev-238.tar.xz
 https://github.com/hughsie/libgusb/releases/download/0.4.9/libgusb-0.4.9.tar.xz
-https://github.com/libical/libical/releases/download/v4.0.3/libical-4.0.3.tar.gz
+https://github.com/libical/libical/releases/download/v4.0.4/libical-4.0.4.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn-1.44.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn2-2.3.8.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.8.0.tar.bz2
@@ -484,7 +484,7 @@ https://download.gnome.org/sources/blueprint-compiler/0.22/blueprint-compiler-0.
 https://files.pythonhosted.org/packages/source/b/build/build-1.5.1.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.4/cachecontrol-0.14.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.4.0.tar.gz
-https://github.com/cython/cython/releases/download/3.2.8/cython-3.2.8.tar.gz
+https://github.com/cython/cython/releases/download/3.2.9/cython-3.2.9.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.38.1.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.4.0.tar.xz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.23.tar.gz
