diff --git a/wget-list b/wget-list
index 5b97615f93..fa9aec3e1e 100644
--- a/wget-list
+++ b/wget-list
@@ -351,7 +351,7 @@ https://www.cpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz
 https://www.cpan.org/authors/id/E/ET/ETJ/Log-Log4perl-1.57.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.15.tar.gz
 https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4234.tar.gz
-https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.55.tar.gz
+https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.56.tar.gz
 https://www.cpan.org/authors/id/J/JT/JTBRAUN/Parse-RecDescent-1.967015.tar.gz
 https://www.cpan.org/authors/id/W/WB/WBRASWELL/Parse-Yapp-1.21.tar.gz
 https://www.cpan.org/authors/id/L/LE/LEONT/PerlIO-utf8_strict-0.010.tar.gz
@@ -673,7 +673,7 @@ https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.10.0.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.4/xf86-input-wacom-1.2.4.tar.bz2
 https://www.x.org/pub/individual/app/twm-1.0.13.1.tar.xz
 https://invisible-mirror.net/archives/xterm/xterm-410.tgz
-https://www.x.org/pub/individual/app/xclock-1.2.0.tar.xz
+https://www.x.org/pub/individual/app/xclock-1.2.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://github.com/WayfireWM/wayfire/releases/download/v0.10.1/wayfire-0.10.1.tar.xz
 https://github.com/WayfireWM/wayfire-plugins-extra/releases/download/v0.10.0/wayfire-plugins-extra-0.10.0.tar.xz
