diff --git a/wget-list b/wget-list
index 36fe159ea4..78db599c1b 100644
--- a/wget-list
+++ b/wget-list
@@ -124,7 +124,7 @@ https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.12/libproxy-0.5.12.tar.gz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.11.0/libqalculate-5.11.0.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.12.0/libqalculate-5.12.0.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.38.0/libqmi-1.38.0.tar.gz
 https://github.com/seccomp/libseccomp/releases/download/v2.6.1/libseccomp-2.6.1.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -479,7 +479,7 @@ https://www.python.org/ftp/python/doc/3.14.6/python-3.14.6-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/Python-3.11.1-openssl_4_fixes-1.patch
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
-https://download.gnome.org/sources/blueprint-compiler/0.20/blueprint-compiler-0.22.2.tar.xz
+https://download.gnome.org/sources/blueprint-compiler/0.22/blueprint-compiler-0.22.2.tar.xz
 https://files.pythonhosted.org/packages/source/b/build/build-1.5.0.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.4/cachecontrol-0.14.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.4.0.tar.gz
@@ -558,7 +558,7 @@ https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.7.0.tar.gz
 https://files.pythonhosted.org/packages/source/u/uv_build/uv_build-0.11.24.tar.gz
 https://files.pythonhosted.org/packages/source/v/vcs_versioning/vcs_versioning-2.1.2.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
-https://cache.ruby-lang.org/pub/ruby/4.0/ruby-4.0.5.tar.xz
+https://cache.ruby-lang.org/pub/ruby/4.0/ruby-4.0.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/ruby-4.0.5-openssl4_fixes-1.patch
 https://static.rust-lang.org/dist/rustc-1.96.0-src.tar.xz
 https://github.com/lfs-book/rust-openssl/archive/v0.10.78/rust-openssl-0.10.78.tar.gz
