diff --git a/wget-list b/wget-list
index a707f67094..aef9bf5e49 100644
--- a/wget-list
+++ b/wget-list
@@ -57,7 +57,7 @@ https://gitlab.gnome.org/World/gedit/libgedit-tepl/-/archive/6.13.0/libgedit-tep
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/25.08.0/src/kate-25.08.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.5.tar.xz
-https://www.nano-editor.org/dist/v8/nano-8.5.tar.xz
+https://www.nano-editor.org/dist/v8/nano-8.6.tar.xz
 https://github.com/vim/vim/archive/v9.1.1629/vim-9.1.1629.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
@@ -571,10 +571,10 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.11/bind-9.20.11.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.12/bind-9.20.12.tar.xz
 https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.54.0/downloads/NetworkManager-1.54.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
-https://nmap.org/dist/nmap-7.97.tar.bz2
+https://nmap.org/dist/nmap-7.98.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.4/whois-5.6.4.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.4.8.tar.xz
@@ -999,7 +999,7 @@ https://download.videolan.org/vlc/3.0.21/vlc-3.0.21.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.21-taglib-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/vlc-3.0.21-fedora_ffmpeg7-1.patch
 https://downloads.sourceforge.net/xine/xine-ui-0.99.14.tar.xz
-https://downloads.sourceforge.net/cdrdao/cdrdao-1.2.4.tar.bz2
+https://downloads.sourceforge.net/cdrdao/cdrdao-1.2.5.tar.bz2
 https://downloads.sourceforge.net/cdrtools/cdrtools-3.02a09.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/dvd+rw-tools/dvd+rw-tools-7.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/dvd+rw-tools-7.1-consolidated_fixes-1.patch
