diff --git a/wget-list b/wget-list
index a77dc571dd..2406a1ef87 100644
--- a/wget-list
+++ b/wget-list
@@ -15,8 +15,7 @@ https://anduin.linuxfromscratch.org/BLFS/Linux-PAM/Linux-PAM-1.7.1-docs.tar.xz
 https://downloads.sourceforge.net/liboauth/liboauth-1.0.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3.patch
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
-https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/mitkrb-1.21.3-gcc15_fixes-1.patch
+https://kerberos.org/dist/krb5/1.22/krb5-1.22.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.10.2.tar.gz
 https://archive.mozilla.org/pub/security/nss/releases/NSS_3_114_RTM/src/nss-3.114.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/nss-standalone-1.patch
