diff --git a/wget-list b/wget-list
index 325f739cbb..87228e7fc0 100644
--- a/wget-list
+++ b/wget-list
@@ -144,7 +144,7 @@ https://dist.libuv.org/dist/v1.51.0/libuv-v1.51.0.tar.gz
 https://github.com/lfs-book/libxkbcommon/archive/v1.10.0/libxkbcommon-1.10.0.tar.gz
 https://download.gnome.org/sources/libxml2/2.14/libxml2-2.14.5.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
-https://github.com/hughsie/libxmlb/releases/download/0.3.22/libxmlb-0.3.22.tar.xz
+https://github.com/hughsie/libxmlb/releases/download/0.3.23/libxmlb-0.3.23.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.43.tar.xz
 https://github.com/linuxwacom/libwacom/releases/download/libwacom-2.16.1/libwacom-2.16.1.tar.xz
 https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz
@@ -537,8 +537,7 @@ https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.g
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.3.0.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.4/ruby-3.4.5.tar.xz
-https://static.rust-lang.org/dist/rustc-1.88.0-src.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/rustc-1.88.0-testsuite_fix-1.patch
+https://static.rust-lang.org/dist/rustc-1.89.0-src.tar.xz
 https://github.com/rust-lang/rust-bindgen/archive/v0.72.0/rust-bindgen-0.72.0.tar.gz
 https://downloads.sourceforge.net/scons/SCons-4.9.1.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
