diff --git a/wget-list b/wget-list
index 4716f65d1c..a466a17474 100644
--- a/wget-list
+++ b/wget-list
@@ -17,7 +17,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
 https://kerberos.org/dist/krb5/1.22/krb5-1.22.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.10.2.tar.gz
-https://archive.mozilla.org/pub/security/nss/releases/NSS_3_114_RTM/src/nss-3.114.tar.gz
+https://archive.mozilla.org/pub/security/nss/releases/NSS_3_115_RTM/src/nss-3.115.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/nss-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-10.0p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.25.5/p11-kit-0.25.5.tar.xz
@@ -58,7 +58,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/25.08.0/src/kate-25.08.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.5.tar.xz
 https://www.nano-editor.org/dist/v8/nano-8.5.tar.xz
-https://github.com/vim/vim/archive/v9.1.1552/vim-9.1.1552.tar.gz
+https://github.com/vim/vim/archive/v9.1.1629/vim-9.1.1629.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
@@ -230,7 +230,7 @@ https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
 http://www.andre-simon.de/zip/highlight-4.16.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.32/ibus-1.5.32.tar.gz
 https://www.unicode.org/Public/zipped/16.0.0/UCD.zip
-https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-47.tar.xz
+https://www.imagemagick.org/archive/releases/ImageMagick-7.1.2-1.tar.xz
 https://salsa.debian.org/iso-codes-team/iso-codes/-/archive/v4.18.0/iso-codes-v4.18.0.tar.gz
 https://github.com/lsof-org/lsof/releases/download/4.99.5/lsof-4.99.5.tar.gz
 https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.3.2.tar.bz2
@@ -275,7 +275,7 @@ https://download.librdf.org/source/raptor2-2.0.16.tar.gz
 https://download.librdf.org/source/rasqal-0.9.33.tar.gz
 https://download.librdf.org/source/redland-1.0.17.tar.gz
 https://sysstat.github.io/sysstat-packages/sysstat-12.7.8.tar.xz
-https://github.com/systemd/systemd/archive/v257.6/systemd-257.6.tar.gz
+https://github.com/systemd/systemd/archive/v257.8/systemd-257.8.tar.gz
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
 https://www.rarlab.com/rar/unrarsrc-7.1.10.tar.gz
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.9/upower-v1.90.9.tar.bz2
@@ -311,8 +311,8 @@ https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.8/compiler-r
 https://www.lua.org/ftp/lua-5.4.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.8-shared_library-1.patch
 https://www.lua.org/tests/lua-5.4.8-tests.tar.gz
-https://anduin.linuxfromscratch.org/BLFS/luajit/luajit-20250212.tar.xz
-https://www.mercurial-scm.org/release/mercurial-7.0.1.tar.gz
+https://anduin.linuxfromscratch.org/BLFS/luajit/luajit-20250816.tar.xz
+https://www.mercurial-scm.org/release/mercurial-7.1.tar.gz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03.tar.xz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.03/nasm-2.16.03-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
@@ -462,14 +462,14 @@ https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
 https://www.php.net/distributions/php-8.4.11.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
-https://www.python.org/ftp/python/3.13.6/Python-3.13.6.tar.xz
-https://www.python.org/ftp/python/doc/3.13.6/python-3.13.6-docs-html.tar.bz2
+https://www.python.org/ftp/python/3.13.7/Python-3.13.7.tar.xz
+https://www.python.org/ftp/python/doc/3.13.7/python-3.13.7-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
-https://files.pythonhosted.org/packages/source/b/build/build-1.2.2.post1.tar.gz
+https://files.pythonhosted.org/packages/source/b/build/build-1.3.0.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.3.0.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.3/cachecontrol-0.14.3.tar.gz
-https://github.com/cython/cython/releases/download/3.1.2/cython-3.1.2.tar.gz
+https://github.com/cython/cython/releases/download/3.1.3/cython-3.1.3.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.36.0.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.4.0.tar.xz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.2.tar.gz
@@ -484,7 +484,7 @@ https://files.pythonhosted.org/packages/source/p/ply/ply-3.11.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.0.0.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
 https://download.gnome.org/sources/pyatspi/2.46/pyatspi-2.46.1.tar.xz
-https://github.com/pygobject/pycairo/releases/download/v1.26.1/pycairo-1.26.1.tar.gz
+https://github.com/pygobject/pycairo/releases/download/v1.28.0/pycairo-1.28.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pygdbmi/pygdbmi-0.11.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.19.2.tar.gz
 https://download.gnome.org/sources/pygobject/3.52/pygobject-3.52.3.tar.gz
@@ -497,16 +497,16 @@ https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1
 https://files.pythonhosted.org/packages/source/r/requests/requests-2.32.4.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/requests-use_system_certs-1.patch
 https://github.com/scour-project/scour/archive/v0.38.2/scour-0.38.2.tar.gz
-https://github.com/getsentry/sentry-python/releases/download/2.29.1/sentry_sdk-2.29.1.tar.gz
+https://github.com/getsentry/sentry-python/releases/download/2.35.0/sentry_sdk-2.35.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/six/six-1.17.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx/sphinx-8.2.3.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinx_rtd_theme/sphinx_rtd_theme-3.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/a/alabaster/alabaster-1.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/a/attrs/attrs-25.3.0.tar.gz
 https://files.pythonhosted.org/packages/source/B/Babel/babel-2.17.0.tar.gz
-https://files.pythonhosted.org/packages/source/c/certifi/certifi-2025.1.31.tar.gz
+https://files.pythonhosted.org/packages/source/c/certifi/certifi-2025.8.3.tar.gz
 https://files.pythonhosted.org/packages/source/c/chardet/chardet-5.2.0.tar.gz
-https://files.pythonhosted.org/packages/source/c/charset-normalizer/charset_normalizer-3.4.1.tar.gz
+https://files.pythonhosted.org/packages/source/c/charset-normalizer/charset_normalizer-3.4.3.tar.gz
 https://files.pythonhosted.org/packages/source/c/commonmark/commonmark-0.9.1.tar.gz
 https://files.pythonhosted.org/packages/source/e/editables/editables-0.5.tar.gz
 https://files.pythonhosted.org/packages/source/h/hatchling/hatchling-1.27.0.tar.gz
@@ -517,15 +517,16 @@ https://files.pythonhosted.org/packages/source/i/imagesize/imagesize-1.4.1.tar.g
 https://files.pythonhosted.org/packages/source/i/iniconfig/iniconfig-2.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/M/Markdown/markdown-3.8.2.tar.gz
 https://files.pythonhosted.org/packages/source/m/meson_python/meson_python-0.18.0.tar.gz
-https://files.pythonhosted.org/packages/source/m/msgpack/msgpack-1.1.0.tar.gz
+https://files.pythonhosted.org/packages/source/m/msgpack/msgpack-1.1.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pathspec/pathspec-0.12.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pluggy/pluggy-1.6.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyproject-hooks/pyproject_hooks-1.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyproject-metadata/pyproject_metadata-0.9.1.tar.gz
-https://files.pythonhosted.org/packages/source/p/pytz/pytz-2025.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pytz/pytz-2025.2.tar.gz
+https://files.pythonhosted.org/packages/source/r/roman-numerals-py/roman_numerals_py-3.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/setuptools_scm/setuptools_scm-8.3.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/smartypants/smartypants-2.0.2.tar.gz
-https://files.pythonhosted.org/packages/source/s/snowballstemmer/snowballstemmer-2.2.0.tar.gz
+https://files.pythonhosted.org/packages/source/s/snowballstemmer/snowballstemmer-3.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-applehelp/sphinxcontrib_applehelp-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-devhelp/sphinxcontrib_devhelp-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-htmlhelp/sphinxcontrib_htmlhelp-2.1.0.tar.gz
@@ -535,7 +536,7 @@ https://files.pythonhosted.org/packages/source/s/sphinxcontrib-qthelp/sphinxcont
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib_serializinghtml/sphinxcontrib_serializinghtml-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/t/trove_classifiers/trove_classifiers-2025.8.6.13.tar.gz
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.gz
-https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.3.0.tar.gz
+https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.5.0.tar.gz
 https://files.pythonhosted.org/packages/source/w/webencodings/webencodings-0.5.1.tar.gz
 https://cache.ruby-lang.org/pub/ruby/3.4/ruby-3.4.5.tar.xz
 https://static.rust-lang.org/dist/rustc-1.89.0-src.tar.xz
@@ -621,7 +622,7 @@ https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/sendmail-8.18.1-gcc15_fixes-1.patch
 https://git.openldap.org/openldap/openldap/-/archive/LMDB_0.9.33.tar.bz2
 https://downloads.mariadb.org/interstitial/mariadb-11.8.3/source/mariadb-11.8.3.tar.gz
-https://ftp.postgresql.org/pub/source/v17.5/postgresql-17.5.tar.bz2
+https://ftp.postgresql.org/pub/source/v17.6/postgresql-17.6.tar.bz2
 https://sqlite.org/2025/sqlite-autoconf-3500400.tar.gz
 https://sqlite.org/2025/sqlite-doc-3500400.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.10.tgz
@@ -817,22 +818,22 @@ https://download.gnome.org/sources/seahorse/47/seahorse-47.0.1.tar.xz
 https://download.gnome.org/sources/snapshot/48/snapshot-48.0.1.tar.xz
 https://archive.xfce.org/src/xfce/libxfce4util/4.20/libxfce4util-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfconf/4.20/xfconf-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/libxfce4ui/4.20/libxfce4ui-4.20.1.tar.bz2
+https://archive.xfce.org/src/xfce/libxfce4ui/4.20/libxfce4ui-4.20.2.tar.bz2
 https://archive.xfce.org/src/xfce/exo/4.20/exo-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/garcon/4.20/garcon-4.20.0.tar.bz2
 https://download.gnome.org/sources/libwnck/43/libwnck-43.2.tar.xz
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.20/xfce4-dev-tools-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.3.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.4.tar.bz2
+https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.4.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.5.tar.bz2
 https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.4.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.20/thunar-volman-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.20/tumbler-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.20/xfce4-appfinder-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-power-manager/4.20/xfce4-power-manager-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-settings/4.20/xfce4-settings-4.20.1.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-settings/4.20/xfce4-settings-4.20.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfdesktop/4.20/xfdesktop-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfwm4/4.20/xfwm4-4.20.0.tar.bz2
-https://archive.xfce.org/src/xfce/xfce4-session/4.20/xfce4-session-4.20.2.tar.bz2
+https://archive.xfce.org/src/xfce/xfce4-session/4.20/xfce4-session-4.20.3.tar.bz2
 https://archive.xfce.org/src/apps/parole/4.20/parole-4.20.0.tar.xz
 https://archive.xfce.org/src/apps/xfce4-terminal/1.1/xfce4-terminal-1.1.5.tar.xz
 https://archive.xfce.org/src/apps/xfburn/0.8/xfburn-0.8.0.tar.bz2
@@ -976,7 +977,7 @@ https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
 https://taglib.org/releases/taglib-2.1.1.tar.gz
 https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.30.1.tar.xz
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.10/wireplumber-0.5.10.tar.bz2
-https://anduin.linuxfromscratch.org/BLFS/x264/x264-20250212.tar.xz
+https://anduin.linuxfromscratch.org/BLFS/x264/x264-20250815.tar.xz
 https://bitbucket.org/multicoreware/x265_git/downloads/x265_4.1.tar.gz
 https://downloads.sourceforge.net/xine/xine-lib-1.2.13.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xine-lib-1.2.13-upstream_fixes-1.patch
@@ -1049,7 +1050,7 @@ https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-texmf.tar.xz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-extra.tar.xz
 https://downloads.sourceforge.net/asymptote/asymptote-3.05.src.tgz
-https://github.com/plk/biber/archive/v2.20/biber-2.20.tar.gz
+https://github.com/plk/biber/archive/v2.21/biber-2.21.tar.gz
 https://github.com/mgieseki/dvisvgm/releases/download/3.5/dvisvgm-3.5.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
