diff --git a/wget-list b/wget-list
index 7bea378dae..ddc689bc9a 100644
--- a/wget-list
+++ b/wget-list
@@ -882,11 +882,10 @@ https://github.com/lxqt/qtermwidget/releases/download/2.2.0/qtermwidget-2.2.0.ta
 https://github.com/lxqt/qterminal/releases/download/2.2.1/qterminal-2.2.1.tar.xz
 https://github.com/lxqt/screengrab/releases/download/3.0.0/screengrab-3.0.0.tar.xz
 https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.59.tar.xz
-https://download.documentfoundation.org/libreoffice/src/25.2.5/libreoffice-25.2.5.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/25.2.5/libreoffice-dictionaries-25.2.5.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/25.2.5/libreoffice-help-25.2.5.2.tar.xz
-https://download.documentfoundation.org/libreoffice/src/25.2.5/libreoffice-translations-25.2.5.2.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-25.2.5.2-icu76_fixes-1.patch
+https://download.documentfoundation.org/libreoffice/src/25.8.0/libreoffice-25.8.0.4.tar.xz
+https://download.documentfoundation.org/libreoffice/src/25.8.0/libreoffice-dictionaries-25.8.0.4.tar.xz
+https://download.documentfoundation.org/libreoffice/src/25.8.0/libreoffice-help-25.8.0.4.tar.xz
+https://download.documentfoundation.org/libreoffice/src/25.8.0/libreoffice-translations-25.8.0.4.tar.xz
 https://download.gnome.org/sources/epiphany/48/epiphany-48.5.tar.xz
 https://download.kde.org/stable/release-service/25.08.0/src/falkon-25.08.0.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.21/source/seamonkey-2.53.21.source.tar.xz
