diff --git a/wget-list b/wget-list
index 87228e7fc0..28ad4e257d 100644
--- a/wget-list
+++ b/wget-list
@@ -296,7 +296,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.p
 https://cmake.org/files/v4.0/cmake-4.0.3.tar.gz
 https://doxygen.nl/files/doxygen-1.14.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
-https://ftp.gnu.org/gnu/gcc/gcc-15.1.0/gcc-15.1.0.tar.xz
+https://ftp.gnu.org/gnu/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.8/gc-8.2.8.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-16.3.tar.xz
 https://www.kernel.org/pub/software/scm/git/git-2.50.1.tar.xz
@@ -461,8 +461,8 @@ https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
 https://www.php.net/distributions/php-8.4.11.tar.xz
 https://www.php.net/distributions/manual/php_manual_en.html.gz
 https://www.php.net/distributions/manual/php_manual_en.tar.gz
-https://www.python.org/ftp/python/3.13.5/Python-3.13.5.tar.xz
-https://www.python.org/ftp/python/doc/3.13.5/python-3.13.5-docs-html.tar.bz2
+https://www.python.org/ftp/python/3.13.6/Python-3.13.6.tar.xz
+https://www.python.org/ftp/python/doc/3.13.6/python-3.13.6-docs-html.tar.bz2
 https://www.python.org/ftp/python/3.11.1/Python-3.11.1.tar.xz
 https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/b/build/build-1.2.2.post1.tar.gz
