diff --git a/wget-list b/wget-list
index db61fa28a..d9d553f9d 100644
--- a/wget-list
+++ b/wget-list
@@ -605,7 +605,7 @@ https://www.dovecot.org/releases/2.3/dovecot-2.3.21.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-openssl3_fixes-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/dovecot-2.3.21-security_fix-1.patch
 https://ftp.exim.org/pub/exim/exim4/exim-4.97.1.tar.xz
-https://ghostarchive.org/postfix/postfix-release/official/postfix-3.8.5.tar.gz
+https://ghostarchive.org/postfix/postfix-release/official/postfix-3.9.0.tar.gz
 https://ftp.sendmail.org/sendmail.8.18.1.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-10.11.7/source/mariadb-10.11.7.tar.gz
@@ -806,7 +806,7 @@ https://download.gnome.org/sources/snapshot/45/snapshot-45.2.tar.xz
 https://download.gnome.org/sources/vinagre/3.22/vinagre-3.22.0.tar.xz
 https://archive.xfce.org/src/xfce/libxfce4util/4.18/libxfce4util-4.18.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfconf/4.18/xfconf-4.18.3.tar.bz2
-https://archive.xfce.org/src/xfce/libxfce4ui/4.18/libxfce4ui-4.18.5.tar.bz2
+https://archive.xfce.org/src/xfce/libxfce4ui/4.18/libxfce4ui-4.18.6.tar.bz2
 https://archive.xfce.org/src/xfce/exo/4.18/exo-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/garcon/4.18/garcon-4.18.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-panel/4.18/xfce4-panel-4.18.6.tar.bz2
@@ -1000,7 +1000,7 @@ https://files.libburnia-project.org/releases/libisofs-1.5.6.tar.gz
 https://github.com/OpenPrinting/cups/releases/download/v2.4.7/cups-2.4.7-source.tar.gz
 https://github.com/OpenPrinting/cups-browsed/releases/download/2.0.0/cups-browsed-2.0.0.tar.xz
 https://github.com/OpenPrinting/cups-filters/releases/download/2.0.0/cups-filters-2.0.0.tar.xz
-https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10021/ghostscript-10.02.1.tar.xz
+https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10030/ghostscript-10.03.0.tar.xz
 https://downloads.sourceforge.net/gs-fonts/ghostscript-fonts-std-8.11.tar.gz
 https://downloads.sourceforge.net/gs-fonts/gnu-gs-fonts-other-6.0.tar.gz
 https://downloads.sourceforge.net/gimp-print/gutenprint-5.3.4.tar.xz
@@ -1039,7 +1039,7 @@ https://ftp.tu-chemnitz.de/pub/tug/historic/systems/texlive/2023/texlive-2023031
 https://downloads.sourceforge.net/asymptote/asymptote-2.86.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/asymptote-2.86-build_fix-1.patch
 https://github.com/plk/biber/archive/v2.19/biber-2.19.tar.gz
-https://github.com/mgieseki/dvisvgm/releases/download/3.2/dvisvgm-3.2.tar.gz
+https://github.com/mgieseki/dvisvgm/releases/download/3.2.2/dvisvgm-3.2.2.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20240209.tar.xz
