diff --git a/wget-list b/wget-list
index a2b5f2aad..67db15802 100644
--- a/wget-list
+++ b/wget-list
@@ -49,7 +49,7 @@ https://ftp.gnu.org/gnu/ed/ed-1.20.1.tar.lz
 https://ftp.gnu.org/gnu/emacs/emacs-29.2.tar.xz
 https://download.gnome.org/sources/gedit/46/gedit-46.2.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
-https://download.kde.org/stable/release-service/23.08.5/src/kate-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/kate-24.02.0.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.6/mousepad-0.6.2.tar.bz2
 https://www.nano-editor.org/dist/v7/nano-7.2.tar.xz
 https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
@@ -708,7 +708,7 @@ https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/3.4.5/icewm-3.4.5.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://download.gnome.org/sources/adwaita-icon-theme/45/adwaita-icon-theme-45.0.tar.xz
-https://download.kde.org/stable/frameworks/5.115/breeze-icons-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/breeze-icons-6.0.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-symbolic/3.12/gnome-icon-theme-symbolic-3.12.0.tar.xz
@@ -716,28 +716,28 @@ https://download.gnome.org/sources/gnome-themes-extra/3.28/gnome-themes-extra-3.
 https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.17.tar.xz
 http://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
-https://download.kde.org/stable/frameworks/5.115/oxygen-icons-5.115.0.tar.xz
-https://download.kde.org/stable/frameworks/5.115/extra-cmake-modules-5.115.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/oxygen-icons-6.0.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/extra-cmake-modules-6.0.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.200.0.tar.xz
 https://launchpad.net/ubuntu/+archive/primary/+files/libdbusmenu-qt_0.9.3+16.04.20160218.orig.tar.gz
 https://download.kde.org/stable/plasma-wayland-protocols/plasma-wayland-protocols-1.12.0.tar.xz
 https://download.kde.org/stable/kuserfeedback/kuserfeedback-1.3.0.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/ark-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/kdenlive-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/kmix-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/kio-extras-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/khelpcenter-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/konsole-23.08.5.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/konsole-23.08.5-scrollbar-1.patch
-https://download.kde.org/stable/release-service/23.08.5/src/libkexiv2-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/okular-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/libkdcraw-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/gwenview-23.08.5.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/gwenview-23.08.5-build_fixes-1.patch
-https://download.kde.org/stable/release-service/23.08.5/src/libkcddb-23.08.5.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/k3b-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/ark-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/kdenlive-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/kmix-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/kio-extras-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/khelpcenter-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/konsole-24.02.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/konsole-24.02.0-scrollbar-1.patch
+https://download.kde.org/stable/release-service/24.02.0/src/libkexiv2-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/okular-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/libkdcraw-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/gwenview-24.02.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/gwenview-24.02.0-build_fixes-1.patch
+https://download.kde.org/stable/release-service/24.02.0/src/libkcddb-24.02.0.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/k3b-24.02.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.2/gcr-4.2.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/45/gsettings-desktop-schemas-45.0.tar.xz
@@ -830,12 +830,12 @@ https://archive.xfce.org/src/apps/ristretto/0.13/ristretto-0.13.2.tar.bz2
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.18/xfce4-dev-tools-4.18.1.tar.bz2
 https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.4.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.4/xfce4-pulseaudio-plugin-0.4.8.tar.bz2
-https://download.kde.org/stable/frameworks/5.115/kwindowsystem-5.115.0.tar.xz
-https://download.kde.org/stable/frameworks/5.115/kwayland-5.115.0.tar.xz
-https://download.kde.org/stable/frameworks/5.115/kconfig-5.115.0.tar.xz
-https://download.kde.org/stable/frameworks/5.115/solid-5.115.0.tar.xz
-https://download.kde.org/stable/frameworks/5.115/kidletime-5.115.0.tar.xz
-http://download.kde.org/stable/plasma/5.27.10/libkscreen-5.27.10.tar.xz
+https://download.kde.org/stable/frameworks/6.0/kwindowsystem-6.0.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/kwayland-6.0.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/kconfig-6.0.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/solid-6.0.0.tar.xz
+https://download.kde.org/stable/frameworks/6.0/kidletime-6.0.0.tar.xz
+http://download.kde.org/stable/plasma/6.0.0/libkscreen-6.0.0.tar.xz
 https://github.com/beltoforion/muparser/archive/v2.3.4/muparser-2.3.4.tar.gz
 https://github.com/lxqt/lxqt-build-tools/releases/download/0.13.0/lxqt-build-tools-0.13.0.tar.xz
 https://github.com/lxqt/libqtxdg/releases/download/3.12.0/libqtxdg-3.12.0.tar.xz
@@ -880,7 +880,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-help-
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-translations-24.2.0.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.2.0.3-consolidated_fixes-1.patch
 https://download.gnome.org/sources/epiphany/45/epiphany-45.3.tar.xz
-https://download.kde.org/stable/release-service/23.08.5/src/falkon-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/falkon-24.02.0.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.18/source/seamonkey-2.53.18.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
@@ -977,7 +977,7 @@ https://distfiles.audacious-media-player.org/audacious-4.3.1.tar.bz2
 https://distfiles.audacious-media-player.org/audacious-plugins-4.3.1.tar.bz2
 https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
-https://download.kde.org/stable/release-service/23.08.5/src/kwave-23.08.5.tar.xz
+https://download.kde.org/stable/release-service/24.02.0/src/kwave-24.02.0.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
 https://downloads.sourceforge.net/mpg123/mpg123-1.32.5.tar.bz2
 https://freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-5.0.tar.xz
