diff --git a/wget-list b/wget-list
index 94bef2373..e968c4df4 100644
--- a/wget-list
+++ b/wget-list
@@ -17,7 +17,7 @@ https://kerberos.org/dist/krb5/1.21/krb5-1.21.2.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.9.1.tar.gz
 https://archive.mozilla.org/pub/security/nss/releases/NSS_3_98_RTM/src/nss-3.98.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.98-standalone-1.patch
-https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.6p1.tar.gz
+https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.7p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.25.3/p11-kit-0.25.3.tar.xz
 https://gitlab.freedesktop.org/polkit/polkit/-/archive/124/polkit-124.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
@@ -593,7 +593,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/lynx-2.8.9rel.1-security_fix-1
 https://downloads.sourceforge.net/fetchmail/fetchmail-6.4.38.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/mailx/heirloom-mailx_12.5.orig.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/heirloom-mailx-12.5-fixes-1.patch
-https://bitbucket.org/mutt/mutt/downloads/mutt-2.2.12.tar.gz
+https://bitbucket.org/mutt/mutt/downloads/mutt-2.2.13.tar.gz
 https://ftp.osuosl.org/pub/blfs/conglomeration/procmail/procmail-3.22.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/procmail-3.22-consolidated_fixes-1.patch
 https://archive.apache.org/dist/httpd/httpd-2.4.58.tar.bz2
