diff --git a/wget-list b/wget-list
index 892837224..d784e52f8 100644
--- a/wget-list
+++ b/wget-list
@@ -574,7 +574,7 @@ https://downloads.sourceforge.net/traceroute/traceroute-2.1.5.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.2.3.tar.xz
 https://c-ares.haxx.se/download/c-ares-1.27.0.tar.gz
-https://curl.se/download/curl-8.6.0.tar.xz
+https://curl.se/download/curl-8.7.1.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.80/glib-networking-2.80.0.tar.xz
 https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.2.0/kdsoap-2.2.0.tar.gz
