diff --git a/wget-list b/wget-list
index 67db15802..75f9d6a4a 100644
--- a/wget-list
+++ b/wget-list
@@ -56,7 +56,7 @@ https://github.com/vim/vim/archive/v9.1.0041/vim-9.1.0041.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.10.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
-https://download.qemu.org/qemu-8.2.1.tar.xz
+https://download.qemu.org/qemu-8.2.2.tar.xz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.2.tar.xz
 https://archive.apache.org/dist/apr/apr-1.7.4.tar.bz2
 https://archive.apache.org/dist/apr/apr-util-1.6.3.tar.bz2
@@ -195,7 +195,7 @@ https://github.com/opencv/opencv/archive/4.9.0/opencv-4.9.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.9.0/opencv_contrib-4.9.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.2/openjpeg-2.5.2.tar.gz
 https://www.cairographics.org/releases/pixman-0.43.4.tar.gz
-https://poppler.freedesktop.org/poppler-24.02.0.tar.xz
+https://poppler.freedesktop.org/poppler-24.03.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v11.9.0/qpdf-11.9.0.tar.gz
@@ -671,8 +671,8 @@ https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.
 https://download.gnome.org/sources/gtksourceview/4.8/gtksourceview-4.8.4.tar.xz
 https://download.gnome.org/sources/gtksourceview/5.10/gtksourceview-5.10.0.tar.xz
 https://downloads.sourceforge.net/enlightenment/imlib2-1.12.2.tar.xz
-https://github.com/ksnip/kColorPicker/archive/refs/tags/v0.3.0/kColorPicker-0.3.0.tar.gz
-https://github.com/ksnip/kImageAnnotator/archive/refs/tags/v0.7.0/kImageAnnotator-0.7.0.tar.gz
+https://github.com/ksnip/kColorPicker/archive/refs/tags/v0.3.1/kColorPicker-0.3.1.tar.gz
+https://github.com/ksnip/kImageAnnotator/archive/refs/tags/v0.7.1/kImageAnnotator-0.7.1.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
 https://download.gnome.org/sources/libadwaita/1.4/libadwaita-1.4.3.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.2.1/libei-1.2.1.tar.gz
@@ -735,7 +735,6 @@ https://download.kde.org/stable/release-service/24.02.0/src/libkexiv2-24.02.0.ta
 https://download.kde.org/stable/release-service/24.02.0/src/okular-24.02.0.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/libkdcraw-24.02.0.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/gwenview-24.02.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/gwenview-24.02.0-build_fixes-1.patch
 https://download.kde.org/stable/release-service/24.02.0/src/libkcddb-24.02.0.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/k3b-24.02.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
@@ -874,11 +873,10 @@ https://github.com/lxqt/screengrab/releases/download/2.7.0/screengrab-2.7.0.tar.
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-3.0.5.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/abiword/abiword-docs-3.0.2.tar.gz
 https://download.gnome.org/sources/gnumeric/1.12/gnumeric-1.12.57.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-24.2.0.3.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-dictionaries-24.2.0.3.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-help-24.2.0.3.tar.xz
-https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-translations-24.2.0.3.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.2.0.3-consolidated_fixes-1.patch
+https://download.documentfoundation.org/libreoffice/src/24.2.1/libreoffice-24.2.1.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.2.1/libreoffice-dictionaries-24.2.1.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.2.1/libreoffice-help-24.2.1.2.tar.xz
+https://download.documentfoundation.org/libreoffice/src/24.2.1/libreoffice-translations-24.2.1.2.tar.xz
 https://download.gnome.org/sources/epiphany/45/epiphany-45.3.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/falkon-24.02.0.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.18/source/seamonkey-2.53.18.source.tar.xz
