diff --git a/wget-list b/wget-list
index 0d0577c46..c29e22d4c 100644
--- a/wget-list
+++ b/wget-list
@@ -149,7 +149,7 @@ https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.43/pcre2-10.43.
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.10.0/qcoro-0.10.0.tar.gz
-https://archive.mozilla.org/pub/firefox/releases/115.8.0esr/source/firefox-115.8.0esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/115.9.0esr/source/firefox-115.9.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.280.0/SPIRV-Headers-1.3.280.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Tools-1.3.275.0.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.2.tar.gz
@@ -298,7 +298,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.4.6-shared_library-1.pat
 https://www.lua.org/tests/lua-5.4.6-tests.tar.gz
 https://www.lua.org/ftp/lua-5.2.4.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/lua-5.2.4-shared_library-1.patch
-https://www.mercurial-scm.org/release/mercurial-6.7.tar.gz
+https://www.mercurial-scm.org/release/mercurial-6.7.1.tar.gz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.01/nasm-2.16.01.tar.xz
 https://www.nasm.us/pub/nasm/releasebuilds/2.16.01/nasm-2.16.01-xdoc.tar.xz
 https://github.com/NixOS/patchelf/releases/download/0.18.0/patchelf-0.18.0.tar.gz
@@ -876,7 +876,7 @@ https://download.documentfoundation.org/libreoffice/src/24.2.1/libreoffice-help-
 https://download.documentfoundation.org/libreoffice/src/24.2.1/libreoffice-translations-24.2.1.2.tar.xz
 https://download.gnome.org/sources/epiphany/45/epiphany-45.3.tar.xz
 https://download.kde.org/stable/release-service/24.02.0/src/falkon-24.02.0.tar.xz
-https://archive.mozilla.org/pub/firefox/releases/115.9.0esr/source/firefox-115.9.0esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/115.9.1esr/source/firefox-115.9.1esr.source.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.18.1/source/seamonkey-2.53.18.1.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
