diff --git a/wget-list b/wget-list
index f871697ae..fb3f385ec 100644
--- a/wget-list
+++ b/wget-list
@@ -57,7 +57,7 @@ http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.10.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
 https://download.qemu.org/qemu-8.2.1.tar.xz
-https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.1.tar.xz
+https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.2.tar.xz
 https://archive.apache.org/dist/apr/apr-1.7.4.tar.bz2
 https://archive.apache.org/dist/apr/apr-util-1.6.3.tar.bz2
 https://ftp.gnu.org/gnu/aspell/aspell-0.60.8.1.tar.gz
@@ -144,10 +144,10 @@ https://nodejs.org/dist/v20.11.1/node-v20.11.1.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.6.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.35/src/nspr-4.35.tar.gz
 https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.bz2
-https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.42/pcre2-10.42.tar.bz2
+https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.43/pcre2-10.43.tar.bz2
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
-https://archive.mozilla.org/pub/firefox/releases/115.7.0esr/source/firefox-115.7.0esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/115.8.0esr/source/firefox-115.8.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Headers-1.3.275.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Tools-1.3.275.0.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.2.tar.gz
@@ -168,12 +168,13 @@ https://downloads.sourceforge.net/freetype/freetype-doc-2.13.2.tar.xz
 https://www.freedesktop.org/software/fontconfig/release/fontconfig-2.15.0.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.13/fribidi-1.0.13.tar.xz
 https://download.gimp.org/pub/gegl/0.4/gegl-0.4.48.tar.xz
-https://sourceforge.net/projects/giflib/files/giflib-5.2.1.tar.gz
+https://sourceforge.net/projects/giflib/files/giflib-5.2.2.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/giflib-5.2.2-upstream_fixes-1.patch
 https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.5/glad-2.0.5.tar.gz
-https://github.com/g-truc/glm/archive/1.0.0/glm-1.0.0.tar.gz
+https://github.com/g-truc/glm/archive/1.0.1/glm-1.0.1.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/8.3.0/harfbuzz-8.3.0.tar.xz
-https://github.com/jasper-software/jasper/archive/version-4.2.0/jasper-4.2.0.tar.gz
+https://github.com/jasper-software/jasper/archive/version-4.2.1/jasper-4.2.1.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.14/lcms2-2.14.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.0.4/libavif-1.0.4.tar.gz
 https://github.com/libexif/libexif/releases/download/v0.6.24/libexif-0.6.24.tar.bz2
@@ -192,7 +193,7 @@ https://github.com/mypaint/mypaint-brushes/releases/download/v1.3.1/mypaint-brus
 https://releases.pagure.org/newt/newt-0.52.24.tar.gz
 https://github.com/opencv/opencv/archive/4.9.0/opencv-4.9.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.9.0/opencv_contrib-4.9.0.tar.gz
-https://github.com/uclouvain/openjpeg/archive/v2.5.0/openjpeg-2.5.0.tar.gz
+https://github.com/uclouvain/openjpeg/archive/v2.5.2/openjpeg-2.5.2.tar.gz
 https://www.cairographics.org/releases/pixman-0.43.2.tar.gz
 https://poppler.freedesktop.org/poppler-24.02.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
@@ -203,7 +204,7 @@ https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
 https://github.com/aruiz/webp-pixbuf-loader/archive/0.2.7/webp-pixbuf-loader-0.2.7.tar.gz
 https://github.com/google/woff2/archive/v1.0.2/woff2-1.0.2.tar.gz
-https://github.com/asciidoctor/asciidoctor/archive/v2.0.20/asciidoctor-2.0.20.tar.gz
+https://github.com/asciidoctor/asciidoctor/archive/v2.0.21/asciidoctor-2.0.21.tar.gz
 https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.27.tar.xz
@@ -264,7 +265,7 @@ https://sysstat.github.io/sysstat-packages/sysstat-12.7.5.tar.xz
 https://github.com/systemd/systemd/archive/v255/systemd-255.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/systemd-255-upstream_fixes-1.patch
 https://github.com/storaged-project/udisks/releases/download/udisks-2.10.1/udisks-2.10.1.tar.bz2
-https://www.rarlab.com/rar/unrarsrc-6.2.12.tar.gz
+https://www.rarlab.com/rar/unrarsrc-7.0.7.tar.gz
 https://downloads.sourceforge.net/infozip/unzip60.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/unzip-6.0-consolidated_fixes-1.patch
 https://gitlab.freedesktop.org/upower/upower/-/archive/v1.90.2/upower-v1.90.2.tar.bz2
@@ -476,7 +477,7 @@ https://download.gnome.org/sources/pygobject/3.46/pygobject-3.46.0.tar.xz
 https://download.gnome.org/sources/pygtk/2.24/pygtk-2.24.0.tar.bz2
 https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.1.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
-https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.0.0.tar.gz
+https://files.pythonhosted.org/packages/source/p/pytest/pytest-8.0.2.tar.gz
 https://files.pythonhosted.org/packages/b0/25/7998cd2dec731acbd438fbf91bc619603fc5188de0a9a17699a781840452/pyxdg-0.28.tar.gz
 https://files.pythonhosted.org/packages/source/P/PyYAML/PyYAML-6.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/r/recommonmark/recommonmark-0.7.1.tar.gz
@@ -557,16 +558,16 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wpa_supplicant-2.10-security_f
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
 https://ftp.isc.org/isc/bind9/9.18.24/bind-9.18.24.tar.xz
-https://download.gnome.org/sources/NetworkManager/1.44/NetworkManager-1.44.2.tar.xz
+https://download.gnome.org/sources/NetworkManager/1.46/NetworkManager-1.46.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.94.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.5.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.2.3.tar.xz
-https://c-ares.haxx.se/download/c-ares-1.26.0.tar.gz
+https://c-ares.haxx.se/download/c-ares-1.27.0.tar.gz
 https://curl.se/download/curl-8.6.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
-https://download.gnome.org/sources/glib-networking/2.78/glib-networking-2.78.0.tar.xz
+https://download.gnome.org/sources/glib-networking/2.78/glib-networking-2.78.1.tar.xz
 https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.2.0/kdsoap-2.2.0.tar.gz
 https://www.nlnetlabs.nl/downloads/ldns/ldns-1.8.3.tar.gz
 https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
@@ -694,8 +695,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-ffmpeg5_fi
 https://download.qt.io/archive/qt/6.6/6.6.2/single/qt-everywhere-src-6.6.2.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.277/Vulkan-Headers-1.3.277.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.277/Vulkan-Loader-1.3.277.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.279/Vulkan-Headers-1.3.279.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.279/Vulkan-Loader-1.3.279.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.42.5.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.18.2/xdg-desktop-portal-1.18.2.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.1/xdg-desktop-portal-gtk-1.15.1.tar.xz
@@ -740,7 +741,7 @@ https://download.kde.org/stable/release-service/23.08.5/src/k3b-23.08.5.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.2/gcr-4.2.0.tar.xz
 https://download.gnome.org/sources/gsettings-desktop-schemas/45/gsettings-desktop-schemas-45.0.tar.xz
-https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.3.tar.xz
+https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.4.tar.xz
 https://download.gnome.org/sources/rest/0.9/rest-0.9.1.tar.xz
 https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.6.tar.xz
 https://gitlab.gnome.org/GNOME/vte/-/archive/0.74.2/vte-0.74.2.tar.gz
@@ -878,9 +879,8 @@ https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-dicti
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-help-24.2.0.3.tar.xz
 https://download.documentfoundation.org/libreoffice/src/24.2.0/libreoffice-translations-24.2.0.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-24.2.0.3-consolidated_fixes-1.patch
-https://download.gnome.org/sources/epiphany/45/epiphany-45.2.tar.xz
+https://download.gnome.org/sources/epiphany/45/epiphany-45.3.tar.xz
 https://download.kde.org/stable/release-service/23.08.5/src/falkon-23.08.5.tar.xz
-https://archive.mozilla.org/pub/firefox/releases/115.8.0esr/source/firefox-115.8.0esr.source.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.18/source/seamonkey-2.53.18.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
@@ -979,12 +979,13 @@ https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
 https://download.kde.org/stable/release-service/23.08.5/src/kwave-23.08.5.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
-https://downloads.sourceforge.net/mpg123/mpg123-1.32.4.tar.bz2
+https://downloads.sourceforge.net/mpg123/mpg123-1.32.5.tar.bz2
 https://freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-5.0.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.2.tar.gz
 https://ffmpeg.org/releases/ffmpeg-6.1.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.1.1-chromium_method-1.patch
+https://www.linuxfromscratch.org/patches/blfs/svn/ffmpeg-6.1.1-vulkan_fixes-1.patch
 https://github.com/mpv-player/mpv/archive/refs/tags/v0.37.0/mpv-0.37.0.tar.gz
 https://mplayerhq.hu/MPlayer/releases/MPlayer-1.5.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/MPlayer-1.5-upstream_ffmpg6_fixes-1.patch
