diff --git a/wget-list b/wget-list
index d784e52f8..191dfb298 100644
--- a/wget-list
+++ b/wget-list
@@ -26,7 +26,7 @@ https://github.com/shadow-maint/shadow/releases/download/4.15.0/shadow-4.15.0.ta
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.72.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.15p5.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
-https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.7.1.tar.xz
+https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.8.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
 https://github.com/libfuse/libfuse/releases/download/fuse-3.16.2/fuse-3.16.2.tar.gz
 https://jfs.sourceforge.net/project/pub/jfsutils-1.1.15.tar.gz
@@ -93,7 +93,7 @@ https://github.com/libarchive/libarchive/releases/download/v3.7.2/libarchive-3.7
 https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.7.tar.bz2
 https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/ivmai/libatomic_ops/releases/download/v7.8.2/libatomic_ops-7.8.2.tar.gz
-https://github.com/storaged-project/libblockdev/releases/download/3.1.0-1/libblockdev-3.1.0.tar.gz
+https://github.com/storaged-project/libblockdev/releases/download/3.1.1-1/libblockdev-3.1.1.tar.gz
 https://github.com/storaged-project/libbytesize/releases/download/2.10/libbytesize-2.10.tar.gz
 https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
 https://launchpad.net/ubuntu/+archive/primary/+files/libdbusmenu-qt_0.9.3+16.04.20160218.orig.tar.gz
@@ -236,7 +236,7 @@ https://downloads.sourceforge.net/acpid2/acpid-2.0.34.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/at/at_3.2.5.orig.tar.gz
 https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.9.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/bluez-5.73.tar.xz
-https://github.com/containers/bubblewrap/releases/download/v0.8.0/bubblewrap-0.8.0.tar.xz
+https://github.com/containers/bubblewrap/releases/download/v0.9.0/bubblewrap-0.9.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-1.4.7.tar.xz
 https://ftp.gnu.org/gnu/cpio/cpio-2.15.tar.bz2
 https://www.freedesktop.org/software/cups-pk-helper/releases/cups-pk-helper-0.2.7.tar.xz
@@ -346,7 +346,7 @@ https://www.cpan.org/authors/id/I/IS/ISHIGAKI/Text-CSV-2.04.tar.gz
 https://www.cpan.org/authors/id/S/SY/SYP/Text-Roman-3.5.tar.gz
 https://www.cpan.org/authors/id/S/SA/SADAHIRO/Unicode-Collate-1.31.tar.gz
 https://www.cpan.org/authors/id/N/NE/NEZUMI/Unicode-LineBreak-2019.001.tar.gz
-https://www.cpan.org/authors/id/O/OA/OALDERS/URI-5.27.tar.gz
+https://www.cpan.org/authors/id/O/OA/OALDERS/URI-5.28.tar.gz
 https://www.cpan.org/authors/id/M/MA/MARKOV/XML-LibXML-Simple-1.01.tar.gz
 https://www.cpan.org/authors/id/S/SH/SHLOMIF/XML-LibXSLT-2.002001.tar.gz
 https://www.cpan.org/authors/id/G/GR/GRANTM/XML-Simple-2.25.tar.gz
@@ -626,7 +626,7 @@ https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.7-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.19.3.tar.gz
 https://www.x.org/pub/individual/util/util-macros-1.20.0.tar.xz
-https://xorg.freedesktop.org/archive/individual/proto/xorgproto-2023.2.tar.xz
+https://xorg.freedesktop.org/archive/individual/proto/xorgproto-2024.1.tar.xz
 https://www.x.org/pub/individual/lib/libXau-1.0.11.tar.xz
 https://www.x.org/pub/individual/lib/libXdmcp-1.1.5.tar.xz
 https://xorg.freedesktop.org/archive/individual/proto/xcb-proto-1.16.0.tar.xz
@@ -951,7 +951,7 @@ https://github.com/haasn/libplacebo/archive/refs/tags/v6.338.2/libplacebo-6.338.
 https://downloads.sourceforge.net/libquicktime/libquicktime-1.2.4.tar.gz
 https://github.com/libsndfile/libsamplerate/releases/download/0.2.2/libsamplerate-0.2.2.tar.xz
 https://github.com/libsndfile/libsndfile/releases/download/1.2.2/libsndfile-1.2.2.tar.xz
-https://github.com/intel/libva/releases/download/2.20.0/libva-2.20.0.tar.bz2
+https://github.com/intel/libva/archive/2.21.0/libva-2.21.0.tar.gz
 https://gitlab.freedesktop.org/vdpau/libvdpau/-/archive/1.5/libvdpau-1.5.tar.bz2
 https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.2/libvdpau-va-gl-0.4.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.7.tar.xz
