diff --git a/wget-list b/wget-list
index 051647c07..0d0577c46 100644
--- a/wget-list
+++ b/wget-list
@@ -95,7 +95,7 @@ https://0pointer.de/public/libatasmart-0.19.tar.xz
 https://github.com/ivmai/libatomic_ops/releases/download/v7.8.2/libatomic_ops-7.8.2.tar.gz
 https://github.com/storaged-project/libblockdev/releases/download/3.1.0-1/libblockdev-3.1.0.tar.gz
 https://github.com/storaged-project/libbytesize/releases/download/2.10/libbytesize-2.10.tar.gz
-https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.5.tar.xz
+https://download.gnome.org/sources/libcloudproviders/0.3/libcloudproviders-0.3.6.tar.xz
 https://launchpad.net/ubuntu/+archive/primary/+files/libdbusmenu-qt_0.9.3+16.04.20160218.orig.tar.gz
 https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
 https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.1.1/downloads//libdisplay-info-0.1.1.tar.xz
@@ -150,7 +150,7 @@ http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.10.0/qcoro-0.10.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/115.8.0esr/source/firefox-115.8.0esr.source.tar.xz
-https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Headers-1.3.275.0.tar.gz
+https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.280.0/SPIRV-Headers-1.3.280.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Tools-1.3.275.0.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.2.tar.gz
 https://telepathy.freedesktop.org/releases/telepathy-glib/telepathy-glib-0.24.2.tar.gz
@@ -158,7 +158,7 @@ https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.18.0/umockdev-0.18.0.tar.xz
 https://github.com/nemtrif/utfcpp/archive/v4.0.5/utfcpp-4.0.5.tar.gz
 https://gitlab.freedesktop.org/wayland/wayland/-/releases/1.22.0/downloads/wayland-1.22.0.tar.xz
-https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.33/downloads/wayland-protocols-1.33.tar.xz
+https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.34/downloads/wayland-protocols-1.34.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/1.4.25/xapian-core-1.4.25.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
@@ -171,7 +171,7 @@ https://github.com/fribidi/fribidi/releases/download/v1.0.13/fribidi-1.0.13.tar.
 https://download.gimp.org/pub/gegl/0.4/gegl-0.4.48.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/giflib-5.2.2-upstream_fixes-1.patch
-https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.5/glad-2.0.5.tar.gz
+https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.6/glad-2.0.6.tar.gz
 https://github.com/g-truc/glm/archive/1.0.1/glm-1.0.1.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/8.3.1/harfbuzz-8.3.1.tar.xz
@@ -219,7 +219,7 @@ https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-28.tar.xz
 https://ftp.debian.org/debian/pool/main/i/iso-codes/iso-codes_4.16.0.orig.tar.xz
 https://github.com/lsof-org/lsof/releases/download/4.99.0/lsof-4.99.0.tar.gz
 https://mandoc.bsd.lv/snapshots/mandoc-1.14.6.tar.gz
-https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.2.1.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.3.0.tar.bz2
 https://ftp.gnu.org/gnu/screen/screen-4.9.1.tar.gz
 https://gitlab.freedesktop.org/xdg/shared-mime-info/-/archive/2.4/shared-mime-info-2.4.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/xdgmime/xdgmime.tar.xz
@@ -560,7 +560,7 @@ https://w1.fi/releases/wpa_supplicant-2.10.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/wpa_supplicant-2.10-security_fix-1.patch
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.18.24/bind-9.18.24.tar.xz
+https://ftp.isc.org/isc/bind9/9.18.25/bind-9.18.25.tar.xz
 https://download.gnome.org/sources/NetworkManager/1.46/NetworkManager-1.46.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.94.tar.bz2
@@ -709,7 +709,7 @@ https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
 https://github.com/ice-wm/icewm/archive/3.4.6/icewm-3.4.6.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
-https://download.gnome.org/sources/adwaita-icon-theme/45/adwaita-icon-theme-45.0.tar.xz
+https://download.gnome.org/sources/adwaita-icon-theme/46/adwaita-icon-theme-46.0.tar.xz
 https://download.kde.org/stable/frameworks/6.0/breeze-icons-6.0.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
@@ -1031,7 +1031,7 @@ https://releases.pagure.org/xmlto/xmlto-0.0.28.tar.bz2
 https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview-gtk3/epdfview-gtk3-20200814.tar.xz
 https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.9-src.tar.gz
-https://www.mupdf.com/downloads/archive/mupdf-1.23.11-source.tar.gz
+https://www.mupdf.com/downloads/archive/mupdf-1.24.0-source.tar.gz
 https://github.com/dov/paps/releases/download/v0.8.0/paps-0.8.0.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2024/texlive-20240312-source.tar.xz
