diff --git a/wget-list b/wget-list
index e968c4df4..503f37de2 100644
--- a/wget-list
+++ b/wget-list
@@ -276,6 +276,7 @@ https://ftp.gnu.org/gnu/clisp/latest/clisp-2.49.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/clisp-2.49-readline7_fixes-1.patch
 https://cmake.org/files/v3.28/cmake-3.28.3.tar.gz
 https://doxygen.nl/files/doxygen-1.10.0.src.tar.gz
+https://kernel.org/pub/software/utils/dtc/dtc-1.7.0.tar.xz
 https://ftp.gnu.org/gnu/gcc/gcc-13.2.0/gcc-13.2.0.tar.xz
 https://github.com/ivmai/bdwgc/releases/download/v8.2.6/gc-8.2.6.tar.gz
 https://ftp.gnu.org/gnu/gdb/gdb-14.2.tar.xz
