diff --git a/wget-list b/wget-list
index efaf18cc42..896bb06b4b 100644
--- a/wget-list
+++ b/wget-list
@@ -23,7 +23,7 @@ https://github.com/polkit-org/polkit/archive/126/polkit-126.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
 https://github.com/shadow-maint/shadow/releases/download/4.17.4/shadow-4.17.4.tar.xz
-https://www.stunnel.org/downloads/archive/5.x/stunnel-5.74.tar.gz
+https://www.stunnel.org/downloads/archive/5.x/stunnel-5.75.tar.gz
 https://www.sudo.ws/dist/sudo-1.9.16p2.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.14.tar.xz
