diff --git a/wget-list b/wget-list
index 053922490e..a229b4c228 100644
--- a/wget-list
+++ b/wget-list
@@ -60,7 +60,7 @@ https://github.com/vim/vim/archive/v9.1.1263/vim-9.1.1263.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.12.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.15.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
-https://download.qemu.org/qemu-10.0.0.tar.xz
+https://download.qemu.org/qemu-10.0.2.tar.xz
 https://github.com/abseil/abseil-cpp/releases/download/20250512.0/abseil-cpp-20250512.0.tar.gz
 https://www.freedesktop.org/software/appstream/releases/AppStream-1.0.5.tar.xz
 http://people.freedesktop.org/~hughsient/appstream-glib/releases/appstream-glib-0.8.3.tar.xz
@@ -153,11 +153,11 @@ https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.36/src/nspr-4.36.tar.gz
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.45/pcre2-10.45.tar.bz2
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
-https://github.com/protocolbuffers/protobuf/releases/download/v31.0/protobuf-31.0.tar.gz
+https://github.com/protocolbuffers/protobuf/releases/download/v31.1/protobuf-31.1.tar.gz
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.2/protobuf-c-1.5.2.tar.gz
 https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.12.0/qcoro-0.12.0.tar.gz
-https://archive.mozilla.org/pub/firefox/releases/128.10.1esr/source/firefox-128.10.1esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/128.11.0esr/source/firefox-128.11.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.313.0/SPIRV-Headers-vulkan-sdk-1.4.313.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.313.0/SPIRV-Tools-vulkan-sdk-1.4.313.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v20.1.2/SPIRV-LLVM-Translator-20.1.2.tar.gz
@@ -575,7 +575,8 @@ https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.1/whois-5.6.1.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.4.6.tar.xz
 https://github.com/c-ares/c-ares/releases/download/v1.34.5/c-ares-1.34.5.tar.gz
-https://curl.se/download/curl-8.13.0.tar.xz
+https://curl.se/download/curl-8.14.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/curl-8.14.0-upstream_fix-1.patch
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.2/geoclue-2.7.2.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.80/glib-networking-2.80.1.tar.xz
 https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.2.0/kdsoap-2.2.0.tar.gz
@@ -607,8 +608,7 @@ https://github.com/BuGlessRB/procmail/archive/v3.24/procmail-3.24.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/procmail-3.24-consolidated_fixes-1.patch
 https://archive.apache.org/dist/httpd/httpd-2.4.63.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/httpd-2.4.63-blfs_layout-1.patch
-https://downloads.isc.org/isc/kea/2.6.2/kea-2.6.2.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/kea-2.6.2-fix_boost_1_87-1.patch
+https://downloads.isc.org/isc/kea/2.6.3/kea-2.6.3.tar.gz
 https://github.com/proftpd/proftpd/archive/v1.3.9/proftpd-1.3.9.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.0.tar.gz
 https://ftp.exim.org/pub/exim/exim4/exim-4.98.2.tar.xz
@@ -695,7 +695,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-6.9.0-fix_gpu_acce
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.4.313/Vulkan-Headers-1.4.313.tar.gz
 https://github.com/KhronosGroup/Vulkan-Loader/archive/v1.4.313/Vulkan-Loader-1.4.313.tar.gz
-https://webkitgtk.org/releases/webkitgtk-2.48.2.tar.xz
+https://webkitgtk.org/releases/webkitgtk-2.48.3.tar.xz
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.20.3/xdg-desktop-portal-1.20.3.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.3/xdg-desktop-portal-gtk-1.15.3.tar.xz
 https://download.gnome.org/sources/gdm/48/gdm-48.0.tar.xz
@@ -747,7 +747,7 @@ https://download.gnome.org/sources/gsettings-desktop-schemas/48/gsettings-deskto
 https://download.gnome.org/sources/libsecret/0.21/libsecret-0.21.7.tar.xz
 https://download.gnome.org/sources/rest/0.9/rest-0.9.1.tar.xz
 https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.6.tar.xz
-https://gitlab.gnome.org/GNOME/vte/-/archive/0.80.1/vte-0.80.1.tar.gz
+https://gitlab.gnome.org/GNOME/vte/-/archive/0.80.2/vte-0.80.2.tar.gz
 https://download.gnome.org/sources/yelp-xsl/42/yelp-xsl-42.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/yelp-xsl-42.1-security_fix-1.patch
 https://download.gnome.org/sources/geocode-glib/3.26/geocode-glib-3.26.4.tar.xz
@@ -755,14 +755,14 @@ https://download.gnome.org/sources/gjs/1.84/gjs-1.84.2.tar.xz
 https://download.gnome.org/sources/gnome-autoar/0.4/gnome-autoar-0.4.5.tar.xz
 https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.1.tar.xz
 https://download.gnome.org/sources/gnome-menus/3.36/gnome-menus-3.36.0.tar.xz
-https://download.gnome.org/sources/gnome-online-accounts/3.54/gnome-online-accounts-3.54.2.tar.xz
+https://download.gnome.org/sources/gnome-online-accounts/3.54/gnome-online-accounts-3.54.3.tar.xz
 https://download.gnome.org/sources/libgee/0.20/libgee-0.20.8.tar.xz
 https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libgweather-4.4.4-upstream_fix-1.patch
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.4/libshumate-1.4.0.tar.xz
-https://download.gnome.org/sources/evolution-data-server/3.56/evolution-data-server-3.56.1.tar.xz
+https://download.gnome.org/sources/evolution-data-server/3.56/evolution-data-server-3.56.2.tar.xz
 https://download.gnome.org/sources/tinysparql/3.9/tinysparql-3.9.2.tar.xz
 https://download.gnome.org/sources/localsearch/3.9/localsearch-3.9.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/localsearch-3.9.0-libxml2_2_14-1.patch
@@ -770,22 +770,22 @@ https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
 https://download.gnome.org/sources/xdg-desktop-portal-gnome/48/xdg-desktop-portal-gnome-48.0.tar.xz
 https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
-https://download.gnome.org/sources/gnome-backgrounds/48/gnome-backgrounds-48.0.tar.xz
+https://download.gnome.org/sources/gnome-backgrounds/48/gnome-backgrounds-48.2.1.tar.xz
 https://download.gnome.org/sources/gvfs/1.56/gvfs-1.56.1.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.5.tar.xz
-https://download.gnome.org/sources/nautilus/48/nautilus-48.1.tar.xz
+https://download.gnome.org/sources/nautilus/48/nautilus-48.2.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/47/gnome-bluetooth-47.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-bluetooth-47.1-build_fix-1.patch
 https://download.gnome.org/sources/gnome-keyring/48/gnome-keyring-48.0.tar.xz
 https://download.gnome.org/sources/gnome-settings-daemon/48/gnome-settings-daemon-48.1.tar.xz
 https://download.gnome.org/sources/tecla/48/tecla-48.0.2.tar.xz
-https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.1.tar.xz
-https://download.gnome.org/sources/mutter/48/mutter-48.2.tar.xz
-https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.1.tar.xz
-https://download.gnome.org/sources/gnome-shell-extensions/48/gnome-shell-extensions-48.1.tar.xz
+https://download.gnome.org/sources/gnome-control-center/48/gnome-control-center-48.2.tar.xz
+https://download.gnome.org/sources/mutter/48/mutter-48.3.tar.xz
+https://download.gnome.org/sources/gnome-shell/48/gnome-shell-48.2.tar.xz
+https://download.gnome.org/sources/gnome-shell-extensions/48/gnome-shell-extensions-48.2.tar.xz
 https://download.gnome.org/sources/gnome-session/48/gnome-session-48.0.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/46/gnome-tweaks-46.1.tar.xz
-https://download.gnome.org/sources/gnome-user-docs/48/gnome-user-docs-48.1.tar.xz
+https://download.gnome.org/sources/gnome-user-docs/48/gnome-user-docs-48.2.tar.xz
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/yelp-42.2-security_fix-1.patch
 https://download.gnome.org/sources/baobab/48/baobab-48.0.tar.xz
@@ -793,14 +793,14 @@ https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/brasero-3.12.3-upstream_fixes-1.patch
 https://download.gnome.org/sources/eog/47/eog-47.0.tar.xz
 https://download.gnome.org/sources/evince/48/evince-48.0.tar.xz
-https://download.gnome.org/sources/evolution/3.56/evolution-3.56.1.tar.xz
+https://download.gnome.org/sources/evolution/3.56/evolution-3.56.2.tar.xz
 https://download.gnome.org/sources/file-roller/44/file-roller-44.5.tar.xz
 https://download.gnome.org/sources/gnome-calculator/48/gnome-calculator-48.1.tar.xz
 https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-3.36.2.tar.xz
 https://download.gnome.org/sources/gnome-connections/48/gnome-connections-48.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/46/gnome-disk-utility-46.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/45/gnome-logs-45.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/48/gnome-maps-48.2.tar.xz
+https://download.gnome.org/sources/gnome-maps/48/gnome-maps-48.3.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/43/gnome-power-manager-43.0.tar.xz
@@ -900,7 +900,7 @@ https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
 https://inkscape.org/gallery/item/56344/inkscape-1.4.2.tar.xz
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/128.10.2esr/source/thunderbird-128.10.2esr.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/128.11.0esr/source/thunderbird-128.11.0esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.15.0/tigervnc-1.15.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.15.0-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
@@ -1043,9 +1043,9 @@ https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-source.tar.xz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-texmf.tar.xz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-extra.tar.xz
-https://downloads.sourceforge.net/asymptote/asymptote-3.03.src.tgz
+https://downloads.sourceforge.net/asymptote/asymptote-3.04.src.tgz
 https://github.com/plk/biber/archive/v2.20/biber-2.20.tar.gz
-https://github.com/mgieseki/dvisvgm/releases/download/3.4.4/dvisvgm-3.4.4.tar.gz
+https://github.com/mgieseki/dvisvgm/releases/download/3.5/dvisvgm-3.5.tar.gz
 https://tug.ctan.org/support/xindy/base/xindy-2.5.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.patch
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20250225.tar.xz
