diff --git a/wget-list b/wget-list
index 9a2fd6ae02..7956f3e073 100644
--- a/wget-list
+++ b/wget-list
@@ -738,7 +738,7 @@ https://download.kde.org/stable/release-service/25.04.0/src/libkdcraw-25.04.0.ta
 https://download.kde.org/stable/release-service/25.04.0/src/gwenview-25.04.0.tar.xz
 https://download.kde.org/stable/release-service/25.04.0/src/libkcddb-25.04.0.tar.xz
 https://download.kde.org/stable/release-service/25.04.0/src/k3b-25.04.0.tar.xz
-https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.4.0.tar.xz
+https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.7.0.tar.xz
 https://download.kde.org/stable/pulseaudio-qt/pulseaudio-qt-1.7.0.tar.xz
 https://download.gnome.org/sources/gcr/3.41/gcr-3.41.2.tar.xz
 https://download.gnome.org/sources/gcr/4.4/gcr-4.4.0.1.tar.xz
