diff --git a/wget-list b/wget-list
index 2ebda6ff26..5395fb6ef9 100644
--- a/wget-list
+++ b/wget-list
@@ -5,7 +5,7 @@ https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.5.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.8.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/gnutls/v3.8/gnutls-3.8.9.tar.xz
-https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.2.tar.bz2
+https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-1.24.3.tar.bz2
 https://www.netfilter.org/projects/iptables/files/iptables-1.8.11.tar.xz
 https://www.kernel.org/pub/linux/libs/security/linux-privs/libcap2/libcap-2.76.tar.xz
 https://github.com/linux-pam/linux-pam/releases/download/v1.7.0/Linux-PAM-1.7.0.tar.xz
@@ -80,7 +80,7 @@ https://libopenraw.freedesktop.org/download/exempi-2.6.6.tar.xz
 https://github.com/fastfloat/fast_float/archive/v8.0.2/fast_float-8.0.2.tar.gz
 https://www.fftw.org/fftw-3.3.10.tar.gz
 https://github.com/fmtlib/fmt/archive/11.2.0/fmt-11.2.0.tar.gz
-https://download.gnome.org/sources/glib/2.84/glib-2.84.1.tar.xz
+https://download.gnome.org/sources/glib/2.84/glib-2.84.2.tar.xz
 https://download.gnome.org/sources/gobject-introspection/1.84/gobject-introspection-1.84.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.8.tar.xz
@@ -118,7 +118,7 @@ https://ftp.gnu.org/gnu/libidn/libidn2-2.3.8.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v248/liblinear-248.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.32.0/libmbim-1.32.0.tar.gz
-https://github.com/linux-nvme/libnvme/archive/v1.13/libnvme-1.13.tar.gz
+https://github.com/linux-nvme/libnvme/archive/v1.14/libnvme-1.14.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.6/libpaper-2.2.6.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
@@ -138,7 +138,7 @@ https://github.com/libunwind/libunwind/releases/download/v1.8.1/libunwind-1.8.1.
 https://lttng.org/files/urcu/userspace-rcu-0.15.3.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.28/libusb-1.0.28.tar.bz2
 https://dist.libuv.org/dist/v1.51.0/libuv-v1.51.0.tar.gz
-https://github.com/lfs-book/libxkbcommon/archive/v1.9.2/libxkbcommon-1.9.2.tar.gz
+https://github.com/lfs-book/libxkbcommon/archive/v1.10.0/libxkbcommon-1.10.0.tar.gz
 https://download.gnome.org/sources/libxml2/2.14/libxml2-2.14.3.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.22/libxmlb-0.3.22.tar.xz
@@ -224,7 +224,7 @@ https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
 https://github.com/google/shaderc/archive/v2025.2/shaderc-2025.2.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/12.2.1/graphviz-12.2.1.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz
-http://www.andre-simon.de/zip/highlight-4.15.tar.bz2
+http://www.andre-simon.de/zip/highlight-4.16.tar.bz2
 https://github.com/ibus/ibus/archive/1.5.32/ibus-1.5.32.tar.gz
 https://www.unicode.org/Public/zipped/16.0.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-43.tar.xz
@@ -247,7 +247,7 @@ https://www.freedesktop.org/software/accountsservice/accountsservice-23.13.9.tar
 https://downloads.sourceforge.net/acpid2/acpid-2.0.34.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/at/at_3.2.5.orig.tar.gz
 https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.9.tar.xz
-https://www.kernel.org/pub/linux/bluetooth/bluez-5.79.tar.xz
+https://www.kernel.org/pub/linux/bluetooth/bluez-5.82.tar.xz
 https://github.com/containers/bubblewrap/releases/download/v0.11.0/bubblewrap-0.11.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-1.4.7.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/colord-1.4.7-upstream_fixes-1.patch
@@ -465,7 +465,7 @@ https://files.pythonhosted.org/packages/source/a/asciidoc/asciidoc-10.2.1.tar.gz
 https://files.pythonhosted.org/packages/source/b/build/build-1.2.2.post1.tar.gz
 https://files.pythonhosted.org/packages/source/c/cssselect/cssselect-1.3.0.tar.gz
 https://github.com/psf/cachecontrol/archive/v0.14.3/cachecontrol-0.14.3.tar.gz
-https://github.com/cython/cython/releases/download/3.1.0/cython-3.1.0.tar.gz
+https://github.com/cython/cython/releases/download/3.1.1/cython-3.1.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/python-dbusmock/python_dbusmock-0.34.3.tar.gz
 https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.4.0.tar.xz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.21.2.tar.gz
@@ -543,7 +543,7 @@ https://downloads.sourceforge.net/swig/swig-4.3.1.tar.gz
 https://downloads.sourceforge.net/tcl/tk8.6.16-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.18.tar.xz
-https://sourceware.org/pub/valgrind/valgrind-3.25.0.tar.bz2
+https://sourceware.org/pub/valgrind/valgrind-3.25.1.tar.bz2
 https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-24.0.1/OpenJDK-24.0.1+9-i686-bin.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/OpenJDK/OpenJDK-24.0.1/OpenJDK-24.0.1+9-x86_64-bin.tar.xz
@@ -973,7 +973,7 @@ https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
 https://taglib.org/releases/taglib-2.0.2.tar.gz
 https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.30.1.tar.xz
-https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.8/wireplumber-0.5.8.tar.bz2
+https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.10/wireplumber-0.5.10.tar.bz2
 https://anduin.linuxfromscratch.org/BLFS/x264/x264-20250212.tar.xz
 https://bitbucket.org/multicoreware/x265_git/downloads/x265_4.1.tar.gz
 https://downloads.sourceforge.net/xine/xine-lib-1.2.13.tar.xz
@@ -1037,7 +1037,7 @@ https://pagure.io/xmlto/archive/0.0.29/xmlto-0.0.29.tar.gz
 https://ftp.gnu.org/gnu/enscript/enscript-1.6.6.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/epdfview-gtk3/epdfview-gtk3-20200814.tar.xz
 https://archive.apache.org/dist/xmlgraphics/fop/source/fop-2.11-src.tar.gz
-https://www.mupdf.com/downloads/archive/mupdf-1.25.6-source.tar.gz
+https://www.mupdf.com/downloads/archive/mupdf-1.26.1-source.tar.gz
 https://github.com/dov/paps/releases/download/v0.8.0/paps-0.8.0.tar.gz
 https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz
 https://ftp.math.utah.edu/pub/tex/historic/systems/texlive/2025/texlive-20250308-source.tar.xz
