diff --git a/wget-list b/wget-list
index 8a1f251fc7..68c5717f92 100644
--- a/wget-list
+++ b/wget-list
@@ -139,7 +139,7 @@ https://lttng.org/files/urcu/userspace-rcu-0.15.2.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.28/libusb-1.0.28.tar.bz2
 https://dist.libuv.org/dist/v1.51.0/libuv-v1.51.0.tar.gz
 https://github.com/lfs-book/libxkbcommon/archive/v1.9.2/libxkbcommon-1.9.2.tar.gz
-https://download.gnome.org/sources/libxml2/2.14/libxml2-2.14.2.tar.xz
+https://download.gnome.org/sources/libxml2/2.14/libxml2-2.14.3.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.22/libxmlb-0.3.22.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.43.tar.xz
@@ -170,19 +170,19 @@ https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.44/downloa
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/1.4.29/xapian-core-1.4.29.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
-https://download.gimp.org/pub/babl/0.1/babl-0.1.112.tar.xz
+https://download.gimp.org/pub/babl/0.1/babl-0.1.114.tar.xz
 https://github.com/Exiv2/exiv2/archive/v0.28.5/exiv2-0.28.5.tar.gz
 https://downloads.sourceforge.net/freetype/freetype-2.13.3.tar.xz
 https://downloads.sourceforge.net/freetype/freetype-doc-2.13.3.tar.xz
 https://gitlab.freedesktop.org/api/v4/projects/890/packages/generic/fontconfig/2.16.2/fontconfig-2.16.2.tar.xz
 https://github.com/fribidi/fribidi/releases/download/v1.0.16/fribidi-1.0.16.tar.xz
-https://download.gimp.org/pub/gegl/0.4/gegl-0.4.58.tar.xz
+https://download.gimp.org/pub/gegl/0.4/gegl-0.4.62.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/giflib-5.2.2-upstream_fixes-1.patch
 https://github.com/Dav1dde/glad/archive/v2.0.8/glad-2.0.8.tar.gz
 https://github.com/g-truc/glm/archive/1.0.1/glm-1.0.1.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
-https://github.com/harfbuzz/harfbuzz/releases/download/11.2.0/harfbuzz-11.2.0.tar.xz
+https://github.com/harfbuzz/harfbuzz/releases/download/11.2.1/harfbuzz-11.2.1.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.2.5/jasper-version-4.2.5.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.17/lcms2-2.17.tar.gz
 https://github.com/AOMediaCodec/libavif/archive/v1.3.0/libavif-1.3.0.tar.gz
@@ -568,7 +568,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_
 https://ftp.isc.org/isc/bind9/9.20.8/bind-9.20.8.tar.xz
 https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.52.0/downloads/NetworkManager-1.52.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
-https://nmap.org/dist/nmap-7.96.tar.bz2
+https://nmap.org/dist/nmap-7.97.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.0/whois-5.6.0.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.4.6.tar.xz
@@ -900,10 +900,9 @@ https://github.com/freerdp/freerdp/archive/3.15.0/FreeRDP-3.15.0.tar.gz
 https://download.gimp.org/gimp/v3.0/gimp-3.0.2.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
-https://inkscape.org/gallery/item/53679/inkscape-1.4.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/inkscape-1.4-poppler_fixes-2.patch
+https://inkscape.org/gallery/item/56344/inkscape-1.4.2.tar.xz
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/128.10.0esr/source/thunderbird-128.10.0esr.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/128.10.1esr/source/thunderbird-128.10.1esr.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.15.0/tigervnc-1.15.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.15.0-configuration_fixes-1.patch
 https://github.com/transmission/transmission/releases/download/4.0.6/transmission-4.0.6.tar.xz
