diff --git a/wget-list b/wget-list
index 2d92ad1dde..8b92cf6af9 100644
--- a/wget-list
+++ b/wget-list
@@ -75,7 +75,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/clucene-2.3.3.4-contribs_lib-1
 https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.114.tar.gz
 https://github.com/google/double-conversion/archive/v3.3.1/double-conversion-3.3.1.tar.gz
 https://duktape.org/duktape-2.7.0.tar.xz
-https://github.com/rrthomas/enchant/releases/download/v2.8.3/enchant-2.8.3.tar.gz
+https://github.com/rrthomas/enchant/releases/download/v2.8.4/enchant-2.8.4.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.6.tar.xz
 https://github.com/fastfloat/fast_float/archive/v8.0.2/fast_float-8.0.2.tar.gz
 https://www.fftw.org/fftw-3.3.10.tar.gz
@@ -138,7 +138,7 @@ https://github.com/libunwind/libunwind/releases/download/v1.8.1/libunwind-1.8.1.
 https://lttng.org/files/urcu/userspace-rcu-0.15.2.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.28/libusb-1.0.28.tar.bz2
 https://dist.libuv.org/dist/v1.51.0/libuv-v1.51.0.tar.gz
-https://github.com/lfs-book/libxkbcommon/archive/v1.9.0/libxkbcommon-1.9.0.tar.gz
+https://github.com/lfs-book/libxkbcommon/archive/v1.9.1/libxkbcommon-1.9.1.tar.gz
 https://download.gnome.org/sources/libxml2/2.14/libxml2-2.14.2.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.22/libxmlb-0.3.22.tar.xz
