diff --git a/wget-list b/wget-list
index b3939e66e2..670bef8e86 100644
--- a/wget-list
+++ b/wget-list
@@ -725,7 +725,7 @@ https://github.com/flatpak/xdg-desktop-portal/releases/download/1.22.1/xdg-deskt
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.3/xdg-desktop-portal-gtk-1.15.3.tar.xz
 https://github.com/emersion/xdg-desktop-portal-wlr/archive/v0.8.3/xdg-desktop-portal-wlr-0.8.3.tar.gz
 https://download.gnome.org/sources/gdm/50/gdm-50.2.tar.xz
-https://github.com/ubuntu/lightdm/archive/1.32.0/lightdm-1.32.0.tar.gz
+https://github.com/ubuntu/lightdm/archive/1.33.1/lightdm-1.33.1.tar.gz
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
@@ -733,7 +733,7 @@ https://github.com/ice-wm/icewm/archive/4.1.0/icewm-4.1.0.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openbox-3.6.1-py3-1.patch
 https://download.gnome.org/sources/adwaita-icon-theme/50/adwaita-icon-theme-50.0.tar.xz
-https://download.kde.org/stable/frameworks/6.26/breeze-icons-6.26.0.tar.xz
+https://download.kde.org/stable/frameworks/6.29/breeze-icons-6.29.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme/3.12/gnome-icon-theme-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-extras/3.12/gnome-icon-theme-extras-3.12.0.tar.xz
 https://download.gnome.org/sources/gnome-icon-theme-symbolic/3.12/gnome-icon-theme-symbolic-3.12.0.tar.xz
@@ -742,7 +742,7 @@ https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.18.tar.xz
 https://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
 https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.1.0.tar.xz
-https://download.kde.org/stable/frameworks/6.26/extra-cmake-modules-6.26.0.tar.xz
+https://download.kde.org/stable/frameworks/6.29/extra-cmake-modules-6.29.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
 https://download.kde.org/stable/polkit-qt-1/polkit-qt-1-0.201.1.tar.xz
@@ -863,10 +863,10 @@ https://archive.xfce.org/src/apps/xfburn/0.8/xfburn-0.8.0.tar.bz2
 https://archive.xfce.org/src/apps/ristretto/0.14/ristretto-0.14.0.tar.xz
 https://archive.xfce.org/src/apps/xfce4-notifyd/0.9/xfce4-notifyd-0.9.7.tar.bz2
 https://archive.xfce.org/src/panel-plugins/xfce4-pulseaudio-plugin/0.5/xfce4-pulseaudio-plugin-0.5.1.tar.xz
-https://download.kde.org/stable/frameworks/6.26/kwindowsystem-6.26.0.tar.xz
-https://download.kde.org/stable/frameworks/6.26/kconfig-6.26.0.tar.xz
-https://download.kde.org/stable/frameworks/6.26/solid-6.26.0.tar.xz
-https://download.kde.org/stable/frameworks/6.26/kidletime-6.26.0.tar.xz
+https://download.kde.org/stable/frameworks/6.29/kwindowsystem-6.29.0.tar.xz
+https://download.kde.org/stable/frameworks/6.29/kconfig-6.29.0.tar.xz
+https://download.kde.org/stable/frameworks/6.29/solid-6.29.0.tar.xz
+https://download.kde.org/stable/frameworks/6.29/kidletime-6.29.0.tar.xz
 https://download.kde.org/stable/plasma/6.6.5/kwayland-6.6.5.tar.xz
 https://download.kde.org/stable/plasma/6.6.5/libkscreen-6.6.5.tar.xz
 https://download.kde.org/stable/plasma/6.6.5/layer-shell-qt-6.6.5.tar.xz
