diff --git a/wget-list b/wget-list
index 670bef8e86..bb69dd597f 100644
--- a/wget-list
+++ b/wget-list
@@ -55,7 +55,7 @@ https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/26.04.1/src/kate-26.04.1.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.7/mousepad-0.7.0.tar.xz
 https://www.nano-editor.org/dist/v9/nano-9.2.tar.xz
-https://github.com/vim/vim/archive/v9.2.0858/vim-9.2.0858.tar.gz
+https://github.com/vim/vim/archive/v9.2.0954/vim-9.2.0954.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.5.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.2.tar.xz
@@ -587,7 +587,7 @@ https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p18.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/ntp-4.2.8p18-openssl_4-1.patch
 https://downloads.sourceforge.net/rpcbind/rpcbind-1.2.9.tar.bz2
 https://www.samba.org/ftp/rsync/src/rsync-3.5.0.tar.gz
-https://download.samba.org/pub/samba/stable/samba-4.24.5.tar.gz
+https://download.samba.org/pub/samba/stable/samba-4.24.6.tar.gz
 https://ftpmirror.gnu.org/wget/wget-1.25.0.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.17.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
@@ -595,8 +595,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_s
 https://w1.fi/releases/wpa_supplicant-2.12.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
-https://ftp.isc.org/isc/bind9/9.20.26/bind-9.20.26.tar.xz
-https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.58.0/downloads/NetworkManager-1.58.0.tar.xz
+https://ftp.isc.org/isc/bind9/9.20.27/bind-9.20.27.tar.xz
+https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.58.1/downloads/NetworkManager-1.58.1.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
 https://nmap.org/dist/nmap-7.991.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
@@ -645,7 +645,7 @@ https://ghostarchive.org/postfix/postfix-release/official/postfix-3.11.6.tar.gz
 https://git.openldap.org/openldap/openldap/-/archive/LMDB_1.0.1/openldap-LMDB_1.0.1.tar.bz2
 https://downloads.mariadb.org/interstitial/mariadb-12.3.2/source/mariadb-12.3.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/mariadb-12.3.2-openssl4_fixes-1.patch
-https://ftp.postgresql.org/pub/source/v18.4/postgresql-18.4.tar.bz2
+https://ftp.postgresql.org/pub/source/v18.6/postgresql-18.6.tar.bz2
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.7.0.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.7.0-consolidated-1.patch
 https://www.x.org/pub/individual/util/util-macros-1.20.2.tar.xz
@@ -676,8 +676,7 @@ https://invisible-mirror.net/archives/xterm/xterm-410.tgz
 https://www.x.org/pub/individual/app/xclock-1.2.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://github.com/WayfireWM/wayfire/releases/download/v0.11.0/wayfire-0.11.0.tar.xz
-https://github.com/WayfireWM/wayfire-plugins-extra/releases/download/v0.10.0/wayfire-plugins-extra-0.10.0.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/wayfire-plugins-extra-0.10.0-upstream_fixes-1.patch
+https://github.com/WayfireWM/wayfire-plugins-extra/releases/download/v0.11.2/wayfire-plugins-extra-0.11.2.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.5.tar.xz
 https://download.gnome.org/sources/at-spi2-core/2.60/at-spi2-core-2.60.6.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.4.tar.xz
@@ -741,7 +740,7 @@ https://download.gnome.org/sources/gnome-themes-extra/3.28/gnome-themes-extra-3.
 https://icon-theme.freedesktop.org/releases/hicolor-icon-theme-0.18.tar.xz
 https://tango.freedesktop.org/releases/icon-naming-utils-0.8.90.tar.bz2
 https://downloads.sourceforge.net/lxde/lxde-icon-theme-0.5.1.tar.xz
-https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.1.0.tar.xz
+https://download.kde.org/stable/oxygen-icons/oxygen-icons-6.2.0.tar.xz
 https://download.kde.org/stable/frameworks/6.29/extra-cmake-modules-6.29.0.tar.xz
 https://download.kde.org/stable/phonon/4.12.0/phonon-4.12.0.tar.xz
 https://download.kde.org/stable/phonon/phonon-backend-vlc/0.12.0/phonon-backend-vlc-0.12.0.tar.xz
@@ -810,11 +809,11 @@ https://download.gnome.org/sources/tecla/50/tecla-50.0.tar.xz
 https://download.gnome.org/sources/gnome-control-center/50/gnome-control-center-50.4.tar.xz
 https://download.gnome.org/sources/mutter/50/mutter-50.4.tar.xz
 https://download.gnome.org/sources/gnome-shell/50/gnome-shell-50.4.tar.xz
-https://download.gnome.org/sources/gnome-shell-extensions/50/gnome-shell-extensions-50.2.tar.xz
+https://download.gnome.org/sources/gnome-shell-extensions/50/gnome-shell-extensions-50.3.tar.xz
 https://download.gnome.org/sources/gnome-session/50/gnome-session-50.1.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/49/gnome-tweaks-49.0.tar.xz
 https://download.gnome.org/sources/gnome-user-docs/50/gnome-user-docs-50.4.tar.xz
-https://download.gnome.org/sources/yelp/49/yelp-49.1.tar.xz
+https://download.gnome.org/sources/yelp/49/yelp-49.2.tar.xz
 https://download.gnome.org/sources/baobab/50/baobab-50.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/brasero-3.12.3-upstream_fixes-1.patch
@@ -826,7 +825,7 @@ https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-
 https://download.gnome.org/sources/gnome-connections/50/gnome-connections-50.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/46/gnome-disk-utility-46.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/45/gnome-logs-45.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/50/gnome-maps-50.3.tar.xz
+https://download.gnome.org/sources/gnome-maps/50/gnome-maps-50.4.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/50/gnome-power-manager-50.0.tar.xz
@@ -911,7 +910,7 @@ https://download.documentfoundation.org/libreoffice/src/26.2.5/libreoffice-26.2.
 https://download.documentfoundation.org/libreoffice/src/26.2.5/libreoffice-dictionaries-26.2.5.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/26.2.5/libreoffice-help-26.2.5.2.tar.xz
 https://download.documentfoundation.org/libreoffice/src/26.2.5/libreoffice-translations-26.2.5.2.tar.xz
-https://download.gnome.org/sources/epiphany/50/epiphany-50.4.tar.xz
+https://download.gnome.org/sources/epiphany/50/epiphany-50.5.tar.xz
 https://download.kde.org/stable/release-service/26.04.1/src/falkon-26.04.1.tar.xz
 https://archive.mozilla.org/pub/firefox/releases/153.1.0esr/source/firefox-153.1.0esr.source.tar.xz
 https://archive.seamonkey-project.org/releases/2.53.24/source/seamonkey-2.53.24.source.tar.xz
@@ -947,7 +946,7 @@ https://github.com/lfs-book/faac/archive/v2.1/faac-2.1.tar.gz
 https://github.com/knik0/faad2/archive/2.11.3/faad2-2.11.3.tar.gz
 https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://github.com/xiph/flac/releases/download/1.5.0/flac-1.5.0.tar.xz
-https://github.com/dyne/frei0r/archive/v3.3.0/frei0r-3.3.0.tar.gz
+https://github.com/dyne/frei0r/archive/v3.4.0/frei0r-3.4.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
 https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.28.6.tar.xz
 https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.28.6.tar.xz
@@ -1085,7 +1084,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/xindy-2.5.1-upstream_fixes-2.p
 https://anduin.linuxfromscratch.org/BLFS/blfs-bootscripts/blfs-bootscripts-20251220.tar.xz
 https://github.com/elogind/elogind/archive/v257.16/elogind-257.16.tar.gz
 https://github.com/lfs-book/blocaled/releases/download/v0.7/blocaled-0.7.tar.xz
-https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.5.0/dhcpcd-10.5.0.tar.xz
+https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.5.2/dhcpcd-10.5.2.tar.xz
 https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.xz
 https://nlnetlabs.nl/downloads/unbound/unbound-1.25.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/unbound-1.25.1-openssl4-1.patch
