diff --git a/wget-list b/wget-list
index 2a94888dfe..8df1b40086 100644
--- a/wget-list
+++ b/wget-list
@@ -153,7 +153,7 @@ https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
 https://nodejs.org/dist/v24.19.0/node-v24.19.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
-https://archive.mozilla.org/pub/nspr/releases/v4.39/src/nspr-4.39.tar.gz
+https://archive.mozilla.org/pub/nspr/releases/v4.40/src/nspr-4.40.tar.gz
 https://github.com/AcademySoftwareFoundation/OpenTimelineIO/archive/v0.18.1/OpenTimelineIO-0.18.1.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/OpenTimelineIO/OpenTimelineIO-rapidjson-20260513.tar.xz
 https://ftp.osuosl.org/pub/rpm/popt/releases/popt-1.x/popt-1.19.tar.gz
@@ -180,7 +180,7 @@ https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/2.0.0/xapian-core-2.0.0.tar.xz
 https://github.com/ibireme/yyjson/archive/0.12.0/yyjson-0.12.0.tar.gz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
-https://download.gimp.org/pub/babl/0.1/babl-0.1.126.tar.xz
+https://download.gimp.org/pub/babl/0.1/babl-0.1.128.tar.xz
 https://github.com/Exiv2/exiv2/archive/v0.28.8/exiv2-0.28.8.tar.gz
 https://downloads.sourceforge.net/freetype/freetype-2.14.3.tar.xz
 https://downloads.sourceforge.net/freetype/freetype-doc-2.14.3.tar.xz
@@ -223,7 +223,7 @@ https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
 https://poppler.freedesktop.org/poppler-26.08.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
-https://github.com/qpdf/qpdf/releases/download/v12.3.2/qpdf-12.3.2.tar.gz
+https://github.com/qpdf/qpdf/releases/download/v12.4.0/qpdf-12.4.0.tar.gz
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.7.2.tar.gz
@@ -496,11 +496,11 @@ https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/html5lib-1.1-python_3.14_buildfix-1.patch
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.1.1.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.4.1.tar.gz
-https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.5.1.tar.gz
+https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.5.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.2.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
 https://download.gnome.org/sources/pyatspi/2.58/pyatspi-2.58.2.tar.xz
-https://github.com/pygobject/pycairo/releases/download/v1.29.0/pycairo-1.29.0.tar.gz
+https://github.com/pygobject/pycairo/releases/download/v1.29.1/pycairo-1.29.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pygdbmi/pygdbmi-0.11.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.20.0.tar.gz
 https://download.gnome.org/sources/pygobject/3.56/pygobject-3.56.3.tar.gz
@@ -591,15 +591,13 @@ https://ftpmirror.gnu.org/wget/wget-1.25.0.tar.gz
 https://www.kernel.org/pub/software/network/iw/iw-6.17.tar.xz
 https://hewlettpackard.github.io/wireless-tools/wireless_tools.29.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/wireless_tools-29-fix_iwlist_scanning-1.patch
-https://w1.fi/releases/wpa_supplicant-2.11.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/wpa_supplicant-2.11-openssl_4_fixes-1.patch
+https://w1.fi/releases/wpa_supplicant-2.12.tar.gz
 https://github.com/lathiat/avahi/releases/download/v0.8/avahi-0.8.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/avahi-0.8-ipv6_race_condition_fix-1.patch
 https://ftp.isc.org/isc/bind9/9.20.26/bind-9.20.26.tar.xz
 https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/releases/1.58.0/downloads/NetworkManager-1.58.0.tar.xz
 https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-applet-1.34.0.tar.xz
-https://nmap.org/dist/nmap-7.99.tar.bz2
-https://www.linuxfromscratch.org/patches/blfs/svn/nmap-7.99-openssl4_fixes-1.patch
+https://nmap.org/dist/nmap-7.991.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.6.tar.gz
 https://github.com/rfc1036/whois/archive/v5.6.6/whois-5.6.6.tar.gz
 https://www.wireshark.org/download/src/all-versions/wireshark-4.6.7.tar.xz
@@ -643,7 +641,7 @@ https://github.com/proftpd/proftpd/archive/v1.3.9b/proftpd-1.3.9b.tar.gz
 https://www.dovecot.org/releases/2.4/dovecot-2.4.4.tar.gz
 https://ftp.exim.org/pub/exim/exim4/exim-4.99.4.tar.xz
 https://ghostarchive.org/postfix/postfix-release/official/postfix-3.11.5.tar.gz
-https://git.openldap.org/openldap/openldap/-/archive/LMDB_1.0.0/openldap-LMDB_1.0.0.tar.bz2
+https://git.openldap.org/openldap/openldap/-/archive/LMDB_1.0.1/openldap-LMDB_1.0.1.tar.bz2
 https://downloads.mariadb.org/interstitial/mariadb-12.3.2/source/mariadb-12.3.2.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/mariadb-12.3.2-openssl4_fixes-1.patch
 https://ftp.postgresql.org/pub/source/v18.4/postgresql-18.4.tar.bz2
@@ -726,12 +724,12 @@ https://gitlab.freedesktop.org/wlroots/wlroots/-/archive/0.20.2/wlroots-0.20.2.t
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.22.1/xdg-desktop-portal-1.22.1.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.3/xdg-desktop-portal-gtk-1.15.3.tar.xz
 https://github.com/emersion/xdg-desktop-portal-wlr/archive/v0.8.3/xdg-desktop-portal-wlr-0.8.3.tar.gz
-https://download.gnome.org/sources/gdm/50/gdm-50.1.tar.xz
+https://download.gnome.org/sources/gdm/50/gdm-50.2.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
 https://github.com/Xubuntu/lightdm-gtk-greeter/releases/download/lightdm-gtk-greeter-2.0.9/lightdm-gtk-greeter-2.0.9.tar.gz
 https://github.com/sddm/sddm/archive/v0.21.0/sddm-0.21.0.tar.gz
 https://downloads.sourceforge.net/fluxbox/fluxbox-1.3.7.tar.xz
-https://github.com/ice-wm/icewm/archive/4.0.0/icewm-4.0.0.tar.gz
+https://github.com/ice-wm/icewm/archive/4.1.0/icewm-4.1.0.tar.gz
 http://openbox.org/dist/openbox/openbox-3.6.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/openbox-3.6.1-py3-1.patch
 https://download.gnome.org/sources/adwaita-icon-theme/50/adwaita-icon-theme-50.0.tar.xz
@@ -810,8 +808,8 @@ https://download.gnome.org/sources/gnome-keyring/50/gnome-keyring-50.0.tar.xz
 https://download.gnome.org/sources/gnome-settings-daemon/50/gnome-settings-daemon-50.1.tar.xz
 https://download.gnome.org/sources/tecla/50/tecla-50.0.tar.xz
 https://download.gnome.org/sources/gnome-control-center/50/gnome-control-center-50.4.tar.xz
-https://download.gnome.org/sources/mutter/50/mutter-50.3.tar.xz
-https://download.gnome.org/sources/gnome-shell/50/gnome-shell-50.3.tar.xz
+https://download.gnome.org/sources/mutter/50/mutter-50.4.tar.xz
+https://download.gnome.org/sources/gnome-shell/50/gnome-shell-50.4.tar.xz
 https://download.gnome.org/sources/gnome-shell-extensions/50/gnome-shell-extensions-50.2.tar.xz
 https://download.gnome.org/sources/gnome-session/50/gnome-session-50.1.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/49/gnome-tweaks-49.0.tar.xz
