diff --git a/wget-list b/wget-list
index 1c16a01714..5f7b1613a3 100644
--- a/wget-list
+++ b/wget-list
@@ -54,7 +54,7 @@ https://ftpmirror.gnu.org/emacs/emacs-30.2.tar.xz
 https://downloads.sourceforge.net/joe-editor/joe-4.6.tar.gz
 https://download.kde.org/stable/release-service/26.04.1/src/kate-26.04.1.tar.xz
 https://archive.xfce.org/src/apps/mousepad/0.7/mousepad-0.7.0.tar.xz
-https://www.nano-editor.org/dist/v9/nano-9.1.tar.xz
+https://www.nano-editor.org/dist/v9/nano-9.2.tar.xz
 https://github.com/vim/vim/archive/v9.2.0858/vim-9.2.0858.tar.gz
 http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.13.5.tar.gz
 https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
@@ -169,7 +169,7 @@ https://archive.mozilla.org/pub/firefox/releases/140.13.0esr/source/firefox-140.
 https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.13.0-python_3.14_fixes-1.patch
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.357.0/SPIRV-Headers-vulkan-sdk-1.4.357.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.357.0/SPIRV-Tools-vulkan-sdk-1.4.357.0.tar.gz
-https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.4/SPIRV-LLVM-Translator-22.1.4.tar.gz
+https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.5/SPIRV-LLVM-Translator-22.1.5.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.4.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.8/umockdev-0.19.8.tar.xz
@@ -193,7 +193,7 @@ https://github.com/g-truc/glm/archive/1.0.3/glm-1.0.3.tar.gz
 https://download.gnome.org/sources/glycin/2.1/glycin-2.1.5.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glycin-2.1.5-xorg_prefix-1.patch
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
-https://github.com/harfbuzz/harfbuzz/releases/download/14.2.1/harfbuzz-14.2.1.tar.xz
+https://github.com/harfbuzz/harfbuzz/releases/download/14.3.0/harfbuzz-14.3.0.tar.xz
 https://github.com/AcademySoftwareFoundation/Imath/releases/download/v3.2.2/Imath-3.2.2.tar.gz
 https://github.com/jasper-software/jasper/archive/version-4.2.9/jasper-version-4.2.9.tar.gz
 https://github.com/mm2/Little-CMS/releases/download/lcms2.19.1/lcms2-2.19.1.tar.gz
@@ -227,7 +227,7 @@ https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
 https://download.osgeo.org/libtiff/tiff-4.7.2.tar.gz
 https://github.com/google/woff2/archive/v1.0.2/woff2-1.0.2.tar.gz
-https://github.com/zxing-cpp/zxing-cpp/releases/download/v3.1.0/zxing-cpp-3.1.0.tar.gz
+https://github.com/zxing-cpp/zxing-cpp/releases/download/v3.1.1/zxing-cpp-3.1.1.tar.gz
 https://github.com/asciidoctor/asciidoctor/archive/v2.0.26/asciidoctor-2.0.26.tar.gz
 https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
@@ -271,7 +271,7 @@ http://fcron.free.fr/archives/fcron-3.4.1.src.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/gpm/gpm-1.20.7.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-consolidated-1.patch
 https://www.linuxfromscratch.org/patches/blfs/svn/gpm-1.20.7-gcc15_fixes-1.patch
-https://github.com/vcrhonek/hwdata/archive/v0.409/hwdata-0.409.tar.gz
+https://github.com/vcrhonek/hwdata/archive/v0.410/hwdata-0.410.tar.gz
 https://github.com/lfs-book/LSB-Tools/releases/download/v0.12/LSB-Tools-0.12.tar.gz
 https://github.com/logrotate/logrotate/releases/download/3.22.0/logrotate-3.22.0.tar.xz
 http://ftp.midnight-commander.org/mc-4.8.33.tar.xz
@@ -303,7 +303,7 @@ https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-019.tar.xz
 https://github.com/lu-zero/cargo-c/archive/v0.10.24/cargo-c-0.10.24.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.29.4/cbindgen-0.29.4.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/clisp/clisp-2.49.95.tar.xz
-https://cmake.org/files/v4.4/cmake-4.4.1.tar.gz
+https://cmake.org/files/v4.4/cmake-4.4.2.tar.gz
 https://doxygen.nl/files/doxygen-1.17.0.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.8.1.tar.xz
 https://ftpmirror.gnu.org/gcc/gcc-16.1.0/gcc-16.1.0.tar.xz
@@ -675,8 +675,9 @@ https://www.x.org/pub/individual/app/twm-1.0.13.1.tar.xz
 https://invisible-mirror.net/archives/xterm/xterm-410.tgz
 https://www.x.org/pub/individual/app/xclock-1.2.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
-https://github.com/WayfireWM/wayfire/releases/download/v0.10.1/wayfire-0.10.1.tar.xz
+https://github.com/WayfireWM/wayfire/releases/download/v0.11.0/wayfire-0.11.0.tar.xz
 https://github.com/WayfireWM/wayfire-plugins-extra/releases/download/v0.10.0/wayfire-plugins-extra-0.10.0.tar.xz
+https://www.linuxfromscratch.org/patches/blfs/svn/wayfire-plugins-extra-0.10.0-upstream_fixes-1.patch
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.5.tar.xz
 https://download.gnome.org/sources/at-spi2-core/2.60/at-spi2-core-2.60.5.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.4.tar.xz
@@ -717,10 +718,10 @@ https://download.qt.io/archive/qt/6.11/6.11.1/single/qt-everywhere-src-6.11.1.ta
 https://www.linuxfromscratch.org/patches/blfs/svn/qt-everywhere-src-6.11.1-openssl4_fix-1.patch
 https://download.qt.io/official_releases/qt/6.11/6.11.1/submodules/qtwebengine-everywhere-src-6.11.1.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/vulkan-sdk-1.4.350.1/Vulkan-Headers-vulkan-sdk-1.4.350.1.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/vulkan-sdk-1.4.350.1/Vulkan-Loader-vulkan-sdk-1.4.350.1.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/vulkan-sdk-1.4.357.0/Vulkan-Headers-vulkan-sdk-1.4.357.0.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/vulkan-sdk-1.4.357.0/Vulkan-Loader-vulkan-sdk-1.4.357.0.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.52.5.tar.xz
-https://gitlab.freedesktop.org/wlroots/wlroots/-/archive/0.19.3/wlroots-0.19.3.tar.bz2
+https://gitlab.freedesktop.org/wlroots/wlroots/-/archive/0.20.2/wlroots-0.20.2.tar.bz2
 https://github.com/flatpak/xdg-desktop-portal/releases/download/1.22.1/xdg-desktop-portal-1.22.1.tar.xz
 https://github.com/flatpak/xdg-desktop-portal-gtk/releases/download/1.15.3/xdg-desktop-portal-gtk-1.15.3.tar.xz
 https://github.com/emersion/xdg-desktop-portal-wlr/archive/v0.8.3/xdg-desktop-portal-wlr-0.8.3.tar.gz
@@ -800,14 +801,14 @@ https://download.gnome.org/sources/xdg-desktop-portal-gnome/50/xdg-desktop-porta
 https://download.gnome.org/sources/dconf/0.49/dconf-0.49.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/49/dconf-editor-49.0.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/50/gnome-backgrounds-50.0.tar.xz
-https://download.gnome.org/sources/gvfs/1.60/gvfs-1.60.1.tar.xz
+https://download.gnome.org/sources/gvfs/1.60/gvfs-1.60.2.tar.xz
 https://download.gnome.org/sources/gexiv2/0.16/gexiv2-0.16.1.tar.xz
 https://download.gnome.org/sources/nautilus/50/nautilus-50.2.2.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/47/gnome-bluetooth-47.2.tar.xz
 https://download.gnome.org/sources/gnome-keyring/50/gnome-keyring-50.0.tar.xz
 https://download.gnome.org/sources/gnome-settings-daemon/50/gnome-settings-daemon-50.1.tar.xz
 https://download.gnome.org/sources/tecla/50/tecla-50.0.tar.xz
-https://download.gnome.org/sources/gnome-control-center/50/gnome-control-center-50.3.tar.xz
+https://download.gnome.org/sources/gnome-control-center/50/gnome-control-center-50.4.tar.xz
 https://download.gnome.org/sources/mutter/50/mutter-50.3.tar.xz
 https://download.gnome.org/sources/gnome-shell/50/gnome-shell-50.3.tar.xz
 https://download.gnome.org/sources/gnome-shell-extensions/50/gnome-shell-extensions-50.2.tar.xz
@@ -969,7 +970,7 @@ https://downloads.sourceforge.net/libcddb/libcddb-1.3.2.tar.bz2
 https://ftpmirror.gnu.org/libcdio/libcdio-2.1.0.tar.bz2
 https://ftpmirror.gnu.org/libcdio/libcdio-paranoia-10.2+2.0.2.tar.bz2
 https://github.com/strukturag/libde265/releases/download/v1.1.1/libde265-1.1.1.tar.gz
-https://get.videolan.org/libdvdcss/1.5.0/libdvdcss-1.5.0.tar.xz
+https://get.videolan.org/libdvdcss/1.6.0/libdvdcss-1.6.0.tar.xz
 https://get.videolan.org/libdvdread/7.1.1/libdvdread-7.1.1.tar.xz
 https://get.videolan.org/libdvdnav/7.0.0/libdvdnav-7.0.0.tar.xz
 https://downloads.sourceforge.net/libdv/libdv-1.0.0.tar.gz
