diff --git a/wget-list b/wget-list
index d5a3170fcf..3e2e5dcdfa 100644
--- a/wget-list
+++ b/wget-list
@@ -151,7 +151,7 @@ https://github.com/pantoniou/libfyaml/releases/download/v0.9.6/libfyaml-0.9.6.ta
 https://github.com/log4cplus/log4cplus/releases/download/REL_2_2_0_1/log4cplus-2.2.0.1.tar.xz
 https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
 https://bitmath.org/code/mtdev/mtdev-1.1.7.tar.bz2
-https://nodejs.org/dist/v24.18.1/node-v24.18.1.tar.xz
+https://nodejs.org/dist/v24.19.0/node-v24.19.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/npth/npth-1.8.tar.bz2
 https://archive.mozilla.org/pub/nspr/releases/v4.39/src/nspr-4.39.tar.gz
 https://github.com/AcademySoftwareFoundation/OpenTimelineIO/archive/v0.18.1/OpenTimelineIO-0.18.1.tar.gz
@@ -170,7 +170,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.13.0-python_3
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.357.0/SPIRV-Headers-vulkan-sdk-1.4.357.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/vulkan-sdk-1.4.357.0/SPIRV-Tools-vulkan-sdk-1.4.357.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/v22.1.5/SPIRV-LLVM-Translator-22.1.5.tar.gz
-https://www.samba.org/ftp/talloc/talloc-2.4.4.tar.gz
+https://www.samba.org/ftp/talloc/talloc-2.5.0.tar.gz
 https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.8.tar.xz
 https://github.com/martinpitt/umockdev/releases/download/0.19.8/umockdev-0.19.8.tar.xz
 https://github.com/nemtrif/utfcpp/archive/v4.1.1/utfcpp-4.1.1.tar.gz
@@ -219,7 +219,7 @@ https://github.com/opencv/opencv/archive/4.13.0/opencv-4.13.0.tar.gz
 https://github.com/opencv/opencv_contrib/archive/4.13.0/opencv_contrib-4.13.0.tar.gz
 https://github.com/uclouvain/openjpeg/archive/v2.5.4/openjpeg-2.5.4.tar.gz
 https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
-https://poppler.freedesktop.org/poppler-26.07.0.tar.xz
+https://poppler.freedesktop.org/poppler-26.08.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
 https://github.com/qpdf/qpdf/releases/download/v12.3.2/qpdf-12.3.2.tar.gz
@@ -688,7 +688,7 @@ https://github.com/fltk/fltk/releases/download/release-1.3.11/fltk-1.3.11-source
 https://downloads.sourceforge.net/freeglut/freeglut-3.8.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.7.tar.xz
 https://downloads.sourceforge.net/glew/glew-2.3.1.tgz
-https://github.com/KhronosGroup/glslang/archive/16.4.0/glslang-16.4.0.tar.gz
+https://github.com/KhronosGroup/glslang/archive/16.5.0/glslang-16.5.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.61.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
@@ -704,7 +704,7 @@ https://downloads.sourceforge.net/enlightenment/imlib2-1.12.6.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.2/kImageAnnotator-0.7.2.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
-https://download.gnome.org/sources/libadwaita/1.9/libadwaita-1.9.2.tar.xz
+https://download.gnome.org/sources/libadwaita/1.9/libadwaita-1.9.3.tar.xz
 https://gitlab.freedesktop.org/libinput/libei/-/archive/1.6.0/libei-1.6.0.tar.bz2
 https://download.gnome.org/sources/libhandy/1.8/libhandy-1.8.3.tar.xz
 https://dri.freedesktop.org/libdrm/libdrm-2.4.134.tar.xz
@@ -993,8 +993,8 @@ https://gitlab.freedesktop.org/pipewire/pipewire/-/archive/1.6.8/pipewire-1.6.8.
 https://www.freedesktop.org/software/pulseaudio/releases/pulseaudio-17.0.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/sbc-2.2.tar.xz
 https://www.libsdl.org/release/sdl2-compat-2.32.70.tar.gz
-https://www.libsdl.org/release/SDL3-3.4.12.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/SDL3-3.4.12-upstream_fixes-1.patch
+https://www.libsdl.org/release/SDL3-3.4.14.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/SDL3-3.4.14-upstream_fixes-1.patch
 https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.4.1.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
@@ -1016,7 +1016,7 @@ https://downloads.xiph.org/releases/cdparanoia/cdparanoia-III-10.2.src.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/cdparanoia-III-10.2-gcc_fixes-1.patch
 https://download.kde.org/stable/release-service/26.04.1/src/kwave-26.04.1.tar.xz
 https://downloads.sourceforge.net/lame/lame-3.100.tar.gz
-https://downloads.sourceforge.net/mpg123/mpg123-1.33.6.tar.bz2
+https://downloads.sourceforge.net/mpg123/mpg123-1.33.7.tar.bz2
 https://www.freedesktop.org/software/pulseaudio/pavucontrol/pavucontrol-6.2.tar.xz
 https://github.com/nicklan/pnmixer/releases/download/v0.7.2/pnmixer-v0.7.2.tar.gz
 https://downloads.xiph.org/releases/vorbis/vorbis-tools-1.4.3.tar.gz
