diff --git a/wget-list b/wget-list
index c878c96..b746ef2 100644
--- a/wget-list
+++ b/wget-list
@@ -25,7 +25,7 @@ https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
 https://github.com/shadow-maint/shadow/releases/download/4.14.2/shadow-4.14.2.tar.xz
 https://www.stunnel.org/downloads/archive/5.x/stunnel-5.71.tar.gz
-https://www.sudo.ws/dist/sudo-1.9.15p4.tar.gz
+https://www.sudo.ws/dist/sudo-1.9.15p5.tar.gz
 https://github.com/Tripwire/tripwire-open-source/releases/download/2.4.3.7/tripwire-open-source-2.4.3.7.tar.gz
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v6.6.1.tar.xz
 https://github.com/dosfstools/dosfstools/releases/download/v4.2/dosfstools-4.2.tar.gz
@@ -524,7 +524,7 @@ https://static.rust-lang.org/dist/rustc-1.74.1-src.tar.xz
 https://downloads.sourceforge.net/scons/SCons-4.6.0.tar.gz
 https://www.jedsoft.org/releases/slang/slang-2.3.3.tar.bz2
 https://archive.apache.org/dist/subversion/subversion-1.14.3.tar.bz2
-https://downloads.sourceforge.net/swig/swig-4.1.1.tar.gz
+https://downloads.sourceforge.net/swig/swig-4.2.0.tar.gz
 https://downloads.sourceforge.net/tcl/tk8.6.13-src.tar.gz
 https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
 https://download.gnome.org/sources/vala/0.56/vala-0.56.14.tar.xz
@@ -805,7 +805,7 @@ https://archive.xfce.org/src/xfce/libxfce4ui/4.18/libxfce4ui-4.18.4.tar.bz2
 https://archive.xfce.org/src/xfce/exo/4.18/exo-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/garcon/4.18/garcon-4.18.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-panel/4.18/xfce4-panel-4.18.5.tar.bz2
-https://archive.xfce.org/src/xfce/thunar/4.18/thunar-4.18.9.tar.bz2
+https://archive.xfce.org/src/xfce/thunar/4.18/thunar-4.18.10.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.18/thunar-volman-4.18.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.18/tumbler-4.18.2.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.18/xfce4-appfinder-4.18.1.tar.bz2
