diff --git a/wget-list b/wget-list
index 9ee124f4..160365cf 100644
--- a/wget-list
+++ b/wget-list
@@ -113,7 +113,7 @@ https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.5.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v247/liblinear-247.tar.gz
 https://www.freedesktop.org/software/libmbim/libmbim-1.26.4.tar.xz
 https://github.com/linux-nvme/libnvme/archive/refs/tags/v1.7.1/libnvme-1.7.1.tar.gz
-https://github.com/rrthomas/libpaper/releases/download/v2.1.2/libpaper-2.1.2.tar.gz
+https://github.com/rrthomas/libpaper/releases/download/v2.1.3/libpaper-2.1.3.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.7.1/libportal-0.7.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.7.1-testsuite_fix-1.patch
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
@@ -150,7 +150,7 @@ https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.42/pcre2-10.42.
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://ftp.gnu.org/gnu/pth/pth-2.0.7.tar.gz
 https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
-https://archive.mozilla.org/pub/firefox/releases/115.6.0esr/source/firefox-115.6.0esr.source.tar.xz
+https://archive.mozilla.org/pub/firefox/releases/115.7.0esr/source/firefox-115.7.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Headers-1.3.275.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/vulkan-sdk-1.3.275.0/SPIRV-Tools-1.3.275.0.tar.gz
 https://www.samba.org/ftp/talloc/talloc-2.4.1.tar.gz
@@ -203,13 +203,13 @@ https://github.com/qpdf/qpdf/releases/download/v11.8.0/qpdf-11.8.0.tar.gz
 https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.bz2
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
-https://github.com/aruiz/webp-pixbuf-loader/archive/0.2.4/webp-pixbuf-loader-0.2.4.tar.gz
+https://github.com/aruiz/webp-pixbuf-loader/archive/0.2.5/webp-pixbuf-loader-0.2.5.tar.gz
 https://github.com/google/woff2/archive/v1.0.2/woff2-1.0.2.tar.gz
 https://github.com/asciidoctor/asciidoctor/archive/v2.0.20/asciidoctor-2.0.20.tar.gz
 https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.27.tar.xz
-https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.1.tar.gz
+https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.2.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/9.0.0/graphviz-9.0.0.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.33/gtk-doc-1.33.2.tar.xz
 http://www.andre-simon.de/zip/highlight-4.10.tar.bz2
@@ -237,7 +237,7 @@ https://anduin.linuxfromscratch.org/BLFS/at/at_3.2.5.orig.tar.gz
 https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.9.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/bluez-5.72.tar.xz
 https://github.com/containers/bubblewrap/releases/download/v0.8.0/bubblewrap-0.8.0.tar.xz
-https://www.freedesktop.org/software/colord/releases/colord-1.4.6.tar.xz
+https://www.freedesktop.org/software/colord/releases/colord-1.4.7.tar.xz
 https://ftp.gnu.org/gnu/cpio/cpio-2.15.tar.bz2
 https://www.freedesktop.org/software/cups-pk-helper/releases/cups-pk-helper-0.2.7.tar.xz
 https://dbus.freedesktop.org/releases/dbus/dbus-1.14.10.tar.xz
@@ -317,16 +317,16 @@ https://www.cpan.org/authors/id/L/LE/LEONT/File-Slurper-0.014.tar.gz
 https://www.cpan.org/authors/id/P/PL/PLICEASE/File-Which-1.27.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/HTML-Parser-3.81.tar.gz
 https://www.cpan.org/authors/id/O/OA/OALDERS/HTTP-Daemon-6.16.tar.gz
-https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.084.tar.gz
+https://www.cpan.org/authors/id/S/SU/SULLR/IO-Socket-SSL-2.085.tar.gz
 https://www.cpan.org/authors/id/G/GA/GAAS/IO-String-1.08.tar.gz
 https://www.cpan.org/authors/id/R/RJ/RJBS/IPC-Run3-0.049.tar.gz
 https://www.cpan.org/authors/id/I/IS/ISHIGAKI/JSON-4.10.tar.gz
 https://www.cpan.org/authors/id/A/AL/ALINKE/Lingua-Translit-0.29.tar.gz
-https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.73.tar.gz
+https://www.cpan.org/authors/id/O/OA/OALDERS/libwww-perl-6.75.tar.gz
 https://www.cpan.org/authors/id/D/DR/DROLSKY/List-AllUtils-0.19.tar.gz
 https://www.cpan.org/authors/id/R/RE/REHSACK/List-MoreUtils-0.430.tar.gz
 https://www.cpan.org/authors/id/E/ET/ETJ/Log-Log4perl-1.57.tar.gz
-https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.11.tar.gz
+https://www.cpan.org/authors/id/O/OA/OALDERS/LWP-Protocol-https-6.12.tar.gz
 https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4232.tar.gz
 https://www.cpan.org/authors/id/N/NL/NLNETLABS/Net-DNS-1.42.tar.gz
 https://www.cpan.org/authors/id/J/JT/JTBRAUN/Parse-RecDescent-1.967015.tar.gz
@@ -463,7 +463,7 @@ https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.g
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
 https://files.pythonhosted.org/packages/source/g/gi-docgen/gi-docgen-2023.3.tar.gz
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-4.9.4.tar.gz
-https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.0.tar.gz
+https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.1.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.3.tar.gz
 https://files.pythonhosted.org/packages/source/p/packaging/packaging-23.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
@@ -877,7 +877,6 @@ https://download.documentfoundation.org/libreoffice/src/7.6.4/libreoffice-transl
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-7.6.4.1-consolidated_fixes-1.patch
 https://download.gnome.org/sources/epiphany/45/epiphany-45.2.tar.xz
 https://download.kde.org/stable/release-service/23.08.4/src/falkon-23.08.4.tar.xz
-https://archive.mozilla.org/pub/firefox/releases/115.7.0esr/source/firefox-115.7.0esr.source.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.18/source/seamonkey-2.53.18.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/balsa-2.6.4-upstream_fixes-2.patch
@@ -891,7 +890,7 @@ https://inkscape.org/gallery/item/44615/inkscape-1.3.2.tar.xz
 https://downloads.sourceforge.net/pidgin/pidgin-2.14.12.tar.bz2
 https://downloads.sourceforge.net/rox/rox-filer-2.11.tar.bz2
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/115.6.1/source/thunderbird-115.6.1.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/115.7.0/source/thunderbird-115.7.0.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.13.1/tigervnc-1.13.1.tar.gz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.13.1-configuration_fixes-1.patch
@@ -945,7 +944,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/libmusicbrainz-2.1.5-missing-i
 https://github.com/metabrainz/libmusicbrainz/releases/download/release-5.1.0/libmusicbrainz-5.1.0.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/libmusicbrainz-5.1.0-cmake_fixes-1.patch
 https://downloads.xiph.org/releases/ogg/libogg-1.3.5.tar.xz
-https://github.com/haasn/libplacebo/archive/refs/tags/v6.338.1/libplacebo-6.338.1.tar.gz
+https://github.com/haasn/libplacebo/archive/refs/tags/v6.338.2/libplacebo-6.338.2.tar.gz
 https://downloads.sourceforge.net/libquicktime/libquicktime-1.2.4.tar.gz
 https://github.com/libsndfile/libsamplerate/releases/download/0.2.2/libsamplerate-0.2.2.tar.xz
 https://github.com/libsndfile/libsndfile/releases/download/1.2.2/libsndfile-1.2.2.tar.xz
