diff --git a/wget-list b/wget-list
index 8b4d6670..1e1f9ec1 100644
--- a/wget-list
+++ b/wget-list
@@ -131,7 +131,7 @@ https://lttng.org/files/urcu/userspace-rcu-0.14.0.tar.bz2
 https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26.tar.bz2
 https://dist.libuv.org/dist/v1.47.0/libuv-v1.47.0.tar.gz
 https://xkbcommon.org/download/libxkbcommon-1.6.0.tar.xz
-https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.3.tar.xz
+https://download.gnome.org/sources/libxml2/2.12/libxml2-2.12.4.tar.xz
 https://www.w3.org/XML/Test/xmlts20130923.tar.gz
 https://github.com/hughsie/libxmlb/releases/download/0.3.15/libxmlb-0.3.15.tar.xz
 https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.39.tar.xz
@@ -607,8 +607,8 @@ https://ftp.sendmail.org/sendmail.8.17.2.tar.gz
 https://github.com/LMDB/lmdb/archive/LMDB_0.9.31.tar.gz
 https://downloads.mariadb.org/interstitial/mariadb-10.11.6/source/mariadb-10.11.6.tar.gz
 https://ftp.postgresql.org/pub/source/v16.1/postgresql-16.1.tar.bz2
-https://sqlite.org/2023/sqlite-autoconf-3440200.tar.gz
-https://sqlite.org/2023/sqlite-doc-3440000.zip
+https://sqlite.org/2024/sqlite-autoconf-3450000.tar.gz
+https://sqlite.org/2024/sqlite-doc-3450000.zip
 https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.6.tgz
 https://www.linuxfromscratch.org/patches/blfs/svn/openldap-2.6.6-consolidated-1.patch
 https://nlnetlabs.nl/downloads/unbound/unbound-1.19.0.tar.gz
@@ -635,7 +635,7 @@ https://www.x.org/pub/individual/xserver/xorg-server-21.1.10.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/xorg-server-21.1.10-tearfree_backport-10.patch
 https://www.freedesktop.org/software/libevdev/libevdev-1.13.1.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-evdev-2.10.6.tar.bz2
-https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.24.0/libinput-1.24.0.tar.gz
+https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.25.0/libinput-1.25.0.tar.gz
 https://www.x.org/pub/individual/driver/xf86-input-libinput-1.4.0.tar.xz
 https://www.x.org/pub/individual/driver/xf86-input-synaptics-1.9.2.tar.xz
 https://github.com/linuxwacom/xf86-input-wacom/releases/download/xf86-input-wacom-1.2.0/xf86-input-wacom-1.2.0.tar.bz2
