diff --git a/wget-list b/wget-list
index 9bb08aee..9daead0d 100644
--- a/wget-list
+++ b/wget-list
@@ -466,7 +466,7 @@ https://files.pythonhosted.org/packages/source/M/Mako/Mako-1.3.0.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-1.26.3.tar.gz
 https://files.pythonhosted.org/packages/source/p/packaging/packaging-23.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
-https://download.gnome.org/sources/pyatspi/2.46/pyatspi-2.46.0.tar.xz
+https://download.gnome.org/sources/pyatspi/2.46/pyatspi-2.46.1.tar.xz
 https://github.com/pygobject/pycairo/releases/download/v1.18.2/pycairo-1.18.2.tar.gz
 https://github.com/pygobject/pycairo/releases/download/v1.25.1/pycairo-1.25.1.tar.gz
 https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.17.2.tar.gz
@@ -645,7 +645,7 @@ https://invisible-mirror.net/archives/xterm/xterm-389.tgz
 https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.2.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.3.tar.xz
-https://download.gnome.org/sources/at-spi2-core/2.50/at-spi2-core-2.50.0.tar.xz
+https://download.gnome.org/sources/at-spi2-core/2.50/at-spi2-core-2.50.1.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.0.tar.xz
 https://www.cairographics.org/releases/cairomm-1.14.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-gtk-0.3.0.tar.xz
@@ -691,8 +691,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.17-ffmpeg5_fi
 https://download.qt.io/archive/qt/6.6/6.6.1/single/qt-everywhere-src-6.6.1.tar.xz
 https://www.freedesktop.org/software/startup-notification/releases/startup-notification-0.12.tar.gz
 https://download.gnome.org/sources/tepl/6.8/tepl-6.8.0.tar.xz
-https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.274/Vulkan-Headers-1.3.274.tar.gz
-https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.274/Vulkan-Loader-1.3.274.tar.gz
+https://github.com/KhronosGroup/Vulkan-Headers/archive/refs/tags/v1.3.275/Vulkan-Headers-1.3.275.tar.gz
+https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.275/Vulkan-Loader-1.3.275.tar.gz
 https://webkitgtk.org/releases/webkitgtk-2.42.4.tar.xz
 https://download.gnome.org/sources/gdm/45/gdm-45.0.1.tar.xz
 https://github.com/CanonicalLtd/lightdm/releases/download/1.32.0/lightdm-1.32.0.tar.xz
@@ -740,7 +740,7 @@ https://download.gnome.org/sources/totem-pl-parser/3.26/totem-pl-parser-3.26.6.t
 https://gitlab.gnome.org/GNOME/vte/-/archive/0.74.2/vte-0.74.2.tar.gz
 https://download.gnome.org/sources/yelp-xsl/42/yelp-xsl-42.1.tar.xz
 https://download.gnome.org/sources/geocode-glib/3.26/geocode-glib-3.26.4.tar.xz
-https://download.gnome.org/sources/gjs/1.78/gjs-1.78.1.tar.xz
+https://download.gnome.org/sources/gjs/1.78/gjs-1.78.2.tar.xz
 https://download.gnome.org/sources/gnome-autoar/0.4/gnome-autoar-0.4.4.tar.xz
 https://download.gnome.org/sources/gnome-desktop/44/gnome-desktop-44.0.tar.xz
 https://download.gnome.org/sources/gnome-menus/3.36/gnome-menus-3.36.0.tar.xz
@@ -754,14 +754,14 @@ https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.0.tar.xz
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
 https://download.gnome.org/sources/libshumate/1.1/libshumate-1.1.2.tar.xz
 https://download.gnome.org/sources/libwnck/43/libwnck-43.0.tar.xz
-https://download.gnome.org/sources/evolution-data-server/3.50/evolution-data-server-3.50.2.tar.xz
+https://download.gnome.org/sources/evolution-data-server/3.50/evolution-data-server-3.50.3.tar.xz
 https://download.gnome.org/sources/tracker/3.6/tracker-3.6.0.tar.xz
 https://download.gnome.org/sources/tracker-miners/3.6/tracker-miners-3.6.2.tar.xz
 https://download.gnome.org/sources/gsound/1.0/gsound-1.0.3.tar.xz
 https://download.gnome.org/sources/dconf/0.40/dconf-0.40.0.tar.xz
 https://download.gnome.org/sources/dconf-editor/45/dconf-editor-45.0.1.tar.xz
 https://download.gnome.org/sources/gnome-backgrounds/45/gnome-backgrounds-45.0.tar.xz
-https://download.gnome.org/sources/gvfs/1.52/gvfs-1.52.1.tar.xz
+https://download.gnome.org/sources/gvfs/1.52/gvfs-1.52.2.tar.xz
 https://download.gnome.org/sources/gexiv2/0.14/gexiv2-0.14.2.tar.xz
 https://download.gnome.org/sources/nautilus/45/nautilus-45.2.1.tar.xz
 https://download.gnome.org/sources/gnome-bluetooth/42/gnome-bluetooth-42.7.tar.xz
@@ -769,8 +769,8 @@ https://download.gnome.org/sources/gnome-keyring/42/gnome-keyring-42.1.tar.xz
 https://download.gnome.org/sources/gnome-settings-daemon/45/gnome-settings-daemon-45.1.tar.xz
 https://download.gnome.org/sources/tecla/45/tecla-45.0.tar.xz
 https://download.gnome.org/sources/gnome-control-center/45/gnome-control-center-45.2.tar.xz
-https://download.gnome.org/sources/mutter/45/mutter-45.2.tar.xz
-https://download.gnome.org/sources/gnome-shell/45/gnome-shell-45.2.tar.xz
+https://download.gnome.org/sources/mutter/45/mutter-45.3.tar.xz
+https://download.gnome.org/sources/gnome-shell/45/gnome-shell-45.3.tar.xz
 https://download.gnome.org/sources/gnome-shell-extensions/45/gnome-shell-extensions-45.2.tar.xz
 https://download.gnome.org/sources/gnome-session/45/gnome-session-45.0.tar.xz
 https://download.gnome.org/sources/gnome-tweaks/40/gnome-tweaks-40.10.tar.xz
@@ -778,15 +778,15 @@ https://download.gnome.org/sources/gnome-user-docs/45/gnome-user-docs-45.1.tar.x
 https://download.gnome.org/sources/yelp/42/yelp-42.2.tar.xz
 https://download.gnome.org/sources/baobab/45/baobab-45.0.tar.xz
 https://download.gnome.org/sources/brasero/3.12/brasero-3.12.3.tar.xz
-https://download.gnome.org/sources/eog/45/eog-45.1.tar.xz
+https://download.gnome.org/sources/eog/45/eog-45.2.tar.xz
 https://download.gnome.org/sources/evince/45/evince-45.0.tar.xz
-https://download.gnome.org/sources/evolution/3.50/evolution-3.50.2.tar.xz
+https://download.gnome.org/sources/evolution/3.50/evolution-3.50.3.tar.xz
 https://download.gnome.org/sources/file-roller/43/file-roller-43.1.tar.xz
 https://download.gnome.org/sources/gnome-calculator/45/gnome-calculator-45.0.2.tar.xz
 https://download.gnome.org/sources/gnome-color-manager/3.36/gnome-color-manager-3.36.0.tar.xz
 https://download.gnome.org/sources/gnome-disk-utility/45/gnome-disk-utility-45.1.tar.xz
 https://download.gnome.org/sources/gnome-logs/43/gnome-logs-43.0.tar.xz
-https://download.gnome.org/sources/gnome-maps/45/gnome-maps-45.2.tar.xz
+https://download.gnome.org/sources/gnome-maps/45/gnome-maps-45.3.tar.xz
 https://download.gnome.org/sources/gnome-nettool/42/gnome-nettool-42.0.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/gnome-nettool-42.0-ping_and_netstat_fixes-1.patch
 https://download.gnome.org/sources/gnome-power-manager/43/gnome-power-manager-43.0.tar.xz
@@ -869,7 +869,7 @@ https://download.documentfoundation.org/libreoffice/src/7.6.4/libreoffice-dictio
 https://download.documentfoundation.org/libreoffice/src/7.6.4/libreoffice-help-7.6.4.1.tar.xz
 https://download.documentfoundation.org/libreoffice/src/7.6.4/libreoffice-translations-7.6.4.1.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libreoffice-7.6.4.1-consolidated_fixes-1.patch
-https://download.gnome.org/sources/epiphany/45/epiphany-45.1.tar.xz
+https://download.gnome.org/sources/epiphany/45/epiphany-45.2.tar.xz
 https://download.kde.org/stable/release-service/23.08.4/src/falkon-23.08.4.tar.xz
 https://archive.mozilla.org/pub/seamonkey/releases/2.53.18/source/seamonkey-2.53.18.source.tar.xz
 https://pawsa.fedorapeople.org/balsa/balsa-2.6.4.tar.xz
