diff --git a/wget-list b/wget-list
index bf3970f6..8f65f6a1 100644
--- a/wget-list
+++ b/wget-list
@@ -75,7 +75,7 @@ https://duktape.org/duktape-2.7.0.tar.xz
 https://github.com/AbiWord/enchant/releases/download/v2.6.4/enchant-2.6.4.tar.gz
 https://libopenraw.freedesktop.org/download/exempi-2.6.5.tar.xz
 https://www.fftw.org/fftw-3.3.10.tar.gz
-https://github.com/fmtlib/fmt/archive/refs/tags/10.2.0/fmt-10.2.0.tar.gz
+https://github.com/fmtlib/fmt/archive/refs/tags/10.2.1/fmt-10.2.1.tar.gz
 https://download.gnome.org/sources/glib/2.78/glib-2.78.3.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/glib-skip_warnings-1.patch
 https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.5.tar.xz
@@ -561,8 +561,8 @@ https://download.gnome.org/sources/network-manager-applet/1.34/network-manager-a
 https://nmap.org/dist/nmap-7.94.tar.bz2
 https://downloads.sourceforge.net/traceroute/traceroute-2.1.5.tar.gz
 https://github.com/rfc1036/whois/archive/v5.4.3/whois-5.4.3.tar.gz
-https://www.wireshark.org/download/src/all-versions/wireshark-4.2.1.tar.xz
-https://c-ares.haxx.se/download/c-ares-1.24.0.tar.gz
+https://www.wireshark.org/download/src/all-versions/wireshark-4.2.2.tar.xz
+https://c-ares.haxx.se/download/c-ares-1.25.0.tar.gz
 https://curl.se/download/curl-8.5.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.78/glib-networking-2.78.0.tar.xz
