diff --git a/wget-list b/wget-list
index 749ffcac..8b4d6670 100644
--- a/wget-list
+++ b/wget-list
@@ -215,7 +215,7 @@ http://www.andre-simon.de/zip/highlight-4.10.tar.bz2
 https://github.com/ibus/ibus/archive/refs/tags/1.5.29/ibus-1.5.29.tar.gz
 https://www.unicode.org/Public/zipped/15.1.0/UCD.zip
 https://www.imagemagick.org/archive/releases/ImageMagick-7.1.1-15.tar.xz
-https://ftp.debian.org/debian/pool/main/i/iso-codes/iso-codes_4.15.0.orig.tar.xz
+https://ftp.debian.org/debian/pool/main/i/iso-codes/iso-codes_4.16.0.orig.tar.xz
 https://github.com/lsof-org/lsof/releases/download/4.99.0/lsof-4.99.0.tar.gz
 https://mandoc.bsd.lv/snapshots/mandoc-1.14.6.tar.gz
 https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.2.1.tar.bz2
@@ -237,7 +237,7 @@ https://www.kernel.org/pub/linux/daemons/autofs/v5/autofs-5.1.9.tar.xz
 https://www.kernel.org/pub/linux/bluetooth/bluez-5.72.tar.xz
 https://github.com/containers/bubblewrap/releases/download/v0.8.0/bubblewrap-0.8.0.tar.xz
 https://www.freedesktop.org/software/colord/releases/colord-1.4.6.tar.xz
-https://ftp.gnu.org/gnu/cpio/cpio-2.14.tar.bz2
+https://ftp.gnu.org/gnu/cpio/cpio-2.15.tar.bz2
 https://www.freedesktop.org/software/cups-pk-helper/releases/cups-pk-helper-0.2.7.tar.xz
 https://dbus.freedesktop.org/releases/dbus/dbus-1.14.10.tar.xz
 http://fcron.free.fr/archives/fcron-3.2.1.src.tar.gz
@@ -538,8 +538,7 @@ https://archive.apache.org/dist/ant/source/apache-ant-1.10.14-src.tar.xz
 https://github.com/NetworkConfiguration/dhcpcd/releases/download/v10.0.6/dhcpcd-10.0.6.tar.xz
 https://www.kernel.org/pub/linux/utils/net/bridge-utils/bridge-utils-1.7.1.tar.xz
 https://www.samba.org/ftp/linux-cifs/cifs-utils/cifs-utils-7.0.tar.bz2
-https://www.ncftp.com/downloads/ncftp/ncftp-3.2.6-src.tar.xz
-https://www.linuxfromscratch.org/patches/blfs/svn/ncftp-3.2.6-configure_fixes-1.patch
+https://www.ncftp.com/downloads/ncftp/ncftp-3.2.7-src.tar.xz
 https://downloads.sourceforge.net/project/net-tools/net-tools-2.10.tar.xz
 https://www.kernel.org/pub/linux/utils/nfs-utils/2.6.4/nfs-utils-2.6.4.tar.xz
 https://www.eecis.udel.edu/~ntp/ntp_spool/ntp4/ntp-4.2/ntp-4.2.8p17.tar.gz
@@ -574,7 +573,7 @@ https://github.com/thom311/libnl/releases/download/libnl3_9_0/libnl-doc-3.9.0.ta
 https://download.gnome.org/sources/libnma/1.10/libnma-1.10.6.tar.xz
 https://github.com/thkukuk/libnsl/releases/download/v2.0.1/libnsl-2.0.1.tar.xz
 https://www.tcpdump.org/release/libpcap-1.10.4.tar.gz
-https://github.com/rockdaboot/libpsl/releases/download/0.21.2/libpsl-0.21.2.tar.gz
+https://github.com/rockdaboot/libpsl/releases/download/0.21.5/libpsl-0.21.5.tar.gz
 http://libndp.org/files/libndp-1.8.tar.gz
 https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.7.0/libslirp-v4.7.0.tar.bz2
 https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz
@@ -947,7 +946,6 @@ https://github.com/haasn/libplacebo/archive/refs/tags/v6.338.1/libplacebo-6.338.
 https://downloads.sourceforge.net/libquicktime/libquicktime-1.2.4.tar.gz
 https://github.com/libsndfile/libsamplerate/releases/download/0.2.2/libsamplerate-0.2.2.tar.xz
 https://github.com/libsndfile/libsndfile/releases/download/1.2.2/libsndfile-1.2.2.tar.xz
-https://downloads.xiph.org/releases/theora/libtheora-1.1.1.tar.xz
 https://github.com/intel/libva/releases/download/2.20.0/libva-2.20.0.tar.bz2
 https://gitlab.freedesktop.org/vdpau/libvdpau/-/archive/1.5/libvdpau-1.5.tar.bz2
 https://github.com/i-rinat/libvdpau-va-gl/archive/v0.4.0/libvdpau-va-gl-0.4.0.tar.gz
