diff --git a/wget-list b/wget-list
index 160365cf..b5a77073 100644
--- a/wget-list
+++ b/wget-list
@@ -1,7 +1,7 @@
 https://www.linuxfromscratch.org/blfs/downloads/systemd/blfs-systemd-units-20231205.tar.xz
 https://github.com/lfs-book/make-ca/releases/download/v1.13/make-ca-1.13.tar.xz
 https://github.com/cracklib/cracklib/releases/download/v2.9.11/cracklib-2.9.11.tar.xz
-https://www.kernel.org/pub/linux/utils/cryptsetup/v2.6/cryptsetup-2.6.1.tar.xz
+https://www.kernel.org/pub/linux/utils/cryptsetup/v2.7/cryptsetup-2.7.0.tar.xz
 https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.28/cyrus-sasl-2.1.28.tar.gz
 https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.4.3.tar.bz2
 https://www.linuxfromscratch.org/patches/blfs/svn/gnupg-2.4.3-emacs-1.patch
@@ -83,7 +83,7 @@ https://download.gnome.org/sources/gobject-introspection/1.78/gobject-introspect
 https://ftp.gnu.org/gnu/gsl/gsl-2.7.1.tar.gz
 https://download.gnome.org/sources/gspell/1.12/gspell-1.12.2.tar.xz
 https://github.com/unicode-org/icu/releases/download/release-74-1/icu4c-74_1-src.tgz
-https://github.com/benhoyt/inih/archive/r57/inih-r57.tar.gz
+https://github.com/benhoyt/inih/archive/r58/inih-r58.tar.gz
 https://github.com/intel/gmmlib/archive/refs/tags/intel-gmmlib-22.3.16.tar.gz
 https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
 https://s3.amazonaws.com/json-c_releases/releases/json-c-0.17.tar.gz
@@ -172,7 +172,7 @@ https://github.com/fribidi/fribidi/releases/download/v1.0.13/fribidi-1.0.13.tar.
 https://download.gimp.org/pub/gegl/0.4/gegl-0.4.46.tar.xz
 https://sourceforge.net/projects/giflib/files/giflib-5.2.1.tar.gz
 https://github.com/Dav1dde/glad/archive/refs/tags/v2.0.4/glad-2.0.4.tar.gz
-https://github.com/g-truc/glm/archive/0.9.9.8/glm-0.9.9.8.tar.gz
+https://github.com/g-truc/glm/archive/1.0.0/glm-1.0.0.tar.gz
 https://github.com/silnrsi/graphite/releases/download/1.3.14/graphite2-1.3.14.tgz
 https://github.com/harfbuzz/harfbuzz/releases/download/8.3.0/harfbuzz-8.3.0.tar.xz
 https://github.com/jasper-software/jasper/archive/version-4.1.2/jasper-4.1.2.tar.gz
@@ -203,7 +203,7 @@ https://github.com/qpdf/qpdf/releases/download/v11.8.0/qpdf-11.8.0.tar.gz
 https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.bz2
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
-https://github.com/aruiz/webp-pixbuf-loader/archive/0.2.5/webp-pixbuf-loader-0.2.5.tar.gz
+https://github.com/aruiz/webp-pixbuf-loader/archive/0.2.6/webp-pixbuf-loader-0.2.6.tar.gz
 https://github.com/google/woff2/archive/v1.0.2/woff2-1.0.2.tar.gz
 https://github.com/asciidoctor/asciidoctor/archive/v2.0.20/asciidoctor-2.0.20.tar.gz
 https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
@@ -626,7 +626,7 @@ https://xcb.freedesktop.org/dist/xcb-util-keysyms-0.4.1.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-renderutil-0.3.10.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-wm-0.4.2.tar.xz
 https://xcb.freedesktop.org/dist/xcb-util-cursor-0.1.4.tar.xz
-https://mesa.freedesktop.org/archive/mesa-23.3.3.tar.xz
+https://mesa.freedesktop.org/archive/mesa-23.3.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/mesa-add_xdemos-2.patch
 https://www.x.org/pub/individual/data/xbitmaps-1.1.3.tar.xz
 https://www.x.org/pub/individual/data/xcursor-themes-1.0.7.tar.xz
@@ -912,13 +912,13 @@ https://downloads.sourceforge.net/opencore-amr/fdk-aac-2.0.3.tar.gz
 https://downloads.xiph.org/releases/flac/flac-1.4.3.tar.xz
 https://files.dyne.org/frei0r/releases/frei0r-plugins-1.8.0.tar.gz
 https://downloads.sourceforge.net/gmerlin/gavl-1.4.0.tar.gz
-https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.8.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.22.8.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.22.8.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.22.8.tar.xz
-https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.22.8.tar.xz
-https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.22.8.tar.xz
-https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.22.8.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.9.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-base/gst-plugins-base-1.22.9.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-good/gst-plugins-good-1.22.9.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-bad/gst-plugins-bad-1.22.9.tar.xz
+https://gstreamer.freedesktop.org/src/gst-plugins-ugly/gst-plugins-ugly-1.22.9.tar.xz
+https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.22.9.tar.xz
+https://gstreamer.freedesktop.org/src/gstreamer-vaapi/gstreamer-vaapi-1.22.9.tar.xz
 https://downloads.sourceforge.net/id3lib/id3lib-3.8.3.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/id3lib-3.8.3-consolidated_patches-1.patch
 https://github.com/intel/media-driver/archive/refs/tags/intel-media-23.4.3.tar.gz
