diff --git a/wget-list b/wget-list
index d91cb19f..354f1cfb 100644
--- a/wget-list
+++ b/wget-list
@@ -15,8 +15,8 @@ https://www.linuxfromscratch.org/patches/blfs/svn/liboauth-1.0.3-openssl-1.1.0-3
 https://github.com/libpwquality/libpwquality/releases/download/libpwquality-1.4.5/libpwquality-1.4.5.tar.bz2
 https://kerberos.org/dist/krb5/1.21/krb5-1.21.2.tar.gz
 https://ftp.gnu.org/gnu/nettle/nettle-3.9.1.tar.gz
-https://archive.mozilla.org/pub/security/nss/releases/NSS_3_96_1_RTM/src/nss-3.96.1.tar.gz
-https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.96.1-standalone-1.patch
+https://archive.mozilla.org/pub/security/nss/releases/NSS_3_97_RTM/src/nss-3.97.tar.gz
+https://www.linuxfromscratch.org/patches/blfs/svn/nss-3.97-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-9.6p1.tar.gz
 https://github.com/p11-glue/p11-kit/releases/download/0.25.3/p11-kit-0.25.3.tar.xz
 https://gitlab.freedesktop.org/polkit/polkit/-/archive/124/polkit-124.tar.gz
@@ -658,7 +658,7 @@ https://download.gnome.org/sources/goffice/0.10/goffice-0.10.56.tar.xz
 https://github.com/steveire/grantlee/releases/download/v5.3.1/grantlee-5.3.1.tar.gz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
 https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.33.tar.xz
-https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.40.tar.xz
+https://download.gnome.org/sources/gtk+/3.24/gtk+-3.24.41.tar.xz
 https://download.gnome.org/sources/gtk/4.12/gtk-4.12.5.tar.xz
 https://download.gnome.org/sources/gtk-engines/2.20/gtk-engines-2.20.2.tar.bz2
 https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.8.tar.xz
