diff --git a/wget-list b/wget-list
index 9daead0d..b5350e0a 100644
--- a/wget-list
+++ b/wget-list
@@ -148,7 +148,7 @@ https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.bz2
 https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.42/pcre2-10.42.tar.bz2
 http://ftp.rpm.org/popt/releases/popt-1.x/popt-1.19.tar.gz
 https://ftp.gnu.org/gnu/pth/pth-2.0.7.tar.gz
-https://download.kde.org/stable/qca/2.3.7/qca-2.3.7.tar.xz
+https://download.kde.org/stable/qca/2.3.8/qca-2.3.8.tar.xz
 https://archive.mozilla.org/pub/firefox/releases/115.6.0esr/source/firefox-115.6.0esr.source.tar.xz
 https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/vulkan-sdk-1.3.268.0/SPIRV-Headers-1.3.268.0.tar.gz
 https://github.com/KhronosGroup/SPIRV-Tools/archive/refs/tags/v2023.5.rc1/SPIRV-Tools-2023.5.rc1.tar.gz
@@ -565,7 +565,7 @@ https://c-ares.haxx.se/download/c-ares-1.25.0.tar.gz
 https://curl.se/download/curl-8.5.0.tar.xz
 https://gitlab.freedesktop.org/geoclue/geoclue/-/archive/2.7.1/geoclue-2.7.1.tar.bz2
 https://download.gnome.org/sources/glib-networking/2.78/glib-networking-2.78.0.tar.xz
-https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.1.1/kdsoap-2.1.1.tar.gz
+https://github.com/KDAB/KDSoap/releases/download/kdsoap-2.2.0/kdsoap-2.2.0.tar.gz
 https://www.nlnetlabs.nl/downloads/ldns/ldns-1.8.3.tar.gz
 https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
 https://netfilter.org/projects/libmnl/files/libmnl-1.0.5.tar.bz2
@@ -884,7 +884,7 @@ https://inkscape.org/gallery/item/44615/inkscape-1.3.2.tar.xz
 https://downloads.sourceforge.net/pidgin/pidgin-2.14.12.tar.bz2
 https://downloads.sourceforge.net/rox/rox-filer-2.11.tar.bz2
 http://dist.schmorp.de/rxvt-unicode/Attic/rxvt-unicode-9.31.tar.bz2
-https://archive.mozilla.org/pub/thunderbird/releases/115.6.0/source/thunderbird-115.6.0.source.tar.xz
+https://archive.mozilla.org/pub/thunderbird/releases/115.6.1/source/thunderbird-115.6.1.source.tar.xz
 https://github.com/TigerVNC/tigervnc/archive/v1.13.1/tigervnc-1.13.1.tar.gz
 https://www.x.org/pub/individual/xserver/xorg-server-21.1.6.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/tigervnc-1.13.1-configuration_fixes-1.patch
