diff --git a/wget-list b/wget-list
index 0d1c234ef4..62f946b9b0 100644
--- a/wget-list
+++ b/wget-list
@@ -297,7 +297,7 @@ https://github.com/lu-zero/cargo-c/archive/v0.10.19/cargo-c-0.10.19.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.29.2/cbindgen-0.29.2.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/clisp/clisp-2.49.95.tar.xz
 https://cmake.org/files/v4.2/cmake-4.2.1.tar.gz
-https://doxygen.nl/files/doxygen-1.16.0.src.tar.gz
+https://doxygen.nl/files/doxygen-1.16.1.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftpmirror.gnu.org/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
 https://github.com/bdwgc/bdwgc/releases/download/v8.2.10/gc-8.2.10.tar.gz
