diff --git a/wget-list b/wget-list
index ce29b17378..1f878cc1a0 100644
--- a/wget-list
+++ b/wget-list
@@ -119,7 +119,7 @@ https://ftpmirror.gnu.org/libidn/libidn-1.43.tar.gz
 https://ftpmirror.gnu.org/libidn/libidn2-2.3.8.tar.gz
 https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.6.7.tar.bz2
 https://github.com/cjlin1/liblinear/archive/v250/liblinear-250.tar.gz
-https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.32.0/libmbim-1.32.0.tar.gz
+https://gitlab.freedesktop.org/mobile-broadband/libmbim/-/archive/1.34.0/libmbim-1.34.0.tar.gz
 https://github.com/linux-nvme/libnvme/archive/v1.16.1/libnvme-1.16.1.tar.gz
 https://github.com/rrthomas/libpaper/releases/download/v2.2.7/libpaper-2.2.7.tar.gz
 https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar.xz
@@ -668,7 +668,7 @@ https://www.x.org/pub/individual/app/xclock-1.1.1.tar.xz
 https://www.x.org/pub/individual/app/xinit-1.4.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.4.tar.xz
 https://download.gnome.org/sources/atkmm/2.36/atkmm-2.36.3.tar.xz
-https://download.gnome.org/sources/at-spi2-core/2.58/at-spi2-core-2.58.2.tar.xz
+https://download.gnome.org/sources/at-spi2-core/2.58/at-spi2-core-2.58.3.tar.xz
 https://www.cairographics.org/releases/cairo-1.18.4.tar.xz
 https://www.cairographics.org/releases/cairomm-1.14.5.tar.xz
 https://www.cairographics.org/releases/cairomm-1.18.0.tar.xz
@@ -690,7 +690,7 @@ https://download.gnome.org/sources/gtk-vnc/1.5/gtk-vnc-1.5.0.tar.xz
 https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
 https://download.gnome.org/sources/gtksourceview/4.8/gtksourceview-4.8.4.tar.xz
 https://download.gnome.org/sources/gtksourceview/5.18/gtksourceview-5.18.0.tar.xz
-https://downloads.sourceforge.net/enlightenment/imlib2-1.12.5.tar.xz
+https://downloads.sourceforge.net/enlightenment/imlib2-1.12.6.tar.xz
 https://github.com/ksnip/kColorPicker/archive/v0.3.1/kColorPicker-0.3.1.tar.gz
 https://github.com/ksnip/kImageAnnotator/archive/v0.7.2/kImageAnnotator-0.7.2.tar.gz
 https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
