diff --git a/wget-list b/wget-list
index af7831e1e0..23cd6b58f6 100644
--- a/wget-list
+++ b/wget-list
@@ -111,7 +111,7 @@ https://0pointer.de/lennart/projects/libdaemon/libdaemon-0.14.tar.gz
 https://gitlab.freedesktop.org/emersion/libdisplay-info/-/releases/0.3.0/downloads/libdisplay-info-0.3.0.tar.xz
 https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.11.2.tar.bz2
 https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.58.tar.bz2
-https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.54.tar.xz
+https://download.gnome.org/sources/libgsf/1.14/libgsf-1.14.55.tar.xz
 https://download.gnome.org/sources/libgudev/238/libgudev-238.tar.xz
 https://github.com/hughsie/libgusb/releases/download/0.4.9/libgusb-0.4.9.tar.xz
 https://github.com/libical/libical/releases/download/v3.0.20/libical-3.0.20.tar.gz
@@ -160,7 +160,7 @@ https://github.com/protocolbuffers/protobuf/releases/download/v33.4/protobuf-33.
 https://github.com/protobuf-c/protobuf-c/releases/download/v1.5.2/protobuf-c-1.5.2.tar.gz
 https://download.kde.org/stable/qca/2.3.10/qca-2.3.10.tar.xz
 https://github.com/danvratil/qcoro/archive/v0.12.0/qcoro-0.12.0.tar.gz
-https://github.com/simdutf/simdutf/archive/v7.7.1/simdutf-7.7.1.tar.gz
+https://github.com/simdutf/simdutf/archive/v8.0.0/simdutf-8.0.0.tar.gz
 https://archive.mozilla.org/pub/firefox/releases/140.7.0esr/source/firefox-140.7.0esr.source.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/spidermonkey-140.7.0-python_3.14_fixes-1.patch
 https://github.com/KhronosGroup/SPIRV-Headers/archive/vulkan-sdk-1.4.335.0/SPIRV-Headers-vulkan-sdk-1.4.335.0.tar.gz
@@ -489,7 +489,7 @@ https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.4.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/ply/ply-3.11.tar.gz
 https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.2.1.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
-https://download.gnome.org/sources/pyatspi/2.58/pyatspi-2.58.0.tar.xz
+https://download.gnome.org/sources/pyatspi/2.58/pyatspi-2.58.1.tar.xz
 https://github.com/pygobject/pycairo/releases/download/v1.29.0/pycairo-1.29.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pygdbmi/pygdbmi-0.11.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.19.2.tar.gz
@@ -516,7 +516,7 @@ https://files.pythonhosted.org/packages/source/c/chardet/chardet-5.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/c/charset-normalizer/charset_normalizer-3.4.4.tar.gz
 https://files.pythonhosted.org/packages/source/c/commonmark/commonmark-0.9.1.tar.gz
 https://files.pythonhosted.org/packages/source/e/editables/editables-0.5.tar.gz
-https://files.pythonhosted.org/packages/source/h/hatchling/hatchling-1.27.0.tar.gz
+https://files.pythonhosted.org/packages/source/h/hatchling/hatchling-1.28.0.tar.gz
 https://files.pythonhosted.org/packages/source/h/hatch-fancy-pypi-readme/hatch_fancy_pypi_readme-25.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/h/hatch-vcs/hatch_vcs-0.5.0.tar.gz
 https://files.pythonhosted.org/packages/source/i/idna/idna-3.11.tar.gz
@@ -524,14 +524,14 @@ https://files.pythonhosted.org/packages/source/i/imagesize/imagesize-1.4.1.tar.g
 https://files.pythonhosted.org/packages/source/i/iniconfig/iniconfig-2.3.0.tar.gz
 https://github.com/PyO3/maturin/archive/v1.10.2/maturin-1.10.2.tar.gz
 https://files.pythonhosted.org/packages/source/M/Markdown/markdown-3.10.tar.gz
-https://files.pythonhosted.org/packages/source/m/meson_python/meson_python-0.18.0.tar.gz
+https://files.pythonhosted.org/packages/source/m/meson_python/meson_python-0.19.0.tar.gz
 https://files.pythonhosted.org/packages/source/m/msgpack/msgpack-1.1.2.tar.gz
-https://files.pythonhosted.org/packages/source/p/pathspec/pathspec-0.12.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pathspec/pathspec-1.0.3.tar.gz
 https://files.pythonhosted.org/packages/source/p/pluggy/pluggy-1.6.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyproject-hooks/pyproject_hooks-1.2.0.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyproject-metadata/pyproject_metadata-0.9.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/pytz/pytz-2025.2.tar.gz
-https://files.pythonhosted.org/packages/source/r/roman-numerals/roman_numerals-3.1.0.tar.gz
+https://files.pythonhosted.org/packages/source/r/roman-numerals/roman_numerals-4.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/semantic_version/semantic_version-2.10.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/setuptools_rust/setuptools_rust-1.12.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/setuptools_scm/setuptools_scm-9.2.2.tar.gz
@@ -544,7 +544,7 @@ https://files.pythonhosted.org/packages/source/s/sphinxcontrib-jquery/sphinxcont
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-jsmath/sphinxcontrib-jsmath-1.0.1.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib-qthelp/sphinxcontrib_qthelp-2.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/s/sphinxcontrib_serializinghtml/sphinxcontrib_serializinghtml-2.0.0.tar.gz
-https://files.pythonhosted.org/packages/source/t/trove_classifiers/trove_classifiers-2025.11.14.15.tar.gz
+https://files.pythonhosted.org/packages/source/t/trove_classifiers/trove_classifiers-2026.1.14.14.tar.gz
 https://files.pythonhosted.org/packages/source/t/typogrify/typogrify-2.1.0.tar.gz
 https://files.pythonhosted.org/packages/source/u/urllib3/urllib3-2.6.0.tar.gz
 https://files.pythonhosted.org/packages/source/u/uv_build/uv_build-0.9.10.tar.gz
