diff --git a/wget-list b/wget-list
index e6fbec10f0..e6126f7dde 100644
--- a/wget-list
+++ b/wget-list
@@ -296,7 +296,7 @@ https://kernel.org/pub/linux/utils/usb/usbutils/usbutils-019.tar.xz
 https://github.com/lu-zero/cargo-c/archive/v0.10.19/cargo-c-0.10.19.tar.gz
 https://github.com/mozilla/cbindgen/archive/v0.29.2/cbindgen-0.29.2.tar.gz
 https://anduin.linuxfromscratch.org/BLFS/clisp/clisp-2.49.95.tar.xz
-https://cmake.org/files/v4.2/cmake-4.2.1.tar.gz
+https://cmake.org/files/v4.2/cmake-4.2.2.tar.gz
 https://doxygen.nl/files/doxygen-1.16.1.src.tar.gz
 https://kernel.org/pub/software/utils/dtc/dtc-1.7.2.tar.xz
 https://ftpmirror.gnu.org/gcc/gcc-15.2.0/gcc-15.2.0.tar.xz
@@ -833,7 +833,7 @@ https://download.gnome.org/sources/libwnck/43/libwnck-43.3.tar.xz
 http://archive.xfce.org/src/xfce/xfce4-dev-tools/4.20/xfce4-dev-tools-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/libxfce4windowing/4.20/libxfce4windowing-4.20.5.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-panel/4.20/xfce4-panel-4.20.6.tar.bz2
-https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.6.tar.bz2
+https://archive.xfce.org/src/xfce/thunar/4.20/thunar-4.20.7.tar.bz2
 https://archive.xfce.org/src/xfce/thunar-volman/4.20/thunar-volman-4.20.0.tar.bz2
 https://archive.xfce.org/src/xfce/tumbler/4.20/tumbler-4.20.1.tar.bz2
 https://archive.xfce.org/src/xfce/xfce4-appfinder/4.20/xfce4-appfinder-4.20.0.tar.bz2
