diff --git a/wget-list b/wget-list
index e6126f7dde..55c8a603b3 100644
--- a/wget-list
+++ b/wget-list
@@ -228,7 +228,7 @@ https://downloads.sourceforge.net/bogofilter/bogofilter-1.2.5.tar.xz
 https://anduin.linuxfromscratch.org/BLFS/compface/compface-1.5.2.tar.gz
 https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.28.tar.xz
 https://downloads.sourceforge.net/dos2unix/dos2unix-7.5.3.tar.gz
-https://github.com/google/shaderc/archive/v2025.5/shaderc-2025.5.tar.gz
+https://github.com/google/shaderc/archive/v2026.1/shaderc-2026.1.tar.gz
 https://gitlab.com/graphviz/graphviz/-/archive/14.1.1/graphviz-14.1.1.tar.bz2
 https://download.gnome.org/sources/gtk-doc/1.35/gtk-doc-1.35.1.tar.xz
 http://www.andre-simon.de/zip/highlight-4.18.tar.bz2
@@ -480,7 +480,7 @@ https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.4.0.tar.xz
 https://files.pythonhosted.org/packages/source/d/docutils/docutils-0.22.4.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxypypy/doxypypy-0.8.8.7.tar.gz
 https://files.pythonhosted.org/packages/source/d/doxyqml/doxyqml-0.5.3.tar.gz
-https://files.pythonhosted.org/packages/source/g/gi-docgen/gi_docgen-2025.5.tar.gz
+https://files.pythonhosted.org/packages/source/g/gi-docgen/gi_docgen-2026.1.tar.gz
 https://files.pythonhosted.org/packages/source/h/html5lib/html5lib-1.1.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/html5lib-1.1-python_3.14_buildfix-1.patch
 https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.0.2.tar.gz
@@ -676,8 +676,8 @@ https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-source.t
 https://github.com/fltk/fltk/releases/download/release-1.4.4/fltk-1.4.4-docs-html.tar.gz
 https://downloads.sourceforge.net/freeglut/freeglut-3.8.0.tar.gz
 https://download.gnome.org/sources/gdk-pixbuf/2.44/gdk-pixbuf-2.44.4.tar.xz
-https://downloads.sourceforge.net/glew/glew-2.3.0.tgz
-https://github.com/KhronosGroup/glslang/archive/16.1.0/glslang-16.1.0.tar.gz
+https://downloads.sourceforge.net/glew/glew-2.3.1.tgz
+https://github.com/KhronosGroup/glslang/archive/16.2.0/glslang-16.2.0.tar.gz
 https://archive.mesa3d.org/glu/glu-9.0.3.tar.xz
 https://download.gnome.org/sources/goffice/0.10/goffice-0.10.59.tar.xz
 https://download.gnome.org/sources/graphene/1.10/graphene-1.10.8.tar.xz
