diff --git a/wget-list b/wget-list
index d3a62f82a7..e6fbec10f0 100644
--- a/wget-list
+++ b/wget-list
@@ -494,7 +494,7 @@ https://github.com/pygobject/pycairo/releases/download/v1.29.0/pycairo-1.29.0.ta
 https://files.pythonhosted.org/packages/source/p/pygdbmi/pygdbmi-0.11.0.0.tar.gz
 https://files.pythonhosted.org/packages/source/P/Pygments/pygments-2.19.2.tar.gz
 https://download.gnome.org/sources/pygobject/3.54/pygobject-3.54.5.tar.gz
-https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.3.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/pyparsing/pyparsing-3.3.2.tar.gz
 https://files.pythonhosted.org/packages/source/p/pyserial/pyserial-3.5.tar.gz
 https://files.pythonhosted.org/packages/source/p/pytest/pytest-9.0.2.tar.gz
 https://files.pythonhosted.org/packages/b0/25/7998cd2dec731acbd438fbf91bc619603fc5188de0a9a17699a781840452/pyxdg-0.28.tar.gz
