diff --git a/wget-list b/wget-list
index 0e68d2ba3f..88e672d258 100644
--- a/wget-list
+++ b/wget-list
@@ -64,11 +64,11 @@ https://astron.com/pub/tcsh/tcsh-6.24.16.tar.gz
 https://www.zsh.org/pub/zsh-5.9.tar.xz
 https://download.qemu.org/qemu-10.2.0.tar.xz
 https://github.com/abseil/abseil-cpp/releases/download/20260107.0/abseil-cpp-20260107.0.tar.gz
-https://www.freedesktop.org/software/appstream/releases/AppStream-1.1.1.tar.xz
+https://www.freedesktop.org/software/appstream/releases/AppStream-1.1.2.tar.xz
 http://people.freedesktop.org/~hughsient/appstream-glib/releases/appstream-glib-0.8.3.tar.xz
 https://archive.apache.org/dist/apr/apr-1.7.6.tar.bz2
 https://archive.apache.org/dist/apr/apr-util-1.6.3.tar.bz2
-https://ftpmirror.gnu.org/aspell/aspell-0.60.8.1.tar.gz
+https://ftpmirror.gnu.org/aspell/aspell-0.60.8.2.tar.gz
 https://ftpmirror.gnu.org/aspell/dict/en/aspell6-en-2020.12.07-0.tar.bz2
 https://github.com/boostorg/boost/releases/download/boost-1.90.0/boost-1.90.0-b2-nodocs.tar.xz
 https://github.com/google/brotli/archive/v1.2.0/brotli-1.2.0.tar.gz
@@ -175,7 +175,7 @@ https://gitlab.freedesktop.org/wayland/wayland-protocols/-/releases/1.47/downloa
 https://anduin.linuxfromscratch.org/BLFS/wv/wv-1.2.9.tar.gz
 https://oligarchy.co.uk/xapian/1.4.30/xapian-core-1.4.30.tar.xz
 https://downloads.sourceforge.net/aa-project/aalib-1.4rc5.tar.gz
-https://download.gimp.org/pub/babl/0.1/babl-0.1.120.tar.xz
+https://download.gimp.org/pub/babl/0.1/babl-0.1.122.tar.xz
 https://github.com/Exiv2/exiv2/archive/v0.28.7/exiv2-0.28.7.tar.gz
 https://downloads.sourceforge.net/freetype/freetype-2.14.1.tar.xz
 https://downloads.sourceforge.net/freetype/freetype-doc-2.14.1.tar.xz
@@ -487,7 +487,7 @@ https://files.pythonhosted.org/packages/source/l/lxml/lxml-6.0.2.tar.gz
 https://files.pythonhosted.org/packages/source/M/Mako/mako-1.3.10.tar.gz
 https://files.pythonhosted.org/packages/source/n/numpy/numpy-2.4.1.tar.gz
 https://files.pythonhosted.org/packages/source/p/ply/ply-3.11.tar.gz
-https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.2.1.tar.gz
+https://files.pythonhosted.org/packages/source/p/psutil/psutil-7.2.2.tar.gz
 https://github.com/encukou/py3c/archive/v1.4/py3c-1.4.tar.gz
 https://download.gnome.org/sources/pyatspi/2.58/pyatspi-2.58.1.tar.xz
 https://github.com/pygobject/pycairo/releases/download/v1.29.0/pycairo-1.29.0.tar.gz
@@ -774,7 +774,7 @@ https://download.gnome.org/sources/libgtop/2.41/libgtop-2.41.3.tar.xz
 https://download.gnome.org/sources/libgweather/4.4/libgweather-4.4.4.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/libgweather-4.4.4-upstream_fix-1.patch
 https://download.gnome.org/sources/libpeas/1.36/libpeas-1.36.0.tar.xz
-https://download.gnome.org/sources/libshumate/1.5/libshumate-1.5.2.tar.xz
+https://download.gnome.org/sources/libshumate/1.5/libshumate-1.5.3.tar.xz
 https://download.gnome.org/sources/evolution-data-server/3.58/evolution-data-server-3.58.2.tar.xz
 https://download.gnome.org/sources/tinysparql/3.10/tinysparql-3.10.1.tar.xz
 https://download.gnome.org/sources/localsearch/3.10/localsearch-3.10.2.tar.xz
@@ -984,7 +984,7 @@ https://people.freedesktop.org/~mccann/dist/sound-theme-freedesktop-0.8.tar.bz2
 https://www.surina.net/soundtouch/soundtouch-2.4.0.tar.gz
 https://downloads.xiph.org/releases/speex/speex-1.2.1.tar.gz
 https://downloads.xiph.org/releases/speex/speexdsp-1.2.1.tar.gz
-https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v4.0.0/SVT-AV1-v4.0.0.tar.gz
+https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v4.0.1/SVT-AV1-v4.0.1.tar.gz
 https://taglib.org/releases/taglib-2.1.1.tar.gz
 https://www.linuxtv.org/downloads/v4l-utils/v4l-utils-1.32.0.tar.xz
 https://gitlab.freedesktop.org/pipewire/wireplumber/-/archive/0.5.13/wireplumber-0.5.13.tar.bz2
