diff --git a/wget-list b/wget-list
index 23cd6b58f6..5b863ca79b 100644
--- a/wget-list
+++ b/wget-list
@@ -20,7 +20,7 @@ https://ftpmirror.gnu.org/nettle/nettle-3.10.2.tar.gz
 https://archive.mozilla.org/pub/security/nss/releases/NSS_3_120_RTM/src/nss-3.120.tar.gz
 https://www.linuxfromscratch.org/patches/blfs/svn/nss-standalone-1.patch
 https://ftp.openbsd.org/pub/OpenBSD/OpenSSH/portable/openssh-10.2p1.tar.gz
-https://github.com/p11-glue/p11-kit/releases/download/0.26.0/p11-kit-0.26.0.tar.xz
+https://github.com/p11-glue/p11-kit/releases/download/0.26.1/p11-kit-0.26.1.tar.xz
 https://github.com/polkit-org/polkit/archive/127/polkit-127.tar.gz
 https://download.gnome.org/sources/polkit-gnome/0.105/polkit-gnome-0.105.tar.xz
 https://www.linuxfromscratch.org/patches/blfs/svn/polkit-gnome-0.105-consolidated_fixes-1.patch
@@ -126,7 +126,7 @@ https://github.com/flatpak/libportal/releases/download/0.9.1/libportal-0.9.1.tar
 https://www.linuxfromscratch.org/patches/blfs/svn/libportal-0.9.1-qt6.9_fixes-1.patch
 https://github.com/libproxy/libproxy/archive/0.5.12/libproxy-0.5.12.tar.gz
 http://dist.schmorp.de/libptytty/libptytty-2.0.tar.gz
-https://github.com/Qalculate/libqalculate/releases/download/v5.8.2/libqalculate-5.8.2.tar.gz
+https://github.com/Qalculate/libqalculate/releases/download/v5.9.0/libqalculate-5.9.0.tar.gz
 https://gitlab.freedesktop.org/mobile-broadband/libqmi/-/archive/1.38.0/libqmi-1.38.0.tar.gz
 https://github.com/seccomp/libseccomp/releases/download/v2.6.0/libseccomp-2.6.0.tar.gz
 https://download.gnome.org/sources/libsigc++/2.12/libsigc++-2.12.1.tar.xz
@@ -218,7 +218,7 @@ https://www.cairographics.org/releases/pixman-0.46.4.tar.gz
 https://poppler.freedesktop.org/poppler-26.01.0.tar.xz
 https://poppler.freedesktop.org/poppler-data-0.4.12.tar.gz
 https://downloads.sourceforge.net/potrace/potrace-1.16.tar.gz
-https://github.com/qpdf/qpdf/releases/download/v12.3.0/qpdf-12.3.0.tar.gz
+https://github.com/qpdf/qpdf/releases/download/v12.3.1/qpdf-12.3.1.tar.gz
 https://github.com/sass/sassc/archive/3.6.2/sassc-3.6.2.tar.gz
 https://github.com/sass/libsass/archive/3.6.6/libsass-3.6.6.tar.gz
 https://github.com/google/woff2/archive/v1.0.2/woff2-1.0.2.tar.gz
@@ -905,7 +905,7 @@ https://www.linuxfromscratch.org/patches/blfs/svn/seamonkey-2.53.23-cxx17-1.patc
 https://gitlab.gnome.org/GNOME/balsa/-/archive/2.6.5/balsa-2.6.5.tar.gz
 https://feh.finalrewind.org/feh-3.11.2.tar.bz2
 https://github.com/fontforge/fontforge/releases/download/20251009/fontforge-20251009.tar.xz
-https://github.com/freerdp/freerdp/archive/3.20.2/FreeRDP-3.20.2.tar.gz
+https://github.com/freerdp/freerdp/archive/3.21.0/FreeRDP-3.21.0.tar.gz
 https://download.gimp.org/gimp/v3.0/gimp-3.0.6.tar.xz
 https://download.gimp.org/pub/gimp/help/gimp-help-3.0.0.tar.bz2
 https://downloads.sourceforge.net/gparted/gparted-1.7.0.tar.gz
